<?xml version="1.0" encoding="UTF-8"?>
<div xmlns="http://www.w3.org/1999/xhtml"><p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.1.1.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.1.1.svg" target="separate">SVG</a></div><div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.1.2.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.1.2.svg" target="separate">SVG</a></div><h1><div id="g.1.3"><p/><b>Gene conversion and purifying selection shape</b> <b>nucleotide variation in gibbon L/M opsin genes</b> </div></h1>Tomohide Hiwatashi <sup>1</sup> , Akichika Mikami <sup>2,8</sup> , Takafumi Katsumura <sup>1</sup> , Bambang Suryobroto <sup>3</sup> , Dyah Perwitasari-Farajallah <sup>3,4</sup> , Suchinda Malaivijitnond <sup>5</sup> , Boripat Siriaroonrat <sup>6</sup> , Hiroki Oota <sup>1,9</sup> , Shunji Goto <sup>7,10 </sup> and Shoji Kawamura <sup>1*</sup> <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.1.5.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.1.5.svg" target="separate">SVG</a></div><p/><b>Background</b> In catarrhine primates (humans, apes and Old World monkeys) the L and M opsin genes are closely juxta-posed on the X chromosome and, in combination with the autosomal S opsin gene, enable routinely trichro-matic color vision [1,2]. The L and M opsin genes have a close evolutionary relationship and are highly similar in nucleotide sequence (~96% identity). Among 15 <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.1.8.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.1.8.svg" target="separate">SVG</a></div>amino acid differences between the human L and M opsin genes, three account for the main shifts in spectral sensitivities and tuning [3-9]. The organization of the L and M opsin genes among humans is known to be variable and includes the absence of an L or M opsin gene or the presence of L/M hybrid genes with an intermediate spectral sensitivity. A high incidence (approximately 3-8%) of color vision  <b>&#8220; </b> deficien-cies <b>&#8221; </b> in males results as a consequence [10]. This variation is caused by unequal meiotic recombination between L and M opsin genes. When the nucleotide sequence of a part of one gene appears to be replaced with the <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.1.10.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.1.10.svg" target="separate">SVG</a></div><p>see LICENSE</p><p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 2 of 14</b> corresponding sequence of the other, this type of recombi-nation is often called gene conversion. Gene conversion is suggested to have occurred frequently between the human L and M opsin genes [11-15] and to have played a crucial role in generating hybrids of the two genes with altered spectral sensitivities [16-18]. Even among individuals with normal color vision, the allele frequency of the L opsin gene with Ala at the site 180 instead of Ser is reported to be 30-38% in non-African populations [16-18]. Compared to humans, the incidence of color vision variation is reported to be rare in other catarrhines [19-21]. Among 744 male long-tailed macaques ( <i>Macaca</i> <i>fascicularis</i> ) examined, only three were found to have a single L/M hybrid gene with an intermediate spectral sensitivity and to be dichromats [19,22,23]. Among 58 male chimpanzees ( <i>Pan troglodytes</i> ), one was found to have an L/M hybrid gene with an intermediate spectral sensitivity in addition to one normal M opsin gene on the X chromosome and to be a protanomalous trichro-mat [21,24]. Thus, frequencies of color vision variants in male long-tailed macaques and male chimpanzees can be calculated to be ~0.4% and ~1.7%, respectively. These frequencies could be overestimated because no variants were found in 455 male monkeys from other macaque species [19,23] and because the chimpanzees examined were from limited numbers of breeding colonies [21]. Other studies have reported an absence of color vision defects in Old World monkeys and apes [20,25]. Nevertheless, gene conversion is suggested to have occurred frequently between the L and M opsin genes in non-human catarrhines on the basis of the following observations: (1) the intraspecific nucleotide divergence between the L and M opsin genes (paralogous divergence) tends to be smaller than divergence of the same gene between species (orthologous divergence) [26,27], (2) alle-lic polymorphism is often shared between L and M opsin genes and between species [28,29], (3) paralogous nucleo-tide divergence in introns and peripheral exons (exons 1 and 6) is significantly smaller than those in the centrally located exons (exons 2-5), which contain the amino acid sites affecting absorption spectra of the L and M photopig-ments [13,14]. These studies suggest that gene conversions at nucleotide sites relevant for the spectral difference between the L and M opsins have been effectively elimi-nated from the population by purifying natural selection. If gene conversion occurred frequently between the L and M opsin genes and if purifying selection was active in non-human catarrhines, we would also expect another intraspecific pattern of nucleotide variation: higher nucleotide divergence between the L and M opsin genes in central exons than in introns in addition to lower nucleotide diversity within these exons than within introns. However, the within-species nucleotide variation of both exons and introns has been evaluated for only the L opsin gene of two African hominoids, humans [18] and chimpanzees (primarily  <i>P. t. verus</i> ) [25]. In the present study, we focused on gibbons (Family Hylobatidae), com-monly known as the lesser apes, for which normal tri-chromacy is reported [30]. Gibbons occur in Asia and are the most diverse and speciose of all living apes [31], mak-ing them an ideal group with which to assess the range of L/M opsin genetic variation. We examined the nucleotide variation of both the L and M opsin genes by sequencing the 3.6~3.9-kb genomic region encompassing exon 3 to exon 5 from individuals in five species and three genera of gibbons. <p/><b>Methods</b> <p/><b>Gibbon DNA samples</b> Blood samples were collected from a total of 157 indivi-duals of the following species: Agile ( <i>Hylobates agilis</i> ; N = 37), Kloss <b>&#8217; </b> ( <i>H. klossii</i> ; N = 2), White-handed ( <i>H. lar</i> ; N = 40), Silvery Javan ( <i>H. moloch</i> ; N = 6), Mueller <b>&#8217; </b> s Bor-nean gray ( <i>H. muelleri</i> ; N = 6), Pileated ( <i>H. pileatus</i> ; N = 19), Chinese White-cheeked ( <i>Nomascus leucogenys</i> ; N = 16) and Siamang ( <i>Symphalangus syndactylus</i> ; N = 31). Sampling was conducted at the Ragunan Zoo and the Pontianak Zoo in Indonesia, and the Chiang Mai Zoo, the Bangkok Zoo and the Khao Kheow Open Zoo in Thailand. We also sampled gibbons reared by local resi-dents in Kalimantan, Indonesia. Genomic DNA was extracted from blood samples using the DNA Microex-traction Kit (Stratagene, Santa Clara, CA) or the QIAamp DNA Blood Mini Kit (Qiagen, Duesseldorf, Germany). Research permissions were granted by each country and sampling was conducted according to the Guide for the Care and Use of Laboratory Animals by the National Institute of Health, U.S.A. (1985) and the Guide for the Care and Use of Laboratory Primates by the Primate Research Institute, Kyoto University (1986, 2002). All procedures were approved by the animal ethics commit-tee of the Primate Research Institute, Kyoto University. Among the 157 individuals, 152 were subjected to the genotyping of the L/M opsin genes (Additional file 1, Table S1). The remaining 5 individuals (two  <i>H. agilis</i> , one  <i>H. lar</i> , and two  <i>S. syndactylus</i> ) were included in the analysis of the neutral reference genes. Among the 152 individuals, 94 were subjected to DNA sequencing of the entire 3.6~3.9-kb region encompassing exon 3 to exon 5 (Table 1). <p/><b>Genotyping and sequencing of the gibbon L and M opsin</b> <b>genes</b> In primates, the L and M opsin genes are arrayed in the same orientation on the X-chromosome and separated by approximately 24 kb [32]. Both genes consist of six exons that encode a protein 364 amino acids long, which spans approximately 15 kb [1,33,34]. The first (most <p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 3 of 14</b> <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.3.2.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.3.2.svg" target="separate">SVG</a></div>upstream) position in the gene array is typically occupied by the L opsin gene, followed by one or more M opsin genes downstream [35,36]. However, the long-range polymerase-chain reaction (PCR) necessary to determine the position of an L or M opsin gene in the array is not feasible for a large number of samples due to the paucity of gene-specific nucleotide sites and of gene-position spe-cific nucleotide sites [21,37-40]. We thus identified a gene not by the position in the gene array, but rather by its inferred spectral property. The peak absorption spectra ( &#955;  max) of the catarrhine L and M opsin photopigments are approximately 560 nm and 530 nm, respectively [41]. This spectral difference is mainly attributed to the amino acid differences at three sites: the residue 180 encoded in the exon 3 and the resi-dues 277 and 285 in the exon 5 [3,4]. Amino acid changes from Ser to Ala at the site 180, (denoted Ser180Ala), Tyr277Phe, and Thr285Ala shift the  &#955;  max values by -7, -8, and -15 nm respectively, in nearly an additive manner, and the reverse amino acid changes cause opposite spectral shifts by approximately the same extent [5-8]. Therefore, the contribution of exon 5 to the difference of absorption spectra between the L and M opsins is greater than that of exon 3. Based on the inferred absorption spectra, we defined the gibbon L opsin gene as having Tyr and Thr and the M opsin gene as having Phe and Ala at the residues 277 and 285, respectively. This differs slightly from previous studies in which gene identity is also based on other nucleotide differences with little or no spectral effects [16]. We defined the spectral hybrid as having Ala at the reside 180 in the exon 3 of the L opsin gene and as having Ser at the exon 3 of the M opsin gene. The average size of the amplified region among the M opsin sequences was approximately 3.6 kb in each spe-cies, while that of the L opsin gene was approximately 3.9, 3.8 and 3.6 kb in the  <i>Hylobates </i> species,  <i>N. leuco-</i><i>genys </i> and  <i>S. syndactylus</i> , respectively. The size differ-ence of the genomic region between the L and M opsin genes was due to an Alu element in the intron 3 of the L opsin gene. We isolated this genomic region by two sets of PCR schemes. For the first PCR set, we designed an L-specific and an M-specific reverse primer, Ex5L_Rev (5 <b>&#8217; </b> - CCCCAGCA-GACGCAGTACGCAAAGAT -3 <b>&#8217; </b> ) and Ex5M_Rev (5 <b>&#8217; </b> -CCCCAGCAGAAGCAGAATGCCAGGAC -3 <b>&#8217; </b> ), at a region including residue 277 and five surrounding nucleo-tide sites at which the human L (GenBank Accession number Z68193) and M (GenBank AC092402) opsin genes differ. This primer site does not contain the residue 285, which is attributable to the relatively few nucleotide differences reported between L and M immediately down-stream of residue 285. Combined with this, a forward pri-mer common to the human L and M opsin genes was set in intron 2, just upstream of exon 3 (Ex3_For: 5 <b>&#8217; </b> -GGAT-CACAGGTCTCTGGTCTCTG-3 <b>&#8217; </b> ). The nucleotide sequence of exon 3 was then determined for the PCR pro-ducts using a sequencing primer common to the L and M opsin genes (Ex3_Rev: 5 <b>&#8217; </b> -GAGCGTGCAATGTCTAT-CAA-3 <b>&#8217; </b> ) located in intron 3. The nucleotide differences in the region used for the gene-specific reverse primers and in the residue 285 downstream of the primers in the exon 5 were confirmed by a second PCR set. Our second PCR set was comprised of a reverse primer common to the human L and M opsin genes. It was designed in intron 5 just downstream of exon 5 (Ex5_Rev: 5 <b>&#8217; </b> -GAAATGACCGGGAAAGGCTC-3 <b>&#8217; </b> ). An L- and an M-specific forward primer were designed in exon 3 at a region upstream of the site 180, Ex3L_For (5 <b>&#8217; </b> -TTTCCT GGGAGAGGTGGCTGGTGGTG-3 <b>&#8217; </b> ) and Ex3M_For (5 <b>&#8217; </b> -TTTCCTGGGAGAGATGGATGGTGGT C-3 <b>&#8217; </b> ), so as to include three differing nucleotide sites between the pub-lished human L and M opsin genes. The nucleotide sequence of exon 5 was then determined for the PCR pro-duct using a common sequencing primer between the L and M opsin genes (Ex5_For: 5 <b>&#8217; </b> -CACTCAGGGCTGGAA-GATGGC-3 <b>&#8217; </b> ) designed in intron 4. The nucleotide differ-ences in the region used for the gene-specific forward primers in exon 3 were confirmed by the first PCR set described above. By using the combination of these two PCR sets, we avoided the possible misinterpretation of the presence or absence of a gene type by false-positive or false-negative amplification by the gene-specific primers. This is an improvement from previous studies relying on the suc-cess of PCR using gene-specific primers [16,21]. PCRs were carried out in 25  &#956;  l containing 1.5 units of Ex Taq polymerase hot start version (Takara, Tokyo) with 1&#215; Ex Taq Buffer, 0.2 mM each of dNTPs, 1  &#956;  M each of the forward and reverse primers, and ca. 5 ng of the gibbon genomic DNA at 94&#176;C for 10 min followed by 35 cycles at 94&#176;C for 30 sec, 65&#176;C for 30 sec, and 72&#176;C for 4 min. Distilled water was used as the template for the negative control in every reaction. The amplification was confirmed by 0.5% agarose-gel electrophoresis and the <p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 4 of 14</b> amplified DNA fragments were purified using the Mon-tage PCR Centrifugal Filter Device (Millipore, Tokyo). Pruified DNA samples were directly sequenced using Applied Biosystems model 3130 automatic sequencer with Big Dye Terminator v3.1 Cycle Sequencing kit (Applied Biosystems Japan, Tokyo) and sequencing primers (Ex3_Rev, Ex5_For and other primers set inside the ampli-fied region). Operationally, we first sequenced the L and M opsin genes of both DNA strands from a sample of a male as only one X-chromosome is present. We sequenced only one strand for other samples if there was no nucleotide difference from the ones for which both strands were sequenced. When we found nucleotide differ-ences in two or more individuals at the same site, we sequenced both strands of at least one of the samples to confirm the variation. When we found a different nucleo-tide in only one sample,  <i>i.e</i> . as a singleton among the sam-ples, we repeated the PCR for the sample and sequenced the region for both strands to rule out a PCR error and to confirm the variation. When a sample from a male showed two nucleotide peaks at one or more sites ( <b>&#8220; </b> heterozygous <b>&#8221; </b> sites) in the L or M opsin gene, the individual should have two or more loci of the gene with different sequences. In this case, we conservatively inferred two loci for this gene. When a female showed heterozygous sites, the individual could have allelic differences or two or more copies of the gene. We regarded such females as having one locus with differ-ent alleles for this opsin type because this arrangement gives the minimum possible and the most conservative estimate of the number of sequences. Thus, the sequence number could be underrepresented in our results and one gene group ( <i>i.e</i> . the L or M opsin group) does not necessa-rily consist of orthologous members. When we confined an analysis to the orthologous gene groups, we used only male samples with single L and M opsin genes. We did not attempt to separate the sequences by DNA cloning. In the alignment of the nucleotide sequences (Additional files 2, 3, 4, 5 and 6), we treated these sequences as two alternate gene sequences and arbitrarily assigned the nucleotides at the heterozygous sites into the two sequences. For calculations of the nucleotide diversity within a gene group and the nucleotide diver-gence between gene groups (see below for details), the nucleotide combination at the heterozygous sites (haplo-type) is not necessary. When we conducted an analysis requiring the haplotype information, such as the GENE-CONV procedure (described below) or the evaluation of linkage disequilibrium, we used only male samples with single L and M opsin genes. <p/><b>Nucleotide sequencing of the neutral reference regions</b> The eta-globin gene is suitable as a neutral reference because it is a single copy pseudogene well characterized for primates [42] and the ortholog for a gibbon ( <i>H. lar</i> ) has been reported (GenBank M54985) [43]. A pair of PCR primers was designed according to the gibbon sequence to amplify ca. 0.5 kb in the 5 <b>&#8217; </b> -flanking region that is free of repeat sequences: forward, 5 <b>&#8217; </b> -AGAGGA-GAAGTAAAAAGCCAC-3 <b>&#8217; </b> ; reverse, 5 <b>&#8217; </b> -ACATAACTCT-CAAAATCCCAC-3 <b>&#8217; </b> . The S opsin gene intron 4 was also used as a neutral reference. A pair of PCR primers was designed according to the human S opsin sequence (Gen-Bank L32835) to amplify ca. 0.5 kb that is free of repeat sequences: forward, 5 <b>&#8217; </b> -CCCTGCCAACTTTTAGCTTG-CAC-3 <b>&#8217; </b> ; reverse, 5 <b>&#8217; </b> -TTCCCGCACCATCTCCATGAT-3 <b>&#8217; </b> . The PCRs were carried out with the same composition of the reaction mixture as used for the L/M opsin gene. Purification and sequencing were carried out as previously described for the L/M opsin gene. As was the case with the L/M opsin gene, we did not clone the neutral refer-ences and thus the two allelic DNA sequences of indivi-duals were not separated. In the alignment (Additional files 7, 8), two heterozygous alleles were tentatively sorted into two allele sequences. In each of the five gibbon species, the nucleotide sequences of the L and M opsin genes from one male sample with single L and M opsin genes have been depos-ited to the GenBank/EMBL/DDBJ database. The nucleo-tide sequences of the eta-globin 5 <b>&#8217; </b> flanking and the S opsin intron 4 sequences were also deposited to the data-base from one individual each from the five species who had no heterozygous sites. Their sequence ID in this study (see also Additional files 2, 7, 8) and the accession num-bers in the database are listed in Table 2. <p/><b>Data analysis</b> Alignment of the nucleotide sequences was conducted using the Clustal W software [44] and was adjusted manually. The average number of between-group nucleo-tide differences per nucleotide site was designated the nucleotide divergence ( <i>d</i> ). The  <i>d </i> value between the L and M opsin genes within a species was calculated by elimi-nating all positions containing gaps from the alignment. The average number of within-group nucleotide differ-ences per nucleotide site between two sequences was designated the nucleotide diversity ( &#960;  ) and was calculated by eliminating all positions containing gaps from the alignment. The average numbers of between- and within-group synonymous differences per synonymous site and non-synonymous differences per non-synonymous site were calculated using the Nei-Gojobori method [45]. We connected the alignments of the eta-globin pseudogene and the S opsin gene intron 4 to calculate the  &#960;  value of the combined neutral references for the individuals for which both genes were sequenced. The standard errors of the  &#960;  and  <i>d </i> values including those of their synon-ymous and non-synonymous components were estimated <p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 5 of 14</b> <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.5.2.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.5.2.svg" target="separate">SVG</a></div>by a bootstrap procedure (1000 replicates). All of these computations were conducted using MEGA version 5 [46]. The statistical significance of the difference between two  <i>d </i> value or two  &#960;  values was evaluated by the one-tailed Z test. For constructing an among-group phylogenetic tree, we corrected the  <i>d </i> values for multiple substitutions by the Jukes-Cantor formula [47] and used the neighbor-joining method [48] by eliminating all positions contain-ing gaps from the alignment. The exon and intron sequences of a human L (GenBank Z68193) and M (GenBank AC092402) opsin genes and the exon sequence of a crab-eating macaque L (GenBank AF158968) and M (GenBank AF158975) opsin genes [23] were also included as well as the exon sequence of a mouse M opsin gene (GenBank AF011389) as an out-group. In the case of the human, macaque and mouse sequences, one gene sequence was regarded as one group. The reliability of the topology of the phylogenetic tree was evaluated by bootstrap resampling (1000x) using the  <b>&#8220; </b> sendbs2 <b>&#8221; </b> program [49]. To compute the bootstrap values, the human, macaque and mouse sequences were input twice in the alignment because the sendbs2 does not accept one sequence as one group. For testing gene conversion, we applied the GENE-CONV 1.81 program [50] implementing Sawyer <b>&#8217; </b> s statisti-cal method [51]. GENECONV searches for a consecutively identical region between two sequences and evaluates the probability of achieving homogeneity when nucleotide dif-ferences are assumed to be randomly distributed through-out the sequence region. <p/><b>Results</b> <p/><b>L/M opsin genotyping</b> On the basis of the amino acid composition at site 180 encoded in the exon 3 and 277 and 285 encoded in exon 5, the L/M opsin genotype was determined for 152 indivi-duals of gibbons from the eight species (Additional file 1, Table S1). All individuals had standard L and M opsin genes. No L/M hybrid genes were found in terms of the three-site composition. <p/><b>Multiple M opsin copies</b> The nucleotide variation was analyzed for five species, <i>H. agilis</i> ,  <i>H. lar, H. pileatus</i> ,  <i>N. leucogenys </i> and  <i>S. syndacty-</i><i>lus </i> for the genomic region encompassing the exon 3 through the exon 5, approximately 3.6~3.9 kb. We excluded  <i>H. klossii</i> ,  <i>H. moloch </i> and  <i>H. muelleri </i> from the subsequent analysis because of their small sample size. A total of 94 individuals including 51 males were subjected for the analysis (Table 1). Among the males, there were no individuals detected that had multiple L opsin genes. On the other hand, multiple M opsin genes were detected in 80.0% of  <i>N. leucogenys</i> , 28.6% of  <i>H. lar</i> , 16.7% of  <i>H. agilis</i> , and 10.0% of  <i>H. pileatus </i> and  <i>S. syndactylus </i> males. <p/><b>Nucleotide divergence between L and M opsin genes</b> <b>within species</b> Figure 1 summarizes the nucleotide divergence between the L and M opsin genes in the exons and introns in the five gibbon species (Additional file 1, Table S2 for the values). Exons were generally more divergent than introns. In particular, the divergence of exons 3 and 5 was signifi-cantly higher than that in the introns. Within the exons, however, non-synonymous divergence was lower or not significantly higher than synonymous divergence in any of the three exons in all five species (Additional file 1, Table S3). Both synonymous and non-synonymous diver-gences of exons were significantly larger than the intron divergence in all five species (Figure 1). We also examined male samples with a single M as well as a single L opsin sequence types. There were ten <i>H. agilis</i> , ten  <i>H. lar</i> , nine  <i>H. pileatus</i> , one  <i>N. leucogenys</i> , and nine  <i>S. syndactylus </i> males that had single L and M opsin genes. We excluded the one  <i>N. leucogenys </i> from the calculation because of its inapplicability to the population analysis. As shown in Additional file 1, Figure S1, the pat-tern of nucleotide divergence between the L and M opsin <p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 6 of 14</b> <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.6.2.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.6.2.svg" target="separate">SVG</a></div>genes in the exons and introns in these individuals was essentially the same as the pattern shown in Figure 1. <p/><b>Nucleotide diversity of L and M opsin genes within</b> <b>species</b> Figure 2 summarizes the nucleotide diversity of the L and M opsin exons and introns and of the neutral refer-ences (see Additional file 1, Tables S4 and S5 for the values). As for the neutral references, we also incorpo-rated the nucleotide diversity reported by Kim et al (2011) [52] for the30~75 kb of autosomal regions and the 2~13 kb of X chromosomal regions of these gibbon species. In Figure 2, the nucleotide diversity values of the autosomal neutral references are adjusted (multi-plied by 3/4) for comparison with X-chromosomal L/M opsin genes (see Discussion). In both the L and M opsin genes, contrary to the case with the nucleotide divergence, the nucleotide diversity of the exons was lower overall than that of the introns and in many cases was even zero (Additional file 1, Table S4). Alternatively, introns were as variable as the neutral references (Figure 2). When the three exons and the two introns were combined respectively, the exons were sig-nificantly less variable than the introns (Figure 2). When using only males with single L and M opsin genes, the pattern of nucleotide diversity of L and M opsin genes (Additional file 1, Figure S2) was also the same with that shown in Figure 2. Within the exons, variation consisted mainly of synon-ymous variation (Additional file 1, Figure S3 and Table S6). The synonymous variation was as high as, or lower than, the intron variation. However, the synonymous var-iation showed a large stochastic error compared to the nucleotide diversity in the introns (Figure S3) and the synonymous nucleotide divergence (Figure 1). This is because of the much smaller number of the synonymous nucleotide sites (137~138 bp) in the coding region than in the total length of the introns (3.0~3.3 kb) and the lower level of synonymous nucleotide diversity (0~0.4%) than the synonymous divergence between the L and M opsin genes (6.1~6.4%). <p/><b>Comparison of inter-LM divergence and intra-L/M</b> <b>polymorphism between synonymous sites and introns</b> Using the male samples with single L and M opsin genes, we counted the number of sites fixed with different nucleotides between the L and M opsin genes (inter-LM divergence) and the number of polymorphic nucleotide sites in the L, M or both genes within a species (intra-L/ M polymorphism). We then categorized these sites into non-synonymous and synonymous sites in the exons and intron sites. As shown in Table 3, the relative amount of divergence to polymorphism was significantly smaller in introns than in exon synonymous sites in all of the four species examined ( <i>P</i> &lt;0.02, Fisher <b>&#8217; </b> s exact test). On the other hand, between non-synonymous and synonymous sites within exons, there was no significant difference <p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 7 of 14</b> <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.7.2.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.7.2.svg" target="separate">SVG</a></div>detected in the relative amount of divergence to poly-morphism (data not presented). <p/><b>Among-group phylogenetic tree</b> By combining the three exons and regarding one gene type ( <i>i.e</i> . L or M opsin) in a species as a group, an among-group phylogenetic tree was constructed based <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.7.6.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.7.6.svg" target="separate">SVG</a></div>on the nucleotide divergence between the groups (Fig-ure 3A). The L and M opsin genes were separated at the common ancestor of the human, gibbons and maca-que. In contrast, when the two introns were combined and an among-group phylogenetic tree was constructed for gibbons and human, the L and M opsin groups were clustered in  <i>N. leucogenys</i> ,  <i>S. syndactylus</i> , human, and in the genus  <i>Hylobates </i> (Figure 3B). In both the exon and intron trees,  <i>Nomascus </i> was placed at the most basal position among the gibbons (Figure 3), being consistent with the current knowledge of the gibbon molecular phylogeny [53-57]. <p/><b>Test of gene conversion by GENECONV</b> For the GENECONV analysis we considered only male samples including the one  <i>N. leucogenys </i> sample with sin-gle L and M opsin genes. We detected tracts of the gene conversion in each of the five species predominantly in intron regions with the global  <i>P </i> values of less than 0.05 based on permutations for the entire alignment of the 78 sequences and corrected for multiple comparisons (Figure 4). <p/><b>Discussion</b> In the present study we examined intraspecific genetic variation of gibbon L and M opsin genes. This study also enabled comparison of the intraspecific nucleotide diver-sity between the L/M opsin gene regions and other auto-somal regions as neutral references. In theory, when mutations are selectively neutral and population size is constant through generations, the nucleotide diversity is expected to be equal to the population mutation rate (4 <i>N</i> <sub><i>e</i></sub> &#956;  for autosomal and 3  <i>N</i> <sub><i>e</i></sub> &#956;  for X-chromosomal <p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 8 of 14</b> <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.8.2.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.8.2.svg" target="separate">SVG</a></div><p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 9 of 14</b> <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.9.2.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.9.2.svg" target="separate">SVG</a></div>genes of diploid organisms, where  <i>N</i> <sub><i>e </i></sub> is the effective population size and  &#956;  is the mutation rate per nucleotide site per chromosome per generation) [58]. The nucleo-tide diversity of the gibbon neutral references was in the range of 0.10~0.41% (0.28% on average) when the two references were combined (Additional file 1, Table S5). These values are comparable to the nucleotide diversity of gibbon genomes recently reported by Kim et al. (2011) [52]; 0.16~0.29% (0.25% on average) in the autosomal regions and 0.04~0.18% (0.11% on average) in the X chromosomal regions of the same five species examined in this study (Figure 2 and Additional file 1, Table S5). Consistent with Kim et al. (2011) [52], they are also higher overall than reported nucleotide diversity of auto-somal genes in humans (0.12%), chimpanzees (0.19%), bonobos (0.10%) and gorillas (0.15%) and are comparable with that in orangutans (0.36%) [59-61], suggesting a generally larger  <i>N</i> <sub><i>e </i></sub> in Asian apes than in African apes. Although, our samples were collected from zoos and pri-vately owned animals, the consistency with the large gen-ome data set [52] implies that these samples could fairly represent the diversity of a natural population. The high nucleotide diversity of the reference genes also safely pre-cludes the possibility that our samples are biased to a close kin group. All 152 individuals from the eight species examined had both of the normal L and M opsin genes. Based on the composition of the three amino acid sites relevant to absorption spectra, there was no L/M hybrid gene detected with an intermediate spectral sensitivity. This observation is consistent with previous studies on the genetic variation of the L and M opsin genes in non-human catarrhines reporting no or rare observation of the gene deletion or the hybrid genes [19-21]. Ninety four individuals from five species, including 51 males, were subjected to sequencing of the 3.6~3.9-kb region encompassing exon 3 through exon 5. Among them, plural M opsin genes were detected in 10-80% of males in each species (23.5% of all males). Only a single L opsin sequence was detected in all males. Although more data are desired, these results are comparable to humans where multiple M copies are found in 66% of males of European origin [11] and 56% of Japanese males [62] and more than one L copy is rarely found [63]. Regarding non-human catarrhines, studies concur that a single L opsin copy is generally found in the genomes whereas some studies report that multiple M copies are rare [19,21,23,25] yet other studies report that they are com-mon [26,29]. Thus, among Old World monkeys and apes, there seems to be no clear trend on the copy number var-iation of M opsin gene. There also seems to be no clear correlation between the frequency of X chromosomes carrying multiple copies of the M opsin gene with and the frequency of L or M opsin gene deletion or the hybridization. In this study, we defined the L and M opsin genes on the basis of their deduced amino acid composition at resi-dues 277 and 285 and not on the basis of their position in the L/M opsin gene array on the chromosome. As explained in the Methods, the sequence numbers of the L and M opsin gene groups could be underrepresented and the nucleotide diversity and divergence values for the gene groups could therefore be overestimated. Conceptually, the gene group in this study does not necessarily corre-spond to the  <b>&#8220; </b> orthologous <b>&#8221; </b> group because it allows inclu-sion of duplicated copies. Nevertheless, when we only included the male samples with single L and M opsin genes to confine the analysis to the orthologous gene groups, the pattern of nucleotide divergence and diversity was indistinguishable from that where all samples were used (compare Figure 1 with Additional file 1, Figure S1, and Figure 2 with Additional file 1, Figure S2). Thus, there appears to be little or no distortion made in the pattern of nucleotide diversity and divergence by our collective treat-ment of the gene groups. The within-species nucleotide divergence between L and M opsin genes was significantly higher in the exons than in the introns (Figure 1 and Additional file 1, Figure S1 and Table S2). This is opposite to what is expected for the general pattern of divergence:  <i>i.e</i> . the protein-coding exons are generally more conservative than introns. Within the exons, however, non-synonymous divergence was not sig-nificantly higher than synonymous divergence in any of the three exons in the five species (Figure 1 and Additional file 1, Figure S1 and Table S3). This is an orthodox pattern of nucleotide divergence in coding regions under a func-tional constraint [64]. It is hence unlikely that the higher divergence in exons than in introns is due to divergent natural selection between the two genes acting on the amino acid sequence. The intron divergence was signifi-cantly lower than both of the synonymous and non-synon-ymous divergences in all of the five species (Figure 1 and <p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 10 of 14</b> Additional file 1, Figure S1). The low divergence in introns is likely due to a homogenizing process operating between the two genes,  <i>i.e</i> . gene conversion, and the purifying selection against the gene conversion in the exons. Contrary to the pattern of nucleotide divergence, in both L and M opsin genes the within-species nucleotide diver-sity was lower overall in exons than in introns (Figure 2 and Additional file 1, Figure S2 and Table S4). Within the exons, the variation was largely synonymous (Figure S3 and Additional file 1, Table S6). This is a typical pattern of nucleotide variation in protein-coding exons and introns. The nucleotide diversity of the introns was about the same magnitude with the nucleotide diversity of the autosomal neutral references multiplied by 3/4 and that of the X-chromosomal references in each species (Figure 2 and Additional file 1, Figure S2 and Table S5). Therefore, gene conversion in the introns does not seem to affect the nucleotide diversity, possibly because the numbers of introduced and erased mutations by the gene conversion are canceled out. The contrasting pattern of the nucleotide divergence and diversity between exons and introns was further mani-fested by comparing the relative amount of the inter-LM divergence to the intra-L/M polymorphism between the synonymous sites and the introns. The ratio in the introns was significantly lower than that in the synonymous sites (Table 3). Since the synonymous sites and introns are both expected to be under selective neutrality, this pattern is consistent with the homogenization between the L and M opsin genes by gene conversion in introns. Within the exons, the ratio was not significantly different between synonymous and non-synonymous sites: in both nucleo-tide divergence and diversity the synonymous variation was generally larger than the non-synonymous variation (Figure 1 and Additional file 1, Figures S1 and S3). This suggests that the non-synonymous sites are under selective constraint and the non-synonymous differences between the two genes are protected by purifying selection in con-junction with neighboring synonymous differences from the homogenization by exon gene conversions. The among-group phylogenetic tree for the L and M opsin genes showed a contrasting cluster pattern between cases using exon versus intron regions (Figure 3). In the exon tree, gene duplication appears to have occurred once in the common ancestor of hominoids (gibbons and human) and Old World monkeys (crab-eating macaque) (Figure 3A), whereas in the intron tree, gene duplication appears to have occurred independently in  <i>N. leucogenys</i> , <i>S. syndactylus</i> , a common ancestor of the  <i>Hylobates </i> spe-cies, and humans (Figure 3B). If the phylogenetic represen-tation of the exon tree reflects the true relationships among the gene groups, the topology of the intron tree can be explained by the gene conversion between the duplicated L and M opsin genes at the individual level and by natural selection against the gene conversion involving exons at the population level. On the other hand, if the intron tree reflects the true relationships, the topology of the exon tree is inexplicable. Tracts of gene conversions were assessed by the GENE-CONV statistical method (Figure 4). This method assesses whether the nucleotide differences found between two sequences are randomly distributed along the length of these sequences; it cannot detect gene con-versions if a given region has fewer differences than its flanking regions. It has been advised that other methods, such as phylogenetic trees, have to be used to identify gene conversions ( <i>e.g</i> . see [65]). Our GENECONV result is regarded consistent with the other results since the detected gene conversions were located mostly in the intron areas (Figure 4). In  <i>S. syndactylus</i> , the detected gene conversion included the most part of the exon 4 (Figure 4). Considering that exon 4 contributes much less to the spectral difference between the L and M opsins than the exons 3 and 5 [5] and that the nucleotide divergence between the L and M opsin genes in the exon 4 is smaller than those in the exons 3 and 5 (Figure 1 and Additional file 1, Figure S1), the GENECONV result sug-gest that the purifying selection against gene conversion is weaker in the exons 4 than in the exons 3 and 5. Throughout this study, we found no signature of posi-tive (either directional or balancing) selection acting on the gibbon L and M opsin genes. As shown in Figure 1 and Additional file 1, Figure S1, the nucleotide divergence between the L and M opsin genes at non-synonymous sites was always smaller than that at synonymous sites. The nucleotide divergence between species for the L opsin genes or the M opsin genes at non-synonymous sites was also smaller or not significantly higher than that at synon-ymous sites (data not presented). Using the male samples with single L and M opsin genes ( <i>i.e</i> . the  <b>&#8220; </b> orthologous <b>&#8221;</b> sequence groups of the L and M opsin genes) of  <i>H. agilis</i> , <i>H. lar</i> ,  <i>H. pileatus </i> and  <i>S. syndactylus</i> , we detected no sig-nificant difference in the relative amount of the inter-spe-cific divergence to the intra-specific polymorphism between synonymous and non-synonymous sites in either the L or M opsin gene (the McDonald-Kreitman test [66]) or between the L or M opsin gene and the neutral refer-ences (the HKA test [67]) (data not presented). Using these male samples, we did not find any characteristic hap-lotype structure in terms of linkage disequilibrium (LD) in the approximately 3.6~3.9-kb genomic region of the L and M opsin genes as was not found in a study of chimpanzee L opsin gene [25]. A measure of LD, the D <b>&#8217; </b> , which takes a range from 0 to 1 [58], was nearly 1 throughout the region in all the four species examined (data not presented). This indicates low incidence of recombination between alleles, which is not unusual within such a short region. While the high LD could simply result from a small sample size, <p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 11 of 14</b> it is not contradictory with the frequent incidence of the gene conversion between gene loci because the LD con-cerns recombination between alleles. The present study of within-species nucleotide varia-tion of both the L and M opsin genes in gibbons provides additional support to previous studies suggesting that the gene conversion has homogenized the tandemly dupli-cated L and M opsin genes and their spectral difference is maintained by purifying selection on centrally-located exons in non-human catarrhines [13,14,26-29]. The strict conservation of trichromacy in non-human catarrhines underscores a general belief that trichromacy is selec-tively more advantageous than dichromacy or anomalous trichromacy. Primate trichromacy is hypothesized to be adaptive for detecting anything differing from the back-ground foliage in red-green coloration, the proposed major objects being mature fruits, young leaves, pelage and skin [68-79]. The strict conservation of the trichromacy in non-human catarrhines is in sharp contrast to New World monkeys. In many species of New World monkeys, the spectral variation of the L/M opsin is achieved by an allelic differentiation of a single locus on the X chromosome. This results in polymorphic color vision, consisting of tri-chromacy in females and dichromacy in both sexes [80]. Genetic studies have shown that the allelic difference has been maintained by balancing selection [81-83]. It remains to be elucidated whether the polymorphic color vision is a consequence of selective advantage on trichromats through the overdominance of the L/M opsin heterozy-gotes or the consequence of other mechanisms of balan-cing selection including a mutual benefit among different vision types [83,84]. It has been demonstrated that dichro-mats are superior to trichromats in some visual tasks such as defeating cryptic coloration and motion detection [70,85,86] and have a foraging advantage for surface-dwell-ing insects [87-89]. In non-human catarrhines, however, this benefit of dichromacy could have been overwhelmed by that of trichromacy. It is still an open question whether the difference between the non-human catarrhines (rou-tine trichromacy) and New World monkeys (polymorphic color vision) is attributable to a phenological difference among continents, such as in severity of seasonality and availability of non-colorful and non-seasonal fig and palms [90], to a dietary variation such as in dependency on insets or colorful fruits, or to variation of social color signals [78,79]. The absence of L/M defects in gibbons and other non-human catarrhines reveals the uniqueness of human color vision, in which the deletion of L or M opsin genes and the presence of L/M hybrid genes causes relatively high incidence of dichromacy and anomalous trichro-macy [18,91-94]. Color vision polymorphisms among humans could be explained by frequent gene conversions of exon 3, due to the presence of a recombination hot-spot chi element conserved in the exon [17] and relaxa-tion of the selective constraint to maintain the spectral difference between the L and M opsin genes. Alterna-tively, adaptive explanations are possible. Balancing selec-tion may be acting on subtle changes in light absorption of L [18] and M opsins among females, or a selective advantage could be experienced by dichromatic males during hunting activities or via an increased ability to detect predators via penetrating camouflage [85]. Further population studies of the L/M opsin genes for terrestrial-savanna catarrhines, such as baboons, would be impor-tant for elucidating evolutionary forces acting on devel-opment and maintenance of primate trichromacy. <p/><b>Conclusions</b> The present study of within-species nucleotide variation of the L and M opsin genes in gibbons supports the notion that gene conversion has homogenized the tandemly duplicated L and M opsin genes and that their spectral dif-ference is maintained by the purifying selection on the centrally-located exons in non-human catarrhines. Grow-ing knowledge on genetic variation of opsin genes and visual ecology of primates has led to a paradigm shift toward the view that the adaptive value of trichromacy is conditional rather than universal, depending on the speci-fic ecological demands on animals in their environments. Further population genetic and molecular phylogenic stu-dies of primate visual opsins including humans, as well as observation and experiments on vision-guided primate behaviors, and phenological studies of habitats of wild pri-mates will elucidate the detailed conditions of color vision evolution in primates. <p/><b>Additional material</b> <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.11.7.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.11.7.svg" target="separate">SVG</a></div><p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 12 of 14</b> <div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.12.2.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.12.2.svg" target="separate">SVG</a></div><p>see ACKNOWLEDGMENT</p>This study was supported by Grants-in-Aid for Scientific Research A 19207018 and 22247036 from the Japan Society for the Promotion of Science (JSPS) and Grants-in-Aid for Scientific Research on Priority Areas <p/><b>&#8220; </b> Comparative Genomics <b>&#8221; </b> 20017008 and  <b>&#8220; </b> Cellular Sensor <b>&#8221; </b> 21026007 from the Ministry of Education, Culture, Sports, Science and Technology of Japan to S. K. and by a Grant-in-Aid for Scientific Research B 17405022 from JSPS to AM. We thank Dr. Amanda D. Melin for proofreading and commenting on the manuscript and Dr. Naoki Osada for helpful suggestions. <p>see AUTHORS</p>1 Department of Integrated Biosciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa 277-8562, Japan.  <sup>2</sup> Department of Behavioral and Brain Sciences, Primate Research Institute, Kanrin, Inuyama, Aichi 484-8506, Japan.  <sup>3</sup> Department of Biology, Faculty of Mathematics and Natural Sciences, Bogor Agricultural University, Kampus IPB Darmaga, Bogor 16680, Indonesia.  <sup>4</sup> Primate Research Center, Bogor Agricultural University, Jalan Lodaya II/5, Bogor 16151, Indonesia.  <sup>5</sup> Primate Research Unit, Department of Biology, Faculty of Science, Chulalongkorn University, Bangkok 10330, Thailand.  <sup>6</sup> Research and Conservation Division; Zoological Park Organization (ZPO), Bangkok, Thailand.  <sup>7</sup> Center for Human Evolution Modeling Research, Primate Research Institute, Kanrin, Inuyama, Aichi 484-8506, Japan. 8 Department of Rehabilitaion, Chubu Gakuin University, Kirigaoka 2-1, Seki, Gifu 501-3993, Japan.  <sup>9</sup> Department of Anatomy, Kitasato University School of Medicine, Kitasato 1-15-1, Sagamihara, Kanagawa 252-0374, Japan.  <sup>10</sup> Faculty of Veterinary Medicine, Azabu University, Fuchinobe 1-17-71, Sagamihara, Kanagawa 252-5301 Japan. <p/><b>Authors</b> <b>&#8217; </b> <b>contributions</b> SK and AM conceived of the study. SK designed and supervised the experiments and analyses. MK organized the sample collection and DNA extraction. BSu organized the sample collection in Indonesia. DP conducted DNA extractions in Indonesia. SM organized the sample collection and conducted DNA extractions in Thailand. BSi organized the sample collection in Thailand. SG conducted the veterinary care of the animals during the sampling. TH carried out experiments, data compiling, and analysis. SK and TK performed additional data analyses. SK and TH wrote and illustrated the manuscript. HO advised on the research design and manuscript preparation. All authors read and approved the final manuscript. <p/><b>Received: 26 May 2011 Accepted: 22 October 2011</b> <b>Published: 22 October 2011</b> <p/><b>References</b> <ul><li>1. Nathans J, Thomas D, Hogness DS:  <b>Molecular genetics of human color</b> <b>vision: the genes encoding blue, green, and red pigments. </b> <i>Science </i> 1986, <p/><b>232</b> :193-202. </li><li>2. Jacobs GH:  <b>Primate photopigments and primate color vision. </b> <i>Proc Natl</i> <i>Acad Sci USA </i> 1996,  <b>93</b> :577-581. </li><li>3. Yokoyama R, Yokoyama S:  <b>Convergent evolution of the red- and green-</b><b>like visual pigment genes in fish, </b> <i>Astyanax fasciatus</i> <b>, and human. </b> <i>Proc</i> <i>Natl Acad Sci USA </i> 1990,  <b>87</b> :9315-9318. </li><li>4. Neitz M, Neitz J, Jacobs GH:  <b>Spectral tuning of pigments underlying red-</b><b>green color vision. </b> <i>Science </i> 1991,  <b>252</b> :971-974. </li><li>5. Asenjo AB, Rim J, Oprian DD:  <b>Molecular determinants of human red/</b> <b>green color discrimination. </b> <i>Neuron </i> 1994,  <b>12</b> :1131-1138. </li><li>6. Yokoyama S, Radlwimmer FB:  <b>The </b> <b>&#8220; </b> <b>five-sites</b> <b>&#8221; </b> <b>rule and the evolution of</b> <b>red and green color vision in mammals. </b> <i>Mol Biol Evol </i> 1998,  <b>15</b> :560-567. </li><li>7. Yokoyama S, Radlwimmer FB:  <b>The molecular genetics of red and green</b> <b>color vision in mammals. </b> <i>Genetics </i> 1999,  <b>153</b> :919-932. </li><li>8. Yokoyama S, Radlwimmer FB:  <b>The molecular genetics and evolution of</b> <b>red and green color vision in vertebrates. </b> <i>Genetics </i> 2001,  <b>158</b> :1697-1710. </li><li>9. Chan T, Lee M, Sakmar TP:  <b>Introduction of hydroxyl-bearing amino acids</b> <b>causes bathochromic spectral shifts in rhodopsin. Amino acid</b> </li></ul><ul><li>10. Deeb SS:  <b>Genetics of variation in human color vision and the retinal</b> <b>cone mosaic. </b> <i>Curr Opin Genet Dev </i> 2006,  <b>16</b> :301-307. </li><li>11. Drummond-Borg M, Deeb SS, Motulsky AG:  <b>Molecular patterns of X</b> <b>chromosome-linked color vision genes among 134 men of European</b> <b>ancestry. </b> <i>Proc Natl Acad Sci USA </i> 1989,  <b>86</b> :983-987. </li><li>12. Jorgensen AL, Deeb SS, Motulsky AG:  <b>Molecular genetics of X</b> <b>chromosome-linked color vision among populations of African and</b> <b>Japanese ancestry: high frequency of a shortened red pigment gene</b> <b>among Afro-Americans. </b> <i>Proc Natl Acad Sci USA </i> 1990,  <b>87</b> :6512-6516. </li><li>13. Shyue SK, Li L, Chang BH, Li WH:  <b>Intronic gene conversion in the</b> <b>evolution of human X-linked color vision genes. </b> <i>Mol Biol Evol </i> 1994, <p/><b>11</b> :548-551. </li><li>14. Zhou YH, Li WH:  <b>Gene conversion and natural selection in the evolution</b> <b>of X-linked color vision genes in higher primates. </b> <i>Mol Biol Evol </i> 1996, <p/><b>13</b> :780-783. </li><li>15. Zhao Z, Hewett-Emmett D, Li WH:  <b>Frequent gene conversion between</b> <b>human red and green opsin genes. </b> <i>J Mol Evol </i> 1998,  <b>46</b> :494-496. </li><li>16. Winderickx J, Lindsey DT, Sanocki E, Teller DY, Motulsky AG, Deeb SS: <p/><b>Polymorphism in red photopigment underlies variation in colour</b> <b>matching. </b> <i>Nature </i> 1992,  <b>356</b> :431-433. </li><li>17. Winderickx J, Battisti L, Hibiya Y, Motulsky AG, Deeb SS:  <b>Haplotype diversity</b> <b>in the human red and green opsin genes: evidence for frequent</b> <b>sequence exchange in exon 3. </b> <i>Hum Mol Genet </i> 1993,  <b>2</b> :1413-1421. </li><li>18. Verrelli BC, Tishkoff SA:  <b>Signatures of selection and gene conversion</b> <b>associated with human color vision variation. </b> <i>Am J Hum Genet </i> 2004, <p/><b>75</b> :363-375. </li><li>19. Onishi A, Koike S, Ida M, Imai H, Shichida Y, Takenaka O, Hanazawa A, Komatsu H, Mikami A, Goto S,  <i>et al</i> :  <b>Dichromatism in macaque monkeys.</b> <i>Nature </i> 1999,  <b>402</b> :139-140. </li><li>20. Jacobs GH, Williams GA:  <b>The prevalence of defective color vision in Old</b> <b>World monkeys and apes. </b> <i>Col Res Appl </i> 2001,  <b>26(Suppl)</b> :S123-S127. </li><li>21. Terao K, Mikami A, Saito A, Itoh S, Ogawa H, Takenaka O, Sakai T, Onishi A, Teramoto M, Udono T,  <i>et al</i> :  <b>Identification of a protanomalous</b> <b>chimpanzee by molecular genetic and electroretinogram analyses. </b> <i>Vision</i> <i>Res </i> 2005,  <b>45</b> :1225-1235. </li><li>22. Hanazawa A, Mikami A, Sulistyo Angelika P, Takenaka O, Goto S, Onishi A, Koike S, Yamamori T, Kato K, Kondo A,  <i>et al</i> :  <b>Electroretinogram analysis of</b> <b>relative spectral sensitivity in genetically identified dichromatic</b> <b>macaques. </b> <i>Proc Natl Acad Sci USA </i> 2001,  <b>98</b> :8124-8127. </li><li>23. Onishi A, Koike S, Ida-Hosonuma M, Imai H, Shichida Y, Takenaka O, Hanazawa A, Komatsu H, Mikami A, Goto S,  <i>et al</i> :  <b>Variations in long- and</b> <b>middle-wavelength-sensitive opsin gene loci in crab-eating monkeys.</b> <i>Vision Res </i> 2002,  <b>42</b> :281-292. </li><li>24. Saito A, Mikami A, Hasegawa T, Koida K, Terao K, Koike S, Onishi A, Takenaka O, Teramoto M, Mori Y:  <b>Behavioral evidence of color vision</b> <b>deficiency in a protanomalia chimpanzee (</b> <i>Pan troglodytes</i> <b>). </b> <i>Primates</i> 2003,  <b>44</b> :171-176. </li><li>25. Verrelli BC, Lewis CM jr, Stone AC, Perry GH:  <b>Different selective pressures</b> <b>shape the molecular evolution of color vision in chimpanzee and</b> <b>human populations. </b> <i>Mol Biol Evol </i> 2008,  <b>25</b> :2735-2743. </li><li>26. Ibbotson RE, Hunt DM, Bowmaker JK, Mollon JD:  <b>Sequence divergence</b> <b>and copy number of the middle- and long-wave photopigment genes</b> <b>in Old World monkeys. </b> <i>Proc R Soc Lond B </i> 1992,  <b>247</b> :145-154. </li><li>27. Balding DJ, Nichols RA, Hunt DM:  <b>Detecting gene conversion: primate</b> <b>visual pigment genes. </b> <i>Proc R Soc B </i> 1992,  <b>249</b> :275-280. </li><li>28. Deeb SS, Jorgensen AL, Battisti L, Iwasaki L, Motulsky AG:  <b>Sequence</b> <b>divergence of the red and green visual pigments in great apes and</b> <b>humans. </b> <i>Proc Natl Acad Sci USA </i> 1994,  <b>91</b> :7262-7266. </li><li>29. Dulai KS, Bowmaker JK, Mollon JD, Hunt DM:  <b>Sequence divergence,</b> <b>polymorphism and evolution of the middle-wave and long-wave visual</b> <b>pigment genes of great apes and Old World monkeys. </b> <i>Vision Res </i> 1994, <p/><b>34</b> :2483-2491. </li><li>30. Deegan JF, Jacobs GH:  <b>Spectral sensitivity of gibbons: implications for</b> <b>photopigments and color vision. </b> <i>Folia Primatol </i> 2001,  <b>72</b> :26-29. </li><li>31. Fleagle JG:  <i>Primate Adaptation and Evolution. </i> 2 edition. San Diego: Academic Press; 1999. </li><li>32. Macke JP, Nathans J:  <b>Individual variation in size of the human red and</b> <b>green visual pigment gene array. </b> <i>Invest Ophthalmol Vis Sci </i> 1997, <p/><b>38</b> :1040-1043. </li></ul><p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 13 of 14</b> <ul><li>33. Kawamura S, Hirai M, Takenaka O, Radlwimmer FB, Yokoyama S:  <b>Genomic</b> <b>and spectral analyses of long to middle wavelength-sensitive visual</b> <b>pigments of common marmoset (</b> <i>Callithrix jacchus</i> <b>). </b> <i>Gene </i> 2001,  <b>269</b> :45-51. </li><li>34. Nagao K, Takenaka N, Hirai M, Kawamura S:  <b>Coupling and decoupling of</b> <b>evolutionary mode between X- and Y-chromosomal red-green opsin</b> <b>genes in owl monkeys. </b> <i>Gene </i> 2005,  <b>352</b> :82-91. </li><li>35. Nathans J, Piantanida TP, Eddy RL, Shows TB, Hogness DS:  <b>Molecular</b> <b>genetics of inherited variation in human color vision. </b> <i>Science </i> 1986, <p/><b>232</b> :203-210. </li><li>36. Vollrath D, Nathans J, Davis RW:  <b>Tandem array of human visual pigment</b> <b>genes at Xq28. </b> <i>Science </i> 1988,  <b>240</b> :1669-1672. </li><li>37. Neitz J, Neitz M, Kainz PM:  <b>Visual pigment gene structure and the</b> <b>severity of color vision defects. </b> <i>Science </i> 1996,  <b>274</b> :801-804. </li><li>38. Kainz PM, Neitz M, Neitz J:  <b>Molecular genetic detection of female carriers</b> <b>of protan defects. </b> <i>Vision Res </i> 1998,  <b>38</b> :3365-3369. </li><li>39. Hayashi T, Motulsky AG, Deeb SS:  <b>Position of a </b> <b>&#8216; </b> <b>green-red</b> <b>&#8217; </b> <b>hybrid gene in</b> <b>the visual pigment array determines colour-vision phenotype. </b> <i>Nat Genet</i> 1999,  <b>22</b> :90-93. </li><li>40. Oda S, Ueyama H, Nishida Y, Tanabe S, Yamade S:  <b>Analysis of L-cone/M-</b><b>cone visual pigment gene arrays in females by long-range PCR. </b> <i>Vision</i> <i>Res </i> 2003,  <b>43</b> :489-495. </li><li>41. Oprian DD, Asenjo AB, Lee N, Pelletier SL:  <b>Design, chemical synthesis, and</b> <b>expression of genes for the three human color vision pigments.</b> <i>Biochemistry </i> 1991,  <b>30</b> :11367-11372. </li><li>42. Koop BF, Goodman M, Xu P, Chan K, Slightom JL:  <b>Primate eta-globin DNA</b> <b>sequences and man</b> <b>&#8217; </b> <b>s place among the great apes. </b> <i>Nature </i> 1986, <p/><b>319</b> :234-238. </li><li>43. Bailey WJ, Fitch DH, Tagle DA, Czelusniak J, Slightom JL, Goodman M: <p/><b>Molecular evolution of the psi eta-globin gene locus: gibbon phylogeny</b> <b>and the hominoid slowdown. </b> <i>Mol Biol Evol </i> 1991,  <b>8</b> :155-184. </li><li>44. Thompson JD, Higgins DG, Gibson TJ:  <b>CLUSTAL W: improving the</b> <b>sensitivity of progressive multiple sequence alignment through</b> <b>sequence weighting, position-specific gap penalties and weight matrix</b> <b>choice. </b> <i>Nucleic Acids Res </i> 1994,  <b>22</b> :4673-4680. </li><li>45. Nei M, Gojobori T:  <b>Simple methods for estimating the numbers of</b> <b>synonymous and nonsynonymous nucleotide substitutions. </b> <i>Mol Biol Evol</i> 1986,  <b>3</b> :418-426. </li><li>46. Tamura K, Peterson D, Peterson N, Stecher G, Nei M, Kumar S:  <b>MEGA5:</b> <b>Molecular Evolutionary Genetics Analysis using maximum likelihood,</b> <b>evolutionary distance, and maximum parsimony methods. </b> <i>Mol Biol Evol</i> 2011,  <b>28</b> :2731-2739. </li><li>47. Nei M, Kumar S:  <i>Molecular Evolution and Phylogenetics </i> New York: Oxford university press; 2000. </li><li>48. Saitou N, Nei M:  <b>The neighbor-joining method: a new method for</b> <b>reconstructing phylogenetic trees. </b> <i>Mol Biol Evol </i> 1987,  <b>4</b> :406-425. </li><li>49.  <b>Evolutionary Genomics Lab. </b> [http://www.kms.ac.jp/~genomelb/takezaki. eng.html]. </li><li>50.  <b>GENECONV Molecular Biology Computer Program. </b> [http://www.math. wustl.edu/~sawyer/geneconv/index.html]. </li><li>51. Sawyer S:  <b>Statistical tests for detecting gene conversion. </b> <i>Mol Biol Evol</i> 1989,  <b>6</b> :526-538. </li><li>52. Kim SK, Carbone L, Becquet C, Mootnick AR, Li DJ, de Jong PJ, Wall JD: <p/><b>Patterns of genetic variation within and between gibbon species. </b> <i>Mol</i> <i>Biol Evol </i> 2011,  <b>28</b> :2211-2218. </li><li>53. Roos C, Geissmann T:  <b>Molecular phylogeny of the major hylobatid</b> <b>divisions. </b> <i>Mol Phylogenet Evol </i> 2001,  <b>19</b> :486-494. </li><li>54. Chatterjee HJ:  <b>Phylogeny and biogeography of gibbons: A dispersal-</b><b>vicariance analysis. </b> <i>Int J Primatol </i> 2006,  <b>27</b> :699-712. </li><li>55. Chan YC, Roos C, Inoue-Murayama M, Inoue E, Shih CC, Pei KJ, Vigilant L: <p/><b>Mitochondrial genome sequences effectively reveal the phylogeny of</b> <i>Hylobates </i> <b>gibbons. </b> <i>PLoS ONE </i> 2010,  <b>5</b> :e14419. </li><li>56. Matsudaira K, Ishida T:  <b>Phylogenetic relationships and divergence dates</b> <b>of the whole mitochondrial genome sequences among three gibbon</b> <b>genera. </b> <i>Mol Phylogenet Evol </i> 2010,  <b>55</b> :454-459. </li><li>57. Israfil H, Zehr SM, Mootnick AR, Ruvolo M, Steiper ME:  <b>Unresolved</b> <b>molecular phylogenies of gibbons and siamangs (Family: Hylobatidae)</b> <b>based on mitochondrial, Y-linked, and X-linked loci indicate a rapid</b> <b>Miocene radiation or sudden vicariance event. </b> <i>Mol Phylogenet Evol </i> 2011, <p/><b>58</b> :447-455. </li><li>58. Hartl DL, Clark AG:  <i>Principles of Population Genetics. </i> 4 edition. Sunderland: Sinauer Associates; 2007. </li></ul><ul><li>59. Nachman MW:  <b>Single nucleotide polymorphisms and recombination rate</b> <b>in humans. </b> <i>Trends Genet </i> 2001,  <b>17</b> :481-485. </li><li>60. Tishkoff SA, Verrelli BC:  <b>Patterns of human genetic diversity: implications</b> <b>for human evolutionary history and disease. </b> <i>Annu Rev Genomics Hum</i> <i>Genet </i> 2003,  <b>4</b> :293-340. </li><li>61. Fischer A, Pollack J, Thalmann O, Nickel B, Paabo S:  <b>Demographic history</b> <b>and genetic differentiation in apes. </b> <i>Curr Biol </i> 2006,  <b>16</b> :1133-1138. </li><li>62. Hayashi S, Ueyama H, Tanabe S, Yamade S, Kani K:  <b>Number and variations</b> <b>of the red and green visual pigment genes in Japanese men with</b> <b>normal color vision. </b> <i>Jpn J Ophthalmol </i> 2001,  <b>45</b> :60-67. </li><li>63. Sharpe LT, Stockman A, Jagle H, Nathans J:  <b>Opsin genes, cone</b> <b>photopigments, color vision, and color blindness. </b> In  <i>Color vision: from</i> <i>genes to perception. </i> Edited by: Gegenfurtner KR, Sharpe LT. Cambridge: Cambridge University Press; 1999:3-51. </li><li>64. Graur D, Li W-H:  <i>Fundamentals of Molecular Evolution. </i> 2 edition. Sunderland: Sinauer Publishers; 2000. </li><li>65. Petronella N, Drouin G:  <b>Gene conversions in the growth hormone gene</b> <b>family of primates: Stronger homogenizing effects in the Hominidae</b> <b>lineage. </b> <i>Genomics </i> 2011,  <b>98</b> :173-181. </li><li>66. McDonald JH, Kreitman M:  <b>Adaptive protein evolution at the </b> <i>Adh </i> <b>locus in</b> <i>Drosophila</i> <b>. </b> <i>Nature </i> 1991,  <b>351</b> :652-654. </li><li>67. Hudson RR, Kreitman M, Aguade M:  <b>A test of neutral molecular evolution</b> <b>based on nucleotide data. </b> <i>Genetics </i> 1987,  <b>116</b> :153-159. </li><li>68. Regan BC, Julliot C, Simmen B, Vienot F, Charles-Dominique P, Mollon JD: <p/><b>Frugivory and colour vision in </b> <i>Alouatta seniculus</i> <b>, a trichromatic</b> <b>platyrrhine monkey. </b> <i>Vision Res </i> 1998,  <b>38</b> :3321-3327. </li><li>69. Sumner P, Mollon JD:  <b>Catarrhine photopigments are optimized for</b> <b>detecting targets against a foliage background. </b> <i>J Exp Biol </i> 2000, <p/><b>203</b> :1963-1986. </li><li>70. Regan BC, Julliot C, Simmen B, Vienot F, Charles-Dominique P, Mollon JD: <p/><b>Fruits, foliage and the evolution of primate colour vision. </b> <i>Phil Trans R Soc</i> <i>B </i> 2001,  <b>356</b> :229-283. </li><li>71. Sumner P, Mollon JD:  <b>Colors of primate pelage and skin: objective</b> <b>assessment of conspicuousness. </b> <i>Am J Primatol </i> 2003,  <b>59</b> :67-91. </li><li>72. Lucas PW, Darvell BW, Lee PKD, Yuen TDB, Choong MF:  <b>Colour cues for</b> <b>leaf food selection by long-tailed macaques (</b> <i>Macaca fascicularis</i> <b>) with a</b> <b>new suggestion for the evolution of trichromatic colour vision. </b> <i>Folia</i> <i>Primatol </i> 1998,  <b>69</b> :139-154. </li><li>73. Dominy NJ, Lucas PW:  <b>Ecological importance of trichromatic vision to</b> <b>primates. </b> <i>Nature </i> 2001,  <b>410</b> :363-366. </li><li>74. Lucas PW, Dominy NJ, Riba-Hern&#225;ndez P, Stoner KE, Yamashita N, Loria-Calderon E, Petersen-Pereira W, Rojas-Duran Y, Salas-Pena R, Solis-Madrigal S,  <i>et al</i> :  <b>Evolution and function of routine trichromatic vision in</b> <b>primates. </b> <i>Evolution </i> 2003,  <b>57</b> :2636-2643. </li><li>75. Allen G:  <i>The Color Sense: Its Origin and Development </i> London: Trubner &amp; Co; 1879. </li><li>76. Surridge AK, Osorio D, Mundy NI:  <b>Evolution and selection of trichromatic</b> <b>vision in primates. </b> <i>Trends Ecol Evol </i> 2003,  <b>18</b> :198-205. </li><li>77. Vorobyev M:  <b>Ecology and evolution of primate colour vision. </b> <i>Clin Exp</i> <i>Optom </i> 2004,  <b>87</b> :230-238. </li><li>78. Changizi MA, Zhang Q, Shimojo S:  <b>Bare skin, blood and the evolution of</b> <b>primate color vision. </b> <i>Biol Lett </i> 2006,  <b>2</b> :217-221. </li><li>79. Fernandez AA, Morris MR:  <b>Sexual selection and trichromatic color vision</b> <b>in primates: statistical support for the preexisting-bias hypothesis. </b> <i>Am</i> <i>Nat </i> 2007,  <b>170</b> :10-20. </li><li>80. Jacobs GH:  <b>New World monkeys and color. </b> <i>Int J Primatol </i> 2007, <p/><b>28</b> :729-759. </li><li>81. Boissinot S, Tan Y, Shyue SK, Schneider H, Sampaio I, Neiswanger K, Hewett-Emmett D, Li WH:  <b>Origins and antiquity of X-linked triallelic color vision</b> <b>systems in New World monkeys. </b> <i>Proc Natl Acad Sci USA </i> 1998, <p/><b>95</b> :13749-13754. </li><li>82. Surridge AK, Mundy NI:  <b>Trans-specific evolution of opsin alleles and the</b> <b>maintenance of trichromatic colour vision in Callitrichine primates. </b> <i>Mol</i> <i>Ecol </i> 2002,  <b>11</b> :2157-2169. </li><li>83. Hiwatashi T, Okabe Y, Tsutsui T, Hiramatsu C, Melin AD, Oota H, Schaffner CM, Aureli F, Fedigan LM, Innan H, Kawamura S:  <b>An explicit</b> <b>signature of balancing selection for color-vision variation in new world</b> <b>monkeys. </b> <i>Mol Biol Evol </i> 2010,  <b>27</b> :453-464. </li><li>84. Mollon JD, Bowmaker JK, Jacobs GH:  <b>Variations of colour vision in a New</b> <b>World primate can be explained by polymorphism of retinal</b> <b>photopigments. </b> <i>Proc R Soc Lond B </i> 1984,  <b>222</b> :373-399. </li></ul><p/><b>Hiwatashi </b> <i>et al</i> <b>. </b> <i>BMC Evolutionary Biology </i> <b>2011, </b> <b>11</b> <b>:312</b> <b>http://www.biomedcentral.com/1471-2148/11/312</b> <p/><b>Page 14 of 14</b> <ul><li>85. Morgan MJ, Adam A, Mollon JD:  <b>Dichromats detect colour-camouflaged</b> <b>objects that are not detected by trichromats. </b> <i>Proc R Soc Lond B </i> 1992, <p/><b>248</b> :291-295. </li><li>86. Saito A, Mikami A, Kawamura S, Ueno Y, Hiramatsu C, Widayati KA, Suryobroto B, Teramoto M, Mori Y, Nagano K,  <i>et al</i> :  <b>Advantage of</b> <b>dichromats over trichromats in discrimination of color-camouflaged</b> <b>stimuli in nonhuman primates. </b> <i>Am J Primatol </i> 2005,  <b>67</b> :425-436. </li><li>87. Melin AD, Fedigan LM, Hiramatsu C, Sendall C, Kawamura S:  <b>Effects of</b> <b>colour vision phenotype on insect capture by a free-ranging population</b> <b>of white-faced capuchins (</b> <i>Cebus capucinus</i> <b>). </b> <i>Anim Behav </i> 2007,  <b>73</b> :205-214. </li><li>88. Melin AD, Fedigan LM, Young HC, Kawamura S:  <b>Can color vision variation</b> <b>explain sex differences in invertebrate foraging by capuchin monkeys?</b> <i>Curr Zool </i> 2010,  <b>56</b> :300-312. </li><li>89. Caine NG, Osorio D, Mundy NI:  <b>A foraging advantage for dichromatic</b> <b>marmosets (</b> <i>Callithrix geoffroyi</i> <b>) at low light intensity. </b> <i>Biol Lett </i> 2010, <p/><b>6</b> :36-38. </li><li>90. Dominy NJ, Svenning JC, Li WH:  <b>Historical contingency in the evolution</b> <b>of primate color vision. </b> <i>J Hum Evol </i> 2003,  <b>44</b> :25-45. </li><li>91. Verhulst S, Maes FW:  <b>Scotopic vision in colour-blinds. </b> <i>Vision Res </i> 1998, <p/><b>38</b> :3387-3390. </li><li>92. Bosten JM, Robinson JD, Jordan G, Mollon JD:  <b>Multidimensional scaling</b> <b>reveals a color dimension unique to </b> <b>&#8216; </b> <b>color deficient</b> <b>&#8217; </b> <b>observers. </b> <i>Curr Biol</i> 2005,  <b>15</b> :R950-R952. </li><li>93. Hood SM, Mollon JD, Purves L, Jordan G:  <b>Color discrimination in carriers</b> <b>of color deficiency. </b> <i>Vision Res </i> 2006,  <b>46</b> :2894-2900. </li><li>94. Sharpe LT, de Luca E, Hansen T, Jagle H, Gegenfurtner KR:  <b>Advantages and</b> <b>disadvantages of human dichromacy. </b> <i>J Vis </i> 2006,  <b>6</b> :213-223. </li></ul><div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.14.3.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.14.3.svg" target="separate">SVG</a></div><div><img src="../../../target/output/multiple-1471-2148-11-312/image.g.14.4.png" width="20%"/><a href="../../../target/output/multiple-1471-2148-11-312/image.g.14.4.svg" target="separate">SVG</a></div></div>
