<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html id="js" xmlns="http://www.w3.org/1999/xhtml" xmlns:m="http://www.w3.org/1998/Math/MathML" xmlns:og="http://ogp.me/ns#" xml:lang="en-GB" xmlns:wb="“http://open.weibo.com/wb”" lang="en-GB"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">

   
  
      
  
                  
  
                  
  
            
  
            
            
    <title>BMC Evolutionary Biology | Full text | Revealing the maternal
 demographic history of Panthera leo using ancient DNA and a spatially 
explicit genealogical analysis</title>
<link rel="shortcut icon" type="image/x-icon" href="http://www.biomedcentral.com/images/icon/1006/favicon.ico">
<link rel="apple-touch-icon" sizes="72x72" href="http://www.biomedcentral.com/sites/1006/images/70.gif">
<meta http-equiv="X-UA-Compatible" content="IE=7">


                
    
		
		
		
		
<!-- <?xml version="1.0" encoding="UTF-8"?>
<rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns="http://purl.org/rss/1.0/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/">
<cc:Work rdf:about="http://www.biomedcentral.com/1471-2148/14/70">
<cc:license rdf:resource="http://creativecommons.org/licenses/by/2.0/"/>
</cc:Work>
<cc:License rdf:about="http://creativecommons.org/licenses/by/2.0/">
<cc:permits rdf:resource="http://web.resource.org/cc/Reproduction"/>
<cc:permits rdf:resource="http://web.resource.org/cc/Distribution"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Notice"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Attribution"/>
<cc:permits rdf:resource="http://web.resource.org/cc/DerivativeWorks"/>
</cc:License>
<item rdf:about="http://www.biomedcentral.com/1471-2148/14/70">
<title>Revealing the maternal demographic history of Panthera leo using ancient DNA and a spatially explicit genealogical analysis</title>
<dc:title>Revealing the maternal demographic history of Panthera leo using ancient DNA and a spatially explicit genealogical analysis</dc:title>
<dc:creator>Barnett, Ross</dc:creator>
<dc:creator>Yamaguchi, Nobuyuki</dc:creator>
<dc:creator>Shapiro, Beth</dc:creator>
<dc:creator>Ho, Simon YW</dc:creator>
<dc:creator>Barnes, Ian</dc:creator>
<dc:creator>Sabin, Richard</dc:creator>
<dc:creator>Werdelin, Lars</dc:creator>
<dc:creator>Cuisin, Jacques</dc:creator>
<dc:creator>Larson, Greger</dc:creator>
<dc:identifier>info:doi/10.1186/1471-2148-14-70</dc:identifier>
<dc:identifier>info:pmid/24690312</dc:identifier>
<dc:source>BMC Evolutionary Biology 2014, 14:70</dc:source>
<dc:date>2014-04-02</dc:date>

<prism:publicationName>BMC Evolutionary Biology</prism:publicationName>
<prism:publicationDate>2014-04-02</prism:publicationDate>
<prism:volume>14</prism:volume>
<prism:number>1</prism:number>
<prism:section>Research article</prism:section>
<prism:startingPage>70</prism:startingPage>
<prism:copyright>2014 Barnett et al.; licensee BioMed Central Ltd.</prism:copyright>
</item>
</rdf:RDF>
 -->
 
 
<meta name="citation_journal_title" content="BMC Evolutionary Biology">
<meta name="citation_publisher" content="BioMed Central Ltd">


<meta name="citation_author" content="Ross Barnett">
<meta name="citation_author_institution" content="Current Address: Centre for GeoGenetics, København Universitet, The Natural History Museum Øster Voldgade, Copenhagen 5-7 DK-1350, Denmark">
<meta name="citation_author_institution" content="Durham Evolution and Ancient DNA, Department of Archaeology, Durham University, Durham DH1 3LE, UK">
<meta name="citation_author" content="Nobuyuki Yamaguchi">
<meta name="citation_author_institution" content="Department of Biological and Environmental Sciences, Qatar University, Doha, Qatar">
<meta name="citation_author" content="Beth Shapiro">
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of California Santa Cruz, Santa Cruz, CA 95064, USA">
<meta name="citation_author" content="Simon YW Ho">
<meta name="citation_author_institution" content="School of Biological Sciences, University of Sydney, Sydney, NSW 2006, Australia">
<meta name="citation_author" content="Ian Barnes">
<meta name="citation_author_institution" content="School of Biological Sciences, Royal Holloway University of London, Egham, Surrey TW20 0EX, UK">
<meta name="citation_author" content="Richard Sabin">
<meta name="citation_author_institution" content="Department of Life Science, Natural History Museum, Cromwell Road, London SW7 5BD, UK">
<meta name="citation_author" content="Lars Werdelin">
<meta name="citation_author_institution" content="Department of Palaeobiology, Swedish Museum of Natural History, Box 50007, Stockholm SE-10405, Sweden">
<meta name="citation_author" content="Jacques Cuisin">
<meta name="citation_author_institution" content="Muséum National d’Histoire Naturelle, 54 Rue Cuvier, Paris 75005, France">
<meta name="citation_author" content="Greger Larson">
<meta name="citation_author_institution" content="Durham Evolution and Ancient DNA, Department of Archaeology, Durham University, Durham DH1 3LE, UK">


<meta name="citation_issn" content="1471-2148">
<meta name="citation_title" content="Revealing the maternal demographic history of Panthera leo using ancient DNA and a spatially explicit genealogical analysis">
<meta name="citation_volume" content="14">
<meta name="citation_issue" content="1">
<meta name="citation_date" content="2014-04-02">
<meta name="citation_firstpage" content="70">
<meta name="citation_doi" content="10.1186/1471-2148-14-70">
<meta name="citation_pdf_url" content="http://www.biomedcentral.com/content/pdf/1471-2148-14-70.pdf">
<meta name="citation_pmid" content="24690312">
<link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/content/pdf/1471-2148-14-70.pdf">
<meta name="title" content="Revealing the maternal demographic history of Panthera leo using ancient DNA and a spatially explicit genealogical analysis">
<meta name="description" content="Understanding the demographic history of a population is critical to conservation and to our broader understanding of evolutionary processes. For many tropical large mammals, however, this aim is confounded by the absence of fossil material and by the misleading signal obtained from genetic data of recently fragmented and isolated populations. This is particularly true for the lion which as a consequence of millennia of human persecution, has large gaps in its natural distribution and several recently extinct populations.">

<meta name="dc.title" content="Revealing the maternal demographic history of Panthera leo using ancient DNA and a spatially explicit genealogical analysis">
<meta name="dc.creator" content="Barnett, Ross">
<meta name="dc.creator" content="Yamaguchi, Nobuyuki">
<meta name="dc.creator" content="Shapiro, Beth">
<meta name="dc.creator" content="Ho, Simon YW">
<meta name="dc.creator" content="Barnes, Ian">
<meta name="dc.creator" content="Sabin, Richard">
<meta name="dc.creator" content="Werdelin, Lars">
<meta name="dc.creator" content="Cuisin, Jacques">
<meta name="dc.creator" content="Larson, Greger">
<meta name="dc.description" content="Understanding the demographic history of a population is critical to conservation and to our broader understanding of evolutionary processes. For many tropical large mammals, however, this aim is confounded by the absence of fossil material and by the misleading signal obtained from genetic data of recently fragmented and isolated populations. This is particularly true for the lion which as a consequence of millennia of human persecution, has large gaps in its natural distribution and several recently extinct populations.">
<meta name="dc.source" content="BMC Evolutionary Biology 2014 14:70">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central Ltd">
<meta name="dc.date" content="2014-04-02">
<meta name="dc.type" content="Research article">
<meta name="dc.identifier" content="10.1186/1471-2148-14-70">
<meta name="dc.identifier" content="info:pmid/24690312">
<meta name="dc.language" content="en">
<meta name="dc.copyright" content="2014 Barnett et al.; licensee BioMed Central Ltd.">
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

<meta name="prism.issn" content="1471-2148">
<meta name="prism.publicationName" content="BMC Evolutionary Biology">
<meta name="prism.publicationDate" content="2014-04-02">
<meta name="prism.volume" content="14">
<meta name="prism.number" content="1">
<meta name="prism.section" content="Research article">
<meta name="prism.startingPage" content="70">
<meta name="prism.copyright" content="2014 Barnett et al.; licensee BioMed Central Ltd.">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">


        <meta name="citation_abstract_html_url" content="http://www.biomedcentral.com/1471-2148/14/70/abstract">
            <meta name="citation_fulltext_html_url" content="http://www.biomedcentral.com/1471-2148/14/70">
            <link rel="alternate" type="text/xml" title="XML version" href="http://www.biomedcentral.com/content/download/xml/1471-2148-14-70.xml">
            <link rel="image_src" href="http://www.biomedcentral.com/content/figures/1471-2148-14-70-toc.gif">
    
    <link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2148-14-70-s1.pdf" title="Additional file 1: Figure S1">
<link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2148-14-70-s2.pdf" title="Additional file 2: Figure S2">
<link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2148-14-70-s3.docx" title="Additional file 3: Table S1">
<link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2148-14-70-s4.docx" title="Additional file 4: Table S2">
<link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2148-14-70-s5.xls" title="Additional file 5: Table S3">

    
    
                    
    

<script async="" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/cbgapi_002.js"></script><script async="" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/cbgapi.js"></script><script src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/widgets.js" id="twitter-wjs"></script><script gapi_processed="true" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/plusone.js" async="" type="text/javascript"></script><script src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/all.js" id="facebook-jssdk"></script><script src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/ga.js" async="" type="text/javascript"></script><script type="text/javascript">
// must run before css
document.documentElement.id = "js";
</script>

<script src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/jquery.js"></script>

						
   
  
      
  
                  
  
                  
  
            
  
            
            



						


	                <link rel="stylesheet" type="text/css" href="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/main-0.css" media="screen, print">
    
                <link rel="stylesheet" type="text/css" href="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/plugins-0.css" media="screen, print">
    



    

			                <link rel="stylesheet" type="text/css" href="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/articles-0.css" media="screen, print">
    	

    

    <link rel="stylesheet" type="text/css" href="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/1006.css" media="screen, print">


                <link rel="stylesheet" type="text/css" href="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/print-0.css" media="print">
    



<!--[if IE]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie-0.css"/>
    <![endif]-->

<!--[if IE 7]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie7-0.css"/>
    <![endif]-->
			
<script type="text/javascript">
    //configuration
    OAS_url = '//oas.biomedcentral.com/RealMedia/ads/';

	// control of advert's position on page [AMC]
    
    	
	
    OAS_sitepage = "biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/14/70";
    	
		OAS_listpos = 'Top,x96,Right3';
	
	
			
		OAS_query = '';
		
	

    OAS_target = '_top';
    //end of configuration
    OAS_version = 10;
    OAS_rn = '001234567890'; OAS_rns = '1234567890';
    OAS_rn = new String (Math.random()); OAS_rns = OAS_rn.substring (2, 11);
    function OAS_NORMAL(pos) {
		document.write('<a href="' + OAS_url + 'click_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" target="' + OAS_target + '">');
		document.write('<img src="' + OAS_url + 'adstream_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" border="0"/></a>');
    }
    
	function refererTerms(referer) {

	if(referer.match("q=") != null) {
		referer = referer.split("q=");
		referer = referer[1].split("&");
		referer = referer[0];
	} else {
		referer = referer;
	}

		if(referer.match("%20") != null && referer.match("%20").length >= 1) {
			referer =  referer.replace(/%20/gi,"+");
			return referer;
		} else { 
			return referer; 
		}
	}
 </script>


 <script type="text/javascript">
	// Set the version of JavaScript to 10 for a Browser 'Mozilla/3' on webTV 
    OAS_version = 11;
    if ((navigator.userAgent.indexOf('Mozilla/3') != -1) || (navigator.userAgent.indexOf('Mozilla/4.0 WebTV') != -1)) {
     OAS_version = 10; }
    if (OAS_version >= 11) {
    document.write('<scr' + 'ipt type="text/javascript" src="' + OAS_url + 'adstream_mjx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '?' + OAS_query + '"><\/script>'); }
</script><script type="text/javascript" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/1826987696Topx96Right3.js"></script>
	
<script type="text/javascript">
    document.write('');
	
    function OAS_AD(pos) {
        if (OAS_version >= 11)
         OAS_RICH(pos);
        else
         OAS_NORMAL(pos);
    }
 </script>
	





<script type="text/javascript">
    jsonLoaders = function(){
        var _loaders = [];
        return {
            register: function(config){
                config.prefix = config.prefix || '/webapi/1.0/blogs/';
                config.feedLength = config.feedLength || 3;

                _loaders.push(config);
            },
            list: function(){return _loaders}
        };
    }();

    function totext() {
	    document.write('<a href="javascript:returnToText();">Return to text</a>');
    }

    function retrieve_doi(){
        var metas = document.getElementsByTagName('meta');
        var i;
        for (i = 0; i < metas.length; i++)
            if (metas[i].getAttribute('name') == "citation_doi")
                break;
        var citation_doi = metas[i].getAttribute('content');
        return citation_doi;
    }

</script>





    <script src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/wb.js" type="text/javascript" charset="utf-8"></script>
        <link title="Latest articles" rel="alternate" href="http://www.biomedcentral.com/bmcevolbiol/latest/rss" type="application/rss+xml" exposetoheader="true">
    <link title="Most viewed" rel="alternate" href="http://www.biomedcentral.com/bmcevolbiol/mostviewed/rss/" type="application/rss+xml" exposetoheader="true">
    <link title="Latest comments" rel="alternate" href="http://www.biomedcentral.com/bmcevolbiol/latestcomments/rss" type="application/rss+xml" exposetoheader="true">
    


	<script type="text/javascript">
	var _gaq = _gaq || [];
	_gaq.push(['_setAccount', 'UA-72326-1']);
	_gaq.push(['_gat._anonymizeIp']);
	_gaq.push(['_setLocalRemoteServerMode']);
	
	/*for cross domain tracking*/
	//_gaq.push(['_setAllowLinker', true]);
    //_gaq.push(['_setDomainName', 'biomedcentral.com/bmcevolbiol']); //this journal's domain
	/**/
	
		_gaq.push(['_trackPageview']); 

	(function() {
		var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
		ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
		var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);

		//var iframe = document.getElementById('casesdb-search-widget');
		//if(iframe !== null) {
		//	var pageTracker = _gat._getTrackerByName();
		//	iframe.src = pageTracker._getLinkerUrl('http://www.casesdatabase.com/');
		//}
	})();
	</script>

	
        <script src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/bundle.js" charset="UTF-8"></script><script src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/iframeWidget.js" charset="UTF-8"></script><style type="text/css">.fb_hidden{position:absolute;top:-10000px;z-index:10001}.fb_invisible{display:none}.fb_reset{background:none;border:0;border-spacing:0;color:#000;cursor:auto;direction:ltr;font-family:"lucida grande", tahoma, verdana, arial, sans-serif;font-size:11px;font-style:normal;font-variant:normal;font-weight:normal;letter-spacing:normal;line-height:1;margin:0;overflow:visible;padding:0;text-align:left;text-decoration:none;text-indent:0;text-shadow:none;text-transform:none;visibility:visible;white-space:normal;word-spacing:normal}.fb_reset>div{overflow:hidden}.fb_link img{border:none}
.fb_dialog{background:rgba(82, 82, 82, .7);position:absolute;top:-10000px;z-index:10001}.fb_reset .fb_dialog_legacy{overflow:visible}.fb_dialog_advanced{padding:10px;-moz-border-radius:8px;-webkit-border-radius:8px;border-radius:8px}.fb_dialog_content{background:#fff;color:#333}.fb_dialog_close_icon{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/yq/r/IE9JII6Z1Ys.png) no-repeat scroll 0 0 transparent;_background-image:url(http://static.ak.fbcdn.net/rsrc.php/v2/yL/r/s816eWC-2sl.gif);cursor:pointer;display:block;height:15px;position:absolute;right:18px;top:17px;width:15px}.fb_dialog_mobile .fb_dialog_close_icon{top:5px;left:5px;right:auto}.fb_dialog_padding{background-color:transparent;position:absolute;width:1px;z-index:-1}.fb_dialog_close_icon:hover{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/yq/r/IE9JII6Z1Ys.png) no-repeat scroll 0 -15px transparent;_background-image:url(http://static.ak.fbcdn.net/rsrc.php/v2/yL/r/s816eWC-2sl.gif)}.fb_dialog_close_icon:active{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/yq/r/IE9JII6Z1Ys.png) no-repeat scroll 0 -30px transparent;_background-image:url(http://static.ak.fbcdn.net/rsrc.php/v2/yL/r/s816eWC-2sl.gif)}.fb_dialog_loader{background-color:#f6f7f8;border:1px solid #606060;font-size:24px;padding:20px}.fb_dialog_top_left,.fb_dialog_top_right,.fb_dialog_bottom_left,.fb_dialog_bottom_right{height:10px;width:10px;overflow:hidden;position:absolute}.fb_dialog_top_left{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/ye/r/8YeTNIlTZjm.png) no-repeat 0 0;left:-10px;top:-10px}.fb_dialog_top_right{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/ye/r/8YeTNIlTZjm.png) no-repeat 0 -10px;right:-10px;top:-10px}.fb_dialog_bottom_left{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/ye/r/8YeTNIlTZjm.png) no-repeat 0 -20px;bottom:-10px;left:-10px}.fb_dialog_bottom_right{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/ye/r/8YeTNIlTZjm.png) no-repeat 0 -30px;right:-10px;bottom:-10px}.fb_dialog_vert_left,.fb_dialog_vert_right,.fb_dialog_horiz_top,.fb_dialog_horiz_bottom{position:absolute;background:#525252;filter:alpha(opacity=70);opacity:.7}.fb_dialog_vert_left,.fb_dialog_vert_right{width:10px;height:100%}.fb_dialog_vert_left{margin-left:-10px}.fb_dialog_vert_right{right:0;margin-right:-10px}.fb_dialog_horiz_top,.fb_dialog_horiz_bottom{width:100%;height:10px}.fb_dialog_horiz_top{margin-top:-10px}.fb_dialog_horiz_bottom{bottom:0;margin-bottom:-10px}.fb_dialog_iframe{line-height:0}.fb_dialog_content .dialog_title{background:#6d84b4;border:1px solid #3a5795;color:#fff;font-size:14px;font-weight:bold;margin:0}.fb_dialog_content .dialog_title>span{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/yd/r/Cou7n-nqK52.gif) no-repeat 5px 50%;float:left;padding:5px 0 7px 26px}body.fb_hidden{-webkit-transform:none;height:100%;margin:0;overflow:visible;position:absolute;top:-10000px;left:0;width:100%}.fb_dialog.fb_dialog_mobile.loading{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/ya/r/3rhSv5V8j3o.gif) white no-repeat 50% 50%;min-height:100%;min-width:100%;overflow:hidden;position:absolute;top:0;z-index:10001}.fb_dialog.fb_dialog_mobile.loading.centered{max-height:590px;min-height:590px;max-width:500px;min-width:500px}#fb-root #fb_dialog_ipad_overlay{background:rgba(0, 0, 0, .45);position:absolute;left:0;top:0;width:100%;min-height:100%;z-index:10000}#fb-root #fb_dialog_ipad_overlay.hidden{display:none}.fb_dialog.fb_dialog_mobile.loading iframe{visibility:hidden}.fb_dialog_content .dialog_header{-webkit-box-shadow:white 0 1px 1px -1px inset;background:-webkit-gradient(linear, 0% 0%, 0% 100%, from(#738ABA), to(#2C4987));border-bottom:1px solid;border-color:#1d4088;color:#fff;font:14px Helvetica, sans-serif;font-weight:bold;text-overflow:ellipsis;text-shadow:rgba(0, 30, 84, .296875) 0 -1px 0;vertical-align:middle;white-space:nowrap}.fb_dialog_content .dialog_header table{-webkit-font-smoothing:subpixel-antialiased;height:43px;width:100%}.fb_dialog_content .dialog_header td.header_left{font-size:12px;padding-left:5px;vertical-align:middle;width:60px}.fb_dialog_content .dialog_header td.header_right{font-size:12px;padding-right:5px;vertical-align:middle;width:60px}.fb_dialog_content .touchable_button{background:-webkit-gradient(linear, 0% 0%, 0% 100%, from(#4966A6), color-stop(.5, #355492), to(#2A4887));border:1px solid #2f477a;-webkit-background-clip:padding-box;-webkit-border-radius:3px;-webkit-box-shadow:rgba(0, 0, 0, .117188) 0 1px 1px inset, rgba(255, 255, 255, .167969) 0 1px 0;display:inline-block;margin-top:3px;max-width:85px;line-height:18px;padding:4px 12px;position:relative}.fb_dialog_content .dialog_header .touchable_button input{border:none;background:none;color:#fff;font:12px Helvetica, sans-serif;font-weight:bold;margin:2px -12px;padding:2px 6px 3px 6px;text-shadow:rgba(0, 30, 84, .296875) 0 -1px 0}.fb_dialog_content .dialog_header .header_center{color:#fff;font-size:16px;font-weight:bold;line-height:18px;text-align:center;vertical-align:middle}.fb_dialog_content .dialog_content{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/y9/r/jKEcVPZFk-2.gif) no-repeat 50% 50%;border:1px solid #555;border-bottom:0;border-top:0;height:150px}.fb_dialog_content .dialog_footer{background:#f6f7f8;border:1px solid #555;border-top-color:#ccc;height:40px}#fb_dialog_loader_close{float:left}.fb_dialog.fb_dialog_mobile .fb_dialog_close_button{text-shadow:rgba(0, 30, 84, .296875) 0 -1px 0}.fb_dialog.fb_dialog_mobile .fb_dialog_close_icon{visibility:hidden}
.fb_iframe_widget{display:inline-block;position:relative}.fb_iframe_widget span{display:inline-block;position:relative;text-align:justify}.fb_iframe_widget iframe{position:absolute}.fb_iframe_widget_fluid_desktop,.fb_iframe_widget_fluid_desktop span,.fb_iframe_widget_fluid_desktop iframe{max-width:100%}.fb_iframe_widget_fluid_desktop iframe{min-width:220px;position:relative}.fb_iframe_widget_lift{z-index:1}.fb_hide_iframes iframe{position:relative;left:-10000px}.fb_iframe_widget_loader{position:relative;display:inline-block}.fb_iframe_widget_fluid{display:inline}.fb_iframe_widget_fluid span{width:100%}.fb_iframe_widget_loader iframe{min-height:32px;z-index:2;zoom:1}.fb_iframe_widget_loader .FB_Loader{background:url(http://static.ak.fbcdn.net/rsrc.php/v2/y9/r/jKEcVPZFk-2.gif) no-repeat;height:32px;width:32px;margin-left:-16px;position:absolute;left:50%;z-index:4}</style></head>


<body data-twttr-rendered="true" class=" theme    "><iframe id="sina_anywhere_iframe" style="display: none;"></iframe>


                                        



   
  
      
  
                  
  
                  
  
            
  
            
            

                    

<div id="oas-campaign" style="display:none">biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/14/70</div>
<div id="oas-positions" style="display:none">Bottom,Top</div>

<div id="branding" role="banner">
                			
		<dl class="google-ad wide  advert-text">
	<dt style="display: block;" class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Top' begin -->
		<script type="text/javascript">
		OAS_AD('Top');

		</script>
		<!-- OAS AD 'Top' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad wide noscript">  
    <dt class="hide"></dt>
    
    <dd>
        <a href="//oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/14/70/14563185554@Top?">
            <img src="//oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/14/70/19670379718@Top?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
			            <div class="sup-panel-outer">
        <div class="sup-panel-inner">

            <style>
  #nojs li.greeting, #nojs li#loginLink {display:none;}
</style>


<ul id="login" class="login">

    <noscript><li><a href="/log"><img class="greeting" src="/images/log.gif"/></a></li></noscript>
    
    <li id="welcome"><span id="username"></span></li>

    <li id="loginLink"><a href="http://www.biomedcentral.com/logon">Log on</a></li>
    
    
</ul>

<script>

function login() {

/* cookie getter/setter */
var allCookies = {
  getItem: function (sKey) {
    if (!sKey || !this.hasItem(sKey)) { return null; }
    return unescape(document.cookie.replace(new RegExp("(?:^|.*;\\s*)" + escape(sKey).replace(/[\-\.\+\*]/g, "\\$&") + "\\s*\\=\\s*((?:[^;](?!;))*[^;]?).*"), "$1"));
  },
  setItem: function (sKey, sValue, vEnd, sPath, sDomain, bSecure) {
    if (!sKey || /^(?:expires|max\-age|path|domain|secure)$/i.test(sKey)) { return; }
    var sExpires = "";
    if (vEnd) {
      switch (vEnd.constructor) {
        case Number:
          sExpires = vEnd === Infinity ? "; expires=Tue, 19 Jan 2038 03:14:07 GMT" : "; max-age=" + vEnd;
          break;
        case String:
          sExpires = "; expires=" + vEnd;
          break;
        case Date:
          sExpires = "; expires=" + vEnd.toGMTString();
          break;
      }
    }
    document.cookie = escape(sKey) + "=" + escape(sValue) + sExpires + (sDomain ? "; domain=" + sDomain : "") + (sPath ? "; path=" + sPath : "") + (bSecure ? "; secure" : "");
  },
  removeItem: function (sKey, sPath) {
    if (!sKey || !this.hasItem(sKey)) { return; }
    document.cookie = escape(sKey) + "=; expires=Thu, 01 Jan 1970 00:00:00 GMT" + (sPath ? "; path=" + sPath : "");
  },
  hasItem: function (sKey) {
    return (new RegExp("(?:^|;\\s*)" + escape(sKey).replace(/[\-\.\+\*]/g, "\\$&") + "\\s*\\=")).test(document.cookie);
  },
  keys: /* optional method: you can safely remove it! */ function () {
    var aKeys = document.cookie.replace(/((?:^|\s*;)[^\=]+)(?=;|$)|^\s*|\s*(?:\=[^;]*)?(?:\1|$)/g, "").split(/\s*(?:\=[^;]*)?;\s*/);
    for (var nIdx = 0; nIdx < aKeys.length; nIdx++) { aKeys[nIdx] = unescape(aKeys[nIdx]); }
    return aKeys;
  }
};

// converts cookie into js object
function parseCookie(cookie) {

    cookie = cookie.split("&");
    var obj = {};

    for(var i = 0; i < cookie.length; i++) {

        var c = cookie[i].split("=");

        c[0] = c[0].replace('"','');
        
        obj[c[0]] = c[1];

    }
    return obj;
}

    var cookie = allCookies.getItem("bmccookie");

    var obj = (cookie != null) ? parseCookie(cookie) : {};
    

        var welcome = "";

        if(obj.hasOwnProperty("fname")) {

            // insert user welcome
            welcome = obj.fname.replace(/\+/g, " ") + " " + obj['lname'].replace(/\+/g," ");

            if(jQuery("#welcome").length > 0) {
              jQuery("#welcome").prepend("<strong>Welcome</strong> ");
              jQuery("#username").prepend('<strong>' + welcome + '</strong><span class="divider"></span>');
            } 

            // update link url
            jQuery("#loginLink a").text("Log off");
            jQuery("#loginLink a").attr("href", window.location.protocol + "//" + window.location.host + "/logoff");

            window.bmcloggedon = true;


        } else if(obj.hasOwnProperty("institution_name")) {

            // insert institution name
            welcome = obj["institution_name"].replace(/\+/g," ");
            
            if(jQuery("#welcome").length > 0) { 
                jQuery("#welcome").prepend("<strong>Welcome</strong> ");
                jQuery("#username").prepend('<strong>' + welcome + '</strong><span class="divider"></span>');
            } 

            // update link url
            jQuery("#loginLink a").text("Log on");
            jQuery("#loginLink a").attr("href", window.location.protocol + "//" + window.location.host + "/logon");

            window.bmcloggedon = false;


        } else {
            jQuery("#welcome").innerHTML = "";
            jQuery("#loginLink a").text("Log on");
            jQuery("#loginLink a").attr("href", window.location.protocol + "//" + window.location.host + "/logon");

            window.bmcloggedon = false;

        }


        // if athens login cookie doesn't exist and the athens login link is visible then remove it.
        if(allCookies.hasItem("athens") != true && document.getElementById("loginAthensLink") != null) {
          jQuery("#loginAthensLink").hide();
        }

}

login();


</script>

            <ul class="nav-sup">

    
        <li id="BMC" class="current">
            <a href="http://www.biomedcentral.com/"><span class="img-title">&nbsp;</span>
            <span class="text-title">BioMed Central</span></a>
        </li>
		
        		<li><a href="http://www.biomedcentral.com/journals"><span>Journals</span></a></li>
		
								<li>
				<a href="http://www.biomedcentral.com/gateways"><span>Gateways</span></a>
			</li>
					    	
</ul>
        </div>
    </div>
    <div class="branding-inner">

        

    
<style>
.survey {
	margin: 10px 0;
}
.survey, .survey .survey-sentence {
	font-size: 13px;
}
.survey-button a, .survey-button a:active, .survey-button a:visited, .survey-button a:hover  {
    color: #FFFFFF;
    text-decoration: none;
}
.survey-button {
	background-color: #FFA500;
    border-radius: 5px;
    color: #FFFFFF;
    font-weight: bold;
    font-size: 13px;
    font-weight: bold;
    padding: 5px 13px;
    background: #ef8142; /* Old browsers */
	background: -moz-linear-gradient(top,  #ef8142 0%, #ef8142 50%, #a64510 100%, #a64510 100%); /* FF3.6+ */
	background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#ef8142), color-stop(50%,#ef8142), color-stop(100%,#a64510), color-stop(100%,#a64510)); /* Chrome,Safari4+ */
	background: -webkit-linear-gradient(top,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* Chrome10+,Safari5.1+ */
	background: -o-linear-gradient(top,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* Opera 11.10+ */
	background: -ms-linear-gradient(top,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* IE10+ */
	background: linear-gradient(to bottom,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* W3C */
	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ef8142', endColorstr='#a64510',GradientType=0 ); /* IE6-9 */

}
</style>




          <div class="logo">
        <a href="http://www.biomedcentral.com/bmcevolbiol/"><img alt="BMC Evolutionary Biology" title="BMC Evolutionary Biology Logo" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/logo.gif"></a>     
      </div>    
        
                    



            <div id="impact-factor" class="official">
    <img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/official.gif" alt="official impact factor" title="3.41">
    <span id="impact-factor-value">3.41</span>
</div>
    
                            

<div class="module search gray small  " role="search">
    <div class="module-inner padded">
        <form method="get" class="searchForm" action=" /quickSearchRedirectURL " onsubmit="window.location.href=this['url'].value + '?terms=' + encodeURIComponent(this['terms'].value); return false;">
            <fieldset class="search">
			                <span>
                    Search <select name="url">
                                <option value="http://www.biomedcentral.com/bmcevolbiol/search/results" selected="selected">this journal</option>
                                <option value="http://www.biomedcentral.com/search/results">BioMed Central</option>
                           </select> for
                </span>
                            <input style="width: 143px;" id="searchTerms" name="terms" alt="Search terms" class="text " type="text">
                 <button class="w37" value="Go" type="submit">Go</button>
            </fieldset>
        </form>
    </div>
	<a class="advanced-search" href="http://www.biomedcentral.com/bmcevolbiol/search">Advanced search</a>
</div> 

                        <div id="mobile-nav-list">

	<ul class="primary-nav" role="navigation">
		<li><a href="http://www.biomedcentral.com/bmcevolbiol/"><span id="home-tab">Home</span></a></li>
														<li class="current"><a href="http://www.biomedcentral.com/bmcevolbiol/content"><span id="articles-tab">Articles</span></a></li>
										
	

	
            <li><a href="http://www.biomedcentral.com/bmcevolbiol/authors/instructions"><span id="authors-tab">Authors</span></a></li>
        <li><a href="http://www.biomedcentral.com/bmcevolbiol/about/reviewers"><span id="reviewers-tab">Reviewers</span></a></li>
    		<li><a href="http://www.biomedcentral.com/bmcevolbiol/about"><span id="about-tab">About this journal</span></a></li>
	<li><a href="http://www.biomedcentral.com/bmcevolbiol/my"><span id="my-tab">My BMC Evolutionary Biology</span></a></li>
		
    		</ul>
</div>
            </div>
</div>

<hr class="hide">
<div class="container">
	<div id="content" class="overflow-visible" role="main">
			    
	

<!--div class="branding-mobile">
		<h1 class="logo">
	    <a href="/bmcevolbiol/">
		    <img alt="BMC Evolutionary Biology" title="BMC Evolutionary Biology Logo" src="/sites/1006/images/logo.gif" />
		</a>
		</h1>
	</div><br/-->



			
	



<div style="position: fixed; top: 20px;" class="outline-wrapper block mobile-hidden" id="left-article-box">
<ul id="box-outline" class="box">
<ul id="outline" class="outline">
   <li>
      <a href="#">Top</a>
   </li>
   <li>
      <a href="#abs">Abstract</a>
   </li>
   <li>
      <a href="#sec1">Background</a>
   </li>
   <li>
      <a href="#sec2">Methods</a>
   </li>
   <li>
      <a href="#sec3">Results</a>
   </li>
   <li>
      <a href="#sec4">Discussion</a>
   </li>
   <li>
      <a href="#sec5">Conclusion</a>
   </li>
   <li>
      <a href="#sec6">Abbreviations</a>
   </li>
   <li>
      <a href="#sec7">Competing interests</a>
   </li>
   <li>
      <a href="#sec8">Authors’ contributions</a>
   </li>
   <li>
      <a href="#ack">Acknowledgements</a>
   </li>
   <li class="lowest">
      <a href="#refs">References</a>
   </li>
</ul>
</ul>

<dl id="box-outline1">
</dl>

				



</div>



<div class="mobile-hidden" id="right-panel">
 	
<div id="mobile-sidebar">
	<div id="mobile-sidebar-tab">
		<span>
			<a href="#" id="mobile-sidebar-tab-link"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sidebar-hidden.png"><img style="display:none;" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sidebar-shown.png"></a>
		</span>
	</div>	<div id="mobile-sidebar-list">

<div id="article-navigation-bar">
    <div class="issue-information block">
        <h5><a href="http://www.biomedcentral.com/bmcevolbiol"><strong>BMC Evolutionary Biology</strong></a></h5>
 		        <ul class="square normal" id="section-name">
            <li><a href="http://www.biomedcentral.com/bmcevolbiol/sections/phylogenetics_and_phylogeography">Phylogenetics and phylogeography</a></li>
        </ul>
                <ul class="square normal">
            <li><a href="http://www.biomedcentral.com/bmcevolbiol/content/14">Volume 14</a></li>
				
                    </ul>
    </div>

    <div class="article-information block" id="article-info">
        


<div id="viewing-options-links">
	<h5>Viewing options</h5>
	<ul class="square normal">
        
            <li>
                             <a href="http://www.biomedcentral.com/1471-2148/14/70/abstract">Abstract</a>
                        </li>
        				<li>
							<strong>Full text</strong>
					</li>
				
		<li class="pdfFileSize">
			<a onclick="_gaq.push(['_trackEvent', 'PDF download', 'Article Sidebar', '/1471-2148/14/70', 1, true]);" href="http://www.biomedcentral.com/content/pdf/1471-2148-14-70.pdf">PDF</a>
			<span>(826KB)</span>
		</li>

        
				<li>
						<a href="http://www.biomedcentral.com/1471-2148/14/70/additional">Additional files</a>
					</li>
			</ul>

        
</div>
		
<div id="associated-material-links">
    <h5>Associated material</h5>
    <ul class="square normal">
                        <li>
            <a href="http://www.biomedcentral.com/pubmed/24690312">PubMed record</a>
        </li>
                            <li><a href="http://www.biomedcentral.com/1471-2148/14/70/about">Article metrics</a></li>
                <li>
                        <a href="http://www.biomedcentral.com/1471-2148/14/70/comments">Readers' comments </a>
                    </li>
        <li>
                    </li>
    </ul>
</div>
		




<div id="related-literature-links">
    <h5>Related literature</h5>
    <ul class="normal">
                    <li><a href="http://www.biomedcentral.com/1471-2148/14/70/about#citations">Cited by</a></li>
                <li id="google-blog-search"><a href="http://www.google.com/search?tbm=blg&amp;hl=en&amp;source=hp&amp;biw=1280&amp;bih=899&amp;q=%22Revealing+the+maternal+demographic+history+of+Panthera+leo+using+ancient+DNA+and+a+spatially+explicit+genealogical+analysis%22&amp;btnG=Search">Google blog search</a></li>
        <h6>Other articles by authors</h6>
        <li>
            <a class="collapser"><i>&nbsp;</i>on Google Scholar</a>
<ul id="authg" class="hidebeforeload">                         <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22R+Barnett%22">Barnett R</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22N+Yamaguchi%22">Yamaguchi N</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22B+Shapiro%22">Shapiro B</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22SYW+Ho%22">Ho SYW</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22I+Barnes%22">Barnes I</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22R+Sabin%22">Sabin R</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22L+Werdelin%22">Werdelin L</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22J+Cuisin%22">Cuisin J</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22G+Larson%22">Larson G</a>
            </li>
            </ul>          </li>
                <li>
            		<a class="collapser"><i>&nbsp;</i>on PubMed</a>
	<ul id="authpm" class="hidebeforeload"> 									<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Barnett_R%20%5BAuthor%5D">Barnett R</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Yamaguchi_N%20%5BAuthor%5D">Yamaguchi N</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Shapiro_B%20%5BAuthor%5D">Shapiro B</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Ho_SYW%20%5BAuthor%5D">Ho SYW</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Barnes_I%20%5BAuthor%5D">Barnes I</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Sabin_R%20%5BAuthor%5D">Sabin R</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Werdelin_L%20%5BAuthor%5D">Werdelin L</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Cuisin_J%20%5BAuthor%5D">Cuisin J</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Larson_G%20%5BAuthor%5D">Larson G</a>
				</li>
						</ul> 	        </li>
    </ul>     <h6>Related articles/pages</h6>
    <ul class="square normal">
        <li><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2148/14/70">on Google</a></li>
        <li><a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2148/14/70">on Google Scholar</a></li>
                    <li><a href="http://www.biomedcentral.com/pubmed/related/24690312">on PubMed</a></li>
            </ul>

</div>		
        
    
    
        

    	






<h5>Tools</h5>
	<ul class="square normal">
									<li>
						<a href="http://www.biomedcentral.com/1471-2148/14/70/citation">Download references</a>
					</li>
				<li><a href="http://www.biomedcentral.com/content/download/xml/1471-2148-14-70.xml">Download XML</a></li>
				<li>
						<a href="http://www.biomedcentral.com/1471-2148/14/70/email?from=standard">Email to a friend</a>
					</li>
		<li><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186/1471-2148-14-70&amp;volume=14&amp;issue=1&amp;title=Revealing+the+maternal+demographic+history+of+Panthera+leo+using+ancient+DNA+and+a+spatially+explicit+genealogical+analysis&amp;author_name=Ross%20Barnett&amp;start_page=1&amp;end_page=11">Order reprints</a></li>
		<li>
						<a href="http://www.biomedcentral.com/1471-2148/14/70/postcomment">Post a comment</a>
					</li>			
		<ul id="download-to-links">
			<a class="downloadto_title btn" href="#"><span class="icon">&nbsp;</span>Download to ...</a>
			
			<ul id="downloadto" class="iconlist hidebeforeload" style="display: none;">
				<li>
					<a href="http://redirect.papersapp.com/redirect?url=http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F14%2F70" onclick="largepopup(this.href,'papers',800,600);return false"><span class="share-icons papers"></span>Papers</a>
				</li>
				<li>
					<a href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2148/14/70" onclick="largepopup(this.href,'mendeley',800,600);return false"><span class="share-icons mendeley"></span>Mendeley</a>
				</li>
			</ul>
		</ul>
		
		<ul id="mobile-downloadto" class="iconlist">
		
		<h6>Download to ...</h6>
			<li>
				<a href="http://redirect.papersapp.com/redirect?url=http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F14%2F70" onclick="largepopup(this.href,'papers',800,600);return false"><span class="share-icons papers"></span>Papers</a>
			</li>
			
			<li>
				<a href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2148/14/70" onclick="largepopup(this.href,'mendeley',800,600);return false"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
		</ul>

		
	</ul>
		




<h5>Share this article</h5>
<ul id="social-networking-links">
	<li>
	<div fb-iframe-plugin-query="action=recommend&amp;app_id=&amp;href=http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F14%2F70&amp;layout=button_count&amp;locale=en_GB&amp;sdk=joey&amp;send=false&amp;show_faces=false&amp;width=100" fb-xfbml-state="rendered" class="fb-like fb_iframe_widget" data-href="http://www.biomedcentral.com/1471-2148/14/70" data-send="false" data-layout="button_count" data-width="100" data-show-faces="false" data-action="recommend"><span style="vertical-align: bottom; width: 121px; height: 20px;"><iframe class="" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/like.html" style="border: medium none; visibility: visible; width: 121px; height: 20px;" title="fb:like Facebook Social Plugin" scrolling="no" allowtransparency="true" name="f342a2cbb131462" frameborder="0" height="1000px" width="100px"></iframe></span></div>
	</li>
	<li>
		<iframe style="width: 107px; height: 20px;" data-twttr-rendered="true" title="Twitter Tweet Button" class="twitter-share-button twitter-tweet-button twitter-share-button twitter-count-horizontal" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/tweet_button.html" allowtransparency="true" scrolling="no" id="twitter-widget-0" frameborder="0"></iframe>
	</li>
	<li>
			<div id="___plusone_0" style="text-indent: 0px; margin: 0px; padding: 0px; background: none repeat scroll 0% 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline; display: inline-block; width: 90px; height: 20px;"><iframe title="+1" data-gapiattached="true" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/fastbutton.html" name="I0_1422791392459" id="I0_1422791392459" vspace="0" tabindex="0" style="position: static; top: 0px; width: 90px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 20px;" scrolling="no" marginwidth="0" marginheight="0" hspace="0" frameborder="0" width="100%"></iframe></div>
	</li>

        <li>
        <wb:share-button addition="number" type="button" ralateuid="2216240737" title="Revealing the maternal demographic history of Panthera leo using ancient DNA and a spatially explicit genealogical analysis http://www.biomedcentral.com/1471-2148/14/70"><iframe marginheight="0" scrolling="no" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/weibosharev2.html" frameborder="0" height="25" width="128"></iframe></wb:share-button>
    </li>
    
		<a class="sharethis_title btn" href="#"><span class="icon">&nbsp;</span>More options...</a>
	<ul style="display: none;" class="iconlist hidebeforeload" id="sharethis">
			<li>
				<a onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false" href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2148/14/70"><span class="share-icons citeulike"></span>Citeulike</a>
			</li>
				<li>
			<a onclick="largepopup(this.href,'linkedin',800,600);return false" href="http://www.linkedin.com/shareArticle?mini=true&amp;url=%26%2335%3Bbmcevolutionarybiology%0D%0ARevealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F14%2F70&amp;title=Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis"><span class="share-icons linkedin"></span>LinkedIn</a>  
	</li>
			<li>
				<a onclick="largepopup(this.href,'Del.icio.us',800,600);return false" href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2148/14/70&amp;title=Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis"><span class="share-icons delicious"></span>Del.icio.us</a>
	</li>
			<li>
				<a href="http://www.biomedcentral.com/1471-2148/14/70/email"><span class="share-icons email"></span>Email</a>
	</li>
			<li>
				<a onclick="largepopup(this.href,'facebook',800,600);return false" href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2148/14/70"><span class="share-icons facebook"></span>Facebook</a>
	</li>
	<li class="gp">
			<div class="googlehider mobile-hide"> 
				<div id="___plusone_1" style="text-indent: 0px; margin: 0px; padding: 0px; background: none repeat scroll 0% 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline; display: inline-block; width: 38px; height: 24px;"><iframe title="+1" data-gapiattached="true" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/fastbutton_002.html" name="I1_1422791392467" id="I1_1422791392467" vspace="0" tabindex="0" style="position: static; top: 0px; width: 38px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 24px;" scrolling="no" marginwidth="0" marginheight="0" hspace="0" frameborder="0" width="100%"></iframe></div>
			</div>
				<span class="share-icons googleplus mobile-hide"></span>
				<div class="googlehider2 mobile-hide"> 
				<div id="___plusone_2" style="text-indent: 0px; margin: 0px; padding: 0px; background: none repeat scroll 0% 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline; display: inline-block; width: 38px; height: 24px;"><iframe title="+1" data-gapiattached="true" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/fastbutton_003.html" name="I2_1422791392472" id="I2_1422791392472" vspace="0" tabindex="0" style="position: static; top: 0px; width: 38px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 24px;" scrolling="no" marginwidth="0" marginheight="0" hspace="0" frameborder="0" width="100%"></iframe></div>
			</div>
				<span class="share-icons googleplus"></span>
				<span class="gp-link"><a href="#">Google+</a></span>
	</li>
			<li>
				<a onclick="largepopup(this.href,'mendeley',800,600);return false" href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2148/14/70"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
			<li>
			<a onclick="largepopup(this.href,'twitter',800,600);return false" href="http://twitter.com/?status=Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F14%2F70+%23bmcevolutionarybiology"><span class="share-icons twitter"></span>Twitter</a>  
			</li>

			<li><a href="http://www.reddit.com/submit" onclick="window.location = 'http://www.reddit.com/submit?url=http://www.biomedcentral.com/1471-2148/14/70'; return false"> <span class="share-icons reddit"></span>Reddit</a>
    </li>






		</ul>
</ul>

    </div>

    
  
        
                
        


















                                                                                
    
    	<div id="biome-badge" style="width: 100%; display: none;">
              <script>

                window.onmessage = function(e) {
                    if(e.data == "biome-failed") {
                        console.log(e);
                        document.getElementById("biome-badge").style.display = "none";
                    }
                };

              </script>


                <iframe src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/biome-widget.html" height="75px" width="100%">Your browser does not support iframes</iframe>


    </div>
		
	

	    
   
  
      
  
                  
  
                  
  
            
  
            
            

<div id="signup-to-etoc" class="button-collection-margins block mobile-border-bottom">	
					<h3>Email updates</h3>
			<p id="receive_issue_alerts_journal_updates_text">Keep up to date with the latest news and content from BMC Evolutionary Biology and BioMed Central.</p>
			
				
<form id="articleSignupForUpdates" method="post" action="/bmcevolbiol/signuptoupdates.html">
	<div id="message-box"></div>
    <fieldset class="block-form">
        <input name="emailAddress" onblur="javascript: if (this.value == '') { this.value = 'email address'; }" onfocus="javascript: if (this.value == 'email address') { this.value = ''; }" value="email address" class="text " id="email" type="text">
        <input value="/bmcevolbiol" name="returnUrl" id="returnUrl" type="hidden">
        <input value="" name="listName" id="listName" type="hidden">
        <input value="BMC Evolutionary Biology" name="journalName" id="journalName" type="hidden">
        <button onclick="_gaq.push('_trackEvent', 'Signup', 'Signup for Email updates', " bmc="" evolutionary="" biology");"="" class="w74 right" type="submit" onmouseover="this.className='w74 hover right'" onmouseout="this.className='w74 right'" value="Sign up">Sign up</button>
    </fieldset>
</form>	
</div>

	


			<div style="position: absolute;" id="advert-offset">
    					
		<dl class="google-ad  ">
	<dt class="hide">
				
					<a class="skyscraper-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</a>
			</dt>
		<dd>

		<!-- OAS AD 'Right3' begin -->
		<script type="text/javascript">
		OAS_AD('Right3');

		</script>
		<!-- OAS AD 'Right3' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad  noscript">  
    <dt class="hide"><a href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</dt>
    
    <dd>
        <a href="//oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/14/70/12808748423@Right3?">
            <img src="//oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/14/70/16919902285@Right3?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
					</div>
	

	
</div>
		<div class="mobile-sidebar-gradient top"></div>
		<div class="mobile-sidebar-gradient bottom"></div>
	</div>

</div>








 </div>
<div class="rounded custom white article full-text">
<div class="wrap-inner content">
    <div class="padded-inner">            


    
   
  
      
  
                  
  
                  
  
            
  
            
            

<div id="topmatter">
	


	   






 
  



				<a href="http://www.biomedcentral.com/about/access"><img alt="Open Access" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/openaccess-large.png" class="access mr15"></a>
			<a href="http://www.biomedcentral.com/about/mostviewed/"><img alt="Highly Accessed" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/highlyaccessed-large.png" class="access mr15"></a>
				<span class="articletype">Research article</span>

<h1>Revealing the maternal demographic history of <em>Panthera leo</em> using ancient DNA and a spatially explicit genealogical analysis</h1>
        
			<div class="multipleins">
		
	<p class="authors">
        																																																																
																																																																								
																																																																								
																																																																								
										         <strong>Ross Barnett</strong><sup><a href="http://www.biomedcentral.com/1471-2148/14/70/#ins1">1</a></sup><sup><a href="http://www.biomedcentral.com/1471-2148/14/70/#ins9">9</a></sup><sup>*</sup>, <strong>Nobuyuki Yamaguchi</strong><sup><a href="http://www.biomedcentral.com/1471-2148/14/70/#ins2">2</a></sup>, <strong>Beth Shapiro</strong><sup><a href="http://www.biomedcentral.com/1471-2148/14/70/#ins3">3</a></sup>, <strong>Simon YW Ho</strong><sup><a href="http://www.biomedcentral.com/1471-2148/14/70/#ins4">4</a></sup>, <strong>Ian Barnes</strong><sup><a href="http://www.biomedcentral.com/1471-2148/14/70/#ins5">5</a></sup>, <strong>Richard Sabin</strong><sup><a href="http://www.biomedcentral.com/1471-2148/14/70/#ins6">6</a></sup>, <strong>Lars Werdelin</strong><sup><a href="http://www.biomedcentral.com/1471-2148/14/70/#ins7">7</a></sup>, <strong>Jacques Cuisin</strong><sup><a href="http://www.biomedcentral.com/1471-2148/14/70/#ins8">8</a></sup> and <strong>Greger Larson</strong><sup><a href="http://www.biomedcentral.com/1471-2148/14/70/#ins1">1</a></sup>	</p>
	
				<div id="affiliations">
		<div class="module gray  inner">
<div class="module-inner padded-inner">
		<ul>
			<li>                        
                                                                                                                            		<p class="authors">
							<span>*</span>
									 Corresponding author:										            Ross  Barnett <a href="mailto:drrossbarnett@gmail.com">drrossbarnett@gmail.com</a>    					</p>
      
</li>
			
                                                                        
                                                                        
                                                                        
                                                    		</ul>

		
		 <p class="options"><a class="affiliations-toggle" href="#"><i class="arrow active"></i>Author Affiliations</a></p>            <section> 
						<div class="collapsible-content">
							
				<div id="ins_container" style="display: block;">
										    						    <p>
						        <sup id="ins1">1</sup>                                
Durham Evolution and Ancient DNA, Department of Archaeology, Durham 
University, Durham DH1 3LE, UK
                            </p>
																    						    <p>
						        <sup id="ins2">2</sup>                                Department of Biological and Environmental Sciences, Qatar University, Doha, Qatar
                            </p>
																    						    <p>
						        <sup id="ins3">3</sup>                                
Department of Ecology and Evolutionary Biology, University of California
 Santa Cruz, Santa Cruz, CA 95064, USA
                            </p>
																    						    <p>
						        <sup id="ins4">4</sup>                                School of Biological Sciences, University of Sydney, Sydney, NSW 2006, Australia
                            </p>
																    						    <p>
						        <sup id="ins5">5</sup>                                School of Biological Sciences, Royal Holloway University of London, Egham, Surrey TW20 0EX, UK
                            </p>
																    						    <p>
						        <sup id="ins6">6</sup>                                Department of Life Science, Natural History Museum, Cromwell Road, London SW7 5BD, UK
                            </p>
																    						    <p>
						        <sup id="ins7">7</sup>                                
Department of Palaeobiology, Swedish Museum of Natural History, Box 
50007, Stockholm SE-10405, Sweden
                            </p>
																    						    <p>
						        <sup id="ins8">8</sup>                                Muséum National d’Histoire Naturelle, 54 Rue Cuvier, Paris 75005, France
                            </p>
																    						    <p>
						        <sup id="ins9">9</sup>                                
Current Address: Centre for GeoGenetics, København Universitet, The 
Natural History Museum Øster Voldgade, Copenhagen 5-7 DK-1350, Denmark
                            </p>
																
				</div>
										<p id="authoremails">            For all author emails, please <a href="http://www.biomedcentral.com/bmcevolbiol/logon">log on</a>.
    </p>
						</div>
			</section>
				</div>
</div>
		</div>
	  </div>

            <section class="cit">
                        <div class="collapsible-content">
        <p><em>BMC Evolutionary Biology</em> 2014, <strong>14</strong>:70&nbsp;
			<span class="pseudotab">doi:10.1186/1471-2148-14-70</span></p>
	    				    
    
        
                
		<p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1471-2148/14/70">http://www.biomedcentral.com/1471-2148/14/70</a></p>
		<br>
                    <table cellpadding="0" cellspacing="0">
        <tbody>
        <tr><td>Received:</td><td>18 December 2013</td></tr>                <tr><td>Accepted:</td><td>13 March 2014</td></tr>        <tr><td>Published:</td><td>2 April 2014</td></tr>        </tbody>
        </table>
    		<!--<br/>-->
		<div style="line-height:140%">
        <p>
            © 2014 Barnett et al.; licensee BioMed Central Ltd. <br>
        </p>
					<p>
				This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>),
 which permits unrestricted use, distribution, and reproduction in any 
medium, provided the original work is properly credited.
			</p>
		    	</div>
	</div></section>

</div>
<div id="article-body">
    <section><a name="abs"></a><h3>Abstract</h3>
   <div class="collapsible-content">
      <h4>Background</h4>
      <p style="line-height:160%">Understanding the demographic history of a population is critical to conservation
         and to our broader understanding of evolutionary processes. For many tropical large
         mammals, however, this aim is confounded by the absence of fossil material and by
         the misleading signal obtained from genetic data of recently fragmented and isolated
         populations. This is particularly true for the lion which as a consequence of millennia
         of human persecution, has large gaps in its natural distribution and several recently
         extinct populations.
      </p>
      <h4>Results</h4>
      <p style="line-height:160%">We sequenced mitochondrial DNA from museum-preserved individuals, including the extinct
         Barbary lion (<em>Panthera leo leo</em>) and Iranian lion (<em>P. l. persica</em>), as well as lions from West and Central Africa. We added these to a broader sample
         of lion sequences, resulting in a data set spanning the historical range of lions.
         Our Bayesian phylogeographical analyses provide evidence for highly supported, reciprocally
         monophyletic lion clades. Using a molecular clock, we estimated that recent lion lineages
         began to diverge in the Late Pleistocene. Expanding equatorial rainforest probably
         separated lions in South and East Africa from other populations. West African lions
         then expanded into Central Africa during periods of rainforest contraction. Lastly,
         we found evidence of two separate incursions into Asia from North Africa, first into
         India and later into the Middle East.
      </p>
      <h4>Conclusions</h4>
      <p style="line-height:160%">We have identified deep, well-supported splits within the mitochondrial phylogeny
         of African lions, arguing for recognition of some regional populations as worthy of
         independent conservation. More morphological and nuclear DNA data are now needed to
         test these subdivisions.
      </p>
   </div>
</section><span id="keywords">
   <h5 class="inline">Keywords: </h5>Barbary lion; <em>Panthera leo</em>; Extinction; Mitochondrial DNA; Ancient DNA; Phylogeography</span><section><a name="sec1"></a><h3>Background</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">Understanding the population history of a species is critical, not only to gain insight
         into past evolutionary processes, but also as a means of predicting responses to future
         environmental change <a name="d3439e231"></a><a name="d3439e233"></a><a name="d3439e235"></a><a name="d3439e237"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>-<a onclick="LoadInParent('#B4'); return false;" href="#B4">4</a>]. Estimates of demographic history are increasingly reliant on genetic data, particularly
         in many tropical regions where the mammalian fossil record is constrained by poor
         preservation of bone (e.g. <a name="d3439e241"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>]). However, large carnivoran species have usually experienced historical persecution,
         and now occupy only a small subset of their former range <a name="d3439e245"></a><a name="d3439e247"></a><a name="d3439e249"></a><a name="d3439e251"></a>[<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>-<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a>]. This is problematic because sampling solely from extant populations provides an
         incomplete and potentially misleading view of the overall demographic history of a
         species <a name="d3439e255"></a><a name="d3439e257"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>,<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>] Under such circumstances, it is particularly important to incorporate historically
         collected samples into the data set, in order to achieve a more complete record of
         the spatial distribution of past genetic diversity <a name="d3439e261"></a><a name="d3439e263"></a>[<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>,<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>].
      </p>
      <p style="line-height:160%">The lion (<em>Panthera leo, sensu lato</em>) had one of the largest geographical distributions of any terrestrial mammal during
         the Late Pleistocene, ranging from southern Africa (<em>Panthera leo ssp.</em>), through northern Eurasia (<em>Panthera leo spelaea</em>), to Central America (<em>Panthera leo atrox</em>) <a name="d3439e281"></a>[<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>]. However, studies of the lion’s population history are made challenging by its sparse
         African Pleistocene fossil record <a name="d3439e286"></a>[<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>], as well as large-scale, human-driven extirpations in many parts of its historical
         range <a name="d3439e290"></a><a name="d3439e292"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>,<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>]. Widespread hunting and anthropogenic changes to lion habitat are continuing to reduce
         lion populations across their entire range <a name="d3439e296"></a><a name="d3439e298"></a>[<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>,<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>]. On the Kathiawar Peninsula in India, the Asian lion subspecies <em>P. leo persica</em> survives as a population of only ~400 individuals and has been classified as “Endangered”
         by the IUCN <a name="d3439e305"></a>[<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>]. Lions in Africa have experienced a suspected ~30% loss of the free-ranging population
         in the past two decades and are now classified as “Vulnerable” by the IUCN <a name="d3439e309"></a>[<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>]. Of special concern are the populations in West and Central Africa, which may be
         close to extinction in the wild and are underrepresented in the captive zoo population
         <a name="d3439e314"></a><a name="d3439e316"></a><a name="d3439e318"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>-<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>].
      </p>
      <p style="line-height:160%">Despite this regional variation in population dynamics and decline, all African lions
         are grouped under one conservation heading as <em>P. leo leo</em><a name="d3439e326"></a>[<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>]. A lack of robust data is a hindrance to producing an integrated and comprehensive
         evolutionary history for <em>P. leo</em> upon which to test assumptions about evolutionarily significant units <a name="d3439e333"></a>[<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>]. Therefore, a better understanding of the evolutionary relationships among modern
         lion populations is critical to developing evidence-based plans for their conservation
         and management.
      </p>
      <p style="line-height:160%">Previous analyses of short fragments of the mitochondrial control region (hypervariable
         region I) from modern and recently extinct lion populations revealed several distinct
         mitochondrial clades, but were unable to provide strong statistical support for these
         groupings <a name="d3439e339"></a><a name="d3439e341"></a><a name="d3439e343"></a><a name="d3439e345"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>,<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>]. Nonetheless, the extinct North African Barbary lion and the Asian lion (including
         the extinct Middle Eastern population) clearly form a mitochondrial group that is
         distinct from all sub-Saharan African lions <a name="d3439e349"></a><a name="d3439e351"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>]. This pattern is supported by morphological analyses that show similarities in cranial
         characters among North African and Asian lions, and separate these two populations
         from sub-Saharan African lions <a name="d3439e355"></a><a name="d3439e357"></a>[<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>,<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>]. Relationships among modern lions have also been studied using longer sequences of
         mitochondrial cytochrome <em>b</em> (<em>cytb</em>) and nuclear DNA sampled from extant populations in eastern and southern Africa <a name="d3439e368"></a><a name="d3439e370"></a><a name="d3439e372"></a>[<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>-<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>] and West and Central Africa <a name="d3439e376"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>].
      </p>
      <p style="line-height:160%">The results from genetic analyses have generally supported the accepted geographical
         subspecies designations <a name="d3439e382"></a><a name="d3439e384"></a>[<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>,<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a>], though confusion has occasionally arisen when using voucher specimens from zoological
         collections to represent wild populations <a name="d3439e388"></a>[<a onclick="LoadInParent('#B31'); return false;" href="#B31">31</a>]. The Moroccan Royal Lion collection, for example, has been used as a proxy for extinct
         North African Barbary lions [e.g. 28], despite the fact that mitochondrial studies
         have demonstrated that these samples cluster with lions from Central Africa, rather
         than North Africa <a name="d3439e392"></a><a name="d3439e394"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]. These studies have provided crucial information for understanding the phylogeographical
         relationships among modern lions. However, they were based on very short fragments
         of mitochondrial DNA, or samples drawn only from the extant populations, or data sets
         that included individuals with questionable geographical origins. For these reasons,
         there is a clear need for the phylogeography of the modern lion to be investigated
         using a robust molecular data set that includes representatives of extinct populations.
      </p>
      <p style="line-height:160%">In order to investigate the demographic and evolutionary history of African and Asian
         lions (<em>Panthera leo, sensu stricto)</em>, we have used a combination of ancient and modern DNA sequences. We make use of a
         new approach that integrates spatial and Bayesian phylogenetic analyses, allowing
         ancestral geographical states to be co-estimated with the phylogeny, evolutionary
         rate, and coalescence times <a name="d3439e403"></a>[<a onclick="LoadInParent('#B32'); return false;" href="#B32">32</a>]. We place this history into the context of environmental changes in the late Quaternary
         and discuss whether the current taxonomic and management paradigm reflects biological
         reality.
      </p>
   </div>
</section>
<section><a name="sec2"></a><h3>Methods</h3>
   <div class="collapsible-content">
      <h4>DNA extraction and amplification</h4>
      <p style="line-height:160%">We collected bone and tissue samples from known-origin lion specimens kept in natural
         history collections (Table&nbsp;<a name="d3439e416"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/table/T1">1</a>) that had previously produced 130&nbsp;bp sequences of the HVR1 control region <a name="d3439e420"></a><a name="d3439e422"></a><a name="d3439e424"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>,<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a>]. We extracted DNA using an ion-exchange column method <a name="d3439e428"></a>[<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>]. Briefly, we reduced samples of cortical bone to powder in a mikrodismembrator (Sartorius).
         We then digested bone powder overnight at 50°C in 2&nbsp;ml of buffer (0.425&nbsp;M EDTA pH8,
         1&nbsp;mM Tris–HCl pH8, 0.05%&nbsp;w/v SDS, 0.33&nbsp;mg/ml Proteinase K) under constant rotation.
         We concentrated the digested solution to approximately 500&nbsp;μl using centrifugal filters
         with a molecular weight cut-off of 30&nbsp;kDa (Amicon® Ultra, Millipore). We passed the
         concentrated solution through a silica column (QIAquick®, Qiagen) following the manufacturer’s
         protocol, and eluted the final extract in 100&nbsp;μl of TE buffer. We measured DNA concentration
         (Table&nbsp;<a name="d3439e431"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/table/T1">1</a>) using 2&nbsp;μl of extract on the Qubit® platform (Invitrogen), and stored the extracts
         at -20°C.
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/table/T1','T1',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/table/T1"><strong>Table 1.</strong></a> <strong>All ancient samples used in the analyses with their total DNA concentration, together
            with information on origin and previously published control region sequences</strong></p></div></div>
      <p style="line-height:160%">We selected primers to amplify short (&lt;200&nbsp;bp) overlapping fragments of mitochondrial
         <em>cytb</em> in order to maximize the success rate of amplification from historical specimens.
         We designed novel primers specific to <em>Panthera leo</em> that minimized contaminant DNA sequences, and also selected previously designed primers
         that successfully amplified DNA from pantherine cats <a name="d3439e893"></a>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>]. Nuclear inserts of mitochondrial regions are a known problem in pantherine cats
         and we carefully selected primers that preferentially amplified the cytoplasmic copy
         <a name="d3439e897"></a><a name="d3439e899"></a>[<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B37'); return false;" href="#B37">37</a>]. We used either Ampli<em>Taq</em>® Gold (ABI) or KAPA2G™ Robust HotStart (KAPA Biosystems) in PCRs according to manufacturers’
         guidelines, with a 90&nbsp;s activation step at 95°C, followed by 45&nbsp;cycles of 95°C for
         45&nbsp;s, T<sub>A</sub> (Additional file <a name="d3439e909"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S1">1</a>: Figure S1, Additional file <a name="d3439e912"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S2">2</a>: Figure S2, Additional file <a name="d3439e915"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1, Additional file <a name="d3439e918"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S4','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S4">4</a>: Table S2 and Additional file <a name="d3439e921"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S5','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S5">5</a>: Table S3) for 45&nbsp;s, and 72°C for 45&nbsp;s with a final extension at 72°C for 5&nbsp;min.
         PCRs were resolved on an agarose gel. Successful amplifications were purified by the
         addition of exonuclease I (0.1 u/μl, Fermentas, UK), buffer (B16, Fermentas, UK) and
         FastAP (0.0025 u/μl, Fermentas, UK), heated at 37°C for 10&nbsp;min, then inactivated at
         65°C for 15&nbsp;min. Purified PCRs were sequenced using BigDye® chemistry (ABI) and read
         on an ABI3730 at DBS Genomics (Durham, UK).
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S1','S1',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S1"></a><strong>Additional file 1: Figure S1.</strong> Schematic of contig overlaps indicating their amplification positions along 1140bp
         of cytochrome <em>b.</em></p><p>
         	Format: PDF 
         	Size: 226KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-14-70-s1.pdf">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.adobe.com/products/acrobat/readstep.html','Help');return false;" href="http://www.adobe.com/products/acrobat/readstep.html">Adobe Acrobat Reader</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/opendata.gif"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S2','S2',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S2"></a><strong>Additional file 2: Figure S2.</strong> Neighbour-joining trees produced with PAUP* <a name="d3439e944"></a>[<a onclick="LoadInParent('#B72'); return false;" href="#B72">72</a>] of individual amplicons produced in this study, compared to known cymt (AF053040
         &amp; AF053018) and numt (AF053053 &amp; AF053054) sequences from tiger <a name="d3439e948"></a>[<a onclick="LoadInParent('#B73'); return false;" href="#B73">73</a>].</p><p>
         	Format: PDF 
         	Size: 826KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-14-70-s2.pdf">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.adobe.com/products/acrobat/readstep.html','Help');return false;" href="http://www.adobe.com/products/acrobat/readstep.html">Adobe Acrobat Reader</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/opendata.gif"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','S3',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3"></a><strong>Additional file 3: Table S1.</strong> List of all Genbank accessible comparative sequences and relevant information.</p><p>
         	Format: DOCX 
         	Size: 107KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-14-70-s3.docx">Download file</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/opendata.gif"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S4','S4',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S4"></a><strong>Additional file 4: Table S2.</strong> List of all primers used and their annealing temperatures and lengths.</p><p>
         	Format: DOCX 
         	Size: 93KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-14-70-s4.docx">Download file</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/opendata.gif"></a></span></p></div>
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S5','S5',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S5"></a><strong>Additional file 5: Table S3.</strong> Table of number of independent PCRs sequenced for each region. Cells with an outlined
         border indicate regions containing unique mutations not found in other lions. Where
         2 cells share the same outlined border indicates mutation identified in region of
         PCR overlap and present in both contigs.</p><p>
         	Format: XLS 
         	Size: 34KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2148-14-70-s5.xls">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=1cd6acf9-ce06-4e1c-8dcf-f33f669dbc3a','Help');return false;" href="http://www.microsoft.com/downloads/details.aspx?displaylang=en&amp;FamilyID=1cd6acf9-ce06-4e1c-8dcf-f33f669dbc3a">Microsoft Excel Viewer</a><span class="open-data-icon"><a href="http://www.biomedcentral.com/about/access/#opendata"><img title="Open Data" alt="Open Data" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/opendata.gif"></a></span></p></div></div>
      <h4>Data analysis</h4>
      <p style="line-height:160%">Contigs were assembled in Se-Al v2.0a11 <a name="d3439e991"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>] and then aligned with 74 previously published lion sequences (Additional file <a name="d3439e994"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1). For a subset of individuals, both <em>cytb</em> and HVR1 control region sequences were available (Additional file <a name="d3439e1000"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1). We did not identify any nonsense or frameshift mutations in our data.
         In every instance, overlapping contigs were complementary, including overlaps where
         diagnostic mutations were also found (Additional file <a name="d3439e1003"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S1">1</a>: Figure S1, Additional file <a name="d3439e1007"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S2">2</a>: Figure S2, Additional file <a name="d3439e1010"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1, Additional file <a name="d3439e1013"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S4','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S4">4</a>: Table S2 and Additional file <a name="d3439e1016"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S5','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S5">5</a>: Table S1). Both individual amplicons and assembled contigs showed greater similarity
         to mitochondrial sequences than to nuclear inserts of mitochondrial regions (Additional
         file <a name="d3439e1019"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S1">1</a>: Figure S1, Additional file <a name="d3439e1022"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S2">2</a>: Figure S2, Additional file <a name="d3439e1026"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1, Additional file <a name="d3439e1029"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S4','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S4">4</a>: Table S2 and Additional file <a name="d3439e1032"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S5','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S5">5</a>: Table S1).
      </p>
      <p style="line-height:160%">To estimate the relationships among <em>cytb</em> haplotypes, we generated a median network <a name="d3439e1041"></a>[<a onclick="LoadInParent('#B39'); return false;" href="#B39">39</a>] using the median-joining algorithm in the program Network v4.610 (<a href="http://www.fluxus-engineering.com/">http://www.fluxus-engineering.com</a> <a href="http://www.webcitation.org/query.php?url=http://www.fluxus-engineering.com&amp;refdoi=10.1186/1471-2148-14-70" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>). More published sequences were available on GenBank that consisted only of <em>cytb</em> than of <em>cytb</em> + control region. Network analysis allowed us to confirm that congruent trees were
         inferred using both a shorter DNA dataset containing more sequences and our longer
         DNA dataset containing fewer sequences.
      </p>
      <p style="line-height:160%">Bayesian phylogenetic analyses were performed using BEAST v1.7 <a name="d3439e1056"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>]. The <em>cytb</em> and control region data were concatenated (1186&nbsp;bp) and the HKY substitution model
         was selected for each region using the Bayesian information criterion. This criterion
         has been shown to perform well under a variety of simulation scenarios <a name="d3439e1063"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>]. Owing to the intraspecific nature of the data set, a strict molecular clock was
         assumed and a separate rate was allowed for each region. However, calibrating the
         molecular clock can be difficult when the data set involves deep coalescence events
         <a name="d3439e1067"></a><a name="d3439e1069"></a><a name="d3439e1071"></a>[<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a>-<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>]. Although our data set includes ancient DNA samples, they are not old enough to provide
         sufficient calibrating information for the molecular clock <a name="d3439e1075"></a>[<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>]. Instead, we favour an approach that combines two sources of calibrating information.
         First, we employed a previous estimate of the substitution rate based on the control
         region of cave lions, calibrated using the radiocarbon dates of the samples <a name="d3439e1080"></a>[<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>]. This was implemented using a normal prior (mean 5.35×10<sup>−7</sup> substitutions/site/year, standard deviation 2.28×10<sup>−7</sup> substitutions/site/year) for the substitution rate of the control region. The mutation
         rate of <em>cytb</em>, relative to the control region, was estimated in the analysis. Second, we added
         a sequence from a cave lion, <em>P. leo spelaea</em>, which allowed us to specify a calibration for the divergence between <em>P. leo leo</em> and <em>P. leo spelaea</em>. We used a normal prior with a mean of 550,000&nbsp;years and standard deviation of 25,000&nbsp;years
         for the timing of this split, based on the appearance of the ancestral cave lion <em>P. leo fossilis</em> in the European fossil record <a name="d3439e1106"></a><a name="d3439e1108"></a>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>,<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>].
      </p>
      <p style="line-height:160%">An analysis using Bayes factors <a name="d3439e1114"></a>[<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a>] supported a model with constant population size as the best-fitting coalescent model
         for the data set. We obtained posterior estimates of parameters via Markov chain Monte
         Carlo (MCMC) sampling. Samples were drawn every 1,000 steps over 10,000,000 MCMC steps,
         with the first 10% discarded as burn-in. Acceptable sampling and convergence to the
         stationary distribution were checked by inspection of traces using Tracer v1.5 <a name="d3439e1118"></a>[<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>].
      </p>
      <p style="line-height:160%">For the phylogeographical analyses, we ran two independent MCMC simulations of 50,000,000
         steps each using the discrete phylogeographical model described in Edwards <em>et al</em>. <a name="d3439e1127"></a>[<a onclick="LoadInParent('#B32'); return false;" href="#B32">32</a>]. We assigned each sequence to one of seven geographical locations, broadly reflecting
         traditional subspecies designations <a name="d3439e1131"></a><a name="d3439e1133"></a>[<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>,<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a>]: Middle East, Asia, Central Africa, East Africa, West Africa, South Africa, and North
         Africa (Table&nbsp;<a name="d3439e1136"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/table/T1">1</a> and Additional file <a name="d3439e1139"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1), and estimated rates of diffusion to and from each location. Settings
         for the analysis matched those described above, except we replaced the constant-size
         coalescent prior with the flexible Gaussian Markov random field prior (skyride plot;
         <a name="d3439e1144"></a>[<a onclick="LoadInParent('#B49'); return false;" href="#B49">49</a>]).
      </p>
      <h4>Data authenticity</h4>
      <p style="line-height:160%">We employed strict protocols to eliminate contamination and other potential artefacts
         <a name="d3439e1154"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>]. We used only specimens that had been shown previously to contain authentic endogenous
         DNA <a name="d3439e1158"></a><a name="d3439e1160"></a><a name="d3439e1162"></a><a name="d3439e1164"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>,<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>,<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a>]. DNA extractions were performed in a physically isolated laboratory separated from
         any modern molecular biology work, and in which no work on felids had ever been attempted.
         Samples were co-extracted with negative controls in extract:control ratios of between
         2:1 and 9:1. We used sterile reagents, filtered pipette tips, and bleached and UV-irradiated
         surfaces and instruments. We set up PCRs in an isolated facility, under positive air
         pressure in a dedicated sterile hood, incorporating negative PCR and extraction controls.
         Following set-up, we manually transported PCRs to a different laboratory for thermal
         cycling and post-PCR work. We confirmed all putative novel mutations on the basis
         of at least three independent PCRs (Additional file <a name="d3439e1167"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S1">1</a>: Figure S1, Additional file <a name="d3439e1170"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S2">2</a>: Figure S2, Additional file <a name="d3439e1173"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1, Additional file <a name="d3439e1177"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S4','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S4">4</a>: Table S2 and Additional file <a name="d3439e1180"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S5','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S5">5</a>: Table S1). Additionally, several specimens (PL1, PL4, PL13, PL15, and PL16) produced
         sequences that were identical to those previously published for lions from nearby
         geographic regions.
      </p>
   </div>
</section>
<section><a name="sec3"></a><h3>Results</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">We amplified a 1051&nbsp;bp fragment of <em>cytb</em> from 14 museum-preserved lions (Table&nbsp;<a name="d3439e1192"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/table/T1">1</a>). These lions were sampled from extinct populations in North Africa and the Middle
         East and endangered populations in West and Central Africa (Figure&nbsp;<a name="d3439e1195"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F1">1</a>).
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F1','F1',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F1"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/1471-2148-14-70-1.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 1.</strong></a> <strong>A map of the source areas for the lion samples analysed in this study.</strong> Numbers within circles correspond to PL numbers in Table&nbsp;<a name="d3439e1210"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/table/T1">1</a>. Squares correspond to sequences available on Genbank and identified in Additional
         file <a name="d3439e1213"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1. Colours of squares and circles correspond to those used in Figure&nbsp;<a name="d3439e1216"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>. *PL11 and PL12 are medieval English lions that have been identified as North African
         lions (<em>P. leo leo</em>) based on mitochondrial HVR sequences and morphological data.
      </p></div></div>
      <p style="line-height:160%">From these, we identified four new mitochondrial haplotypes: one from North Africa,
         one from a suspected Barbary lion present in medieval London, one from Iran, and one
         from Senegal. All other samples produced haplotypes identical to published data and
         were compatible with their geographical origin (Additional file <a name="d3439e1225"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1). Four of the six Barbary lions exhibited a <em>cytb</em> sequence identical to that of the extant Indian lion (Table&nbsp;<a name="d3439e1231"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/table/T1">1</a> &amp; Additional file <a name="d3439e1234"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1).
      </p>
      <p style="line-height:160%">Network analyses of North African and Asian <em>cytb</em> sequences show a simple starburst pattern with a basal haplotype shared between North
         African and Indian lions (Figure&nbsp;<a name="d3439e1242"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>A). Additional diversity is represented by Iranian lions that share a haplotype separated
         by a single synonymous mutation from the central Indian haplotype, and a wild-shot
         North African lion and putative Barbary lion from medieval England that differ by
         two and three synonymous mutations, respectively.
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','F2',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/1471-2148-14-70-2.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 2.</strong></a> <strong>Phylogenetic analyses of lion sequence data. A)</strong> Median network of 1051&nbsp;bp of <em>cytb</em> for all 88 lion individuals identified from GenBank plus those generated in this
         study. <em>Panthera leo spelaea</em> was used as an outgroup. Circles are proportional to haplotype frequencies and black
         circles represent hypothesized intermediate haplotypes. The number of links represent
         the number of mutations between haplotypes. Haplotypes are labelled from A to S and
         correspond to sequences labelled in Table&nbsp;<a name="d3439e1263"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/table/T1">1</a> and Additional file <a name="d3439e1266"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/suppl/S3">3</a>: Table S1. <strong>B)</strong> Phylogenetic tree from a Bayesian analysis of combined <em>cytb</em> and control region data for all lion taxa where available (n = 54). Posterior probabilities
         of supported clades are shown at nodes. Estimates of divergence times: <strong>(a)</strong> 124,200&nbsp;years (95% credibility: 81,800-183,500); <strong>(b)</strong> 61,500&nbsp;years (32,700-97,300); <strong>(c)</strong> 51,000&nbsp;years (26,600-83,100); <strong>(d)</strong> 81,900&nbsp;years (45,700-122,200); <strong>(e)</strong> 57,800&nbsp;years (26,800-96,600); <strong>(f)</strong> 21,100&nbsp;years (8300–38,800). Branch colours correspond to reconstructed ancestral
         geographic states (Purple, South Africa; Yellow, East Africa; Orange, West Africa;
         Red, Central Africa; Teal, North Africa; Blue, South Asia; Green, Near-East). Tip
         colours correspond to origins of samples.
      </p></div></div>
      <p style="line-height:160%">Bayesian phylogenetic analysis of combined <em>cytb</em> and control region data suggests a clear clustering pattern in which North African/Asian,
         western African, central African, eastern-southern African, and southern African lion
         populations are recognized, although the two Southern African clusters are not as
         internally coherent as the other three clades (Figure&nbsp;<a name="d3439e1301"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>B). The most recent common ancestor of the five groups (node a in Figure&nbsp;<a name="d3439e1304"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>B) was estimated to have existed in the Late Pleistocene (124,200&nbsp;years BP, 95% HPD:
         81,800-183,500). This date overlaps considerably with two published estimates for
         the MRCA of lions, 74,000-203,000&nbsp;years BP <a name="d3439e1308"></a>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>] or 145,000-502,000&nbsp;years BP <a name="d3439e1312"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>]. It should be noted that the external fossil calibration prior used in our analysis
         has had a substantial effect on the estimates of divergence times when compared to
         using only the internally calibrated mutation rate from the control region data. If
         <em>Panthera leo fossilis</em> is a poor indicator of the split between <em>P.leo leo</em> and <em>P.leo spelaea</em>, then this would have concomitant effects on the divergence estimate <a name="d3439e1326"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>].
      </p>
      <p style="line-height:160%">The phylogeographical methods also allowed us to estimate the divergence times of
         individual lion clades (Figure&nbsp;<a name="d3439e1331"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>B). The split between North African/Asian and West and Central African lions (node
         b in Figure&nbsp;<a name="d3439e1334"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>B) took place approximately 61,500&nbsp;years BP (95% HPD: 32,700-97,300). North African/Asian
         and West African lions split (node c in Figure&nbsp;<a name="d3439e1337"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>B) by approximately 51,000&nbsp;years BP (95% HPD: 26,600-83,100). The movement of lions
         out of Africa and into the Near-East/Asia appears to have occurred in two recent waves.
         The first expansion from North Africa (node f in Figure&nbsp;<a name="d3439e1340"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>B) occurred around 21,100&nbsp;years BP (95% HPD: 8,300-38,800) resulting in the major
         Asian lion clade, and the second migration later produced the Middle-Eastern lion
         clade.
      </p>
      <p style="line-height:160%">The geographical component incorporated into the Bayesian analysis also allowed us
         to reconstruct ancestral geographical states (Figures&nbsp;<a name="d3439e1345"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>B and <a name="d3439e1348"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F3">3</a>) and assign probabilities to the movements. It should be noted that these reconstructed
         movements are only based on maternally inherited mitochondrial DNA, and are therefore
         subject to future modification when biparentally inherited nuclear DNA becomes available.
         As suggested previously <a name="d3439e1352"></a><a name="d3439e1354"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>,<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>], the geographical base of the lion tree is located in sub-Saharan Africa, with movement
         between East and South Africa (Bayes factor: 4.83/4.66). The West African population
         probably acted as a source from which a population expanded into Central Africa (Bayes
         factor: 3.00). During the Holocene, the North African population expanded into South
         Asia (Bayes factor: 4.37/4.50), and then again into the Near East (Bayes factor: 21.03).
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F3','F3',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F3"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/1471-2148-14-70-3.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 3.</strong></a> <strong>Reconstructed distribution of the modern lion at different times.</strong> Estimates of spatial diffusion pathways at Marine Isotope Stage (MIS) time points:
         <strong>A.</strong> MIS5 <strong>B.</strong> MIS4-MIS3 <strong>C.</strong> MIS2-MIS1 <strong>D.</strong> Estimated natural distribution prior to anthropogenic disturbance. Black arrows show
         estimated spatial diffusions, with thicknesses proportional to Bayes factors. Movement
         from East Africa to South Africa (4.83), from South Africa to East Africa (4.66),
         from West Africa to Central Africa (3.00), from North Africa to South Asia (4.37),
         from South Asia to North Africa (4.50), from North Africa to Middle East (21.03).
         Tropical rainforest is shown in light grey (present distribution), maximal extent
         during humid periods (black dashed line), and minimal extent during arid periods (white
         dashed line). The Great Rift Valley is shown in dark grey. African rivers are shown
         in blue. Co, Congo; Ng, Niger; Ni, Nile; Se, Senegal.
      </p></div></div>
   </div>
</section>
<section><a name="sec4"></a><h3>Discussion</h3>
   <div class="collapsible-content">
      <h4>Demographic and evolutionary history</h4>
      <p style="line-height:160%">We identified a Late Pleistocene origin for all five major phylogeographical groups
         of the modern lion (Figure&nbsp;<a name="d3439e1392"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>), thus providing additional support for a single-African-origin model. This model
         was first proposed by Yamaguchi <em>et al.</em><a name="d3439e1398"></a>[<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>] based on morphology, distribution, and parietal art and has been reinforced by aDNA
         studies of extinct Late Pleistocene lions <a name="d3439e1402"></a><a name="d3439e1404"></a><a name="d3439e1406"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>,<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>,<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>]. Additional analyses of ancient DNA from historical examples of modern lions suggest
         that sub-Saharan Africa, with a tentatively identified focal point in eastern-southern
         Africa, was the likely evolutionary cradle of the modern lion <a name="d3439e1410"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>]. The eastern-southern origin has been independently supported by Antunes <em>et al.</em><a name="d3439e1417"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>], who based their findings on mtDNA and lion feline immunodeficiency virus (FIV) sequences
         amplified from 357 modern East African, South African and Indian lions. Our results,
         showing longer branches amongst eastern-southern African lions (Figure&nbsp;<a name="d3439e1420"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>), also support the idea that the evolutionary cradle of the modern lion was in eastern-southern
         Africa.
      </p>
      <p style="line-height:160%">Our results suggest a very late date for the most recent common ancestor of all five
         phylogeographical groups. Burger <em>et al.</em><a name="d3439e1428"></a>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>] previously concluded that the most recent common ancestor of the modern lion lived
         74–203 thousand years BP based on mitochondrial <em>cytb</em> sequences calibrated with fossil data from <em>P. leo fossilis</em>. Antunes <em>et al.</em><a name="d3439e1440"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>] suggested c. 145–502 thousand years BP for a major range expansion of the modern
         lion, including the colonization of south-western Eurasia based on mtDNA (12S and
         16S rRNA genes). Although our estimate of the divergence time overlaps with both previous
         estimates, we demonstrate that the modern lion exodus from Africa took place only
         ~21,000&nbsp;years ago. This analysis has not previously been possible due to the extinction
         of sister populations in North Africa and Middle East, now accessible for study through
         ancient DNA.
      </p>
      <p style="line-height:160%">Our use of statistical phylogeography and DNA sequences from purported refugial regions
         allows us to reconstruct the past movements of lion populations and place them in
         the context of palaeoclimatic evidence. Late Pleistocene Africa experienced massive,
         sudden fluctuations in hydrology caused by the glacial/interglacial cycle, with concomitant
         effects on vegetation and biome distribution <a name="d3439e1446"></a><a name="d3439e1448"></a><a name="d3439e1450"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>-<a onclick="LoadInParent('#B53'); return false;" href="#B53">53</a>]. More specifically, several episodes of rapid cycling between high humidity and aridity
         have been identified from the palynological record <a name="d3439e1454"></a><a name="d3439e1456"></a><a name="d3439e1458"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>,<a onclick="LoadInParent('#B53'); return false;" href="#B53">53</a>,<a onclick="LoadInParent('#B54'); return false;" href="#B54">54</a>], with the three most recent humid phases having occurred during Marine Isotope Stage
         5 (MIS5, 120-110Ka), Marine Isotope stage 3 (MIS3, 50-45Ka) and the early Holocene
         (10-6Ka) <a name="d3439e1462"></a><a name="d3439e1464"></a><a name="d3439e1466"></a>[<a onclick="LoadInParent('#B55'); return false;" href="#B55">55</a>-<a onclick="LoadInParent('#B57'); return false;" href="#B57">57</a>]. During periods of high humidity, tropical rainforest expanded across the equatorial
         region, and the Sahara became savanna <a name="d3439e1470"></a><a name="d3439e1472"></a><a name="d3439e1474"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>,<a onclick="LoadInParent('#B55'); return false;" href="#B55">55</a>,<a onclick="LoadInParent('#B56'); return false;" href="#B56">56</a>]. Conversely, during periods of high aridity, tropical rainforest contracted to form
         isolated refugia as the Sahara expanded <a name="d3439e1478"></a><a name="d3439e1480"></a><a name="d3439e1482"></a><a name="d3439e1484"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>,<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>,<a onclick="LoadInParent('#B54'); return false;" href="#B54">54</a>,<a onclick="LoadInParent('#B56'); return false;" href="#B56">56</a>]. The timing and nature of these changes correlate well with inferred patterns of
         lion diversification and suggest a mechanism to explain the phylogeographical patterning
         of lions in Africa and Asia.
      </p>
      <p style="line-height:160%">During the Middle Pleistocene, prior to MIS5, lions were probably widespread over
         Africa, occupying regions of savanna/scrub-woodland. During the humid period of MIS5,
         tropical rainforest expansion eastward from the Gulf of Guinea to the Great Rift Valley
         would have isolated southern and eastern African populations from western and northern
         populations, corresponding to the basal divergence among lion lineages ~124,200&nbsp;years
         BP (95% HPD: 81,800-183,500). As aridity increased, leading into MIS4, the Sahara
         expanded and separated lion populations in North Africa and West Africa. This climatic
         phase corresponds to the bifurcation of these two populations, which is estimated
         to have occurred ~51,000&nbsp;years (95% HPD: 26,600-83,100).
      </p>
      <p style="line-height:160%">The contraction of tropical rainforests, as the continent dried, permitted the expansion
         of lions from West Africa into newly open biomes in central Africa, and a signal of
         this movement is recovered by the statistical phylogeographical analysis. At the same
         time, there is evidence for a complicated interaction between populations in East
         and South Africa, with the Rift Valley potentially acting as a partial barrier to
         dispersal. Lastly, our phylogenetic and demographic reconstructions provide evidence
         for two separate excursions into Asia by lions from North Africa, initially during
         the end-Pleistocene ~21,000&nbsp;years BP (95% HPD: 8,300-38,800). The most recent population
         movement involves Iranian lions that appear to be descended from North African lions
         dispersing during the mid-late Holocene. The recent dates for emigration to Asia are
         interesting because lion remains have been found from throughout the Late Pleistocene
         in the Middle East <a name="d3439e1492"></a><a name="d3439e1494"></a>[<a onclick="LoadInParent('#B58'); return false;" href="#B58">58</a>,<a onclick="LoadInParent('#B59'); return false;" href="#B59">59</a>]. However, the genetic data presented here only reflect the most recently arrived
         populations in the region. Populations that resulted from earlier expansions are unlikely
         to be detected since their genetic signatures would have been overwritten by those
         of later expansions.
      </p>
      <p style="line-height:160%">The identification of well-supported phylogeographical groups of lions naturally prompts
         the question of what has maintained these divisions. Several obvious dispersal barriers
         have been proposed in previous publications, including the Great Rift Valley <a name="d3439e1500"></a>[<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>], tropical rainforests <a name="d3439e1504"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>], and the Sahara <a name="d3439e1508"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>], though other topographic obstacles are also likely to have played a role. The separation
         between West African and Central African lions occurs in the region between Benin
         and Cameroon, an area bisected by the Niger River (Figure&nbsp;<a name="d3439e1511"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F3">3</a>). Similarly, the Central African lions appear to be bounded by the Nile on their
         eastern front, separating them from the Eastern African group (Figure&nbsp;<a name="d3439e1514"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F3">3</a>), though the sparse sampling from this region makes identifying barriers difficult.
         The influence of large rivers on African mammal phylogeography has been demonstrated
         for chimpanzees and bonobos (<em>Pan</em> spp.), where the Niger River may act as a subspecies barrier <a name="d3439e1522"></a>[<a onclick="LoadInParent('#B60'); return false;" href="#B60">60</a>]. Other African savannah ungulates, such as giraffes (<em>Giraffa camelopardalis</em>) also show a congruent pattern <a name="d3439e1529"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>]. Similarly, large rivers appear to act as biogeographic boundaries to a certain extent,
         even for the jaguar (<em>Panthera onca</em>) which is renowned for its ability to swim <a name="d3439e1536"></a>[<a onclick="LoadInParent('#B61'); return false;" href="#B61">61</a>]. It is probable that deserts, valleys, and rivers and watershed boundaries have all
         acted to maintain the phylogeographical structure of lions.
      </p>
      <h4>Conservation</h4>
      <p style="line-height:160%">Our analyses recovered five major phylogeographical groups in the modern lion: North
         African/Asian, West African, Central African, South African, and East-South African.
         All of these could be designated as Evolutionarily Significant Units (ESUs) in the
         absence of conflicting morphological or nuclear DNA data <a name="d3439e1546"></a>[<a onclick="LoadInParent('#B62'); return false;" href="#B62">62</a>] (Figure&nbsp;<a name="d3439e1549"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>). This pattern is consistent with previous studies based on control region data from
         across the species range <a name="d3439e1553"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>], and studies using other mitochondrial regions to examine detailed phylogeographical
         patterns in sub-Saharan Africa <a name="d3439e1557"></a><a name="d3439e1559"></a>[<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>,<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>] and western-central Africa <a name="d3439e1563"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>].
      </p>
      <p style="line-height:160%">International bodies currently recognize only two lion conservation units: African
         and Asian lions <a name="d3439e1569"></a>[<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>] on the basis of early attempts to categorize lions using crude allozyme separation
         <a name="d3439e1573"></a><a name="d3439e1575"></a>[<a onclick="LoadInParent('#B63'); return false;" href="#B63">63</a>,<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>]. DNA sequence studies have questioned these widely accepted legislative conservation
         units because the current dichotomy does not coincide with the intraspecific phylogeny
         estimated using a wider sampling regime <a name="d3439e1579"></a><a name="d3439e1581"></a><a name="d3439e1583"></a><a name="d3439e1585"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>,<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>].
      </p>
      <p style="line-height:160%">Our results further suggest that this dichotomy requires revision (Figure&nbsp;<a name="d3439e1590"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>). The mitochondrial data clearly show that Asian lions are nested within the diversity
         present in Central, West, and North Africa. Perhaps <em>Panthera leo persica</em> should be treated as consubspecific with <em>Panthera leo leo,</em> or alternatively the other phylogeographical groups could be considered for elevation
         to the same status. Of particular concern are the central African and western African
         populations, which may be close to extinction, with estimates of ~800 lions in West
         Africa and ~900 lions in Central Africa <a name="d3439e1600"></a>[<a onclick="LoadInParent('#B65'); return false;" href="#B65">65</a>]. Our data confirm the distinct nature of western African lions and the need to afford
         them appropriate protection <a name="d3439e1604"></a>[<a onclick="LoadInParent('#B66'); return false;" href="#B66">66</a>]. At the same time, we encourage a careful approach when discussing ESUs based on
         mtDNA, rather than morphology or nuclear DNA, due to the scale-dependent and static
         nature of many units (see <a name="d3439e1609"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>] for detailed arguments).
      </p>
      <p style="line-height:160%">The close phylogenetic relationships among Barbary, Iranian, and Indian lion populations
         are noteworthy given their considerable geographical separation (Figures&nbsp;<a name="d3439e1614"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F1">1</a> and <a name="d3439e1617"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>). Nonetheless, the extinct North African Barbary lion harboured appreciable genetic
         diversity prior to extirpation, including unique <em>cytb</em> haplotypes (Table&nbsp;<a name="d3439e1623"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/table/T1">1</a>, Figure&nbsp;<a name="d3439e1626"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/figure/F2">2</a>A). Individuals PL3 and PL12 (Table&nbsp;<a name="d3439e1630"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/14/70/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/14/70/table/T1">1</a>) differ from the majority haplotype, though neither sample is associated with a specific
         provenance. PL3 was collected in “Barbary” according to museum notes, whereas PL12
         was kept in the Tower of London during the 15th Century and its North African origin
         has only recently been identified <a name="d3439e1634"></a>[<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a>]. In comparison to the Barbary lions from Tunisia and Algeria, the divergent haplotypes
         might reflect the historical presence of additional population subdivisions within
         North African lions (e.g., Atlas Mountains and Mediterranean coast populations), evidence
         of a large, diverse, panmictic population, or incomplete lineage sorting. Barbary
         lion samples with precise provenance data would be needed to resolve the issue.
      </p>
      <p style="line-height:160%">The restoration of the extinct North African Barbary lion has attracted the attention
         of conservationists both inside and outside North Africa <a name="d3439e1640"></a><a name="d3439e1642"></a><a name="d3439e1644"></a><a name="d3439e1646"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B31'); return false;" href="#B31">31</a>,<a onclick="LoadInParent('#B67'); return false;" href="#B67">67</a>,<a onclick="LoadInParent('#B68'); return false;" href="#B68">68</a>]. Although circumstantial evidence suggested that the Barbary lion could have survived
         in captivity <a name="d3439e1650"></a><a name="d3439e1652"></a>[<a onclick="LoadInParent('#B67'); return false;" href="#B67">67</a>,<a onclick="LoadInParent('#B68'); return false;" href="#B68">68</a>], the most likely descendants of wild Barbary lions from the Moroccan Royal Menagerie
         do not appear to be (maternally) Barbary [19, this study]. However, there is a close
         mitochondrial relationship between the Barbary lion and the extant Indian lion, and
         this has been tentatively (but independently) supported by non-molecular studies <a name="d3439e1656"></a><a name="d3439e1658"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>,<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a>].
      </p>
      <p style="line-height:160%">In the tiger, another charismatic felid species, studies of ancient mitochondrial
         DNA have suggested a close relationship between the extinct central Asian Caspian
         tiger (<em>Panthera tigris virgata</em>) and the extant Amur tiger (<em>P. t. altaica</em>) <a name="d3439e1670"></a>[<a onclick="LoadInParent('#B69'); return false;" href="#B69">69</a>]. This has allowed conservationists to discuss the translocation of Amur tiger stock
         to occupy the former range of the Caspian tiger <a name="d3439e1674"></a>[<a onclick="LoadInParent('#B70'); return false;" href="#B70">70</a>], with support from the World Tiger Summit <a name="d3439e1678"></a>[<a onclick="LoadInParent('#B71'); return false;" href="#B71">71</a>]. Similarly, if no examples of purebred Barbary lions can be found within the zoo
         population, there might be scope for restoration of the North African lion population
         using the closely related Indian lion.
      </p>
   </div>
</section>
<section><a name="sec5"></a><h3>Conclusion</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">Ancient DNA, in combination with Bayesian phylogeographical analysis, has enabled
         us to infer a maternal evolutionary and demographic history of the lion that would
         be impossible using modern data alone. The inclusion of representatives of populations
         from North Africa, West Africa, Central Africa, and the Middle East fills major gaps
         in our understanding of the past and present distribution of lions and our analysis
         has led to two major findings. Firstly, our identification of the influence of cyclical
         changes in African climate, and of the expansion from refugia in western, southern
         and northern Africa, highlight the importance of these regions for lion diversity.
         Secondly, population subdivisions identified here are sufficiently divergent to warrant
         recognition as ESUs. Our data regarding these ESUs will greatly aid conservation planning
         of these important taxa over the complete natural range of modern lions.
      </p>
      <p style="line-height:160%">These data can now also be used to help establish the likely provenance of zoo lions,
         as an assessment before potential inclusion in <em>ex-situ</em> breeding programmes. The search should now focus on nuclear markers to expand and
         confirm the subdivisions we have identified. The need for this will become increasingly
         clear as the unique lion populations of sub-Saharan Africa edge closer to endangered
         status.
      </p>
      <h4>Availability of data</h4>
      <p style="line-height:160%">The data supporting the results of this article are available from GenBank (DNA sequences,
         accession nos. KJ545522-KJ545535).
      </p>
   </div>
</section>
<section><a name="sec6"></a><h3>Abbreviations</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">MIS: Marine Isotope Stage; ESU: Evolutionarily Significant Unit; HPD: Highest Posterior
         Density; HVR1: Highly Variable Region 1; EDTA: Ethylenediaminetetraacetic acid; SDS:
         Sodium Dodecyl Sulfate; PCR: Polymerase Chain Reaction; IUCN: International Union
         for Conservation of Nature; MCMC: Markov Chain Monte Carlo; cytb: cytochrome b; bp:
         base pairs; kDa: KiloDaltons.
      </p>
   </div>
</section>
<section><a name="sec7"></a><h3>Competing interests</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">The authors declare they have no competing interests.</p>
   </div>
</section>
<section><a name="sec8"></a><h3>Authors’ contributions</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">RB, GL, and NY conceived the study. RB carried out the molecular genetic work. RB,
         BS, and SYWH performed data analysis. NY, RS, JC, and LW coordinated sampling of material.
         NY, IB, and LW interpreted the biological context of results. All authors wrote, read
         and approved the final manuscript.
      </p>
   </div>
</section>
<section><a name="ack"></a><h3>Acknowledgements</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">We thank F. Renoud and D. Robineau (Muséum national d’Histoire naturelle, Paris),
         B. Fernholm and O. Grönwall, (Swedish Museum of Natural History, Stockholm), and C.
         Smeenk (Natural History Museum, Leiden) for access to sample material. We also thank
         J. Burger (Johannes Gutenburg University, Mainz) for helpful comments and discussion.
         This project has received funding from the European Union’s Seventh Framework Programme
         for research, technological development and demonstration under grant agreement no.
         FP7-PEOPLE-2011-IEF-298820.
      </p>
   </div>
</section>
<section><a name="refs"></a><h3>References</h3>
   <div id="article-references" class="collapsible-content">
      <ol id="references">
         <li id="B1">
            <p><a name="B1"></a> Mellows A,  Barnett R,  Dalen L,  Sandoval-Castellanos E,  Linderholm A,  McGovern TH,  Church MJ,  Larson G: <strong> The impact of past climate change on genetic variation and population connectivity
                  in the Icelandic arctic fox. </strong></p><p><em>Proc Biol Sci</em> 2012,  <strong>279</strong>(1747)<strong>:</strong>4568-4573.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/22977155">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22977155">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=22977155">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B1" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B1','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B2">
            <p><a name="B2"></a> Dalén L,  Nyström V,  Valdiosera C,  Germonpre M,  Sablin M,  Turner E,  Angerbjörn A,  Arsuaga JL,  Götherstrom A: <strong> Ancient DNA reveals lack of postglacial habitat tracking in the arctic fox. </strong></p><p><em>Proc Natl Acad Sci U S A</em> 2007,  <strong>104</strong>(16)<strong>:</strong>6726-6729.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/17420452">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17420452">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17420452">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B2" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B2','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B3">
            <p><a name="B3"></a> Crandall KA,  Bininda-Emonds ORP,  Mace GM,  Wayne RK: <strong> Considering evolutionary processes in conservation biology. </strong></p><p><em>Trends Ecol Evol</em> 2000,  <strong>15</strong>(7)<strong>:</strong>290-295.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/10856956">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10856956">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B3" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B3','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B4">
            <p><a name="B4"></a> Lorenzen ED,  Nogues-Bravo D,  Orlando 
L,  Weinstock J,  Binladen J,  Marske KA,  Ugan A,  Borregaard MK,  
Gilbert MT,  Nielsen R,  Ho SYW,  Goebel T,  Graf KE,  Byers D,  
Stenderup JT,  Rasmussen M,  Campos PF,  Leonard JA,  Koepfli KP,  
Froese D,  Zazula G,  Stafford TW Jr,  Aaris-Sorensen K,  Batra P,  
Haywood AM,  Singarayer JS,  Valdes PJ,  Boeskorov G,  Burns JA,  
Davydov SP, <em>et al</em>.: <strong> Species-specific responses of Late Quaternary megafauna to climate and humans. </strong></p><p><em>Nature</em> 2011,  <strong>479</strong>(7373)<strong>:</strong>359-364.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/22048313">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22048313">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B4" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B4','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B5">
            <p><a name="B5"></a> Lorenzen ED,  Heller R,  Siegismund HR: <strong> Comparative phylogeography of African savannah ungulates. </strong></p><p><em>Mol Ecol</em> 2012,  <strong>21</strong>(15)<strong>:</strong>3656-3670.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/22702960">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22702960">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B5" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B5','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B6">
            <p><a name="B6"></a> Nowell K,  Jackson P: <em>Wild Cats: Status Survey and Conservation Action Plan</em>.  Gland, Switzerland and Cambridge, UK: IUCN/SSC Cat Specialist Group;  1996. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B6" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B6','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B7">
            <p><a name="B7"></a> Mills G,  Hofer H: <em>Hyaenas: Status Survey and Conservation Action Plan</em>.  Gland, Switzerland and Cambridge, UK: IUCN/SSC Hyaena Specialist Group;  1998. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B7" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B7','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B8">
            <p><a name="B8"></a> Servheen C,  Herrero S,  Peyton B: <em>Bears: Status Survey and Conservation Action Plan</em>.  Gland, Switzerland and Cambridge, UK: IUCN/SSC Bear Specialise Group IUCN/SSC Polar
               Bear Specialist Group;  1999. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B8" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B8','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B9">
            <p><a name="B9"></a> Sillero-Zubri C,  Hoffmann M,  Macdonald DW: <em>Canids: Foxes, Wolves, Jackals And Dogs: Status Survey and Conservation Action Plan</em>.  Gland, Switzerland and Cambridge, UK: IUCN/SSC Dog Specialist Group;  2004. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B9" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B9','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B10">
            <p><a name="B10"></a> Ramakrishnan U,  Hadly EA: <strong> Using phylochronology to reveal cryptic population histories: review and synthesis
                  of 29 ancient DNA studies. </strong></p><p><em>Mol Ecol</em> 2009,  <strong>18</strong><strong>:</strong>1310-1330.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/19281471">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19281471">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B10" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B10','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B11">
            <p><a name="B11"></a> de Bruyn M,  Hoelzel AR,  Carvalho GR,  Hofreiter M: <strong> Faunal histories from Holocene ancient DNA. </strong></p><p><em>Trends Ecol Evol</em> 2011,  <strong>26</strong><strong>:</strong>405-413.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/21529992">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21529992">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B11" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B11','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B12">
            <p><a name="B12"></a> Miller CR,  Waits LP,  Joyce P: <strong> Phylogeography and mitochondrial diversity of extirpated brown bear (Ursus arctos)
                  populations in the contiguous United States and Mexico. </strong></p><p><em>Mol Ecol</em> 2006,  <strong>15</strong><strong>:</strong>4477-4485.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/17107477">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17107477">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B12" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B12','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B13">
            <p><a name="B13"></a> Calvignac S,  Hughes S,  Tougard C,  Michaux JR,  Thevenot M,  Philippe M,  Hamdine W,  Hanni C: <strong> Ancient DNA evidence for the loss of a highly divergent brown bear clade during historical
                  times. </strong></p><p><em>Mol Ecol</em> 2008,  <strong>17</strong><strong>:</strong>1962-1970.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/18363668">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18363668">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B13" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B13','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B14">
            <p><a name="B14"></a> Yamaguchi N,  Cooper A,  Werdelin L,  MacDonald DW: <strong> Evolution of the mane and group-living in the lion (<em>Panthera leo</em>): a review. </strong></p><p><em>J Zool</em> 2004,  <strong>263</strong><strong>:</strong>329-342.   <a target="_blank" href="http://dx.doi.org/10.1017/S0952836904005242">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B14" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B14','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B15">
            <p><a name="B15"></a> Turner A,  Antón M: <em>The Big Cats and Their Fossil Relatives</em>.  New York: Columbia University Press;  1997. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B15" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B15','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B16">
            <p><a name="B16"></a> Bauer H,  Van Der Merwe S: <strong> Inventory of free-ranging lions <em>Panthera leo</em> in Africa. </strong></p><p><em>Oryx</em> 2004,  <strong>38</strong>(1)<strong>:</strong>26-31.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B16" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B16','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B17">
            <p><a name="B17"></a> Schnitzler AE: <strong> Past and present distribution of the North African-Asian lion subgroup: a review. </strong></p><p><em>Mammal Rev</em> 2011,  <strong>41</strong>(3)<strong>:</strong>220-243.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1365-2907.2010.00181.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B17" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B17','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B18">
            <p><a name="B18"></a><strong> IUCN Red List of Threatened Species </strong>[<a href="http://www.iucnredlist.org/">http://www.iucnredlist.org</a> <a href="http://www.webcitation.org/query.php?url=http://www.iucnredlist.org&amp;refdoi=10.1186/1471-2148-14-70" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>]
            </p>
         </li>
         <li id="B19">
            <p><a name="B19"></a> Barnett R,  Yamaguchi N,  Barnes I,  Cooper A: <strong> Lost populations and preserving genetic diversity in the lion <em>Panthera leo</em>: Implications for its <em>ex situ</em> conservation. </strong></p><p><em>Conserv Genet</em> 2006,  <strong>7</strong>(4)<strong>:</strong>507-514.   <a target="_blank" href="http://dx.doi.org/10.1007/s10592-005-9062-0">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B19" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B19','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B20">
            <p><a name="B20"></a> Bertola LD,  van Hooft WF,  Vrieling 
K,  Uit de Weerd DR,  York DS,  Bauer H,  Prins HHT,  Funston PJ,  de 
Haes HA U,  Leirs H,  van Haeringen WA,  Sogbohossou Tumenta EPN,  de 
Iongh HH: <strong> Genetic diversity, evolutionary history and implications for conservation of the lion
                  (Panthera leo) in West and Central Africa. </strong></p><p><em>J Biogeography</em> 2011,  <strong>38</strong>(7)<strong>:</strong>1356-1367.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1365-2699.2011.02500.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B20" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B20','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B21">
            <p><a name="B21"></a> Henschel P,  Azani D,  Burton C,  Malanda G,  Saidu Y,  Sam M,  Hunter L: <strong> Lion status updates from five range countries in West and Central Africa. </strong></p><p><em>Cat News</em> 2010,  <strong>52</strong>(Spring)<strong>:</strong>34-39.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B21" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B21','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B22">
            <p><a name="B22"></a> Mace GM: <strong> The role of taxonomy in species conservation. </strong></p><p><em>Trans R Soc B</em> 2004,  <strong>359</strong>(1444)<strong>:</strong>711-719.   <a target="_blank" href="http://dx.doi.org/10.1098/rstb.2003.1454">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B22" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B22','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B23">
            <p><a name="B23"></a> Barnett R,  Yamaguchi N,  Barnes I,  Cooper A: <strong> The origin, current diversity, and future conservation of the modern lion (<em>Panthera leo</em>). </strong></p><p><em>Proc R Soc Lond B</em> 2006,  <strong>273</strong>(1598)<strong>:</strong>2159-2168.   <a target="_blank" href="http://dx.doi.org/10.1098/rspb.2006.3578">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B23" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B23','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B24">
            <p><a name="B24"></a> Barnett R,  Shapiro B,  Barnes I,  Ho 
SYW,  Burger J,  Yamaguchi N,  Higham TFG,  Wheeler HT,  Rosendahl W,  
Sher A,  Sotnikova M,  Kuznetsova T,  Baryshnikov G,  Martin LD,  
Harington CR,  Burns JA,  Cooper A: <strong> Phylogeography of lions (Panthera leo ssp.) reveals three distinct taxa and late Pleistocene
                  reduction in genetic diversity. </strong></p><p><em>Mol Ecol</em> 2009,  <strong>18</strong>(8)<strong>:</strong>1668-1677.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/19302360">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19302360">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B24" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B24','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B25">
            <p><a name="B25"></a> Yamaguchi N,  Kitchener AC,  Driscoll CA,  Macdonald DW: <strong> Divided infraorbital foramen in the lion (<em>Panthera leo</em>): its implications for colonisation history, population bottlenecks, and conservation
                  of the Asian lion (<em>P.l.persica</em>). </strong></p><p><em>Contrib Zoology</em> 2009,  <strong>78</strong>(2)<strong>:</strong>77-83.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B25" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B25','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B26">
            <p><a name="B26"></a> Mazak JH: <strong> Geographical variation and phylogenetics of modern lions based on craniometric data. </strong></p><p><em>J Zoology</em> 2010,  <strong>281</strong>(3)<strong>:</strong>194-209.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B26" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B26','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B27">
            <p><a name="B27"></a> Dubach J,  Patterson BD,  Briggs MB,  Venzke K,  Flamand J,  Stander P,  Scheepers L,  Kays RW: <strong> Molecular genetic variation across the southern and eastern ranges of the African
                  lion, <em>Panthera leo</em>. </strong></p><p><em>Conserv Genet</em> 2005,  <strong>6</strong><strong>:</strong>15-24.   <a target="_blank" href="http://dx.doi.org/10.1007/s10592-004-7729-6">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B27" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B27','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B28">
            <p><a name="B28"></a> Antunes A,  Troyer JL,  Roelke ME,  
Pecon-Slattery J,  Packer C,  Winterbach C,  Winterbach H,  Hemson G,  
Frank L,  Stander P,  Siefert L,  Driciru M,  Funston PJ,  Alexander KA,
  Prager KC,  Mills G,  Wildt D,  Bush M,  O'Brien SJ,  Johnson WE: <strong> The evolutionary dynamics of the lion <em>Panthera leo</em> revealed by host and viral population genomics. </strong></p><p><em>PLoS Genet</em> 2008,  <strong>4</strong>(11)<strong>:</strong>e1000251.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/18989457">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18989457">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18989457">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B28" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B28','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B29">
            <p><a name="B29"></a> Dubach JM,  Briggs MB,  White PA,  Ament BA,  Patterson BD: <strong> Genetic perspectives on “Lion Conservation Units” in Eastern and Southern Africa. </strong></p><p><em>Conserv Genet</em> 2013,  <strong>14</strong>(4)<strong>:</strong>741-755.   <a target="_blank" href="http://dx.doi.org/10.1007/s10592-013-0453-3">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B29" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B29','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B30">
            <p><a name="B30"></a> Hemmer H: <strong> Untersuchungen zur Stammesgeschichte der Pantherkatzen (Pantherinae), Teil III: Zur
                  Artgeschichte des Löwen <em>Panthera (Panthera) leo</em> (Linnaeus 1758). </strong></p><p><em>Veroffentlichungen der Zoologischen Staatssammlung Munchen</em> 1974,  <strong>17</strong><strong>:</strong>167-280.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B30" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B30','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B31">
            <p><a name="B31"></a> Black S,  Yamaguchi N,  Harland A,  Groombridge J: <strong> Maintaining the genetic health of putative Barbary lions in captivity: an analysis
                  of Moroccan Royal Lions. </strong></p><p><em>Eur J Wildl Res</em> 2009,  <strong>56</strong><strong>:</strong>21-31.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B31" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B31','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B32">
            <p><a name="B32"></a> Edwards CJ,  Suchard MA,  Lemey P,  
Welch JJ,  Barnes I,  Fulton TL,  Barnett R,  O’Connell TC,  Coxon P,  
Monaghan N,  Valdiosera CE,  Lorenzen ED,  Willerslev E,  Baryshnikov 
GF,  Rambaut A,  Thomas MG,  Bradley DG,  Shapiro B: <strong> Multiple hybridizations between ancient brown and polar bears and an Irish origin
                  for the modern polar bear matriline. </strong></p><p><em>Curr Biol</em> 2011,  <strong>21</strong><strong>:</strong>1251-1258.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/21737280">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21737280">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B32" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B32','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B33">
            <p><a name="B33"></a> Barnett R,  Yamaguchi N,  Shapiro B,  Sabin R: <strong> Ancient DNA analysis indicates the first English lions originated from North Africa. </strong></p><p><em>Contrib Zoology</em> 2008,  <strong>77</strong>(1)<strong>:</strong>7-16.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B33" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B33','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B34">
            <p><a name="B34"></a> Rohland N,  Hofreiter M: <strong> Ancient DNA extraction from bones and teeth. </strong></p><p><em>Nat Protoc</em> 2007,  <strong>2</strong>(7)<strong>:</strong>1756-1762.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/17641642">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17641642">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B34" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B34','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B35">
            <p><a name="B35"></a> Burger J,  Rosendahl W,  Loreille O,  Hemmer H,  Eriksson T,  Götherstrom A,  Hiller J,  Collins MJ,  Wess T,  Alt KW: <strong> Molecular phylogeny of the extinct cave lion <em>Panthera leo spelaea</em>. </strong></p><p><em>Mol Phylogenet Evol</em> 2004,  <strong>30</strong>(3)<strong>:</strong>841-849.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/15012963">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15012963">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B35" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B35','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B36">
            <p><a name="B36"></a> Davis BW,  Li G,  Murphy WJ: <strong> Supermatrix and species tree methods resolve phylogenetic relationships amoung the
                  big cats, <em>Panthera</em> (Carnivora: Felidae). </strong></p><p><em>Mol Phylogenet Evol</em> 2010,  <strong>56</strong>(1)<strong>:</strong>64-76.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/20138224">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20138224">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B36" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B36','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B37">
            <p><a name="B37"></a> Kim J-H,  Antunes A,  Luo S-J,  Menninger J,  Nash WG,  O’Brien SJ,  Johnson WE: <strong> Evolutionary analysis of a large mtDNA translocation (numt) into the nuclear genome
                  of the <em>Panthera</em> genus species. </strong></p><p><em>Gene</em> 2006,  <strong>366</strong>(2)<strong>:</strong>292-302.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/16380222">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16380222">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16380222">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B37" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B37','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B38">
            <p><a name="B38"></a> Rambaut A: <em>Se-Al: Sequence Alignment Editor v2.0a11 Carbon</em>.  Oxford: University of Oxford;  1996. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B38" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B38','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B39">
            <p><a name="B39"></a> Bandelt HJ,  Forster P,  Rohl A: <strong> Median-joining networks for inferring intraspecific phylogenies. </strong></p><p><em>Mol Biol Evol</em> 1999,  <strong>16</strong><strong>:</strong>37-48.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/10331250">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10331250">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B39" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B39','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B40">
            <p><a name="B40"></a> Drummond AJ,  Suchard MA,  Xie D,  Rambaut A: <strong> Bayesian phylogenetics with BEAUti and the BEAST 1.7. </strong></p><p><em>Mol Biol Evol</em> 2012,  <strong>29</strong>(8)<strong>:</strong>1969-1973.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/22367748">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22367748">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=22367748">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B40" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B40','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B41">
            <p><a name="B41"></a> Luo A,  Qiao H,  Zhang Y,  Shi W,  Ho SYW,  Xu W,  Zhang A,  Zhu C: <strong> Performance criteria for selecting evolutionary models in phylogenetics: a comprehensive
                  study based on simulated datasets. </strong></p><p><em>BMC Evol Biol</em> 2010,  <strong>10</strong><strong>:</strong>242.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/20696057">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2148-10-242">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20696057">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B41" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B41','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B42">
            <p><a name="B42"></a> Sheng G-L,  Soubrier J,  Liu J,  Werdelin L,  Llamas B,  Thomson VA,  Tuke J,  Wu L-J,  Hou X-D,  Chen Q-J,  Lai X,  Cooper A: <strong> Pleistocene Chinese cave hyenas and the recent Eurasian history of the spotted hyena,
                  Crocuta Crocuta. </strong></p><p><em>Mol Ecol</em> 2013,  <strong>23</strong>(3)<strong>:</strong>522-533.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/24320717">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=24320717">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B42" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B42','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B43">
            <p><a name="B43"></a> Shapiro B,  Ho SYW: <strong> Ancient hyenas highlight the old problem of estimating evolutionary rates. </strong></p><p><em>Mol Eco</em> 2014,  <strong>23</strong>(3)<strong>:</strong>499-501.   <a target="_blank" href="http://dx.doi.org/10.1111/mec.12621">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B43" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B43','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B44">
            <p><a name="B44"></a> Ho SYW,  Lanfear R,  Bromham L,  Phillips MJ,  Soubrier J,  Rodrigo AG,  Cooper A: <strong> Time-dependent rates of molecular evolution. </strong></p><p><em>Mol Ecol</em> 2011,  <strong>20</strong>(15)<strong>:</strong>3087-3101.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/21740474">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21740474">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B44" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B44','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B45">
            <p><a name="B45"></a> Ho SYW,  Lanfear R,  Phillips MJ,  Barnes I,  Thomas JA,  Kolokotronis S,  Shapiro B: <strong> Bayesian estimation of substitution rates from ancient DNA sequences with low information
                  content. </strong></p><p><em>Syst Biol</em> 2011,  <strong>60</strong>(3)<strong>:</strong>366-375.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/21296909">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21296909">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B45" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B45','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B46">
            <p><a name="B46"></a> García García N: <strong> Los Carnivoros de los Yacimientos Pleistocenos de la Sierra de Atapuerca. </strong>In <em>Tésis Doctoral Madrid</em>.  Spain: Universidad Complutense de Madrid;  2001. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B46" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B46','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B47">
            <p><a name="B47"></a> Suchard MA,  Weiss RE,  Sinsheimer JS: <strong> Bayesian selection of continuous-time Markov chain evolutionary models. </strong></p><p><em>Mol Biol Evol</em> 2001,  <strong>18</strong><strong>:</strong>1001-1013.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11371589">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11371589">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B47" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B47','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B48">
            <p><a name="B48"></a> Rambaut A,  Drummond AJ: <em>Tracer v1.5</em>.  1.5 edn;  2009.
               </p><p><a href="http://tree.bio.ed.ac.uk/software/tracer/">http://tree.bio.ed.ac.uk/software/tracer/</a> <a href="http://www.webcitation.org/query.php?url=http://tree.bio.ed.ac.uk/software/tracer/&amp;refdoi=10.1186/1471-2148-14-70" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B48" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B48','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a><p></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B49">
            <p><a name="B49"></a> Minin VN,  Bloomquist EW,  Suchard MA: <strong> Smooth skyride through a rough skyline: Bayesian coalescent-based inference of population
                  dynamics. </strong></p><p><em>Mol Biol Evol</em> 2008,  <strong>25</strong><strong>:</strong>1459-1471.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/18408232">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18408232">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18408232">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B49" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B49','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B50">
            <p><a name="B50"></a> Gilbert MTP,  Bandelt H-J,  Hofreiter M,  Barnes I: <strong> Assessing ancient DNA studies. </strong></p><p><em>Trends Ecol Evol</em> 2005,  <strong>20</strong>(10)<strong>:</strong>541-544.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/16701432">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16701432">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B50" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B50','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B51">
            <p><a name="B51"></a> Dupont L: <strong> Orbital scale vegetation change in Africa. </strong></p><p><em>Quaternary Sci Rev</em> 2011,  <strong>30</strong>(25–26)<strong>:</strong>3589-3602.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B51" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B51','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B52">
            <p><a name="B52"></a> De Vivo M,  Carmignotto AP: <strong> Holocene vegetation change and the mammal faunas of South America and Africa. </strong></p><p><em>J Biogeography</em> 2004,  <strong>31</strong><strong>:</strong>943-957.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1365-2699.2004.01068.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B52" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B52','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B53">
            <p><a name="B53"></a> Meadows ME,  Chase BM: <strong> Pollen Records, Late Pleistocene/Africa. </strong>In <em>Encyclopaedia of Quaternary Science</em>.  Edited by Elias SA.  Elsevier Science Ltd. North Holland;  2007:2606-2613.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B53" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B53','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B54">
            <p><a name="B54"></a> Cohen AS,  Stone JR,  Beuning KRM,  
Park LE,  Reinthal PN,  Dettmar D,  Scholz CA,  Johnsor TC,  King JW,  
Talbot MR,  Brown ET,  Ivory SJ: <strong> Ecological consequences of early Late Pleistocene megadroughts in tropical Africa. </strong></p><p><em>Proc Natl Acad Sci U S A</em> 2007,  <strong>104</strong>(42)<strong>:</strong>16422-16427.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/17925446">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17925446">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17925446">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B54" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B54','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B55">
            <p><a name="B55"></a> Castaneda IS,  Mulitza S,  Schefuss E,  dos Santos RAL,  Damste JSS,  Schouten S: <strong> Wet phases in the Sahara/Sahel region and human migration patterns in North Africa. </strong></p><p><em>Proc Natl Acad Sci U S A</em> 2009,  <strong>106</strong>(48)<strong>:</strong>20159-20163.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/19910531">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19910531">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19910531">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B55" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B55','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B56">
            <p><a name="B56"></a> Blome MW,  Cohen AS,  Tryon CA,  Brooks AS,  Russell J: <strong> The environmental context for the origins of modern human diversity: A synthesis of
                  regional variability in African climate 150,000-30,000&nbsp;years ago. </strong></p><p><em>J Hum Evol</em> 2012,  <strong>62</strong>(5)<strong>:</strong>563-592.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/22513381">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22513381">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B56" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B56','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B57">
            <p><a name="B57"></a> Osborne AH,  Vance D,  Rohling EJ,  Barton N,  Rogerson M,  Fello N: <strong> A humid corridor across the Sahara for the migration of early modern humans out of
                  Africa 120,000&nbsp;years ago. </strong></p><p><em>Proc Natl Acad Sci U S A</em> 2008,  <strong>105</strong>(43)<strong>:</strong>16444-16447.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/18936490">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18936490">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18936490">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B57" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B57','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B58">
            <p><a name="B58"></a> Mashkour M,  Monchot H,  Trinkaus E,  Reyss J,  Biglari F,  Bailon S,  Heydari S,  Abdi K: <strong> Carnivores and their prey in the Wezmeh cave (Kermanshah, Iran): a late Pleistocene
                  refuge in the Zagros. </strong></p><p><em>Int J Osteoarchaeology</em> 2008,  <strong>19</strong>(6)<strong>:</strong>678-694.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B58" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B58','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B59">
            <p><a name="B59"></a> Tchernov E,  Tsoukala E: <strong> Middle Pleistocene (Early Toringian) Carnivore Remains from Northern Israel. </strong></p><p><em>Quaternary Res</em> 1997,  <strong>48</strong><strong>:</strong>122-136.   <a target="_blank" href="http://dx.doi.org/10.1006/qres.1997.1901">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B59" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B59','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B60">
            <p><a name="B60"></a> Eriksson J,  Hohmann G,  Boesch C,  Vigilant L: <strong> Rivers influence the population genetic structure of bonobos (<em>Pan paniscus</em>). </strong></p><p><em>Mol Ecol</em> 2004,  <strong>13</strong>(11)<strong>:</strong>3425-3435.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/15488001">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15488001">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B60" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B60','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B61">
            <p><a name="B61"></a> Eizirik E,  Kim JH,  Menotti-Raymond M,  Crawshaw PG,  O’Brien SJ,  Johnson WE: <strong> Phylogeography, population history and conservation genetics of jaguars (<em>Panthera onca</em>, Mammalia, Felidae). </strong></p><p><em>Mol Ecol</em> 2001,  <strong>10</strong>(1)<strong>:</strong>65-79.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/11251788">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11251788">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B61" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B61','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B62">
            <p><a name="B62"></a> Moritz C: <strong> Applications of mitochondrial DNA analysis in conservation- a critical review. </strong></p><p><em>Mol Ecol</em> 1994,  <strong>3</strong><strong>:</strong>401-411.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1365-294X.1994.tb00080.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B62" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B62','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B63">
            <p><a name="B63"></a> O’Brien SJ,  Martenson JS,  Packer C,  Herbst L,  Devos V,  Joslin P,  Ottjoslin J,  Wildt DE,  Bush M: <strong> Biochemical Genetic-Variation in Geographic Isolates of African and Asiatic Lions. </strong></p><p><em>Natl Geographic Res</em> 1987,  <strong>3</strong>(1)<strong>:</strong>114-124.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B63" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B63','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B64">
            <p><a name="B64"></a> O’Brien SJ,  Joslin P,  Smith GL III, 
 Wolfe R,  Schaffer N,  Heath E,  Ott-Joslin J,  Rawal PP,  
Bhattacharjee KK,  Martenson JS: <strong> Evidence for African origins of founders if the asiatic lion species survival plan. </strong></p><p><em>Zoo Biol</em> 1987,  <strong>6</strong><strong>:</strong>99-116.   <a target="_blank" href="http://dx.doi.org/10.1002/zoo.1430060202">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B64" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B64','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B65">
            <p><a name="B65"></a> Bauer H,  de Iongh HH,  Princee FPG,  Ngantou D: <strong> Research needs for lion conservation in West and Central Africa. </strong></p><p><em>C R Biologies</em> 2003,  <strong>326</strong><strong>:</strong>S112-S118.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/14558459">PubMed&nbsp;Abstract</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B65" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B65','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B66">
            <p><a name="B66"></a> Bauer H,  Nowell K,  Breitenmoser C,  Breitenmoser U,  Jackson P: <strong> Endangered classification for West African lions. </strong></p><p><em>Cat News</em> 2004,  <strong>41</strong><strong>:</strong>35-36.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B66" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B66','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B67">
            <p><a name="B67"></a> Yamaguchi N,  Haddane B: <strong> The North African Barbary lion and the Atlas Lion Project. </strong></p><p><em>Int Zoo News</em> 2002,  <strong>49</strong><strong>:</strong>465-481.  <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B67" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B67','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B68">
            <p><a name="B68"></a> Burger J,  Hemmer H: <strong> Urgent call for further breeding of the relic zoo population of the critically endangered
                  Barbary lion (<em>Panthera leo leo</em> Linnaeus 1758). </strong></p><p><em>Eur J Wildl Res</em> 2006,  <strong>52</strong><strong>:</strong>54-58.   <a target="_blank" href="http://dx.doi.org/10.1007/s10344-005-0009-z">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B68" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B68','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B69">
            <p><a name="B69"></a> Driscoll CA,  Yamaguchi N,  Bar-Gal GK,  Roca AL,  Luo S,  Macdonald DW,  O’Brien SJ: <strong> Mitochondrial phylogeography illuminates the origin of the extinct Caspian tiger and
                  its relationship to the Amur tiger. </strong></p><p><em>PLoS One</em> 2009,  <strong>4</strong>(1)<strong>:</strong>e4125.  <a target="_blank" href="http://www.biomedcentral.com/pubmed/19142238">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19142238">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19142238">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B69" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B69','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B70">
            <p><a name="B70"></a> Jungius H: <em>Feasibility Study on the Possible Restoration of the Caspian Tiger in Central Asia</em>.  WWF;  2010.
               </p><p><a href="http://www.wwf.ru/resources/publ/book/eng/460">http://www.wwf.ru/resources/publ/book/eng/460</a> <a href="http://www.webcitation.org/query.php?url=http://www.wwf.ru/resources/publ/book/eng/460&amp;refdoi=10.1186/1471-2148-14-70" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B70" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B70','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a><p></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B71">
            <p><a name="B71"></a><strong> Tigers could reappear in Kazakhstan under new plan </strong><a href="http://wwf.panda.org/?uNewsID=200017">http://wwf.panda.org/?uNewsID=200017</a> <a href="http://www.webcitation.org/query.php?url=http://wwf.panda.org/?uNewsID=200017&amp;refdoi=10.1186/1471-2148-14-70" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p>
         </li>
         <li id="B72">
            <p><a name="B72"></a> Swofford DL: <em>PAUP*. Phylogenetic Analysis Using Parsimony (*and Other Methods) v4.0b10</em>.  Sunderland, Massachusetts: Sinauer Associates;  2002. <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B72" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B72','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B73">
            <p><a name="B73"></a> Cracraft J,  Feinstein J,  Vaughn J,  Helm-Bychowski K: <strong> Sorting out tiger (Panthera tigris): mitochondrial sequences, nuclear inserts, systematics,
                  and conservation genetics. </strong></p><p><em>Anim Conserv</em> 1998,  <strong>1</strong><strong>:</strong>139-150.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1469-1795.1998.tb00021.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1471-2148-14-70&amp;bibl=B73" onclick="popup('/sfx_links?ui=1471-2148-14-70&amp;bibl=B73','SFXMenu','460','420'); return false;"><img src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/sfx_links.gif" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
      </ol>
   </div>
</section>
    <br class="clearall">
<div class="article-alert-signup-div">

<a class="close png_bg" id="closeHelpBox">close</a>	
<div class="article-alert-signup rounded"> 
<span class="blurb left">Sign up to receive new article alerts from <em>BMC Evolutionary Biology</em></span>

	
				<button name="articleAlertPreference.status" type="button" class="w74 right" onclick="window.location='/bmcevolbiol/my/preferences'">Sign up</button>
	
</div>
</div></div>
    </div></div>
</div>


		</div>
</div>

    <hr class="hide">
	
    

						
   
  
      
  
                  
  
                  
  
            
  
            
            


<div id="footer">

						
			<span class="views"><a id="mobile-view" href="http://www.biomedcentral.com/1471-2148/14/70?fmt_view=mobile">Mobile view</a> | <strong>Desktop view</strong></span>
			
	<div class="content">
    <div class="rounded ">
<div class="wrap-inner ">
        	
    <ul class="desktop">
                                                    
				        <li class="noborder"><a href="http://www.biomedcentral.com/about/tandc">Terms and Conditions</a></li>
<li><a href="http://www.biomedcentral.com/about/privacy">Privacy statement</a></li>
<li><a href="http://www.biomedcentral.com/presscenter">Press</a></li>
	<li><a href="http://www.biomedcentral.com/advertisers">Information for advertisers</a></li>
<li><a href="http://www.biomedcentral.com/about/bmcjobs">Jobs at BMC</a></li>
<li><a href="http://www.biomedcentral.com/support">Support</a></li>
<li><a href="http://www.biomedcentral.com/about/contact">Contact us</a></li>


    
    	    </ul>
    <p id="copyright">© 2015 
	BioMed Central Ltd unless otherwise stated. Part of Springer Science+Business&nbsp;Media.</p>
        </div>
</div>

	</div>

</div>
<div class="springer">
    <a href="http://www.springer.com/" target="blank"><img style="float:right; margin:0;" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/Springer_Branding_footer_logo.png"></a>
</div>


<script>window.bmcIsMobile = "classic";</script>					    

<script>
// getElementsByClassName Polyfill
var getElementsByClassName=function(e,t,n){if(document.getElementsByClassName){getElementsByClassName=function(e,t,n){n=n||document;var r=n.getElementsByClassName(e),i=t?new RegExp("\\b"+t+"\\b","i"):null,s=[],o;for(var u=0,a=r.length;u<a;u+=1){o=r[u];if(!i||i.test(o.nodeName)){s.push(o)}}return s}}else if(document.evaluate){getElementsByClassName=function(e,t,n){t=t||"*";n=n||document;var r=e.split(" "),i="",s="http://www.w3.org/1999/xhtml",o=document.documentElement.namespaceURI===s?s:null,u=[],a,f;for(var l=0,c=r.length;l<c;l+=1){i+="[contains(concat(' ', @class, ' '), ' "+r[l]+" ')]"}try{a=document.evaluate(".//"+t+i,n,o,0,null)}catch(h){a=document.evaluate(".//"+t+i,n,null,0,null)}while(f=a.iterateNext()){u.push(f)}return u}}else{getElementsByClassName=function(e,t,n){t=t||"*";n=n||document;var r=e.split(" "),i=[],s=t==="*"&&n.all?n.all:n.getElementsByTagName(t),o,u=[],a;for(var f=0,l=r.length;f<l;f+=1){i.push(new RegExp("(^|\\s)"+r[f]+"(\\s|$)"))}for(var c=0,h=s.length;c<h;c+=1){o=s[c];a=false;for(var p=0,d=i.length;p<d;p+=1){a=i[p].test(o.className);if(!a){break}}if(a){u.push(o)}}return u}}return getElementsByClassName(e,t,n)}
</script>

						



<script type="text/javascript">
if (typeof jQuery == 'undefined') { document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.min.js' type='text/javascript'%3E%3C/script%3E")); }


document.documentElement.id = "js";

var site = {
			abbreviation: "/bmcevolbiol",
	        type: "BMC_CORE",
        portalId: "9001",
                section: "browse",
                page: "fulltext",
    		
									homePortal: "biomedcentral.com",
		
			siteShowAds: "true",
				showAds: "true",
				url: "biomedcentral.com/bmcevolbiol",
			portal: false,
	        adpage_url: "biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/14/70",
        id: "1006",
	name: "BMC Evolutionary Biology",
	plugins: {
		scrollable: (getElementsByClassName("scrollable").length > 0),
				flowplayer: ((document.getElementById("flowplayer") != null) || (getElementsByClassName("myPlayer").length > 0) || (document.getElementById("videoarticle") != null) || (getElementsByClassName("flowplayer").length > 0) || (document.getElementById("audio") != null) || (document.getElementById("elifeplayer") != null)),
		lightbox: ((document.getElementById("image-highlight") != null) || (getElementsByClassName("lightbox").length > 0)),
		thickbox: (getElementsByClassName("thickbox").length > 0),
		altmetric: ((document.getElementById("altmetric") != null) || (getElementsByClassName("altmetric").length > 0))
	}
	
};



if(typeof quoteSearchUri == "undefined") { quoteSearchUri=null; }
</script>



        <script type="text/javascript" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/behaviours-0.js"></script>






		<script>
	if (site.plugins.scrollable == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.scrollable-1.0.1.min.js' %3E%3C/script%3E")); 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.tools.min.js'%3E%3C/script%3E"));
	}

	if (site.plugins.flowplayer == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/flowplayer-3.2.6.min.js'%3E%3C/script%3E"));
	}

	if (site.plugins.lightbox == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/browse_imagehighlight.js'%3E%3C/script%3E"));
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.colorbox.js' %3E%3C/script%3E"));
	}

	if (site.plugins.thickbox == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/thickbox-compressed.js' %3E%3C/script%3E"));
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/thickbox-flashhidden-patch.js' %3E%3C/script%3E"));
	}
	</script>





            <script type="text/javascript" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/jquery_004.js"></script>
            <script type="text/javascript" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/jquery_002.js"></script>
                      <script type="text/javascript" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/email_preferences-0.js"></script>
                <script type="text/javascript" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/articles-0.js"></script>
		<div class=" fb_reset" id="fb-root"><div style="position: absolute; top: -10000px; height: 0px; width: 0px;"><div><iframe src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/DU1Ia251o0y.html" style="border: medium none;" tabindex="-1" title="Facebook Cross Domain Communication Frame" aria-hidden="true" id="fb_xdm_frame_http" scrolling="no" allowtransparency="true" name="fb_xdm_frame_http" frameborder="0"></iframe><iframe src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/DU1Ia251o0y_002.html" style="border: medium none;" tabindex="-1" title="Facebook Cross Domain Communication Frame" aria-hidden="true" id="fb_xdm_frame_https" scrolling="no" allowtransparency="true" name="fb_xdm_frame_https" frameborder="0"></iframe></div></div><div style="position: absolute; top: -10000px; height: 0px; width: 0px;"><div></div></div></div>











        <script type="text/javascript" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/jquery_003.js"></script>




<!--[if lte IE 6]>
            <script type="text/javascript" src="/javascript/plugins/DD_belatedPNG_0.0.8a-0.js"></script>
    <script type="text/javascript">
        DD_belatedPNG.fix('.png_bg, ul.primary-nav li a, ul.primary-nav li a span, ul.secondary-nav li a, ul.secondary-nav li a span, ul li i, .plus-button, .facebook, .twitter, .fancy-bg, #fancybox-close');
    </script>
	                <link rel="stylesheet" type="text/css" href="http://www.bmcimg.com/css/hacks/ie6-0.css"/>
    <![endif]-->

<!-- Claudia Whitcombe wanted this comment here -->

       <div class="hide">
      <dl class="google-ad" id="x96banner">
        <dt style="display: block;" class="hide">
          <a class="skyscraper-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</a>
       </dt>
        <dd>
          <!-- OAS AD 'x96' begin -->
          <script type="text/javascript">
            OAS_AD('x96');
          </script>
        </dd>
      </dl>
    </div>  
    



<div id="fancybox-tmp"></div><div id="fancybox-loading"><div></div></div><div id="fancybox-overlay"></div><div id="fancybox-wrap"><div id="fancybox-outer"><div class="fancy-bg" id="fancy-bg-n"></div><div class="fancy-bg" id="fancy-bg-ne"></div><div class="fancy-bg" id="fancy-bg-e"></div><div class="fancy-bg" id="fancy-bg-se"></div><div class="fancy-bg" id="fancy-bg-s"></div><div class="fancy-bg" id="fancy-bg-sw"></div><div class="fancy-bg" id="fancy-bg-w"></div><div class="fancy-bg" id="fancy-bg-nw"></div><div id="fancybox-inner"></div><a id="fancybox-close"></a><a href="javascript:;" id="fancybox-left"><span class="fancy-ico" id="fancybox-left-ico"></span></a><a href="javascript:;" id="fancybox-right"><span class="fancy-ico" id="fancybox-right-ico"></span></a></div></div><iframe tabindex="-1" style="width: 1px; height: 1px; position: absolute; top: -100px;" src="BMC%20Evolutionary%20Biology%20%7C%20Full%20text%20%7C%20Revealing%20the%20maternal%20demographic%20history%20of%20Panthera%20leo%20using%20ancient%20DNA%20and%20a%20spatially%20explicit%20genealogical%20analysis_files/postmessageRelay.html" id="oauth2relay350192015" name="oauth2relay350192015"></iframe></body></html>