<html xmlns="http://www.w3.org/1999/xhtml" lang="en" dir="ltr" version="HTML+RDFa+MathML 1.1"><head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /> <meta charset="UTF-8" /> <style>
         	    div {
         	        border: 1px solid red;
         	        margin: 2 2 2 2;
         	        }
         	    ul, ol {
         	        border: 1px solid green;
         	        margin: 2 2 2 2;
         	        }
         	    a {
         	        border: 1px solid purple;
         	        margin: 2 2 2 2;
         	        }
         	    p {
         	        background: #ffffdd;
         	        margin: 2 2 2 2;
         	        }
         	    span {
         	        background: #ddffff;
         	        margin: 2 2 2 2;
         	        }
         	    table {"target/pubstyle/rs/clean/277_1686_1309/fulltext.xhtml"
         	        border: 1px solid blue;
         	        background: #ddffdd;
         	        margin: 2 2 2 2;
         	        }
         	    tr {
         	        border: 2px solid yellow;
         	        margin: 2 2 2 2;
         	        }
         	    td {
         	        border: 2px solid cyan;
         	        margin: 2 2 2 2;
         	        }
         	    
      </style> <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no" /> <meta name="Generator" content="Drupal 7 (http://drupal.org)" /> <meta http-equiv="x-dns-prefetch-control" content="on" /> <meta name="citation_funding_source" content="citation_funder=the National Natural Science Foundation of China;citation_grant_number=31572647;" /> <meta name="issue_cover_image" content="http://d3hu9binmobce5.cloudfront.net/sites/default/files/highwire/royopensci/3/6.cover-source.jpg" /> <meta name="type" content="article" /> <meta name="category" content="research-article" /> <meta name="HW.identifier" content="/royopensci/3/6/160123.atom" /> <meta name="HW.pisa" content="royopensci;3/6/160123" /> <meta name="DC.Format" content="text/html" /> <meta name="DC.Language" content="en" /> <meta name="DC.Title" content="Enhanced antimicrobial peptide-induced activity in the mollusc Toll-2 family through evolution via tandem Toll/interleukin-1 receptor" /> <meta name="DC.Identifier" content="10.1098/rsos.160123" /> <meta name="DC.Date" content="2016-06-01" /> <meta name="DC.Publisher" content="The Royal Society" /> <meta name="DC.Rights" content=". © 2016 The Authors. Published by the Royal Society under the terms of the Creative Commons Attribution License http://creativecommons.org/licenses/by/4.0/, which permits unrestricted use, provided the original author and source are credited." /> <meta name="DC.AccessRights" content="open-access" /> <meta name="DC.Description" content="Toll receptors play an important role in the innate immunity of invertebrates. All reported Tolls have only one Toll/interleukin-1 receptor (TIR) domain at the C-terminal. In this study, numerous Tolls with tandem TIRs at the C-terminal were found in molluscs. Such Tolls presented an extra TIR (TIR-1) compared with Toll-I. Thus, Toll-I might be the ancestor of tandem TIRs containing Toll. To test this hypothesis, 83 Toll-I and Toll-2 (most have two TIRs, but others seem to be the evolutionary intermediates) genes from 29 shellfish species were identified. These Tolls were divided into nine groups based on phylogenetic analyses. A strong correlation between phylogeny and motif composition was found. All Toll proteins contained the TIR-2 domain, whereas the TIR-1 domain only existed in some Toll-2 protein, suggesting that TIR-1 domain insertion may play an important role in Toll protein evolution. Further analyses of functional divergence and adaptive evolution showed that some of the critical sites responsible for functional divergence may have been under positive selection. An additional intragenic recombination played an important role in the evolution of the Toll-I and Toll-2 genes. To investigate the functional difference of Toll-I and Toll-2, over expression of Hcu\_Toll-I or Hcu\_Toll-2-2 in Drosophila S2 cells was performed. Results showed that Hcu\_Toll-2-2 had stronger antimicrobial peptide (AMP) activity than Hcu\_Toll-I. Therefore, enhanced AMP-induced activity resulted from tandem TIRs in Toll-2s of molluscs during evolution history." /> <meta name="DC.Contributor" content="Jun Cao" /> <meta name="DC.Contributor" content="Yihong Chen" /> <meta name="DC.Contributor" content="Min Jin" /> <meta name="DC.Contributor" content="Qian Ren" /> <meta name="og:title" content="Enhanced antimicrobial peptide-induced activity in the mollusc Toll-2 family through evolution via tandem Toll/interleukin-1 receptor" /> <meta name="og:url" content="http://rsos.royalsocietypublishing.org/content/3/6/160123" /> <meta name="og:site_name" content="Open Science" /> <meta name="og:image" content="http://d3hu9binmobce5.cloudfront.net/sites/default/files/highwire/royopensci/3/6.cover-source.jpg" /> <meta name="og:description" content="Toll receptors play an important role in the innate immunity of invertebrates. All reported Tolls have only one Toll/interleukin-1 receptor (TIR) domain at the C-terminal. In this study, numerous Tolls with tandem TIRs at the C-terminal were found in molluscs. Such Tolls presented an extra TIR (TIR-1) compared with Toll-I. Thus, Toll-I might be the ancestor of tandem TIRs containing Toll. To test this hypothesis, 83 Toll-I and Toll-2 (most have two TIRs, but others seem to be the evolutionary intermediates) genes from 29 shellfish species were identified. These Tolls were divided into nine groups based on phylogenetic analyses. A strong correlation between phylogeny and motif composition was found. All Toll proteins contained the TIR-2 domain, whereas the TIR-1 domain only existed in some Toll-2 protein, suggesting that TIR-1 domain insertion may play an important role in Toll protein evolution. Further analyses of functional divergence and adaptive evolution showed that some of the critical sites responsible for functional divergence may have been under positive selection. An additional intragenic recombination played an important role in the evolution of the Toll-I and Toll-2 genes. To investigate the functional difference of Toll-I and Toll-2, over expression of Hcu\_Toll-I or Hcu\_Toll-2-2 in Drosophila S2 cells was performed. Results showed that Hcu\_Toll-2-2 had stronger antimicrobial peptide (AMP) activity than Hcu\_Toll-I. Therefore, enhanced AMP-induced activity resulted from tandem TIRs in Toll-2s of molluscs during evolution history." /> <meta name="og:type" content="article" /> <meta name="article:published_time" content="2016-06-01" /> <meta name="article:section" content="Cellular and molecular biology" /> <meta name="citation_journal_title" content="Royal Society Open Science" /> <meta name="citation_publisher" content="The Royal Society" /> <meta name="citation_title" content="Enhanced antimicrobial peptide-induced activity in the mollusc Toll-2 family through evolution via tandem Toll/interleukin-1 receptor" /> <meta name="citation_publication_date" content="2016/06/01" /> <meta name="citation_mjid" content="royopensci;3/6/160123" /> <meta name="citation_id" content="3/6/160123" /> <meta name="citation_public_url" content="http://rsos.royalsocietypublishing.org/content/3/6/160123" /> <meta name="citation_abstract_html_url" content="http://rsos.royalsocietypublishing.org/content/3/6/160123.abstract" /> <meta name="citation_full_html_url" content="http://rsos.royalsocietypublishing.org/content/3/6/160123.full" /> <meta name="citation_pdf_url" content="http://rsos.royalsocietypublishing.org/content/royopensci/3/6/160123.full.pdf" /> <meta name="citation_issn" content="2054-5703" /> <meta name="citation_doi" content="10.1098/rsos.160123" /> <meta name="citation_volume" content="3" /> <meta name="citation_issue" content="6" /> <meta name="citation_section" content="Cellular and molecular biology" /> <meta name="citation_firstpage" content="160123" /> <meta name="citation_access" content="all" /> <meta name="citation_author" content="Jun Cao" /> <meta name="citation_author_institution" content="Jiangsu University" /> <meta name="citation_author" content="Yihong Chen" /> <meta name="citation_author_institution" content="Sun Yat-sen University" /> <meta name="citation_author" content="Min Jin" /> <meta name="citation_author_institution" content="Third Institute of Oceanography" /> <meta name="citation_author" content="Qian Ren" /> <meta name="citation_author_institution" content="Nanjing Normal University" /> <meta name="citation_author_email" content="renqian0402@126.com" /> <meta name="citation_reference" content="citation_title=The IACUC handbook;citation_year=2014" /> <meta name="citation_reference" content="citation_title=Invertebrate immunity;citation_year=2012" /> <meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=KV. Anderson;citation_author=G. Jürgens;citation_author=C. Nüsslein-Volhard;citation_title=Establishment of dorsal-ventral polarity in the Drosophila embryo: genetic studies on the role of the Toll gene product.;citation_pages=779-789;citation_volume=42;citation_year=1985;citation_issue=3;citation_pmid=3931918;citation_doi=10.1016/0092-8674(85)90274-0" /> <meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=B. Lemaitre;citation_author=E. Nicolas;citation_author=L. Michaut;citation_author=JM. Reichhart;citation_author=JA. Hoffmann;citation_title=The dorsoventral regulatory gene cassette spatzle/Toll/cactus controls the potent antifungal response in Drosophila adults.;citation_pages=973-983;citation_volume=86;citation_year=1996;citation_issue=6;citation_pmid=8808632;citation_doi=10.1016/S0092-8674(00)80172-5" /> <meta name="citation_reference" content="citation_journal_title=Sci. Signal.;citation_title=Interleukin-1 (IL-1) pathway;citation_volume=3;citation_year=2010" /> <meta name="citation_reference" content="citation_journal_title=EMBO reports;citation_journal_abbrev=EMBO Rep.;citation_author=JY. Ooi;citation_author=Y. Yagi;citation_author=X. Hu;citation_author=YT. Ip;citation_title=The Drosophila Toll-9 activates a constitutive antimicrobial defense;citation_pages=82-87;citation_volume=3;citation_year=2002;citation_issue=1;citation_pmid=11751574;citation_doi=10.1093/embo-reports/kvf004" /> <meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=T. Taguchi;citation_author=JL. Mitcham;citation_author=SK. Dower;citation_author=JE. Sims;citation_author=JR. Testa;citation_title=Chromosomal localization of TIL, a gene encoding a protein related to the Drosophila transmembrane receptor Toll, to human chromosome 4p14.;citation_pages=486-488;citation_volume=32;citation_year=1996;citation_issue=3;citation_pmid=8838819;citation_doi=10.1006/geno.1996.0150" /> <meta name="citation_reference" content="citation_journal_title=Nature reviews. Immunology;citation_journal_abbrev=Nat Rev Immunol;citation_author=DJ. Rawlings;citation_author=MA. Schwartz;citation_author=SW. Jackson;citation_author=A. Meyer-Bahlburg;citation_title=Integration of B cell responses through Toll-like receptors and antigen receptors.;citation_pages=282-294;citation_volume=12;citation_year=2012;citation_issue=4;citation_pmid=22421786;citation_doi=10.1038/nri3190" /> <meta name="citation_reference" content="citation_journal_title=Clin. Cancer Res.;citation_author=LA. Ridnour;citation_author=RY. Cheng;citation_author=CH. Switzer;citation_author=JL. Heinecke;citation_author=S. Ambs;citation_author=S. Glynn;citation_author=HA. Young;citation_author=G. Trinchieri;citation_author=DA. Wink;citation_title=Molecular pathways: Toll-like receptors in the tumor microenvironment—poor prognosis or new therapeutic opportunity;citation_volume=19;citation_year=2013;citation_pmid=23271799;citation_doi=10.1158/1078-0432.CCR-12-0408" /> <meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=MD. Neal;citation_title=Toll-like receptor 4 is expressed on intestinal stem cells and regulates their proliferation and apoptosis via the p53 up-regulated modulator of apoptosis;citation_volume=287;citation_year=2012;citation_pmid=22955282;citation_doi=10.1074/jbc.M112.375881" /> <meta name="citation_reference" content="citation_journal_title=Seminars in immunology;citation_journal_abbrev=Semin Immunol;citation_author=NM. Green;citation_author=A. Marshak-Rothstein;citation_title=Toll-like receptor driven B cell activation in the induction of systemic autoimmunity.;citation_pages=106-112;citation_volume=23;citation_year=2011;citation_issue=2;citation_pmid=21306913;citation_doi=10.1016/j.smim.2011.01.016" /> <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA;citation_author=JM. Reynolds;citation_author=GJ. Martinez;citation_author=Y. Chung;citation_author=C. Dong;citation_title=Toll-like receptor 4 signaling in T cells promotes autoimmune inflammation;citation_volume=109;citation_year=2012;citation_pmid=22826216;citation_doi=10.1073/pnas.1120585109" /> <meta name="citation_reference" content="citation_journal_abbrev=Front Immunol;citation_author=H. Satake;citation_author=T. Sekiguchi;citation_title=Toll-like receptors of deuterostome invertebrates.;citation_pages=34-34;citation_volume=3;citation_year=2012;citation_pmid=22566918" /> <meta name="citation_reference" content="citation_journal_title=Fish Shellfish Immunol.;citation_author=Z. Wang;citation_author=YH. Chen;citation_author=YJ. Dai;citation_author=JM. Tan;citation_author=Y. Huang;citation_author=JF. Lan;citation_author=Q. Ren;citation_title=A novel vertebrates Toll-like receptor counterpart regulating the anti-microbial peptides expression in the freshwater crayfish, Procambarus clarkii;citation_pages=219-229;citation_volume=43;citation_year=2015" /> <meta name="citation_reference" content="citation_title=Glycoscience: biology and medicine;citation_year=2015" /> <meta name="citation_reference" content="citation_journal_title=Invert. Surviv. J.;citation_author=L. Zheng;citation_author=L. Zhang;citation_author=H. Lin;citation_author=M. McIntosh;citation_author=A. Malacrida;citation_title=Toll-like receptors in invertebrate innate immunity;citation_pages=105-113;citation_volume=2;citation_year=2005" /> <meta name="citation_reference" content="citation_journal_title=Drug Des. Rev.;citation_author=N. Matsushima;citation_author=P. Enkhbayar;citation_author=M. Kamiya;citation_author=M. Osaki;citation_author=RH. Kretsinger;citation_title=Leucine-rich repeats (LRRs): structure, function, evolution and interaction with ligands;citation_pages=305-322;citation_volume=2;citation_year=2005" /> <meta name="citation_reference" content="citation_journal_title=Apoptosis;citation_author=T. Ve;citation_author=SJ. Williams;citation_author=B. Kobe;citation_title=Structure and function of Toll/interleukin-1 receptor/resistance protein (TIR) domains;citation_pages=250-261;citation_volume=20;citation_year=2015;citation_pmid=25451009;citation_doi=10.1007/s10495-014-1064-2" /> <meta name="citation_reference" content="citation_journal_title=Dev. Comp. Immunol.;citation_author=Q. Ren;citation_author=JF. Lan;citation_author=X. Zhong;citation_author=XJ. Song;citation_author=F. Ma;citation_author=KM. Hui;citation_author=W. Wang;citation_author=XQ. Yu;citation_author=JX. Wang;citation_title=A novel Toll like receptor with two TIR domains (HcToll-2) is involved in regulation of antimicrobial peptide gene expression of Hyriopsis cumingii;citation_pages=198-208;citation_volume=45;citation_year=2014;citation_pmid=24631579;citation_doi=10.1016/j.dci.2014.02.020" /> <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=RC. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" /> <meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=G. Stecher;citation_author=D. Peterson;citation_author=A. Filipski;citation_author=S. Kumar;citation_title=MEGA6: molecular evolutionary genetics analysis version 6.0;citation_volume=30;citation_year=2013;citation_pmid=24132122;citation_doi=10.1093/molbev/mst197" /> <meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=X. Gu;citation_title=Statistical methods for testing functional divergence after gene duplication;citation_pages=1664-1674;citation_volume=16;citation_year=1999;citation_issue=12;citation_pmid=10605109;citation_doi=10.1093/oxfordjournals.molbev.a026080" /> <meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=X. Gu;citation_title=Maximum-Likelihood Approach for Gene Family Evolution Under Functional Divergence;citation_pages=453-464;citation_volume=18;citation_year=2001;citation_issue=4;citation_pmid=11264396;citation_doi=10.1093/oxfordjournals.molbev.a003824" /> <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. Stern;citation_author=A. Doron-Faigenboim;citation_author=E. Erez;citation_author=E. Martz;citation_author=E. Bacharach;citation_author=T. Pupko;citation_title=Selecton 2007: advanced models for detecting positive and purifying selection using a Bayesian inference approach;citation_volume=35;citation_year=2007;citation_pmid=17586822;citation_doi=10.1093/nar/gkm382" /> <meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=DP. Martin;citation_author=P. Lemey;citation_author=M. Lott;citation_author=V. Moulton;citation_author=D. Posada;citation_author=P. Lefeuvre;citation_title=RDP3: a flexible and fast computer program for analyzing recombination;citation_volume=26;citation_year=2010;citation_pmid=20798170;citation_doi=10.1093/bioinformatics/btq467" /> <meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=D. Martin;citation_author=E. Rybicki;citation_title=RDP: detection of recombination amongst aligned sequences;citation_pages=562-563;citation_volume=16;citation_year=2000;citation_issue=6;citation_pmid=10980155;citation_doi=10.1093/bioinformatics/16.6.562" /> <meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=M. Padidam;citation_author=S. Sawyer;citation_author=CM. Fauquet;citation_title=Possible emergence of new geminiviruses by frequent recombination.;citation_pages=218-225;citation_volume=265;citation_year=1999;citation_issue=2;citation_pmid=10600594;citation_doi=10.1006/viro.1999.0056" /> <meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=D. Posada;citation_author=KA. Crandall;citation_title=Evaluation of methods for detecting recombination from DNA sequences: Computer simulations;citation_pages=13757-13762;citation_volume=98;citation_year=2001;citation_issue=24;citation_pmid=11717435;citation_doi=10.1073/pnas.241370698" /> <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=TL. Bailey;citation_author=N. Williams;citation_author=C. Misleh;citation_author=WW. Li;citation_title=MEME: discovering and analyzing DNA and protein sequence motifs;citation_pages=369-373;citation_volume=34;citation_year=2006" /> <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Punta;citation_title=The Pfam protein families database;citation_volume=40;citation_year=2012;citation_pmid=22127870;citation_doi=10.1093/nar/gkr1065" /> <meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=AK. Björklund;citation_author=D. Ekman;citation_author=A. Elofsson;citation_title=Expansion of protein domain repeats.;citation_pages=e114-e114;citation_volume=2;citation_year=2006;citation_issue=8;citation_pmid=16933986;citation_doi=10.1371/journal.pcbi.0020114" /> <meta name="citation_reference" content="citation_journal_title=Journal of structural biology;citation_journal_abbrev=J Struct Biol;citation_author=MA. Andrade;citation_author=C. Perez-Iratxeta;citation_author=CP. Ponting;citation_title=Protein repeats: structures, functions, and evolution.;citation_pages=117-131;citation_volume=134;citation_year=2001;citation_issue=2-3;citation_pmid=11551174;citation_doi=10.1006/jsbi.2001.4392" /> <meta name="citation_reference" content="citation_journal_title=Trends in biochemical sciences;citation_journal_abbrev=Trends Biochem Sci;citation_author=LD. D'Andrea;citation_author=L. Regan;citation_title=TPR proteins: the versatile helix.;citation_pages=655-662;citation_volume=28;citation_year=2003;citation_issue=12;citation_pmid=14659697;citation_doi=10.1016/j.tibs.2003.10.007" /> <meta name="citation_reference" content="citation_journal_title=FEBS Journal;citation_journal_abbrev=FEBS J;citation_author=J. Weiner;citation_author=F. Beaussart;citation_author=E. Bornberg-Bauer;citation_title=Domain deletions and substitutions in the modular protein evolution.;citation_pages=2037-2047;citation_volume=273;citation_year=2006;citation_issue=9;citation_pmid=16640566;citation_doi=10.1111/j.1742-4658.2006.05220.x" /> <meta name="citation_reference" content="citation_journal_title=Journal of structural biology;citation_journal_abbrev=J Struct Biol;citation_author=S. Light;citation_author=R. Sagit;citation_author=SS. Ithychanda;citation_author=J. Qin;citation_author=A. Elofsson;citation_title=The evolution of filamin - a protein domain repeat perspective.;citation_pages=289-298;citation_volume=179;citation_year=2012;citation_issue=3;citation_pmid=22414427;citation_doi=10.1016/j.jsb.2012.02.010" /> <meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=LD. Hurst;citation_title=The Ka/Ks ratio: diagnosing the form of sequence evolution.;citation_pages=486-486;citation_volume=18;citation_year=2002;citation_issue=9;citation_pmid=12175810;citation_doi=10.1016/S0168-9525(02)02722-1" /> <meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. Yang;citation_author=R. Nielsen;citation_author=N. Goldman;citation_author=AM. Pedersen;citation_title=Codon-Substitution Models for Heterogeneous Selection Pressure at Amino Acid Sites;citation_pages=431-449;citation_volume=155;citation_year=2000;citation_issue=1;citation_pmid=10790415" /> <meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. Cao;citation_title=Analysis of the prefoldin gene family in 14 plant species;citation_pages=317;citation_volume=7;citation_year=2006" /> <meta name="citation_reference" content="citation_journal_title=Planta;citation_author=J. Cao;citation_author=X. Li;citation_title=Identification and phylogenetic analysis of late embryogenesis abundant proteins family in tomato (Solanum lycopersicum);citation_pages=757-772;citation_volume=241;citation_year=2015;citation_pmid=25491641;citation_doi=10.1007/s00425-014-2215-y" /> <meta name="citation_reference" content="citation_journal_title=Current opinion in plant biology;citation_journal_abbrev=Curr Opin Plant Biol;citation_author=J. Ellis;citation_author=P. Dodds;citation_author=T. Pryor;citation_title=Structure, function and evolution of plant disease resistance genes.;citation_pages=278-284;citation_volume=3;citation_year=2000;citation_issue=4;citation_pmid=10873844;citation_doi=10.1016/S1369-5266(00)00080-7" /> <meta name="citation_fulltext_world_readable" content="" /> </head> <body class="html not-front not-logged-in page-node page-node- page-node-6429 node-type-highwire-article context-content hw-default-jcode-royopensci"> <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col show-peer-review-tab" id="page"> <section id="section-content" class="section section-content"> <div id="zone-content" class="zone zone-content clearfix container-30"> <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content"> <div class="region-inner region-content-inner"> <a id="main-content" /> <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main"> <div class="block-inner clearfix"> <div class="content clearfix"> <div class="panel-display panels-960-layout jcore-2col-layout"> <div class="panel-row-wrapper panel-row-first clearfix"> <div class="top-wrapper"> <div class="panel-panel panel-region-top"> <div class="inside"> <div class="panel-separator" /> <div class="panel-pane pane-highwire-article-citation"> <div class="pane-content"> <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="6429" id="node6429" data-pisa="royopensci;3/6/160123" data-pisa-master="royopensci;rsos.160123" data-apath="/royopensci/3/6/160123.atom"> <div class="highwire-cite highwire-cite-highwire-article highwire-citation-jnl-rs-article-top-a clearfix has-author-tooltip"> <div class="highwire-cite-access"><span class="highwire-citation-access" /></div> <h1 class="highwire-cite-title" id="page-title">Enhanced antimicrobial peptide-induced activity in the mollusc Toll-2 family through evolution via tandem Toll/interleukin-1 receptor </h1> <div class="highwire-cite-authors"><span class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0"><span class="nlm-given-names">Jun</span> <span class="nlm-surname">Cao</span></span>, <span class="highwire-citation-author" data-delta="1"><span class="nlm-given-names">Yihong</span> <span class="nlm-surname">Chen</span></span>, <span class="highwire-citation-author" data-delta="2"><span class="nlm-given-names">Min</span> <span class="nlm-surname">Jin</span></span>, <span class="highwire-citation-author" data-delta="3"><span class="nlm-given-names">Qian</span> <span class="nlm-surname">Ren</span></span></span></div> <div class="highwire-cite-metadata"><span class="highwire-cite-metadata-oa-ind"> </span><span class="highwire-cite-metadata-date">Published 15 June 2016</span>.<span class="highwire-cite-metadata-doi"><span class="label">DOI:</span> 10.1098/rsos.160123 </span></div> </div> <div id="hw-article-author-popups-node6429"> <div class="author-tooltip-0"> <div class="author-tooltip-name">
 Jun Cao </div> <div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff"><span class="nlm-addr-line">Institute of Life Sciences</span>, <span class="nlm-institution" data-hwp-id="institution-2">Jiangsu University</span>, <span class="nlm-addr-line">Zhenjiang, Jiangsu</span>, <span class="nlm-country">People's Republic of China</span></span></span></div> </div> <div class="author-tooltip-1"> <div class="author-tooltip-name">
 Yihong Chen </div> <div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff"><span class="nlm-addr-line">MOE Key Laboratory of Aquatic Product Safety/State Key Laboratory of Biocontrol, School of Marine Sciences</span>, <span class="nlm-institution" data-hwp-id="institution-4">Sun Yat-sen University</span>, <span class="nlm-addr-line">Guangzhou</span>, <span class="nlm-country">People's Republic of China</span></span></span></div> </div> <div class="author-tooltip-2"> <div class="author-tooltip-name">
 Min Jin </div> <div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff"><span class="nlm-addr-line">State Key Laboratory Breeding Base of Marine Genetic Resource</span>, <span class="nlm-institution" data-hwp-id="institution-3">Third Institute of Oceanography</span>, <span class="nlm-addr-line">SOA, Xiamen 361005</span>, <span class="nlm-country">People's Republic of China</span></span></span></div> </div> <div class="author-tooltip-3"> <div class="author-tooltip-name">
 Qian Ren </div> <div class="author-tooltip-affiliation"><span class="author-tooltip-text"><span class="nlm-aff"><span class="nlm-addr-line">Jiangsu Key Laboratory for Biodiversity and Biotechnology and Jiangsu Key Laboratory for Aquatic Crustacean Diseases, College of Life Sciences</span>, <span class="nlm-institution" data-hwp-id="institution-1">Nanjing Normal University</span>, <span class="nlm-addr-line">Nanjing 210046</span>, <span class="nlm-country">People's Republic of China</span></span></span></div> </div> </div> </div> </div> </div> </div> </div> </div> </div> <div class="panel-row-wrapper clearfix"> <div class="main-content-wrapper grid-17 suffix-1 alpha"> <div class="panel-panel panel-region-content"> <div class="inside"> <div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs pane-article"> <div class="pane-content"> <div class="item-list" /> </div> </div> <div class="panel-separator" /> <div class="panel-pane pane-highwire-panel-tabs-container"> <div class="pane-content"> <div data-panels-ajax-tab-preloaded="roysoc_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"> <div class="panels-ajax-tab-loading"><img class="loading" src="http://d3hu9binmobce5.cloudfront.net/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div> <div class="panels-ajax-tab-wrap-roysoc_tab_art"> <div class="panel-display panel-1col clearfix"> <div class="panel-panel panel-col"> <div> <div class="panel-pane pane-highwire-markup"> <div class="pane-content"> <div class="highwire-markup"> <div id="content-block-markup"> <div class="article fulltext-view "><span class="highwire-journal-article-marker-start" /><div class="section abstract" id="abstract-1"> <h2>Abstract</h2> <p id="p-3">Toll receptors play an important role in the innate immunity of invertebrates. All reported Tolls have only one Toll/interleukin-1 receptor (TIR) domain at the C-terminal. In this study, numerous Tolls with tandem TIRs at the C-terminal were found in molluscs. Such Tolls presented an extra TIR (TIR-1) compared with Toll-I. Thus, Toll-I might be the ancestor of tandem TIRs containing Toll. To test this hypothesis, 83 <em>Toll-I</em> and <em>Toll-2</em> (most have two TIRs, but others seem to be the evolutionary intermediates) genes from 29 shellfish species were identified. These Tolls were divided into nine groups based on phylogenetic analyses. A strong correlation between phylogeny and motif composition was found. All Toll proteins contained the TIR-2 domain, whereas the TIR-1 domain only existed in some Toll-2 protein, suggesting that TIR-1 domain insertion may play an important role in Toll protein evolution. Further analyses of functional divergence and adaptive evolution showed that some of the critical sites responsible for functional divergence may have been under positive selection. An additional intragenic recombination played an important role in the evolution of the <em>Toll-I</em> and <em>Toll-2</em> genes. To investigate the functional difference of Toll-I and Toll-2, over expression of Hcu_Toll-I or Hcu_Toll-2-2 in <em>Drosophila</em> S2 cells was performed. Results showed that Hcu_Toll-2-2 had stronger antimicrobial peptide (AMP) activity than Hcu_Toll-I. Therefore, enhanced AMP-induced activity resulted from tandem TIRs in Toll-2s of molluscs during evolution history. </p> </div> <div class="section" id="sec-1"> <h2 class="">1. Introduction</h2> <p id="p-4">Invertebrate species account for 95% of the total number of animals in the world [<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">1</a>]. Similar to vertebrates, they suffer from pathogens, such as microbes and viruses. For survival, invertebrates have developed the ability to resist pathogens during evolution. Invertebrates may lack an adaptive immunity system as that in vertebrates, but they possess a powerful and high-performance innate immunity system [<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">2</a>]. </p> <p id="p-5">Among the signalling pathways of the invertebrate immune system, the Toll signalling pathway is the best characterized one. The <em>Toll</em> gene was originally identified with function for specifying dorsal–ventral polarity of the <em>Drosophila</em> embryo [<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">3</a>]. Tolls are ubiquitous in embryos, but they are only activated by spatially restricted cleavage by Spätzle (Spz) [<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">4</a>]. A previous study reported that the <em>Drosophila</em> Toll pathway shows remarkable similarity to the mammalian interleukin-1 pathway, which activates NF-κB, a protein responsible for inflammatory and immune responses [<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">5</a>]. Toll-9 has been proven to play a role in fungal and Gram-positive bacterial defence of <em>Drosophila</em>, independent of its morphogenetic functions [<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">4</a>,<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">6</a>]. The discovery of immune function for Toll in flies has led to the identification of vertebrate Toll-like receptors (TLRs) [<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">7</a>]. TLRs have been identified in animals ranging from cnidarians to mammals. </p> <p id="p-6">Pathogen-associated molecular patterns (PAMPs) are small molecular motifs conserved within groups of pathogens. Upon pathogen invasion, they are recognized by TLRs and other pattern recognition receptors (PRRs) and initiate immune response. In <em>Drosophila</em>, a cascade reaction system is triggered once PRRs recognize PAMPs. This system contains Spz, myeloid differentiation factor 88 (MyD88), tube, pelle, dorsal, inhibitor of NF-κB kinases (IKKs) and cactus. This cascade activates NF-κB, which enhances the expression of immune factor genes, such as <em>AMPs</em>, <em>lysozyme</em> and <em>anti-lipopolysaccharides</em> (<em>ALFs</em>) for resistance to infection. The <em>Drosophila</em> genome encodes nine Toll proteins, but only Toll-9 has a role in immunity. Some vertebrate species have more than one TLR involved in immune responses; for example, <em>Homo sapiens</em> TLR1, TLR2, TLR3, TLR4, TLR5, TLR6, TLR7 and TLR9 are all engaged in the recognition of different PAMPs and activate distinct immune responses. </p> <p id="p-7">To date, TLRs play a role in infection prevention and other immune-related processes. A previous study found that dual antigen-specific B cell receptor (BCR) and TLR engagement can fine-tune functional B cell responses, directly linking cell-intrinsic innate and adaptive immune programmes [<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">8</a>]. TLR activation upregulates pro-tumourigenic pathways, including the induction of inducible nitric oxide synthase (iNOS2) and cyclooxygenase (COX) 2, to promote a feed-forward loop leading to tumour progression and the development of more aggressive tumour phenotypes [<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">9</a>]. Furthermore, TLR4 is expressed in intestinal stem cells and regulates their proliferation and apoptosis via the p53 upregulated modulator of apoptosis [<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">10</a>]. Notably, an increasing number of studies have indicated that TLRs are engaged in autoimmunization; for example, TLR4 in T cells promotes autoimmune inflammation; TLR7, TLR8 and TLR9 play a key role in promoting the production of autoantibodies reactive with DNA- or RNA-associated autoantigens [<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">11</a>,<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">12</a>]. Functions of TLRs in invertebrates other than <em>Drosophila</em> have also been investigated in recent years, but most of these studies focused on their roles in anti-pathogens [<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">13</a>,<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">14</a>]. </p> <p id="p-8">The reason for the functional diversity in TLRs is implied in their structures. Toll receptors are transmembrane proteins with extracellular leucine-rich repeat (LRR) motifs and the Toll/interleukin-1 receptor (TIR) domain [<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">15</a>]. The ectodomain of TLRs contains LRR motifs, which are defined by a consensus sequence of 24–29 amino acids in length, and the number of LRRs in different TLRs varies from 19 to 26 [<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">16</a>]. Such tandem arrays of LRRs have been found in proteins involved in ribosome and DNA binding, signal transduction, enzyme inhibition and cell adhesion [<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">17</a>]. The sequences of the TLR intracellular domain have similarities to the mammalian interleukin-1 receptors, called the TIR domain. This domain is composed of about 150 amino acid residues. Recognition of PAMPs and cytokines by TLRs leads to the stabilization of a dimeric form of the receptor through TIR domains, as well as providing a scaffold or recruitment of cytosolic adaptor proteins [<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">18</a>]. To date, most TLRs that have been investigated only contain one TIR domain. </p> <p id="p-9">In our previous study, a Toll receptor (Hcu_Toll-2-1) with tandem TIR domains at the C-terminal was found and studied in <em>Hyriopsis cumingii</em> [<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">19</a>]. A single TIR-containing Toll (Hcu_Toll-I) sharing similarities to Toll-I from <em>Mytilus galloprovincialis</em> (AFU48617.1) was also identified from <em>H. cumingii.</em> Hcu_Toll-2-1 and Hcu_Toll-I have similar domain structures but an extra TIR (TIR-1) is found in Hcu_Toll-2-1. Thus, <em>Hcu_Toll-I</em> may be the ancestor gene of <em>Hcu_Toll-2-1</em>. To test this hypothesis, transcriptome sequencing was performed in 29 mollusc species to obtain more sequences similar to the <em>Toll-I</em> or <em>Hcu_Toll-2-1</em> genes. </p> <p id="p-10">A total of 83 <em>Toll-I</em> and <em>Toll-2</em> genes from 29 aquatic molluscs were identified. Phylogenetic analyses revealed that these Tolls could be divided into the Toll-I and Toll-2 families. Most members of Group VII in the Toll-2 family contained two TIR domains. Functional divergence analyses also indicated that the <em>Toll</em> genes diverged functionally from each other, causing different evolutionary rates. Over expression of Hcu_Toll-2-2 or Hcu_Toll-I in <em>Drosophila</em> S2 cells showed that tandem TIR-containing Toll (Hcu_Toll-2-2) had stronger antimicrobial peptide (AMP)-induced activity than single TIR Toll (Hcu_Toll-I). Therefore, tandem TIR Toll genes have been preserved in mollusc species during evolution, which possibly resulted from its enhanced function. </p> </div> <div class="section methods" id="sec-2"> <h2 class="">2. Material and methods</h2> <div id="sec-3" class="subsection"> <h3>2.1. Animals</h3> <p id="p-11">In this study, 29 mollusc species (species tree is shown in the electronic supplementary material, S1) were selected to investigate the evolution of mollusc Toll receptors. Three freshwater mussels, namely, <em>H. cumingii</em>, <em>Sinanodonta woodiana</em> and <em>Cristaria plicata</em>, were purchased from Wuhu City, Anhui Province, China. Another 26 seawater mollusc species were purchased from Nanjing and Hangzhou aquatic markets. Among these 26 species, <em>Peronidia zyonoensis</em>, <em>Qicaibei</em>, <em>Wenbei</em> and <em>Hongbei</em> were purchased from Hangzhou City, Zhejiang Province, China. In addition to these four species, the remaining species were purchased from Nanjing, Jiangsu Province, China. Among these 29 species, only six species (i.e. <em>Haliotis rubra</em>, <em>Neptunea cumingi</em>, <em>Cymbium melo</em>, <em>Babylonia areolata</em>, <em>Rapana bezona</em> and <em>Turritella terebra</em>) belong to Gastropoda. Other species belong to Bivalvia. Detailed information of species in this study can be found in the electronic supplementary material, S2. Given that no Latin name could be obtained, the Chinese Pinyin of <em>Qicaibei, Wenbei</em>, <em>Hongbei</em> and <em>Jinqianbei</em> were used in this study. </p> </div> <div id="sec-4" class="subsection"> <h3>2.2. Transcriptome sequencing of mollusc species and cDNA cloning</h3> <p id="p-12">The transcriptomes of mollusc species were sequenced using Illumina HiSeq™ 2000. The raw reads were de novo assembled using Trinity program. The transcriptome assemblies were searched for sequences similar to <em>Toll-I</em> or <em>Hcu_Toll-2-1</em> using the tBLASTn algorithm. BLAST results showed that 83 <em>Toll-I</em> and <em>Toll-2</em> genes were found. Some of them possessed complete coding regions. However, some of them had no complete coding region and only had 3′ or 5′-ends. To obtain the coding region of these sequences, 3′ or 5′ RACE methods were employed using the SMARTer RACE 5′/3′ Kit (Clontech), following the manufacturer's manual. Detailed methods can be found in our previously published paper [<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">19</a>]. The primers used for RACE are shown in the electronic supplementary material, S3. </p> </div> <div id="sec-5" class="subsection"> <h3>2.3. Phylogenetic analyses of the Toll-I and Toll-2 protein families</h3> <p id="p-13">Multiple sequence alignments of the TIR-2 domain sequences were performed using MUSCLE 3.52, followed by manual comparisons and refinement [<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">20</a>]. Phylogenetic analyses of the Toll protein family, based on TIR-2 domain sequences, were performed with the neighbour-joining (NJ) method using MEGA 6 [<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">21</a>]. We also used TLR1 protein from <em>Mus musculus</em> as an outgroup. Bootstrap support values were estimated using 1000 pseudo-replicates. </p> </div> <div id="sec-6" class="subsection"> <h3>2.4. Functional divergence analyses</h3> <p id="p-14">Some residues are highly conserved and others are highly variable in evolution. We used DIVERGE (v. 2.0) [<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">23</a>] to analyse the type I functional divergence between different groups of Toll receptor proteins. The functional divergence between two groups was measured as the coefficient of functional divergence (<em>θ</em>). A coefficient equal to 0 indicates that the evolutionary rate of the duplicate genes at each site is entirely consistent. When the coefficient is greater than 0, the evolutionary rate of the duplicate genes at some critical amino acid residues is different. The software will predict these sites responding for functional divergence. </p> </div> <div id="sec-7" class="subsection"> <h3>2.5. Site-specific selection assessment and testing</h3> <p id="p-15">We used the Selecton Server (<a href="http://selecton.tau.ac.il/">http://selecton.tau.ac.il/</a>) [<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">24</a>] to calculate site-specific purifying and positive selection. In this study, <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values were used to estimate two types of substitution events by calculating the synonymous rate (<em>K</em><sub>s</sub>) and the non-synonymous rate (<em>K</em><sub>a</sub>) at each codon. Three evolutionary models (M8 (<em>ω<sub>s</sub></em> ≥ 1), M7 (beta) and M5 (gamma)) were used to describe, in probabilistic terms, how the characters evolve. Each of the models used different biological assumptions and the model that best fit the data was selected. These models all assumed a statistical distribution to account for heterogeneous <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values among sites. The distributions were approximated using eight discrete categories and the <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values were computed by calculating the expectation of a posterior distribution [<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">24</a>]. </p> </div> <div id="sec-8" class="subsection"> <h3>2.6. Detection of recombination events</h3> <p id="p-16">Coding sequence (CDS) in different Toll groups was first aligned. The recombination detection program RDP v. 3.44 [<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">25</a>] was used to explore potential recombination events between divergent nucleotide sequences. This software embeds different methods for detecting recombination signals. In this study, three methods (RDP [<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">26</a>], Geneconv [<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">27</a>] and MaxChi [<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">28</a>]) were used to detect signals. The highest acceptable <em>P</em> cut-off value was set to 0.05. Significance was evaluated with 100 permutation tests. </p> </div> <div id="sec-9" class="subsection"> <h3>2.7. Construction of recombinant plasmid</h3> <p id="p-17">The nucleotide sequences corresponding to the amino acid sequences of LRR + TM + TIR-1, LRR + TM + TIR-1 + TIR-2 of <em>Hcu_Toll-2-2</em> and LRR + TM + TIR of <em>Hcu_Toll-I</em> were amplified using the primers in the electronic supplementary material, S3 (Pac-<em>Hcu_Toll-2-2</em>-F, Pac-<em>Hcu_Toll-2-2-</em>R1, Pac-<em>Hcu_Toll-2-2-</em>R2; Pac-<em>Hcu_Toll-I-</em>F, Pac-<em>Hcu_Toll-I-</em>R). The amplified fragments were digested with Kpn I and Apa I and then ligated into a pAc5.1/V5–His B vector (Invitrogen, USA). The recombinant plasmids were sequenced to ensure correct insertion. To construct the recombinant plasmid of LRR + TM + TIR-2, in-fusion technology was used. In detail, two pairs of primers (<em>Hcu_Toll-2-2</em>-LRR-F: 5′-<span class="underline">CCCGGATCGGGGTACC</span>ATGATAAGCGGACTGCAGCATTTAG-3′, <em>Hcu_Toll-2-2</em>-LRR-R: 5′-CTCGTCTTTATTTGCCACGTCTTTG-3′; <em>Hcu_Toll-2-2</em>-TIR-2-F: 5′-GCAAATAAAGACGAGAGGATCC ACGCTTTCGTTTC-3′, <em>Hcu_Toll-2-2</em>-TIR-2-R: 5′-<span class="underline">TTCGAACCGCGGGCCC</span>TCTGTGATCTCTATCGCAACATC-3′) were used to amplify the <em>LRR</em> + <em>TM</em> and <em>TIR-2</em> cDNA sequences, respectively. The 16 bp vector-specific sequences added on the 5′-ends of <em>Hcu_Toll-2-2</em>-LRR-F and <em>Hcu_Toll-2-2</em>-TIR-2-R were underlined. Moreover, a 15 bp sequence of <em>Hcu_Toll-2-2</em>-<em>TIR-2</em>-F reverse complementary to the first 15 bp of <em>Hcu_Toll-2-2</em>-<em>LRR</em>-R is underlined with dots. The two purified fragments were then ligated into the linearized pAc5.1/V5–His B vector (double digestion with Kpn I and Apa I). About 1 µl of linearized pAc5.1/V5–His B vector, 3 µl of fragment 1, 3 µl of fragment 2 and 2 µl of In-Fusion HD Master Mix were combined and incubated for 15 min at 50°C. The recombinant plasmid was finally transformed into Trans1-T1 chemically competent cells. The recombinant plasmid was also sequenced to ensure correct insertion. </p> </div> <div id="sec-10" class="subsection"> <h3>2.8. Dual-luciferase activity assay in S2 cells</h3> <p id="p-18">Dual-luciferase activity assay in S2 cells was conducted according to the protocol of a previously published paper. In brief, the abovementioned recombinant plasmid or empty pAc5.1/V5–His B plasmid (0.3 µg) along with pGL-Pen4 or empty pGL3-Basic plasmid (0.2 µg) and 0.02 µg of pRL-TK plasmid (wild-type <em>Renilla</em> luciferase control reporter vector; Promega, USA) were co-transfected into <em>Drosophila</em> S2 cells. The reporter gene plasmid (pGL-<em>Pen4</em>) was constructed using the promoter sequence of shrimp Penaeidin-4 (PEN4). S2 cells were cultured in standard <em>Drosophila</em> medium (serum-free; Invitrogen, USA) containing 10% fetal bovine serum (Invitrogen) at 27°C. Cellfectin II reagent (Invitrogen) was used for DNA transfection into S2 cells. The firefly and <em>Renilla</em> luciferase activities were measured after 48 h of transfection with the Dual-Luciferase Reporter Assay System (Promega), according to the manufacturer's instructions. All these assays were performed from three independent transfections. </p> </div> </div> <div class="section" id="sec-11"> <h2 class="">3. Results and discussion</h2> <div id="sec-12" class="subsection"> <h3>3.1. Transcriptome sequencing and identification of <em>Toll-2</em> and <em>Toll-I</em></h3> <p id="p-19">A total of 83 <em>Toll-I</em> and <em>Toll-2</em> (most have two TIRs) genes from 29 mollusc species were identified using high-throughput sequencing and RACE technology (electronic supplementary material, S1). Based on their living environment, these mollusc species could be divided into freshwater and seawater species. Among these species, only <em>H. cumingii</em>, <em>S. woodiana</em> and <em>C. plicata</em> are freshwater mussels. According to the classification of species, these species could be divided into Gastropoda and Bivalvia. Among these species, <em>H. rubra</em>, <em>N. cumingi</em>, <em>C. melo</em>, <em>B. areolata</em>, <em>R. bezona</em> and <em>T. terebra</em> belong to Gastropoda. In some species, only <em>Toll-I</em> could be found and no <em>Toll-2</em> could be identified. One <em>Toll-I</em> in <em>Pinna rudis</em>, <em>C. melo</em>, <em>Crassostrea gigas</em>, <em>Hongbei</em> and <em>Scapharca subcrenata</em>, two <em>Toll-I</em> in <em>Tegillarca granosa</em> and three <em>Toll-I</em> in <em>Moerella iridescens</em>, <em>Mytilus edulis</em> and <em>Mactra veneriformis</em> were identified. In <em>N. cumingi</em>, only one <em>Toll-2</em> was found and no <em>Toll-I</em> could be identified. In other species, both <em>Toll-I</em> and <em>Toll-2</em> could be found. Based on their isoform number, they could be divided into nine different situations. (i) Only one <em>Toll-I</em> and one <em>Toll-2</em> were identified in <em>H. rubra</em>, <em>R. bezona</em>, <em>Ruditapes philippinarum</em>, <em>B. areolata</em>, <em>T. terebra</em>, <em>Saxidomus purpuratus</em>, <em>Sinonovacula constricta</em>, <em>Qicaibei</em> and <em>Cyclina sinensis</em>. (ii) In <em>H. cumingii</em>, one <em>Toll-I</em> and two <em>Toll-2</em> were found. (iii) In <em>Meretrix meretrix</em>, only one <em>Toll-I</em> but as many as six <em>Toll-2</em> were identified. (iv) In <em>Solen strictus</em> and <em>P. zyonoensis</em>, two <em>Toll-I</em> and one <em>Toll-2</em> were identified. (v) Two <em>Toll-I</em> and two <em>Toll-2</em> were identified from <em>Panopea abrupta</em> and <em>Chlamys farreri</em>, respectively. (vi) In <em>Wenbei</em>, two <em>Toll-I</em> and three <em>Toll-2</em> were identified. (vii) In <em>C. plicata</em>, two <em>Toll-I</em> and five <em>Toll-2</em> were found. (viii) In <em>Jinqianbei</em>, three <em>Toll-I</em> and three <em>Toll-2</em> were identified. (ix) Four <em>Toll-I</em> and two <em>Toll-2</em> were identified from <em>S. woodiana</em>. </p> </div> <div id="sec-13" class="subsection"> <h3>3.2. Phylogenetic and structural analyses of the Toll-I and Toll-2 proteins</h3> <p id="p-20">To predict the evolutionary relationships of the Toll-I and Toll-2 families, we constructed a NJ tree based on alignment of the TIR-2 sequences of the Toll-I and Toll-2 proteins. The majority of the phylogenetic clades had well-supported bootstrap values. These genes were categorized into nine major groups, named Groups I to IX. Groups I, II, III, IV, V and VI belong to the <em>Toll-I</em> gene family, whereas Groups VII, VIII and IX belong to the <em>Toll-2</em> gene family. The largest group was VII, which contained 24 members. Group VI contained only four genes. </p> <p id="p-21">To further confirm the phylogenetic relationships and to examine the diversity of Toll proteins, we further searched for conserved motifs in Toll proteins using the MEME web server (<a href="http://meme.sdsc.edu">http://meme.sdsc.edu</a>) [<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">29</a>]. As shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">figure 1</a>, five conserved motifs (motifs 1–5) were identified in these Toll proteins. All the predicted Tolls had conserved motifs 2 and 3, and the genes in the same group had similar conserved motifs, but some divergence was observed between groups. These results suggested that the Toll genes in the same group may have similar functions and some specific motif architectures may have important effects on group-specific functions. We also noticed that most proteins in Group VII contained motif 4, which did not exist in other groups, except one member (Mme_Toll-2-6) in Group IX, implying that this motif may be related to the specific functions of these proteins. Therefore, motif compositions of the Toll proteins in each group may provide additional support for phylogenetic analyses. We also used Pfam [<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">30</a>] to identify major domains of the Toll protein family. Interestingly, we also found that most members of Group VII possessed two TIR domains (one was TIR-1 (motif 4) and the other was TIR-2 (motif 3)), whereas other Tolls only contained the TIR-2 domain. Among these Toll-2 members, 14 Toll-2s were the evolutionary intermediates between single TIR Tolls and tandem TIR Tolls. Among these 14 Tolls, five Tolls were from Gastropoda. In Gastropoda, no complete tandem TIR Tolls could be found. BLASTP results showed that these 14 Tolls had similarities to Hcu_Toll-2-1, which contained two TIRs. However, SMART analysis showed that these 14 Tolls contained incomplete tandem TTRs. Thus, these 14 Tolls may be evolutionary intermediates. </p> <div id="F1" class="fig pos-float  odd"> <div class="highwire-figure"> <div class="fig-inline-img-wrapper"> <div class="fig-inline-img"><a href="http://d3hu9binmobce5.cloudfront.net/content/royopensci/3/6/160123/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogenetic relationships and structure analyses of the Toll-I and Toll-2 protein family. The numbers above branches show bootstrap values from NJ. All motifs were identified by MEME using the complete amino acid sequence of the Toll-I and Toll-2 family. Details of the motifs refer to the electronic supplementary material, S4." class="fragment-images colorbox-load" rel="gallery-fragment-images-525432710" data-figure-caption="Phylogenetic relationships and structure analyses of the Toll-I and Toll-2 protein family. The numbers above branches show bootstrap values from NJ. All motifs were identified by MEME using the complete amino acid sequence of the Toll-I and Toll-2 family. Details of the motifs refer to the electronic supplementary material, S4." data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="fragment-image lazyload" alt="Figure 1." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="http://d3hu9binmobce5.cloudfront.net/content/royopensci/3/6/160123/F1.medium.gif" width="300" height="440" /></span></a></div> </div> </div> <div class="fig-caption"><span class="fig-label">Figure 1.</span> <p id="p-22" class="first-child">Phylogenetic relationships and structure analyses of the Toll-I and Toll-2 protein family. The numbers above branches show bootstrap values from NJ. All motifs were identified by MEME using the complete amino acid sequence of the Toll-I and Toll-2 family. Details of the motifs refer to the electronic supplementary material, S4. </p> <div class="sb-div caption-clear" /> </div> </div> <p id="p-23">Domain insertion is usually thought to evolve through recombination events and intragenic duplication [<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">31</a>]. The creation of new multi-domain architectures is an important mechanism that provides opportunities for the organism to expand its repertoire of cellular functions, such as transcriptional regulation, protein transport and assembly [<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">32</a>–<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">34</a>]. Furthermore, protein domain insertion may constitute a source of variability. In the human genome, duplications are more common in genes containing repeated domains than in non-repeated ones [<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">31</a>]. Domain insertion is quite important in evolution, as it provides a path where proteins can evolve by removing or adding functionally similar or distinct blocks [<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">35</a>]. In the current study, we identified some multi-TIR domains in Tolls. The presence of two TIR domains contributed to the complexity of this gene family. Its influence on the function of Toll proteins remains to be examined. However, our findings suggested that TIR-1 domain insertion in most members of Group VII may play an important role in Toll protein evolution. </p> </div> <div id="sec-14" class="subsection"> <h3>3.3. Functional divergence analyses of the Toll proteins</h3> <p id="p-24">To further study whether amino acid substitutions cause adaptive functional diversification, we used the program DIVERGE [<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">22</a>,<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">23</a>] to estimate type-I functional divergence between different Toll groups. We compared 36 pairs of paralogous members and estimated the evolutionary rate of each amino acid site. Our results indicated that the coefficient of functional divergence (<em>θ</em>) values among these pairs was less than 1 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">table 1</a>), indicating significant site-specific alteration in selective constraints of most members of the Toll group pairs. We also predicted a few critical residues associated with functional divergence. For example, over 90 critical sites were predicted in the II/VI, II/V, II/VIII, II/IX, III/VII, III/IX, IV/IX, V/VII, V/VIII and V/IX pairs, whereas no site was predicted in the I/VI, II/III, II/VI, III/IV, III/VI, V/VI, VI/VII, VI/VIII, VI/IX and VIII/IX pairs when the cut-off value was 0.5. The highest theta values (<em>θ</em>) were observed in the II/IV, II/IX, III/IX and V/VIII pairs (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">table 1</a>), suggesting a higher evolutionary rate between them. Thus, given the different evolutionary rates, the <em>Toll</em> genes diverged functionally from each other. The amino acid mutations promoted the functional evolution and divergence of the <em>Toll</em> genes as an adaptation of the species to the changing environment. </p> <div id="T1" class="table pos-float"> <div class="table-inline"> <div class="callout"><span>View this table:</span><ul class="callout-links"> <li class="view-inline first"><a href="/" class="table-expand-inline" data-table-url="/highwire/markup/6442/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li> <li class="view-popup last"><a href="/highwire/markup/6442/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li> </ul> </div> </div> <div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-25" class="first-child">Functional divergence estimated between different groups of Toll proteins. Theta, coefficient of functional divergence; s.e., standard error; LRT, likelihood ratio test. </p> <div class="sb-div caption-clear" /> </div> </div> </div> <div id="sec-15" class="subsection"> <h3>3.4. Selective pressure at amino acid sites in the Toll family members</h3> <p id="p-27">The <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratio measures selection pressure on amino acid substitutions. A <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratio greater than 1 suggests positive selection and a ratio less than 1 suggests purifying selection [<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">36</a>]. Amino acids in a protein are usually expected to be under different selective pressures and possess different <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratios [<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">37</a>]. To test for the presence of positive or negative selection at individual amino acids, the <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratios were calculated with the Selecton Server (<a href="http://selecton.tau.ac.il">http://selecton.tau.ac.il</a>) [<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">24</a>]. We used three evolutionary models (M8 (<em>ω</em><sub>s</sub> ≥ 1), M7 (beta), and M5 (gamma)) implemented in this server to perform the tests. The results showed that the <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratios of the sequences from different Toll groups were significantly different (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">table 2</a>). For example, higher <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratios existed in Groups II, VI and IX, indicating a higher evolutionary rate or site-specific selective relaxation within members of the same group. Despite the differences in <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values, all the estimated <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values were substantially lower than 1, suggesting that the Toll sequences within each of the groups were under strong purifying selection pressure. The selection model M7 did not indicate the presence of positively selected sites, whereas the M5 model did in Groups II, V, VI and IX (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">table 2</a>). These observations suggested that selection spurred the potential for amino acid diversity at some residues, whereas other residues evolved under purifying or neutral selection. These positively selected residues might have changed the protein structure, thereby accelerating functional divergence during long periods of evolution. </p> <div id="T2" class="table pos-float"> <div class="table-inline"> <div class="callout"><span>View this table:</span><ul class="callout-links"> <li class="view-inline first"><a href="/" class="table-expand-inline" data-table-url="/highwire/markup/6438/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li> <li class="view-popup last"><a href="/highwire/markup/6438/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li> </ul> </div> </div> <div class="table-caption"><span class="table-label">Table 2.</span> <p id="p-28" class="first-child">Likelihood values and parameter estimates for different groups of the <em>Toll-I</em> and <em>Toll-2</em> genes. </p> <div class="sb-div caption-clear" /> </div> </div> </div> <div id="sec-16" class="subsection"> <h3>3.5. Recombination analysis within <em>Toll</em> genes </h3> <p id="p-29">Recombination plays a key role in the generation of genetic diversity. To determine whether homologous recombination shapes the evolution of the <em>Toll</em> genes, we analysed all <em>Toll</em> CDS segments to test whether some of them underwent an intragenic recombination event. Recombination signals of the <em>Toll</em> genes were investigated with the RDP [<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">26</a>], Geneconv [<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">27</a>] and MaxChi [<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">28</a>] methods embedded in the program RDP v3.44 [<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">25</a>]. All nine groups were found to contain similar mosaic segments, demonstrating that intragenic recombination had occurred. As summarized in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">table 3</a>, 57 <em>Toll</em> genes in these groups exhibited evidence of intragenic recombination (<em>p</em> &amp;lt; 0.05 based on 100 permutations). As an example, we presented a recombination event of <em>Cpl_Toll-2-5</em> and <em>Cpl_Toll-2-4</em> detected by the RDP method (<a id="xref-fig-2-1" class="xref-fig" href="#F2">figure 2</a>). A significant recombination event occurred between the 5′-ends of <em>Cpl_Toll-2-5</em> and <em>Cpl_Toll-2-4</em>. Our results indicated that the <em>Toll</em> genes underwent frequent recombination events. Therefore, intragenic recombination played an important role in the evolution of the <em>Toll</em> genes, similar to other family genes [<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">38</a>,<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">39</a>]. Further studies are required to evaluate the influence of recombination on function and investigate the mechanisms underlying <em>Toll</em> recombination. </p> <div id="F2" class="fig pos-float  odd"> <div class="highwire-figure"> <div class="fig-inline-img-wrapper"> <div class="fig-inline-img"><a href="http://d3hu9binmobce5.cloudfront.net/content/royopensci/3/6/160123/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of recombination events between Cpl_Toll-2-5 and Cpl_Toll-2-4 genes. The plot display of recombination events was detected by the RDP method." class="fragment-images colorbox-load" rel="gallery-fragment-images-525432710" data-figure-caption="&lt;div xmlns=" www.w3.org="">Identification of recombination events between <em>Cpl_Toll-2-5</em> and <em>Cpl_Toll-2-4</em> genes. The plot display of recombination events was detected by the RDP method.</a></div><a href="http://d3hu9binmobce5.cloudfront.net/content/royopensci/3/6/160123/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identification of recombination events between Cpl_Toll-2-5 and Cpl_Toll-2-4 genes. The plot display of recombination events was detected by the RDP method." class="fragment-images colorbox-load" rel="gallery-fragment-images-525432710" data-figure-caption="&lt;div xmlns=" www.w3.org="">" data-icon-position="" data-hide-link-title="0"&amp;gt;<span class="hw-responsive-img"><img class="fragment-image lazyload" alt="Figure 2." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="http://d3hu9binmobce5.cloudfront.net/content/royopensci/3/6/160123/F2.medium.gif" width="440" height="185" /></span></a></div> </div> </div> <div class="fig-caption"><span class="fig-label">Figure 2.</span> <p id="p-30" class="first-child">Identification of recombination events between <em>Cpl_Toll-2-5</em> and <em>Cpl_Toll-2-4</em> genes. The plot display of recombination events was detected by the RDP method. </p> <div class="sb-div caption-clear" /> </div> </div> <div id="T3" class="table pos-float"> <div class="table-inline"> <div class="callout"><span>View this table:</span><ul class="callout-links"> <li class="view-inline first"><a href="/" class="table-expand-inline" data-table-url="/highwire/markup/6435/expansion?postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" html="1" fragment="#" external="1" data-icon-position="" data-hide-link-title="0">View inline</a></li> <li class="view-popup last"><a href="/highwire/markup/6435/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li> </ul> </div> </div> <div class="table-caption"><span class="table-label">Table 3.</span> <p id="p-31" class="first-child">The predicted recombination events for the <em>Toll-I</em> and <em>Toll-2</em> genes in shellfish. </p> <div class="sb-div caption-clear" /> </div> </div> </div> <div id="sec-17" class="subsection"> <h3>3.6. Over expression of <em>Hcu_Toll-2-2</em> and <em>Hcu_Toll-I</em> activates shrimp penaeidin-4 </h3> <p id="p-32">To analyse the functional differences between tandem TIRs (TIR-1 and TIR-2) containing Toll (Hcu_Toll-2-2) and single TIR domain containing Toll (Hcu_Toll-I), over expression of <em>Hcu_Toll-2-2</em> or <em>Hcu_Toll-I</em> in <em>Drosophila</em> S2 cells was performed. Dual-luciferase reporter assays were performed and both Hcu_Toll-2-2 and Hcu_Toll-I could activate the promoter activity of shrimp <em>penaeidin</em>-<em>4</em>. However, Hcu_Toll-2-2 had stronger AMP-induced activity than Hcu_Toll-I. The difference in AMP-induced activity of TIR-1 and TIR-2 of Hcu_Toll-2-2 was further investigated. As shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">figure 3</a><em>b</em>, Hcu_Toll-2-2 (containing LRRs + TM + TIR-1 + TIR-2) had stronger AMP-induced activity than Hcu_Toll-2-2-TIR-1 (including LRRs + TM + TIR-1) and Hcu_Toll-2-2-TIR-2 (LRRs + TM + TIR-2). Hcu_Toll-2-2-TIR-2 had stronger AMP-induced activity than Hcu_Toll-2-2-TIR-1 (<a id="xref-fig-3-2" class="xref-fig" href="#F3">figure 3</a>). </p> <div id="F3" class="fig pos-float  odd"> <div class="highwire-figure"> <div class="fig-inline-img-wrapper"> <div class="fig-inline-img"><a href="http://d3hu9binmobce5.cloudfront.net/content/royopensci/3/6/160123/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Activation of shrimp Pen4 antimicrobial peptide gene by over expression of the full length of Hcu_Toll-I and Hcu_Toll-2-2 (a), Hcu_Toll-2-2-TIR-1, Hcu_Toll-2-2-TIR-2 and Hcu_Toll-2-2-TIR-1 + TIR-2 (b) in S2 cells. The detailed methods could be seen in Material and methods section. All data are representative of three independent experiments. The bars indicate the s.d. of the luciferase activity (n = 3)." class="fragment-images colorbox-load" rel="gallery-fragment-images-525432710" data-figure-caption="&lt;div xmlns=" www.w3.org="">Activation of shrimp <em>Pen4</em> antimicrobial peptide gene by over expression of the full length of <em>Hcu_Toll-I</em> and <em>Hcu_Toll-2-2</em> (<em>a</em>), <em>Hcu_Toll-2-2</em>-TIR-1, <em>Hcu_Toll-2-2</em>-TIR-2 and <em>Hcu_Toll-2-2</em>-TIR-1 + TIR-2 (<em>b</em>) in S2 cells. The detailed methods could be seen in Material and methods section. All data are representative of three independent experiments. The bars indicate the s.d. of the luciferase activity (<em>n</em> = 3).</a></div><a href="http://d3hu9binmobce5.cloudfront.net/content/royopensci/3/6/160123/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Activation of shrimp Pen4 antimicrobial peptide gene by over expression of the full length of Hcu_Toll-I and Hcu_Toll-2-2 (a), Hcu_Toll-2-2-TIR-1, Hcu_Toll-2-2-TIR-2 and Hcu_Toll-2-2-TIR-1 + TIR-2 (b) in S2 cells. The detailed methods could be seen in Material and methods section. All data are representative of three independent experiments. The bars indicate the s.d. of the luciferase activity (n = 3)." class="fragment-images colorbox-load" rel="gallery-fragment-images-525432710" data-figure-caption="&lt;div xmlns=" www.w3.org="">" data-icon-position="" data-hide-link-title="0"&amp;gt;<span class="hw-responsive-img"><img class="fragment-image lazyload" alt="Figure 3." src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="http://d3hu9binmobce5.cloudfront.net/content/royopensci/3/6/160123/F3.medium.gif" width="248" height="440" /></span></a></div> </div> </div> <div class="fig-caption"><span class="fig-label">Figure 3.</span> <p id="p-33" class="first-child">Activation of shrimp <em>Pen4</em> antimicrobial peptide gene by over expression of the full length of <em>Hcu_Toll-I</em> and <em>Hcu_Toll-2-2</em> (<em>a</em>), <em>Hcu_Toll-2-2</em>-TIR-1, <em>Hcu_Toll-2-2</em>-TIR-2 and <em>Hcu_Toll-2-2</em>-TIR-1 + TIR-2 (<em>b</em>) in S2 cells. The detailed methods could be seen in Material and methods section. All data are representative of three independent experiments. The bars indicate the s.d. of the luciferase activity (<em>n</em> = 3). </p> <div class="sb-div caption-clear" /> </div> </div> <p id="p-34">SMART analysis showed that Hcu_Toll-I had four LRRs, followed by one LRR_CT, two LRRs, one LRR_CT, one TM and one TIR named TIR-2. In terms of Hcu_Toll-2-2, in addition to a TIR (TIR-1), the other was similar to Hcu_Toll-I. Phylogenetic analysis demonstrated that Toll-I was the ancestor of Toll-2. Thus, the enhanced AMP-induced activity of Toll-2 may result from its evolution into tandem TIRs. A previous study confirmed that LRR repeat domains in many plant resistance genes play important roles in plant innate immune systems [<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">40</a>]. Activation of the Toll signalling pathway can induce the interaction of Toll and Myd88 adaptor molecules through its TIR domain. Tandem TIRs of Toll-2 may have more interaction sites and may bind more Myd88 molecules, which induce strong AMP-induced activity. The enhanced AMP-induced activity resulted in tandem TIR-containing Tolls existing in molluscs during evolution. </p> </div> </div> <div class="section" id="sec-18"> <h2 class="">Authors' contributions</h2> <p id="p-35">Q.R. and Y.C. carried out the experiments. Q.R. designed the experiments and J.C. and Q.R. analysed the data. Q.R., J.C. and Y.C. wrote the manuscript. All authors have read and approved the manuscript. </p> </div> <div class="section" id="sec-19"> <h2 class="">Competing interests</h2> <p id="p-36">The authors declare no conflict of interest.</p> </div> <div class="section" id="sec-20"> <h2 class="">Funding</h2> <p id="p-37">This study was supported by the National Natural Science Foundation of China (Grant Nos. 31572647), the Natural Science Foundation of Jiangsu Province (BK20131401), Natural Science Fund of Colleges and Universities in Jiangsu Province (14KJA240002) and the Project Funded by the Priority Academic Program Development of Jiangsu Higher Education Institutions (PAPD). </p> </div> <ul class="history-list"> <li class="received"><span class="received-label">Received </span>February 22, 2016. </li> <li class="accepted"><span class="accepted-label">Accepted </span>May 12, 2016. </li> </ul> <ul class="copyright-statement"> <li class="fn" id="copyright-statement-1" /> </ul> <div class="license" id="license-1"> <p id="p-2">© 2016 The Authors. Published by the Royal Society under the terms of the Creative Commons Attribution License <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>, which permits unrestricted use, provided the original author and source are credited. </p> </div> <div class="section ref-list" id="ref-list-1"> <h2 class="">References</h2> <ol class="cit-list"> <li> <div class="cit ref-cit ref-book" id="cit-3.6.160123.1"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Carbone</span> <span class="cit-name-given-names">L</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2014</span> <span class="cit-source">The IACUC handbook</span>. <span class="cit-publ-loc">New York, NY</span>: <span class="cit-publ-name">CRC Press</span>. </div> </div> </div> </li> <li> <div class="cit ref-cit ref-book" id="cit-3.6.160123.2"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Maramorosch</span> <span class="cit-name-given-names">K</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2012</span> <span class="cit-source">Invertebrate immunity</span>. <span class="cit-publ-loc">New York, NY</span>: <span class="cit-publ-name">Academic Press</span>. </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.3" data-doi="10.1016/0092-8674(85)90274-0"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Anderson</span> <span class="cit-name-given-names">KV</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Jürgens</span> <span class="cit-name-given-names">G</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Nüsslein-Volhard</span> <span class="cit-name-given-names">C</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">1985</span> <span class="cit-article-title">Establishment of dorsal-ventral polarity in the <em>Drosophila</em> embryo: genetic studies on the role of the <em>Toll</em> gene product</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">42</span>, <span class="cit-fpage">779</span>–<span class="cit-lpage">789</span>. (<a href="http://dx.doi.org/10.1016/0092-8674(85)90274-0">doi:10.1016/0092-8674(85)90274-0</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.4" data-doi="10.1016/S0092-8674(00)80172-5"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Lemaitre</span> <span class="cit-name-given-names">B</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Nicolas</span> <span class="cit-name-given-names">E</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Michaut</span> <span class="cit-name-given-names">L</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Reichhart</span> <span class="cit-name-given-names">JM</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span> <span class="cit-name-given-names">JA</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">1996</span> <span class="cit-article-title">The dorsoventral regulatory gene cassette <em>spätzle/Toll/cactus</em> controls the potent antifungal response in <em>Drosophila</em> adults</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">86</span>, <span class="cit-fpage">973</span>–<span class="cit-lpage">983</span>. (<a href="http://dx.doi.org/10.1016/S0092-8674(00)80172-5">doi:10.1016/S0092-8674(00)80172-5</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.5"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Weber</span> <span class="cit-name-given-names">A</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Wasiliew</span> <span class="cit-name-given-names">P</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Kracht</span> <span class="cit-name-given-names">M</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2010</span> <span class="cit-article-title">Interleukin-1 (IL-1) pathway</span>. <abbr class="cit-jnl-abbrev">Sci. Signal.</abbr> <span class="cit-vol">3</span>, cm1. (<a href="http://dx.doi.org/10.1126/scisignal.3105cm1">doi:10.1126/scisignal.3105cm1</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.6" data-doi="10.1093/embo-reports/kvf004"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Ooi</span> <span class="cit-name-given-names">JY</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Yagi</span> <span class="cit-name-given-names">Y</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Hu</span> <span class="cit-name-given-names">X</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Ip</span> <span class="cit-name-given-names">YT</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2002</span> <span class="cit-article-title">The <em>Drosophila</em> Toll-9 activates a constitutive antimicrobial defense</span>. <abbr class="cit-jnl-abbrev">EMBO Rep.</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">82</span>–<span class="cit-lpage">87</span>. (<a href="http://dx.doi.org/10.1093/embo-reports/kvf004">doi:10.1093/embo-reports/kvf004</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.7" data-doi="10.1006/geno.1996.0150"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Taguchi</span> <span class="cit-name-given-names">T</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Mitcham</span> <span class="cit-name-given-names">JL</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Dower</span> <span class="cit-name-given-names">SK</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Sims</span> <span class="cit-name-given-names">JE</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Testa</span> <span class="cit-name-given-names">JR</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">1996</span> <span class="cit-article-title">Chromosomal localization of <em>TIL</em>, a gene encoding a protein related to the <em>Drosophila</em> transmembrane receptor Toll, to human chromosome 4p14</span>. <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">486</span>–<span class="cit-lpage">488</span>. (<a href="http://dx.doi.org/10.1006/geno.1996.0150">doi:10.1006/geno.1996.0150</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.8" data-doi="10.1038/nri3190"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Rawlings</span> <span class="cit-name-given-names">DJ</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span> <span class="cit-name-given-names">MA</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Jackson</span> <span class="cit-name-given-names">SW</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Meyer-Bahlburg</span> <span class="cit-name-given-names">A</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integration of B cell responses through Toll-like receptors and antigen receptors</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Immunol.</abbr> <span class="cit-vol">12</span>, <span class="cit-fpage">282</span>–<span class="cit-lpage">294</span>. (<a href="http://dx.doi.org/10.1038/nri3190">doi:10.1038/nri3190</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.9" data-doi="10.1158/1078-0432.CCR-12-0408"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Ridnour</span> <span class="cit-name-given-names">LA</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Cheng</span> <span class="cit-name-given-names">RY</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Switzer</span> <span class="cit-name-given-names">CH</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Heinecke</span> <span class="cit-name-given-names">JL</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Ambs</span> <span class="cit-name-given-names">S</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Glynn</span> <span class="cit-name-given-names">S</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Young</span> <span class="cit-name-given-names">HA</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Trinchieri</span> <span class="cit-name-given-names">G</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Wink</span> <span class="cit-name-given-names">DA</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2013</span> <span class="cit-article-title">Molecular pathways: Toll-like receptors in the tumor microenvironment—poor prognosis or new therapeutic opportunity</span>. <abbr class="cit-jnl-abbrev">Clin. Cancer Res.</abbr> <span class="cit-vol">19</span>, <span class="cit-fpage">1340</span>–<span class="cit-lpage">1346</span>. (<a href="http://dx.doi.org/10.1158/1078-0432.CCR-12-0408">doi:10.1158/1078-0432.CCR-12-0408</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.10" data-doi="10.1074/jbc.M112.375881"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Neal</span> <span class="cit-name-given-names">MD</span></span> <span class="cit-etal">et al.</span></li> </ol> <div class="cite_UNKNOWN"> <span class="cit-pub-date">2012</span> <span class="cit-article-title">Toll-like receptor 4 is expressed on intestinal stem cells and regulates their proliferation and apoptosis via the p53 up-regulated modulator of apoptosis</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">287</span>, <span class="cit-fpage">37 296</span>–<span class="cit-lpage">37 308</span>. (<a href="http://dx.doi.org/10.1074/jbc.M112.375881">doi:10.1074/jbc.M112.375881</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.11" data-doi="10.1016/j.smim.2011.01.016"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Green</span> <span class="cit-name-given-names">NM</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Marshak-Rothstein</span> <span class="cit-name-given-names">A</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2011</span> <span class="cit-article-title">Toll-like receptor driven B cell activation in the induction of systemic autoimmunity</span>. <abbr class="cit-jnl-abbrev">Semin. Immunol.</abbr> <span class="cit-vol">23</span>, <span class="cit-fpage">106</span>–<span class="cit-lpage">112</span>. (<a href="http://dx.doi.org/10.1016/j.smim.2011.01.016">doi:10.1016/j.smim.2011.01.016</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.12" data-doi="10.1073/pnas.1120585109"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Reynolds</span> <span class="cit-name-given-names">JM</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Martinez</span> <span class="cit-name-given-names">GJ</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Chung</span> <span class="cit-name-given-names">Y</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Dong</span> <span class="cit-name-given-names">C</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2012</span> <span class="cit-article-title">Toll-like receptor 4 signaling in T cells promotes autoimmune inflammation</span>. <abbr class="cit-jnl-abbrev">Proc. Natl Acad. Sci. USA</abbr> <span class="cit-vol">109</span>, <span class="cit-fpage">13 064</span>–<span class="cit-lpage">13 069</span>. (<a href="http://dx.doi.org/10.1073/pnas.1120585109">doi:10.1073/pnas.1120585109</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.13"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Satake</span> <span class="cit-name-given-names">H</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Sekiguchi</span> <span class="cit-name-given-names">T</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2012</span> <span class="cit-article-title">Toll-like receptors of deuterostome invertebrates</span>. <abbr class="cit-jnl-abbrev">Front. Immunol.</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">34</span>. (<a href="http://dx.doi.org/10.3389/fimmu.2012.00034">doi:10.3389/fimmu.2012.00034</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.14"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Wang</span> <span class="cit-name-given-names">Z</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Chen</span> <span class="cit-name-given-names">YH</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Dai</span> <span class="cit-name-given-names">YJ</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Tan</span> <span class="cit-name-given-names">JM</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Huang</span> <span class="cit-name-given-names">Y</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Lan</span> <span class="cit-name-given-names">JF</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Ren</span> <span class="cit-name-given-names">Q</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2015</span> <span class="cit-article-title">A novel vertebrates Toll-like receptor counterpart regulating the anti-microbial peptides expression in the freshwater crayfish, <em>Procambarus clarkii</em></span>. <abbr class="cit-jnl-abbrev">Fish Shellfish Immunol.</abbr> <span class="cit-vol">43</span>, <span class="cit-fpage">219</span>–<span class="cit-lpage">229</span>. (<a href="http://dx.doi.org/10.1016/j.fsi.2014.12.038">doi:10.1016/j.fsi.2014.12.038</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-book" id="cit-3.6.160123.15"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Tanimura</span> <span class="cit-name-given-names">N</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Miyake</span> <span class="cit-name-given-names">K</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2015</span> <span class="cit-source">Glycoscience: biology and medicine</span>. <span class="cit-publ-loc">Tokyo, Japan</span>: <span class="cit-publ-name">Springer</span>. </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.16"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Zheng</span> <span class="cit-name-given-names">L</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Zhang</span> <span class="cit-name-given-names">L</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Lin</span> <span class="cit-name-given-names">H</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">McIntosh</span> <span class="cit-name-given-names">M</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Malacrida</span> <span class="cit-name-given-names">A</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2005</span> <span class="cit-article-title">Toll-like receptors in invertebrate innate immunity</span>. <abbr class="cit-jnl-abbrev">Invert. Surviv. J.</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">105</span>–<span class="cit-lpage">113</span>. </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.17"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Matsushima</span> <span class="cit-name-given-names">N</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Enkhbayar</span> <span class="cit-name-given-names">P</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Kamiya</span> <span class="cit-name-given-names">M</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Osaki</span> <span class="cit-name-given-names">M</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Kretsinger</span> <span class="cit-name-given-names">RH</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2005</span> <span class="cit-article-title">Leucine-rich repeats (LRRs): structure, function, evolution and interaction with ligands</span>. <abbr class="cit-jnl-abbrev">Drug Des. Rev.</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">305</span>–<span class="cit-lpage">322</span>. (<a href="http://dx.doi.org/10.2174/1567269054087613">doi:10.2174/1567269054087613</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.18" data-doi="10.1007/s10495-014-1064-2"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Ve</span> <span class="cit-name-given-names">T</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Williams</span> <span class="cit-name-given-names">SJ</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Kobe</span> <span class="cit-name-given-names">B</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2015</span> <span class="cit-article-title">Structure and function of Toll/interleukin-1 receptor/resistance protein (TIR) domains</span>. <abbr class="cit-jnl-abbrev">Apoptosis</abbr> <span class="cit-vol">20</span>, <span class="cit-fpage">250</span>–<span class="cit-lpage">261</span>. (<a href="http://dx.doi.org/10.1007/s10495-014-1064-2">doi:10.1007/s10495-014-1064-2</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.19" data-doi="10.1016/j.dci.2014.02.020"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Ren</span> <span class="cit-name-given-names">Q</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Lan</span> <span class="cit-name-given-names">JF</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Zhong</span> <span class="cit-name-given-names">X</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Song</span> <span class="cit-name-given-names">XJ</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Ma</span> <span class="cit-name-given-names">F</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Hui</span> <span class="cit-name-given-names">KM</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Wang</span> <span class="cit-name-given-names">W</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Yu</span> <span class="cit-name-given-names">XQ</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Wang</span> <span class="cit-name-given-names">JX</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2014</span> <span class="cit-article-title">A novel Toll like receptor with two TIR domains (HcToll-2) is involved in regulation of antimicrobial peptide gene expression of <em>Hyriopsis cumingii</em></span>. <abbr class="cit-jnl-abbrev">Dev. Comp. Immunol.</abbr> <span class="cit-vol">45</span>, <span class="cit-fpage">198</span>–<span class="cit-lpage">208</span>. (<a href="http://dx.doi.org/10.1016/j.dci.2014.02.020">doi:10.1016/j.dci.2014.02.020</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.20" data-doi="10.1093/nar/gkh340"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Edgar</span> <span class="cit-name-given-names">RC</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>, <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>. (<a href="http://dx.doi.org/10.1093/nar/gkh340">doi:10.1093/nar/gkh340</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.21" data-doi="10.1093/molbev/mst197"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Tamura</span> <span class="cit-name-given-names">K</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Stecher</span> <span class="cit-name-given-names">G</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Peterson</span> <span class="cit-name-given-names">D</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Filipski</span> <span class="cit-name-given-names">A</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Kumar</span> <span class="cit-name-given-names">S</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2013</span> <span class="cit-article-title">MEGA6: molecular evolutionary genetics analysis version 6.0</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>, <span class="cit-fpage">2725</span>–<span class="cit-lpage">2729</span>. (<a href="http://dx.doi.org/10.1093/molbev/mst197">doi:10.1093/molbev/mst197</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.22" data-doi="10.1093/oxfordjournals.molbev.a026080"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Gu</span> <span class="cit-name-given-names">X</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">1999</span> <span class="cit-article-title">Statistical methods for testing functional divergence after gene duplication</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">1664</span>–<span class="cit-lpage">1674</span>. (<a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026080">doi:10.1093/oxfordjournals.molbev.a026080</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.23" data-doi="10.1093/oxfordjournals.molbev.a003824"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Gu</span> <span class="cit-name-given-names">X</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2001</span> <span class="cit-article-title">Maximum-likelihood approach for gene family evolution under functional divergence</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">453</span>–<span class="cit-lpage">464</span>. (<a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003824">doi:10.1093/oxfordjournals.molbev.a003824</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.24" data-doi="10.1093/nar/gkm382"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Stern</span> <span class="cit-name-given-names">A</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Doron-Faigenboim</span> <span class="cit-name-given-names">A</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Erez</span> <span class="cit-name-given-names">E</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Martz</span> <span class="cit-name-given-names">E</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Bacharach</span> <span class="cit-name-given-names">E</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Pupko</span> <span class="cit-name-given-names">T</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2007</span> <span class="cit-article-title">Selecton 2007: advanced models for detecting positive and purifying selection using a Bayesian inference approach</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>, <span class="cit-fpage">W506</span>–<span class="cit-lpage">W511</span>. (<a href="http://dx.doi.org/10.1093/nar/gkm382">doi:10.1093/nar/gkm382</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.25" data-doi="10.1093/bioinformatics/btq467"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Martin</span> <span class="cit-name-given-names">DP</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Lemey</span> <span class="cit-name-given-names">P</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Lott</span> <span class="cit-name-given-names">M</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Moulton</span> <span class="cit-name-given-names">V</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Posada</span> <span class="cit-name-given-names">D</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Lefeuvre</span> <span class="cit-name-given-names">P</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2010</span> <span class="cit-article-title">RDP3: a flexible and fast computer program for analyzing recombination</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>, <span class="cit-fpage">2462</span>–<span class="cit-lpage">2463</span>. (<a href="http://dx.doi.org/10.1093/bioinformatics/btq467">doi:10.1093/bioinformatics/btq467</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.26" data-doi="10.1093/bioinformatics/16.6.562"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Martin</span> <span class="cit-name-given-names">D</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Rybicki</span> <span class="cit-name-given-names">E</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2000</span> <span class="cit-article-title">RDP: detection of recombination amongst aligned sequences</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">16</span>, <span class="cit-fpage">562</span>–<span class="cit-lpage">563</span>. (<a href="http://dx.doi.org/10.1093/bioinformatics/16.6.562">doi:10.1093/bioinformatics/16.6.562</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.27" data-doi="10.1006/viro.1999.0056"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Padidam</span> <span class="cit-name-given-names">M</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span> <span class="cit-name-given-names">S</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Fauquet</span> <span class="cit-name-given-names">CM</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">1999</span> <span class="cit-article-title">Possible emergence of new geminiviruses by frequent recombination</span>. <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">265</span>, <span class="cit-fpage">218</span>–<span class="cit-lpage">225</span>. (<a href="http://dx.doi.org/10.1006/viro.1999.0056">doi:10.1006/viro.1999.0056</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.28" data-doi="10.1073/pnas.241370698"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Posada</span> <span class="cit-name-given-names">D</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Crandall</span> <span class="cit-name-given-names">KA</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2001</span> <span class="cit-article-title">Evaluation of methods for detecting recombination from DNA sequences: computer simulations</span>. <abbr class="cit-jnl-abbrev">Proc. Natl Acad. Sci. USA</abbr> <span class="cit-vol">98</span>, <span class="cit-fpage">13 757</span>–<span class="cit-lpage">13 762</span>. (<a href="http://dx.doi.org/10.1073/pnas.241370698">doi:10.1073/pnas.241370698</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.29"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Bailey</span> <span class="cit-name-given-names">TL</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Williams</span> <span class="cit-name-given-names">N</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Misleh</span> <span class="cit-name-given-names">C</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Li</span> <span class="cit-name-given-names">WW</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2006</span> <span class="cit-article-title">MEME: discovering and analyzing DNA and protein sequence motifs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>, <span class="cit-fpage">369</span>–<span class="cit-lpage">373</span>. (<a href="http://dx.doi.org/10.1093/nar/gkl198">doi:10.1093/nar/gkl198</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.30" data-doi="10.1093/nar/gkr1065"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Punta</span> <span class="cit-name-given-names">M</span></span> <span class="cit-etal">et al.</span></li> </ol> <div class="cite_UNKNOWN"> <span class="cit-pub-date">2012</span> <span class="cit-article-title">The Pfam protein families database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>, <span class="cit-fpage">D290</span>–<span class="cit-lpage">D301</span>. (<a href="http://dx.doi.org/10.1093/nar/gkr1065">doi:10.1093/nar/gkr1065</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.31" data-doi="10.1371/journal.pcbi.0020114"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Björklund</span> <span class="cit-name-given-names">AK</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Ekman</span> <span class="cit-name-given-names">D</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Elofsson</span> <span class="cit-name-given-names">A</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2006</span> <span class="cit-article-title">Expansion of protein domain repeats</span>. <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">2</span>, <span class="cit-fpage">e114</span>. (<a href="http://dx.doi.org/10.1371/journal.pcbi.0020114">doi:10.1371/journal.pcbi.0020114</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.32" data-doi="10.1006/jsbi.2001.4392"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Andrade</span> <span class="cit-name-given-names">MA</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Perez-Iratxeta</span> <span class="cit-name-given-names">C</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Ponting</span> <span class="cit-name-given-names">CP</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2001</span> <span class="cit-article-title">Protein repeats: structures, functions, and evolution</span>. <abbr class="cit-jnl-abbrev">J. Struct. Biol.</abbr> <span class="cit-vol">134</span>, <span class="cit-fpage">117</span>–<span class="cit-lpage">131</span>. (<a href="http://dx.doi.org/10.1006/jsbi.2001.4392">doi:10.1006/jsbi.2001.4392</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal no-rev-xref" id="cit-3.6.160123.33" data-doi="10.1016/j.tibs.2003.10.007"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">D'Andrea</span> <span class="cit-name-given-names">LD</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Regan</span> <span class="cit-name-given-names">L</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2003</span> <span class="cit-article-title">TPR proteins: the versatile helix</span>. <abbr class="cit-jnl-abbrev">Trends Biochem. Sci.</abbr> <span class="cit-vol">28</span>, <span class="cit-fpage">655</span>–<span class="cit-lpage">662</span>. (<a href="http://dx.doi.org/10.1016/j.tibs.2003.10.007">doi:10.1016/j.tibs.2003.10.007</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.34" data-doi="10.1111/j.1742-4658.2006.05220.x"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Weiner</span> <span class="cit-name-given-names">J</span> <span class="cit-name-suffix">3rd</span>.</span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Beaussart</span> <span class="cit-name-given-names">F</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Bornberg-Bauer</span> <span class="cit-name-given-names">E</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2006</span> <span class="cit-article-title">Domain deletions and substitutions in the modular protein evolution</span>. <abbr class="cit-jnl-abbrev">FEBS J.</abbr> <span class="cit-vol">273</span>, <span class="cit-fpage">2037</span>–<span class="cit-lpage">2047</span>. (<a href="http://dx.doi.org/10.1111/j.1742-4658.2006.05220.x">doi:10.1111/j.1742-4658.2006.05220.x</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.35" data-doi="10.1016/j.jsb.2012.02.010"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Light</span> <span class="cit-name-given-names">S</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Sagit</span> <span class="cit-name-given-names">R</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Ithychanda</span> <span class="cit-name-given-names">SS</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Qin</span> <span class="cit-name-given-names">J</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Elofsson</span> <span class="cit-name-given-names">A</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2012</span> <span class="cit-article-title">The evolution of filamin: a protein domain repeat perspective</span>. <abbr class="cit-jnl-abbrev">J. Struct. Biol.</abbr> <span class="cit-vol">179</span>, <span class="cit-fpage">289</span>–<span class="cit-lpage">298</span>. (<a href="http://dx.doi.org/10.1016/j.jsb.2012.02.010">doi:10.1016/j.jsb.2012.02.010</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.36" data-doi="10.1016/S0168-9525(02)02722-1"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Hurst</span> <span class="cit-name-given-names">LD</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2002</span> <span class="cit-article-title">The Ka/Ks ratio: diagnosing the form of sequence evolution</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>, <span class="cit-fpage">486</span>–<span class="cit-lpage">487</span>. (<a href="http://dx.doi.org/10.1016/S0168-9525(02)02722-1">doi:10.1016/S0168-9525(02)02722-1</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.37"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Yang</span> <span class="cit-name-given-names">Z</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span> <span class="cit-name-given-names">R</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Goldman</span> <span class="cit-name-given-names">N</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span> <span class="cit-name-given-names">AM</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2000</span> <span class="cit-article-title">Codon substitution models for heterogeneous selection pressure at amino acid sites</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>, <span class="cit-fpage">431</span>–<span class="cit-lpage">449</span>. </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.38"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Cao</span> <span class="cit-name-given-names">J</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2006</span> <span class="cit-article-title">Analysis of the prefoldin gene family in 14 plant species</span>. <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">7</span>, <span class="cit-fpage">317</span>. </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.39" data-doi="10.1007/s00425-014-2215-y"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Cao</span> <span class="cit-name-given-names">J</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Li</span> <span class="cit-name-given-names">X</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2015</span> <span class="cit-article-title">Identification and phylogenetic analysis of late embryogenesis abundant proteins family in tomato (<em>Solanum lycopersicum</em>)</span>. <abbr class="cit-jnl-abbrev">Planta</abbr> <span class="cit-vol">241</span>, <span class="cit-fpage">757</span>–<span class="cit-lpage">772</span>. (<a href="http://dx.doi.org/10.1007/s00425-014-2215-y">doi:10.1007/s00425-014-2215-y</a>) </div> </div> </div> </li> <li> <div class="cit ref-cit ref-journal" id="cit-3.6.160123.40" data-doi="10.1016/S1369-5266(00)00080-7"> <div class="cit-metadata"> <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Ellis</span> <span class="cit-name-given-names">J</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Dodds</span> <span class="cit-name-given-names">P</span></span>, </li> <li><span class="cit-auth"><span class="cit-name-surname">Pryor</span> <span class="cit-name-given-names">T</span></span></li> </ol> <div class="cite_UNKNOWN">
. <span class="cit-pub-date">2000</span> <span class="cit-article-title">Structure, function and evolution of plant disease resistance genes</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Plant Biol.</abbr> <span class="cit-vol">3</span>, <span class="cit-fpage">278</span>–<span class="cit-lpage">284</span>. (<a href="http://dx.doi.org/10.1016/S1369-5266(00)00080-7">doi:10.1016/S1369-5266(00)00080-7</a>) </div> </div> </div> </li> </ol> </div><span class="highwire-journal-article-marker-end" /></div><span id="related-urls" /></div> </div> </div> </div> <div class="panel-separator" /> <div class="panel-pane pane-block pane-disqus-disqus-comments"> <div class="pane-content"> <div id="disqus_thread" /> </div> </div> </div> </div> </div> </div> </div> </div> </div> <div class="panel-separator" /> <div class="panel-separator" /> </div> </div> </div> </div> </div> </div> </div> </div> </div> </section></div>    <div class="region region-page-bottom" id="region-page-bottom"> <div class="region-inner region-page-bottom-inner"> </div> </div>  </body></html>