<html xmlns="http://www.w3.org/1999/xhtml"><head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /> <meta charset="UTF-8" /> <style>
         	    div {
         	        border: 1px solid red;
         	        margin: 2 2 2 2;
         	        }
         	    ul, ol {
         	        border: 1px solid green;
         	        margin: 2 2 2 2;
         	        }
         	    a {
         	        border: 1px solid purple;
         	        margin: 2 2 2 2;
         	        }
         	    p {
         	        background: #ffffdd;
         	        margin: 2 2 2 2;
         	        }
         	    span {
         	        background: #ddffff;
         	        margin: 2 2 2 2;
         	        }
         	    table {"target/pubstyle/rs/clean/277_1686_1309/fulltext.xhtml"
         	        border: 1px solid blue;
         	        background: #ddffdd;
         	        margin: 2 2 2 2;
         	        }
         	    tr {
         	        border: 2px solid yellow;
         	        margin: 2 2 2 2;
         	        }
         	    td {
         	        border: 2px solid cyan;
         	        margin: 2 2 2 2;
         	        }
         	    
      </style> <meta charset="utf-8" /> <meta http-equiv="X-UA-Compatible" content="IE=edge" /> <meta name="robots" content="NOARCHIVE" /> <meta name="robots" content="noodp" /> <meta name="robots" content="noydir" /> <meta name="citation_fulltext_world_readable" content="" /> </head> <body> <div id="header-area"> </div> <div style="clear:both;" /> <div class="topBorder" /> <div id="centerInner" class="centerInner svBigBox" aria-label="Article" role="main"> <div class="publisherLogo"> <span class="spElsevierPubIcon" /> </div> <div class="publicationHead"> <div class="title"><a title="Go to Molecular Phylogenetics and Evolution on ScienceDirect" href="/science/journal/10557903" querystr="?zone=centerPane" class="cLink"><span>Molecular Phylogenetics and Evolution</span></a></div> <p class="volIssue"><a class="S_C_volIss" title="Go to table of contents for this volume/issue" href="/science/journal/10557903/83/supp/C">Volume 83</a>, February 2015, Pages 56–71 </p> </div> <div class="module_topic_paths" /> <h1 class="svTitle" id="tm005">A tribal level phylogeny of Lake Tanganyika cichlid fishes based on a genomic multi-marker approach </h1> <ul class="authorGroup noCollab svAuthor " id="ag005"> <li><a href="#" class="authorName S_C_authorName svAuthor" id="authname_N70c80e90N737d3444" data-t="a" data-fn="Britta S." data-ln="Meyer" data-pos="1" data-tb="">Britta S. Meyer</a><a title="Affiliation: a" href="#af005" class="intra_ref auth_aff" id="baf005"><sup>a</sup></a><sup>, </sup><a title="Affiliation: b" href="#af010" class="intra_ref auth_aff" id="baf010"><sup>b</sup></a><sup>, </sup><sup>, </sup>, </li> <li><a href="#" class="authorName S_C_authorName svAuthor" id="authname_N70c80e90N737d3624" data-t="a" data-fn="Michael" data-ln="Matschiner" data-pos="2" data-tb="">Michael Matschiner</a><a title="Affiliation: a" href="#af005" class="intra_ref auth_aff" id="baf005"><sup>a</sup></a><sup>, </sup><a title="Affiliation: c" href="#af015" class="intra_ref auth_aff" id="baf015"><sup>c</sup></a>, </li> <li><a href="#" class="authorName S_C_authorName svAuthor" id="authname_N70c80e90N737d3750" data-t="a" data-fn="Walter" data-ln="Salzburger" data-pos="3" data-tb="">Walter Salzburger</a><a title="Affiliation: a" href="#af005" class="intra_ref auth_aff" id="baf005"><sup>a</sup></a><sup>, </sup><a title="Affiliation: c" href="#af015" class="intra_ref auth_aff" id="baf015"><sup>c</sup></a><sup>, </sup><sup>, </sup></li> </ul> <div class="endLine" /> <ul class="affiliation authAffil"> <li id="af005"><sup>a</sup> <span id="">Zoological Institute, University of Basel, Vesalgasse 1, 4051 Basel, Switzerland</span></li> <li id="af010"><sup>b</sup> <span id="">Evolutionary Ecology of Marine Fishes, GEOMAR Helmholtz Centre for Ocean Research Kiel, Düsternbrooker Weg 20, 24105 Kiel, Germany</span></li> <li id="af015"><sup>c</sup> <span id="">Centre for Ecological and Evolutionary Synthesis (CEES), Department of Biosciences, University of Oslo, Oslo, Norway</span></li> </ul> <dl class="articleDates"> <dd>
 Received 25 November 2013, Revised 5 October 2014, Accepted 8 October 2014, Available online 26 November 2014 </dd> </dl> <div class="moreInformation" /> <div id="showMoreButtons" /> <div id="ppvPlaceHolder" class="hidden" /> <dl class="extLinks"> <dd class="doiLink" /> <dd class="rightsLink" /> </dl> <div class="articleOAlabelForced" /> <div id="refersToAndreferredToBy"> <dl id="referredToBy" class="documentThread" /> </div> <hr id="abs_author-highlightsab0101" class="artHeader" /> <div class="abstract svAbstract     abstractHighlights    " data-etype="ab"> <h2 class="secHeading" id="author-highlightsab0101">Highlights</h2> <p id="sp0010" /> <dl class="listitem" id="list_l0005"> <dt class="label">
 • </dt> <dd> <p id="p0285">We provide a new phylogeny for Lake Tanganyika cichlids using 42 nuclear makers.</p> </dd> <dt class="label">
 • </dt> <dd> <p id="p0290">Data concatenation and a Bayesian concordance analysis lead to congruent results.</p> </dd> <dt class="label">
 • </dt> <dd> <p id="p0295">Gene tree discordance hints to past hybridization or incomplete lineage sorting.</p> </dd> <dt class="label">
 • </dt> <dd> <p id="p0300">The Lamprologini are the sister-group to the ‘H-lineage’.</p> </dd> <dt class="label">
 • </dt> <dd> <p id="p0305">The Eretmodini are nested within the ‘H-lineage’.</p> </dd> </dl> <p /> </div> <hr id="abs_authorab0151" class="artHeader" /> <div class="abstract svAbstract " data-etype="ab"> <h2 class="secHeading" id="authorab0151">Abstract</h2> <p id="sp0015">The species-flocks of cichlid fishes in the East African Great Lakes Victoria, Malawi and Tanganyika constitute the most diverse extant adaptive radiations in vertebrates. Lake Tanganyika, the oldest of the lakes, harbors the morphologically and genetically most diverse assemblage of cichlids and contains the highest number of endemic cichlid genera of all African lakes. Based on morphological grounds, the Tanganyikan cichlid species have been grouped into 12–16 distinct lineages, so-called tribes. While the monophyly of most of the tribes is well established, the phylogenetic relationships among the tribes remain largely elusive. Here, we present a new tribal level phylogenetic hypothesis for the cichlid fishes of Lake Tanganyika that is based on the so far largest set of nuclear markers and a total alignment length of close to 18 kb. Using next-generation amplicon sequencing with the 454 pyrosequencing technology, we compiled a dataset consisting of 42 nuclear loci in 45 East African cichlid species, which we subjected to maximum likelihood and Bayesian inference phylogenetic analyses. We analyzed the entire concatenated dataset and each marker individually, and performed a Bayesian concordance analysis and gene tree discordance tests. Overall, we find strong support for a position of the Oreochromini, Boulengerochromini, Bathybatini and Trematocarini outside of a clade combining the substrate spawning Lamprologini and the mouthbrooding tribes of the ‘H-lineage’, which are both strongly supported to be monophyletic. The Eretmodini are firmly placed within the ‘H-lineage’, as sister-group to the most species-rich tribe of cichlids, the Haplochromini. The phylogenetic relationships at the base of the ‘H-lineage’ received less support, which is likely due to high speciation rates in the early phase of the radiation. Discordance among gene trees and marker sets further suggests the occurrence of past hybridization and/or incomplete lineage sorting in the cichlid fishes of Lake Tanganyika. </p> </div> <hr id="absgraphicalab0051" class="artHeader" /> <div class="abstract svAbstract" data-etype="ab"> <h2 class="secHeading" id="section_gabs">Graphical abstract</h2> <div id="sp0005"> <p /> <div class="figTblUpiOuter svArticle" id="figure_f0035"> <div class="                     textboxdefault                 "> <dl class="figure" id="f0035" data-t="f"> <dt class="autoScroll" /> <dd id="labelCaptionf0035"> <div class="caption" /> </dd> </dl> </div> </div> <p /> </div> </div> <div id="SD_BA1P" class="sgfNoTitleBar sgfNoGadgetBorder svDoNotLink" /> <h2 id="kwd_1" class="svKeywords">Keywords</h2> <ul class="keyword" id="kg005"> <li id="k0005" class="svKeywords"><span id="">Adaptive radiation</span>; </li> <li id="k0010" class="svKeywords"><span id="">Cichlidae</span>; </li> <li id="k0015" class="svKeywords"><span id="">454 amplicon sequencing</span>; </li> <li id="k0020" class="svKeywords"><span id="">Hybridization</span>; </li> <li id="k0025" class="svKeywords"><span id="">Incomplete lineage sorting</span></li> </ul> <hr class="artHeader" /> <h2 class="svArticle" id="s0005">1. Introduction</h2> <p class="svArticle section" id="p0005">The species-flocks of cichlid fishes in the East African Great Lakes Victoria, Malawi and Tanganyika (LT) represent the most species-rich adaptive radiations known in vertebrates (see e.g. <a href="#b0170" id="bb0170" class="intra_ref">Kocher, 2004</a>, <a href="#b0310" id="bb0310" class="intra_ref">Salzburger, 2009</a> and <a href="#b0375" id="bb0375" class="intra_ref">Seehausen, 2006</a>). Several hundred of endemic cichlid species have evolved in each of these lakes in only the last few million to several thousand of years (see e.g. <a href="#b0100" id="bb0100" class="intra_ref">Genner et al., 2007</a>, <a href="#b0170" id="bb0170" class="intra_ref">Kocher, 2004</a>, <a href="#b0310" id="bb0310" class="intra_ref">Salzburger, 2009</a>, <a href="#b0315" id="bb0315" class="intra_ref">Salzburger and Meyer, 2004</a>, <a href="#b0405" id="bb0405" class="intra_ref">Snoeks, 2000</a>, <a href="#b0465" id="bb0465" class="intra_ref">Turner et al., 2001</a> and <a href="#b0470" id="bb0470" class="intra_ref">Verheyen et al., 2003</a>). Because of their taxonomic diversity, their ecological and morphological disparity and the high proportion of endemism, East African cichlid fishes are a prime model system in evolutionary biology (reviewed in: <a href="#b0170" id="bb0170" class="intra_ref">Kocher, 2004</a>, <a href="#b0310" id="bb0310" class="intra_ref">Salzburger, 2009</a> and <a href="#b0335" id="bb0335" class="intra_ref">Santos and Salzburger, 2012</a>). </p> <p class="svArticle section" id="p0010">With a maximum estimated age of 9–12 million years (my), LT is the oldest lake in Africa (<a href="#b0060" id="bb0060" class="intra_ref">Cohen et al., 1997</a> and <a href="#b9000" id="bb9000" class="intra_ref">Salzburger et al., 2014</a>) and contains the genetically, morphologically and ecologically most diverse group of cichlid fishes counting ca. 200 species in more than 50 genera (<a href="#b0160" id="bb0160" class="intra_ref">Koblmüller et al., 2008b</a>, <a href="#b0320" id="bb0320" class="intra_ref">Salzburger et al., 2002a</a> and <a href="#b0405" id="bb0405" class="intra_ref">Snoeks, 2000</a>). Based on morphological grounds, <span id="bb0285"><a href="#b0285" id="ancbb0285" class="intra_ref">Poll (1986)</a></span> grouped the LT cichlid species into 12 tribes (a taxonomic rank between subfamily and genus): Bathybatini, Cyprichromini, Ectodini, Eretmodini, Haplochromini, Lamprologini, Limnochromini, Perissodini, Tilapiini, Trematocarini, Tropheini, and Tylochromini. <span id="bb0450"><a href="#b0450" id="ancbb0450" class="intra_ref">Takahashi (2003)</a></span> revised Poll’s tribal assignment and suggested to (i) taking <em>Boulengerochromis microlepis</em> out of the Tilapiini into its own tribe, Boulengerochromini, leaving behind <em>Oreochromis tanganicae</em> as the only representative of the Tilapiini in LT; (ii) splitting the Limnochromini into Limnochromini <em>sensu stricto</em>, Benthochromini and Greenwoodochromini; (iii) establishing a separate tribe, Cyphotilapiini, for <em>Cyphotilapia frontosa</em> and <em>C. gibberosa</em>; (iv) moving ‘<em>Ctenochromis</em>’ <em>benthicola</em> into its own tribe; and (v) putting the species of the Trematocarini into the Bathybatini. Only some of these revisions are backed up by molecular data, such as the establishment of the new tribes Benthochromini, Boulengerochromini, and Cyphotilapiini ( <a href="#b0160" id="bb0160" class="intra_ref">Koblmüller et al., 2008b</a>, <a href="#b0265" id="bb0265" class="intra_ref">Muschick et al., 2012</a> and <a href="#b0320" id="bb0320" class="intra_ref">Salzburger et al., 2002a</a>). <em>Greenwoodochromis</em>, on the other hand, is clearly nested within the Limnochromini in molecular phylogenies ( <a href="#b0075" id="bb0075" class="intra_ref">Duftner et al., 2005</a>, <a href="#b0265" id="bb0265" class="intra_ref">Muschick et al., 2012</a> and <a href="#b0130" id="bb0130" class="intra_ref">Kirchberger et al., 2014</a>), and should hence remain within the Limnochromini; the Trematocarini consistently form a separate lineage outside the Bathybatini (see e.g. <a href="#b0145" id="bb0145" class="intra_ref">Koblmüller et al., 2005</a> and <a href="#b0265" id="bb0265" class="intra_ref">Muschick et al., 2012</a>) and should remain in their own tribe (note that <span id="bb0160"><a href="#b0160" id="ancbb0160" class="intra_ref">Koblmüller et al. (2008b)</a></span> suggested splitting the Bathybatini into Bathybatini <em>sensu stricto</em> and Hemibatini); and ‘<em>Ctenochromis</em>’ <em>benthicola</em> has recently been identified as member of the Cyphotilapiini ( <span id="bb0265"><a href="#b0265" id="ancbb0265" class="intra_ref">Muschick et al., 2012</a></span>). Finally, the Tropheini were consistently found to be nested within the Haplochromini ( <a href="#b0330" id="bb0330" class="intra_ref">Salzburger et al., 2005</a> and <a href="#b0320" id="bb0320" class="intra_ref">Salzburger et al., 2002a</a>; see also below) and should, hence, not be considered as separate tribe but as part of the Haplochromini. </p> <p class="svArticle section" id="p0015">Not all of the cichlid tribes occurring in LT are endemic to this lake, though, and four tribes show a distribution range that exceeds the LT basin by far. The Tylochromini have their center of divergence in West Africa (<span id="bb0420"><a href="#b0420" id="ancbb0420" class="intra_ref">Stiassny, 1990</a></span>), and the only LT species, <em>T. polylepis</em>, is likely to have invaded LT only recently ( <span id="bb0165"><a href="#b0165" id="ancbb0165" class="intra_ref">Koch et al., 2007</a></span>). The same might be true for <em>O. tanganicae</em>, the only native representative of the widely distributed Tilapiini in LT ( <span id="bb0135"><a href="#b0135" id="ancbb0135" class="intra_ref">Klett and Meyer, 2002</a></span>). Note that the Tilapiini were recently taxonomically revised and that the genus <em>Oreochromis</em> has been placed into a new tribe, namely the Oreochromini ( <span id="bb0080"><a href="#b0080" id="ancbb0080" class="intra_ref">Dunz and Schliewen, 2013</a></span>). The Lamprologini, the most species-rich tribe of cichlids in LT, contain a few species that have secondarily colonized the Congo and Malagarasi River systems ( <a href="#b0320" id="bb0320" class="intra_ref">Salzburger et al., 2002a</a>, <a href="#b0350" id="bb0350" class="intra_ref">Schelly et al., 2003</a>, <a href="#b0345" id="bb0345" class="intra_ref">Schelly and Stiassny, 2004</a> and <a href="#b0435" id="bb0435" class="intra_ref">Sturmbauer et al., 2010</a>). The Haplochromini (including the Tropheini) represent the most species-rich tribe of cichlids overall, and are distributed across large parts of Africa, where they have seeded various radiations including the ones of Lake Malawi and the Lake Victoria Region ( <a href="#b0155" id="bb0155" class="intra_ref">Koblmüller et al., 2008a</a>, <a href="#b0330" id="bb0330" class="intra_ref">Salzburger et al., 2005</a>, <a href="#b0365" id="bb0365" class="intra_ref">Schwarzer et al., 2012</a>, <a href="#b0470" id="bb0470" class="intra_ref">Verheyen et al., 2003</a> and <a href="#b0475" id="bb0475" class="intra_ref">Wagner et al., 2012</a>). The LT cichlid fishes thus show faunal affinities across a large geographical range to both older cichlid lineages such as the Tylochromini and Tilapiini/Oreochromini and younger ones such as the Haplochromini. </p> <p class="svArticle section" id="p0020">The phylogenetic relationships among East African cichlid tribes has been the subject of various studies over the past two decades, yet remain enigmatic (reviewed in: <span id="bb0160"><a href="#b0160" id="ancbb0160" class="intra_ref">Koblmüller et al., 2008b</a></span>). The first comprehensive phylogenetic study of LT’s cichlid fishes using molecular information dates back to the early 1990s, when <span id="bb0270"><a href="#b0270" id="ancbb0270" class="intra_ref">Nishida (1991)</a></span> used allozyme data to examine the relationships among tribes. He established the so-called ‘H-lineage’ consisting of the tribes Cyprichromini, Ectodini, Eretmodini, Haplochromini/Tropheini (which he already found to be monophyletic), Limnochromini, and Perissodini as sister-group to the Lamprologini; the Bathybatini, Trematocarini plus <em>Boulengerochromis microlepis</em>, <em>Oreochromis tanganicae</em>, and <em>Tylochromis polylepis</em> were placed outside of a clade formed by the ‘H-lineage’ and Lamprologini. Yet, the relative position of the ‘H-lineage’ tribes differed depending on the algorithms used (UPGMA and neighbour-joining; NJ) ( <span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>a). <span id="bb0425"><a href="#b0425" id="ancbb0425" class="intra_ref">Sturmbauer and Meyer (1993)</a></span> used two mitochondrial (mt) DNA markers (cytochrome <em>b</em> and control region) and suggested, based on phylogenetic analyses with NJ and maximum parsimony (MP), a sister-group relationship between the Cyprichromini and the Ectodini and between the Eretmodini and the Haplochromini ( <span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>b). <span id="bb0175"><a href="#b0175" id="ancbb0175" class="intra_ref">Kocher et al. (1995)</a></span> established the mitochondrial NADH dehydrogenase subunit 2 (ND2) gene as marker for phylogenetic analyses in cichlid fishes and provided the most inclusive phylogenetic hypothesis for LT cichlids so far. In their MP and NJ phylogenies, the Bathybatini, the Tylochromini, <em>B. microlepis</em> and <em>O. tanganicae</em> formed a clade, and the Eretmodini were placed outside the ‘H-lineage’, as sister-group to the Lamprologini ( <span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>c). The Cyprichromini were resolved as the sister-group to all remaining ‘H-lineage’ taxa (i.e. without the Eretmodini). Using three mitochondrial markers (control region, cytochrome <em>b</em>, ND2) and NJ, MP and maximum-likelihood (ML) phylogenetic analyses, <span id="bb0320"><a href="#b0320" id="ancbb0320" class="intra_ref">Salzburger et al. (2002a)</a></span> confirmed the position of <em>B. microlepis</em>, the Bathybatini and the Trematocarini outside all other tribes occurring in Lake Tanganyika, with the exception of the Tylochromini, and the Eretmodini were placed as sister-group to the Lamprologini and the remaining ‘H-lineage’ tribes ( <span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>d). Within the ‘H-lineage’, the Ectodini appeared as the sister to the remaining taxa. This study was also the first to establish phylogenetic affinities between the LT cichlid fishes and the riverine genus <em>Orthochromis</em> (not shown in <span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>d; see also <span id="bb0330"><a href="#b0330" id="ancbb0330" class="intra_ref">Salzburger et al., 2005</a></span>). <span id="bb0055"><a href="#b0055" id="ancbb0055" class="intra_ref">Clabaut et al. (2005)</a></span> combined sequences of the mitochondrial ND2 gene and the nuclear recombinase activating gene (<em>rag</em>) and applied ML and Bayesian inference (BI). They placed the Eretmodini as sister-group to the Lamprologini and established the ‘C-lineage’, i.e. the ‘H-lineage’ of <span id="bb0270"><a href="#b0270" id="ancbb0270" class="intra_ref">Nishida (1991)</a></span> but without the Eretmodini. Within this ‘C-lineage’, the Limnochromini plus <em>C. frontosa</em> appeared as the sister-group to the Perissodini, the Ectodini, the Cyprichromini and the Haplochromini ( <span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>e). <span id="bb0065"><a href="#b0065" id="ancbb0065" class="intra_ref">Day et al. (2008)</a></span> provided one of the most comprehensive datasets to date (cytochrome <em>b</em>, ND2) including 157 taxa. Their ML and BI phylogenies supported the existence of the ‘C-lineage’ by placing the Eretmodini as sister-group to the Lamprologini. In their analyses, a clade formed by the Ectodini and Cyprichromini was placed as the sister-group of the remaining ‘C-lineage’ taxa ( <span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>f). In the ML phylogeny of <span id="bb0265"><a href="#b0265" id="ancbb0265" class="intra_ref">Muschick et al. (2012)</a></span>, who used the mitochondrial ND2 gene and two nuclear markers (<em>ednrb1</em>, <em>phpt1</em>), the Eretmodini were placed as sister group to the Lamprologini and the ‘C-lineage’, within which the Limnochromini appeared outside of all other included taxa ( <span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>g). The study of <span id="bb0085"><a href="#b0085" id="ancbb0085" class="intra_ref">Friedman et al. (2013)</a></span>, which was based on ten nuclear makers and did not focus specifically on the species of LT but on a larger cichlid phylogeny, revealed a clade formed by the Lamprologini, the Perissodini plus the Cyprichromini, and the Cyphotilapiini plus the Limnochromini as sister group to the Ectodini, the Eretmodini and the Haplochromini ( <span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>h). </p> <div class="figTblUpiOuter svArticle" id="figure_f0005"> <dl class="figure" id="f0005" data-t="f"> <dt class="autoScroll" data-style="height:807px;width:612px;" /> <dd id="labelCaptionf0005"> <div class="caption">
 Fig. 1. <p id="sp0055">Previous hypotheses for the phylogenetic relationships among cichlid tribes in Lake Tanganyika. The figure depicts simplified cladograms based on the studies of (a) <span id="f0005bb0270"><a href="#b0270" id="ancf0005bb0270" class="intra_ref">Nishida (1991)</a></span>, (b) <span id="f0005bb0425"><a href="#b0425" id="ancf0005bb0425" class="intra_ref">Sturmbauer and Meyer (1993)</a></span>, (c) <span id="f0005bb0175"><a href="#b0175" id="ancf0005bb0175" class="intra_ref">Kocher et al. (1995)</a></span>, (d) <span id="f0005bb0320"><a href="#b0320" id="ancf0005bb0320" class="intra_ref">Salzburger et al. (2002a)</a></span>, (e) <span id="f0005bb0055"><a href="#b0055" id="ancf0005bb0055" class="intra_ref">Clabaut et al. (2005)</a></span>, (f) <span id="f0005bb0065"><a href="#b0065" id="ancf0005bb0065" class="intra_ref">Day et al. (2008)</a></span>, (g) <span id="f0005bb0265"><a href="#b0265" id="ancf0005bb0265" class="intra_ref">Muschick et al. (2012)</a></span>, and (h) <span id="f0005bb0085"><a href="#b0085" id="ancf0005bb0085" class="intra_ref">Friedman et al. (2013)</a></span>. The markers used in the respective study and the phylogenetic algorithms applied are indicated; the color code for cichlid tribes follows that of <span id="f0005bb0265"><a href="#b0265" id="ancf0005bb0265" class="intra_ref">Muschick et al. (2012)</a></span>. (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article.) </p> </div> </dd> </dl> </div> <p class="svArticle section" id="p0025">In summary, after more than 20 years of research, the composition of individual LT tribes has been well investigated, whereas the phylogenetic relationships among these cichlid tribes remain largely elusive. All studies performed so far revealed different results (<span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>), and the support values for many of the deeper nodes were consistently low. While there is consensus about the position of <em>T. polylepis</em>, <em>O. tanganicae</em>, the Bathybatini, Boulengerochromini and Trematocarini outside of the other tribes, the following main areas of uncertainty persist: (i) the relative position of the Bathybatini, Boulengerochromini and Trematocarini to each other; (ii) the placement of the Eretmodini, which were suggested as either being part of the ‘H-lineage’ and sister to the Haplochromini ( <a href="#b0085" id="bb0085" class="intra_ref">Friedman et al., 2013</a>, <a href="#b0270" id="bb0270" class="intra_ref">Nishida, 1991</a> and <a href="#b0425" id="bb0425" class="intra_ref">Sturmbauer and Meyer, 1993</a>), as sister-group to the Lamprologini ( <a href="#b0055" id="bb0055" class="intra_ref">Clabaut et al., 2005</a>, <a href="#b0065" id="bb0065" class="intra_ref">Day et al., 2008</a> and <a href="#b0175" id="bb0175" class="intra_ref">Kocher et al., 1995</a>), or as separate lineage outside the Lamprologini-‘C-lineage’ clade ( <a href="#b0265" id="bb0265" class="intra_ref">Muschick et al., 2012</a> and <a href="#b0320" id="bb0320" class="intra_ref">Salzburger et al., 2002a</a>); and (iii) the relative position of the ‘H-lineage’/’C-lineage’ taxa with respect to each other. </p> <p class="svArticle section" id="p0030">The apparent intricacy with resolving the phylogenetic relationships of the cichlid tribes in LT might have various reasons. First, the conflict between the various phylogenetic hypotheses might in part result from the different phylogenetic algorithms used (see above), although this would not apply to the more recent studies, all of which relied on ML and BI methods. Second, we might face the problem here that the previously used markers do not provide enough power of resolution for the question at hand. Alternatively, the inability to resolve some of the phylogenetic relationships of LT’s cichlid tribes might reflect biological reality in the context of an adaptive radiation, where speciation is not necessarily bifurcating and multiple lineages may evolve nearly contemporaneously from a common ancestor (‘soft polytomy’ versus ‘hard polytomy’ problem: <a href="#b0235" id="bb0235" class="intra_ref">Maddison, 1989</a>, <a href="#b0400" id="bb0400" class="intra_ref">Slowinski, 2001</a>, <a href="#b0430" id="bb0430" class="intra_ref">Sturmbauer et al., 2003</a>, <a href="#b0480" id="bb0480" class="intra_ref">Walsh et al., 1999</a> and <a href="#b0485" id="bb0485" class="intra_ref">Whitfield and Lockhart, 2007</a>). Conflicting topologies may also be the result of reticulate evolution due to (introgressive) hybridization, which is a commonly observed phenomenon in LT’s cichlid assemblage (e.g. <a href="#b0150" id="bb0150" class="intra_ref">Koblmüller et al., 2007</a> and <a href="#b0325" id="bb0325" class="intra_ref">Salzburger et al., 2002b</a>) and might have acted as trigger of cichlid adaptive radiations in the first place (<a href="#b0120" id="bb0120" class="intra_ref">Joyce et al., 2011</a> and <a href="#b0370" id="bb0370" class="intra_ref">Seehausen, 2004</a>). Finally, discordance between different sets of markers could reflect incomplete lineage sorting, which is expected to have a strong impact on phylogenetic inference in rapidly diversifying clades (<span id="bb0180"><a href="#b0180" id="ancbb0180" class="intra_ref">Kubatko and Degnan, 2007</a></span>) and has been demonstrated in LT cichlid fishes before (<span id="bb0455"><a href="#b0455" id="ancbb0455" class="intra_ref">Takahashi et al., 2001</a></span>). </p> <p class="svArticle section" id="p0035">With decreasing sequencing costs and increasing computational resources, single marker and mtDNA-based phylogenies are rapidly being replaced by phylogenies inferred from large-scale nuclear marker sets based on selected loci, transcriptomes, or even whole genomes (<span id="bb0240"><a href="#b0240" id="ancbb0240" class="intra_ref">McCormack et al., 2013</a></span>). This recent development enables comparisons between the phylogenetic histories of multiple sets of individual markers. Here, we analyze the phylogenetic history of cichlid fishes from LT on a tribal level, including representatives from the East African Lakes Victoria and Malawi. We sampled 45 species and 42 nuclear loci and thus assembled the largest DNA sequence dataset available for LT cichlid fishes to date. In order to account for potential hybridization and incomplete lineage sorting, we explore gene tree concordance in addition to concatenation as ways for species tree estimation. We further test the strength of our dataset using random resampling of different numbers of markers. </p> <h2 class="svArticle" id="s0010">2. Material and methods</h2> <h3 class="svArticle" id="s0015">2.1. Sample collection and DNA extraction</h3> <p class="svArticle section" id="p0040">Specimens for this study were collected between 2007 and 2011 at the Kafue River (Kafue National Park) and at LT in the Northern Province of the Republic of Zambia following the standard operating procedure described in <span id="bb0265"><a href="#b0265" id="ancbb0265" class="intra_ref">Muschick et al. (2012)</a></span>. Additional samples were obtained from aquaria stocks at the University of Basel and at EAWAG, Kastanienbaum, Switzerland. In total, we analyzed data for 45 specimens, each representing a different East African cichlid species. Our sampling comprised 34 cichlid species from LT covering all major cichlid lineages in this lake. In addition we included 11 further species of riverine clades and from Lakes Victoria and Malawi, to place the LT cichlid taxa into a larger phylogenetic context. A detailed list of specimens, their IDs and sample locations is provided in <span id="bs0115"><a href="#s0115" id="ancbs0115" class="intra_ref">Table S1</a></span>. Genomic DNA was extracted from ethanol preserved tissue of whole specimens (see <span id="bb0265"><a href="#b0265" id="ancbb0265" class="intra_ref">Muschick et al., 2012</a></span> for details). </p> <h3 class="svArticle" id="s0020">2.2. Marker selection, sequencing and quality control</h3> <p class="svArticle section" id="p0045">To infer the phylogenetic history of the cichlid fishes of LT on the basis of an informative set of nuclear (nc) DNA markers, we selected a set of 42 nuclear loci. Twenty-four primer pairs were taken from earlier studies (<a href="#b0250" id="bb0250" class="intra_ref">Meyer and Salzburger, 2012</a>, <a href="#b0265" id="bb0265" class="intra_ref">Muschick et al., 2012</a> and <a href="#b0495" id="bb0495" class="intra_ref">Won et al., 2005</a>) and 18 primer pairs were newly designed following the strategy described in <span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span>. In short, we selected genes with known functions and aimed for amplification products between 400 and 600 bp in length to enable the application of next-generation amplicon sequencing. Twenty-four of the markers were developed as exon-primed intron crossing (so-called EPIC) primers (<a href="#b0210" id="bb0210" class="intra_ref">Lessa, 1992</a> and <a href="#b0395" id="bb0395" class="intra_ref">Slade et al., 1993</a>). The markers for <em>enc1</em>, <em>ptr</em>, <em>tbr</em> and <em>snx33</em> were taken from <span id="bb0225"><a href="#b0225" id="ancbb0225" class="intra_ref">Li et al. (2007)</a></span>, but modified to meet our requirements. The same strategy was applied for <em>ednrb</em> ( <span id="bb0195"><a href="#b0195" id="ancbb0195" class="intra_ref">Lang et al., 2006</a></span>), <em>bmp4</em> ( <span id="bb0005"><a href="#b0005" id="ancbb0005" class="intra_ref">Albertson et al., 2003</a></span>), and the reverse primer of <em>s7</em> ( <span id="bb0045"><a href="#b0045" id="ancbb0045" class="intra_ref">Chow and Hazama, 1998</a></span>). The genome of the Nile Tilapia (<em>Oreochromis niloticus</em>) ( <span id="bb0030"><a href="#b0030" id="ancbb0030" class="intra_ref">Brawand et al., 2014</a></span>) was used to define exon–intron boundaries and UTRs. A detailed list of all primers, their base composition, the length of the amplification products, their source, the ENSEBML reference of the respective locus in Tilapia, the chromosomal position of the respective locus in the Medaka genome and the number of variable sites are provided in <a href="#t0005" id="bt0005" class="intra_ref">Table 1</a> and <a href="#t0010" id="bt0010" class="intra_ref">Table 2</a>. </p> <div class="figTblUpiOuter svArticle" id="table_t0005"> <dl id="t0005" data-t="t" class="table " data-label="Table 1"> <dd class="lblCap"> <div class="caption"><span class="label">Table 1. </span><p>List of the 42 markers used in this study. The marker name, the forward and reverse sequence of each primer, the Ensembl Gene-ID for the respective locus in Tilapia, the link to the Ensemble entry for Tilapia, the chromosomal position of each locus in Medaka and the reference for the primer sequences are provided. </p> </div> </dd> <dd class="table"> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> </div><table>  <thead> <tr> <th rowspan="1" colspan="1">Name (synonym)</th> <th rowspan="1" colspan="1">Forward primer [5′-3′]</th> <th rowspan="1" colspan="1">Reverse primer [5′-3′]</th> <th rowspan="1" colspan="1">Ensembl-Gene-ID</th> <th rowspan="1" colspan="1">Link to Ensembl</th> <th rowspan="1" colspan="1">Chr Medaka</th> <th rowspan="1" colspan="1">Reference</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1" class="alignTD">rag1</td> <td rowspan="1" colspan="1" class="alignTD">TCGGCGCTTTCGGTACGATGTG</td> <td rowspan="1" colspan="1" class="alignTD">TGCCCCTGAAGTGGAASSGA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000014593</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0030" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000014593">RAG1</span></td> <td rowspan="1" colspan="1" class="alignTD">6</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">b2m</td> <td rowspan="1" colspan="1" class="alignTD">GCCACGTGAGTRATTTCCACCCC</td> <td rowspan="1" colspan="1" class="alignTD">ACGCTAYACRGYGGACYCTGA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000014176</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0035" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000014176">B2M</span></td> <td rowspan="1" colspan="1" class="alignTD">23</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">gapdhs</td> <td rowspan="1" colspan="1" class="alignTD">CCCTGGCCAAAGTCATCCACGATA</td> <td rowspan="1" colspan="1" class="alignTD">CACCACTGACACATCGGCCACT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000007262</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0040" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000007262">GAPDHS</span></td> <td rowspan="1" colspan="1" class="alignTD">16</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">Ptchd4</td> <td rowspan="1" colspan="1" class="alignTD">GCGGGTAGTGAATGTGAGTGCG</td> <td rowspan="1" colspan="1" class="alignTD">ACCCAAGACACCCAGCTCCA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000006708</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0045" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000006708">PTCHD4</span></td> <td rowspan="1" colspan="1" class="alignTD">24</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">enc1</td> <td rowspan="1" colspan="1" class="alignTD">CRGTTCGCCTTGCGCTRTTGC</td> <td rowspan="1" colspan="1" class="alignTD">TGGGTGCCGCCTTTGACCAT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000020511</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0050" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000020511">ENC1</span></td> <td rowspan="1" colspan="1" class="alignTD">12</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">phpt1</td> <td rowspan="1" colspan="1" class="alignTD">AGCAGGGTTGACCTTCTCAA</td> <td rowspan="1" colspan="1" class="alignTD">TGGCTAAAATCCCCGATGTA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000002175</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0055" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000002175">novel gene</span></td> <td rowspan="1" colspan="1" class="alignTD">4</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0265"><a href="#b0265" id="ancbb0265" class="intra_ref">Muschick et al. (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">rps7</td> <td rowspan="1" colspan="1" class="alignTD">CGTGCCATTTTACTCTGGACTKGC</td> <td rowspan="1" colspan="1" class="alignTD">AACTCGTCYGGCTTCTCGCC</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000018698</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0060" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000018698">RPS7</span></td> <td rowspan="1" colspan="1" class="alignTD">24</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">tbr1</td> <td rowspan="1" colspan="1" class="alignTD">ATCGTGCCGGGTGCGAGATA</td> <td rowspan="1" colspan="1" class="alignTD">AGGACGGCGTCTCAATCCAGCT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000008933</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0065" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000008933">TBR1</span></td> <td rowspan="1" colspan="1" class="alignTD">21</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">aqp1a.1</td> <td rowspan="1" colspan="1" class="alignTD">ATCAACCCTGCTCGCTCCTTCG</td> <td rowspan="1" colspan="1" class="alignTD">TGCATCGTTGCCTCCGTTGACG</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000009446</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0070" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000009446">novel gene</span></td> <td rowspan="1" colspan="1" class="alignTD">17</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">hprt1</td> <td rowspan="1" colspan="1" class="alignTD">TCAGYGATGAGGAGCAGGGTTATG</td> <td rowspan="1" colspan="1" class="alignTD">CGACCGTCATTGGGATGGAGC</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000017584</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0075" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000017584">HPRT1</span></td> <td rowspan="1" colspan="1" class="alignTD">10</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">anxa4</td> <td rowspan="1" colspan="1" class="alignTD">TGGACGAGGCCCAGGCTATTCAAG</td> <td rowspan="1" colspan="1" class="alignTD">ACGTCTTCCAGGCAGCCAGACA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000003465</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0080" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000003465">ANXA4</span></td> <td rowspan="1" colspan="1" class="alignTD">12</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">pgk1</td> <td rowspan="1" colspan="1" class="alignTD">CGGTACCTCCCTGTATGACGAGGA</td> <td rowspan="1" colspan="1" class="alignTD">GCAGCCAGATTTGGTCACCTCGA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000017337</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0085" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000017337">PGK1</span></td> <td rowspan="1" colspan="1" class="alignTD">14</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">bmp4</td> <td rowspan="1" colspan="1" class="alignTD">GAGGACCCATGCCCATTCGTTT</td> <td rowspan="1" colspan="1" class="alignTD">GCCACTATCCAGTCATTCCAGCC</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000001366</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0090" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000001366">BMP4</span></td> <td rowspan="1" colspan="1" class="alignTD">22</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">bmp2</td> <td rowspan="1" colspan="1" class="alignTD">AGGCCCTGGCCAGCCTAAAA</td> <td rowspan="1" colspan="1" class="alignTD">TCCTGCGTCTGTGGGCATCCTT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000000958</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0095" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000000958">BMP2</span></td> <td rowspan="1" colspan="1" class="alignTD">24</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">TMO-4C4</td> <td rowspan="1" colspan="1" class="alignTD">TTATGCTGAGGTGTTTGGCCTAC</td> <td rowspan="1" colspan="1" class="alignTD">CCACAGCACCCTCCTCATAAAT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000017439</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0100" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000017439">novel gene</span></td> <td rowspan="1" colspan="1" class="alignTD">–</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">fgf6b</td> <td rowspan="1" colspan="1" class="alignTD">CGCAAAGGTGCCACTACAG</td> <td rowspan="1" colspan="1" class="alignTD">TCGCACTGCACGGATGCAAA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000000017</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0105" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000000017">FGF6 (2 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">23</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">runx2</td> <td rowspan="1" colspan="1" class="alignTD">CGGGGTTGGTGTTTGAGGGCAA</td> <td rowspan="1" colspan="1" class="alignTD">GCTGACATGGTGTCACTGTGCTGA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000001025</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0110" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000001025">RUNX2</span></td> <td rowspan="1" colspan="1" class="alignTD">24</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">furina</td> <td rowspan="1" colspan="1" class="alignTD">GCTGCATGGGGACAGACAGTCA</td> <td rowspan="1" colspan="1" class="alignTD">ATAGTCACTGGCACCCGCCACA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000005696</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0115" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000005696">FURIN (1 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">3</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">wnt7b</td> <td rowspan="1" colspan="1" class="alignTD">GCGTCTCGGGATCCTGTACCACTA</td> <td rowspan="1" colspan="1" class="alignTD">TGCAGGTAAACACCTCCGTCCT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000008839</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0120" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000008839">WNT7B</span></td> <td rowspan="1" colspan="1" class="alignTD">6</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">pax9</td> <td rowspan="1" colspan="1" class="alignTD">TCCCACGGCTGTGTCAGYAA</td> <td rowspan="1" colspan="1" class="alignTD">ACAGAGTGCGAGGAAGGCCA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000000990</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0125" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000000990">PAX9</span></td> <td rowspan="1" colspan="1" class="alignTD">–</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">sox10b</td> <td rowspan="1" colspan="1" class="alignTD">TSCRGGGTCTGGGAAACCTCAT</td> <td rowspan="1" colspan="1" class="alignTD">TGGTGGTCGGCGTATTCTGCAA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000008392</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0130" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000008392">SOX10 (1 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">8</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger, 2012</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">otx2</td> <td rowspan="1" colspan="1" class="alignTD">GCAGAACAAAGTGCGACCTGCC</td> <td rowspan="1" colspan="1" class="alignTD">GTCTGCTGTGGAGTTGAAGCCCA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000020156</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0135" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000020156">OTX2</span></td> <td rowspan="1" colspan="1" class="alignTD">22</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">otx1</td> <td rowspan="1" colspan="1" class="alignTD">TACACCTCCTGCTGTCTCCAGCAC</td> <td rowspan="1" colspan="1" class="alignTD">ATAGATGAGGCCGTCATGGGGC</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000001278</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0140" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000001278">OTX1 (1 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">15</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">dlx2a</td> <td rowspan="1" colspan="1" class="alignTD">ATCGCCAACTCCCGCAGACA</td> <td rowspan="1" colspan="1" class="alignTD">TCCGTTGAAGYGCAGCCAGT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000008722</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0145" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000008722">DLX2</span></td> <td rowspan="1" colspan="1" class="alignTD">21</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">dlx4b</td> <td rowspan="1" colspan="1" class="alignTD">GCGTGGATTTCTTCCAGGCTGTC</td> <td rowspan="1" colspan="1" class="alignTD">CTGTGTGCTCTAATCTGCTGTGGG</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000019896</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0150" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000019896">DLX4 (1 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">19</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">barx1</td> <td rowspan="1" colspan="1" class="alignTD">TCTCGCAGAGTCTCTCGGTCTG</td> <td rowspan="1" colspan="1" class="alignTD">TCGCTGCTGGGGATGGAGTT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000003234</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0155" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000003234">BARX1</span></td> <td rowspan="1" colspan="1" class="alignTD">–</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">ednrb1a</td> <td rowspan="1" colspan="1" class="alignTD">CGTTGGCCTGCACTGCCATT</td> <td rowspan="1" colspan="1" class="alignTD">AGGCAGCCAGCACAGAGCAAA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000018701</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0160" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000018701">EDNRB (1 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">17</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">mc1r</td> <td rowspan="1" colspan="1" class="alignTD">GACCACGGCCTCCTGGATGT</td> <td rowspan="1" colspan="1" class="alignTD">GTTGCAGAAGGGGCTGGTGG</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000021393</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0165" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000021393">MC1R</span></td> <td rowspan="1" colspan="1" class="alignTD">3</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">skia</td> <td rowspan="1" colspan="1" class="alignTD">CGACCAGCTGGAGATCCT</td> <td rowspan="1" colspan="1" class="alignTD">TCCTCTTGTACTTGTTGGCG</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000017935</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0170" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000017935">SKI (1 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">7</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">kita</td> <td rowspan="1" colspan="1" class="alignTD">CAGAGTACTGCTGTTTCGGMGAT</td> <td rowspan="1" colspan="1" class="alignTD">GGCTAAGAACTCCATGCCTTTGGC</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000002981</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0175" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000002981">KIT (1 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">4</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">mitfa</td> <td rowspan="1" colspan="1" class="alignTD">CCTGGCATGAAGCARGTACTGGAC</td> <td rowspan="1" colspan="1" class="alignTD">TTGCYAGAGCACGAACTTCRGC</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000020270</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0180" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000020270">MITF (2 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">5</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">tyr</td> <td rowspan="1" colspan="1" class="alignTD">TGGGTGGACGCAACTCCCTT</td> <td rowspan="1" colspan="1" class="alignTD">TGGCAAATCGGTCCATGGGT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIT00000006471</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0185" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000005144">TYR (1 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">13</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">hagoromo (fbxw4)</td> <td rowspan="1" colspan="1" class="alignTD">AAACTGGTACARYGGGVTCTGC</td> <td rowspan="1" colspan="1" class="alignTD">AGCGRCAGACGTCACCCTTGT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000013182</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0190" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000013182">HAGOROMO</span></td> <td rowspan="1" colspan="1" class="alignTD">15</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">slc45a2 (aim)</td> <td rowspan="1" colspan="1" class="alignTD">GAGCTATGGACTGGGGTCAC</td> <td rowspan="1" colspan="1" class="alignTD">TGGCTGTTTGACACTTGAGG</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000007610</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0195" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000007610">SLC45A2</span></td> <td rowspan="1" colspan="1" class="alignTD">12</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0495"><a href="#b0495" id="ancbb0495" class="intra_ref">Won et al. (2005)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">rh1</td> <td rowspan="1" colspan="1" class="alignTD">TCGCCTTGGCTGCAATCTGG</td> <td rowspan="1" colspan="1" class="alignTD">ACCATGCGGGTGACTTCCCT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000021142</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0200" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000021142">RH1</span></td> <td rowspan="1" colspan="1" class="alignTD">7</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">opn1mw (lws)</td> <td rowspan="1" colspan="1" class="alignTD">ATTGCTGCTCTTTGGTCCCTGACA</td> <td rowspan="1" colspan="1" class="alignTD">AGCCAGAGGGTGGAAGGCAT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000020292</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0205" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000020292">OPN1MW</span></td> <td rowspan="1" colspan="1" class="alignTD">5</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">opn1sw (sws)</td> <td rowspan="1" colspan="1" class="alignTD">TGGGTCACACGCTGTGTGCT</td> <td rowspan="1" colspan="1" class="alignTD">CAGCAGCTGGGAGTAGCAGAARA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000007620</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0210" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000007620">OPN1SW</span></td> <td rowspan="1" colspan="1" class="alignTD">scaffold1021</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">ccng1</td> <td rowspan="1" colspan="1" class="alignTD">CTGCTTGCCCTGGCTCTCCT</td> <td rowspan="1" colspan="1" class="alignTD">AGCTGACTCAGGTATGGTCGGA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000012912</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0215" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000012912">CCNG1</span></td> <td rowspan="1" colspan="1" class="alignTD">10</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">snx33</td> <td rowspan="1" colspan="1" class="alignTD">TGGCTGTACAACCGCCTGCT</td> <td rowspan="1" colspan="1" class="alignTD">CCAAYRTGAATGCSTGGCTGA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000012857</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0220" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000012857">SNX33</span></td> <td rowspan="1" colspan="1" class="alignTD">6</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">rpl13a</td> <td rowspan="1" colspan="1" class="alignTD">ACCTGGCTTTCCTGCGCAAGA</td> <td rowspan="1" colspan="1" class="alignTD">TTGCGAGAGGGCTTCAGACGCA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000003560</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0225" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000003560">RPL13A</span></td> <td rowspan="1" colspan="1" class="alignTD">22</td> <td rowspan="1" colspan="1" class="alignTD">This study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">edar</td> <td rowspan="1" colspan="1" class="alignTD">TGAGCAGCTGTTGAGCCGCA</td> <td rowspan="1" colspan="1" class="alignTD">CRCATKGCARGYYCTGGCATACA</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000004260</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0230" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000004260">EDAR</span></td> <td rowspan="1" colspan="1" class="alignTD">21</td> <td rowspan="1" colspan="1" class="alignTD">this study</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">csf1ra</td> <td rowspan="1" colspan="1" class="alignTD">AAGCACAGATGGGACACGCC</td> <td rowspan="1" colspan="1" class="alignTD">TGTACTGGCCCTGCTCCTGT</td> <td rowspan="1" colspan="1" class="alignTD">ENSONIG00000013065</td> <td rowspan="1" colspan="1" class="alignTD"><span id="ir0235" class="interref" data-locatortype="url" data-locatorkey="http://www.ensembl.org/Oreochromis_niloticus/Gene/Summary?g=ENSONIG00000013065">CSF1R (1 of 2)</span></td> <td rowspan="1" colspan="1" class="alignTD">10</td> <td rowspan="1" colspan="1" class="alignTD"><span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger (2012)</a></span></td> </tr> </tbody> </table> </dd> <dd class="fullsizeTable"><a data-url="/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S1055790314003601&amp;_image=t0005&amp;_cid=272322&amp;_explode=defaultEXP_LIST&amp;_idxType=defaultREF_WORK_INDEX_TYPE&amp;_alpha=defaultALPHA&amp;_ba=&amp;_rdoc=1&amp;_fmt=FULL&amp;_issn=10557903&amp;_pii=S1055790314003601&amp;_isTablePopup=Y&amp;md5=df30bed199ea7c4b4529bfd4f6b22e2a" class="tableAnchor" title="Full-size table - Opens new window" href="#">Full-size table</a></dd> </dl> </div> <div class="figTblUpiOuter svArticle" id="table_t0010"> <dl id="t0010" data-t="t" class="table " data-label="Table 2"> <dd class="lblCap"> <div class="caption"><span class="label">Table 2. </span><p>Characterization of the 42 loci used in this study. The marker name, the alignment length of each marker, the sequenced gene regions, the number of variable (<em>V</em>) and parsimony informative (PI) sites in the ingroup taxa, the mean number of differences (genetic distance) and the <em>p</em>-distance in the ingroup taxa, and the assignment to one of six subsets according to the CONCATERILLAR analysis are specified for each marker. </p> </div> </dd> <dd class="table"> <div class="colgroup_UNKNOWN"> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> <div class="col_UNKNOWN" /> </div><table>  <thead> <tr> <th rowspan="1" colspan="1">Name (synonym)</th> <th rowspan="1" colspan="1">Alignment lengths</th> <th rowspan="1" colspan="1">Gene regions</th> <th rowspan="1" colspan="1">V sites ingroup</th> <th rowspan="1" colspan="1">PI sites ingroup</th> <th rowspan="1" colspan="1">Genetic distance</th> <th rowspan="1" colspan="1"><em>p</em>-distance </th> <th rowspan="1" colspan="1">Subset</th> </tr> </thead> <tbody> <tr> <td rowspan="1" colspan="1" class="alignTD">rag1</td> <td rowspan="1" colspan="1" class="alignTD">418</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">49</td> <td rowspan="1" colspan="1" class="alignTDchar">21</td> <td rowspan="1" colspan="1" class="alignTDchar">5.10</td> <td rowspan="1" colspan="1" class="alignTDchar">0.012</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">b2m</td> <td rowspan="1" colspan="1" class="alignTD">478</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron, UTR</td> <td rowspan="1" colspan="1" class="alignTDchar">93</td> <td rowspan="1" colspan="1" class="alignTDchar">50</td> <td rowspan="1" colspan="1" class="alignTDchar">12.88</td> <td rowspan="1" colspan="1" class="alignTDchar">0.031</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">gapdhs</td> <td rowspan="1" colspan="1" class="alignTD">458</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">57</td> <td rowspan="1" colspan="1" class="alignTDchar">15</td> <td rowspan="1" colspan="1" class="alignTDchar">4.35</td> <td rowspan="1" colspan="1" class="alignTDchar">0.01</td> <td rowspan="1" colspan="1" class="alignTD">4</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">Ptchd4</td> <td rowspan="1" colspan="1" class="alignTD">394</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">32</td> <td rowspan="1" colspan="1" class="alignTDchar">11</td> <td rowspan="1" colspan="1" class="alignTDchar">3.59</td> <td rowspan="1" colspan="1" class="alignTDchar">0.009</td> <td rowspan="1" colspan="1" class="alignTD">4</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">enc1</td> <td rowspan="1" colspan="1" class="alignTD">376</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">21</td> <td rowspan="1" colspan="1" class="alignTDchar">7</td> <td rowspan="1" colspan="1" class="alignTDchar">2.95</td> <td rowspan="1" colspan="1" class="alignTDchar">0.008</td> <td rowspan="1" colspan="1" class="alignTD">5</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">phpt1</td> <td rowspan="1" colspan="1" class="alignTD">459</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">67</td> <td rowspan="1" colspan="1" class="alignTDchar">31</td> <td rowspan="1" colspan="1" class="alignTDchar">7.14</td> <td rowspan="1" colspan="1" class="alignTDchar">0.017</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">rps7</td> <td rowspan="1" colspan="1" class="alignTD">470</td> <td rowspan="1" colspan="1" class="alignTD">UTR</td> <td rowspan="1" colspan="1" class="alignTDchar">77</td> <td rowspan="1" colspan="1" class="alignTDchar">31</td> <td rowspan="1" colspan="1" class="alignTDchar">9.24</td> <td rowspan="1" colspan="1" class="alignTDchar">0.021</td> <td rowspan="1" colspan="1" class="alignTD">4</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">tbr1</td> <td rowspan="1" colspan="1" class="alignTD">466</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">13</td> <td rowspan="1" colspan="1" class="alignTDchar">6</td> <td rowspan="1" colspan="1" class="alignTDchar">1.58</td> <td rowspan="1" colspan="1" class="alignTDchar">0.003</td> <td rowspan="1" colspan="1" class="alignTD">5</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">aqp1a.1</td> <td rowspan="1" colspan="1" class="alignTD">440</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">62</td> <td rowspan="1" colspan="1" class="alignTDchar">24</td> <td rowspan="1" colspan="1" class="alignTDchar">5.69</td> <td rowspan="1" colspan="1" class="alignTDchar">0.014</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">hprt1</td> <td rowspan="1" colspan="1" class="alignTD">402</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">45</td> <td rowspan="1" colspan="1" class="alignTDchar">14</td> <td rowspan="1" colspan="1" class="alignTDchar">5.12</td> <td rowspan="1" colspan="1" class="alignTDchar">0.014</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">anxa4</td> <td rowspan="1" colspan="1" class="alignTD">642</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">56</td> <td rowspan="1" colspan="1" class="alignTDchar">20</td> <td rowspan="1" colspan="1" class="alignTDchar">6.31</td> <td rowspan="1" colspan="1" class="alignTDchar">0.014</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">pgk1</td> <td rowspan="1" colspan="1" class="alignTD">377</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">40</td> <td rowspan="1" colspan="1" class="alignTDchar">16</td> <td rowspan="1" colspan="1" class="alignTDchar">3.55</td> <td rowspan="1" colspan="1" class="alignTDchar">0.01</td> <td rowspan="1" colspan="1" class="alignTD">3</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">bmp4</td> <td rowspan="1" colspan="1" class="alignTD">456</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">47</td> <td rowspan="1" colspan="1" class="alignTDchar">16</td> <td rowspan="1" colspan="1" class="alignTDchar">4.37</td> <td rowspan="1" colspan="1" class="alignTDchar">0.011</td> <td rowspan="1" colspan="1" class="alignTD">4</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">bmp2</td> <td rowspan="1" colspan="1" class="alignTD">372</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">26</td> <td rowspan="1" colspan="1" class="alignTDchar">8</td> <td rowspan="1" colspan="1" class="alignTDchar">1.78</td> <td rowspan="1" colspan="1" class="alignTDchar">0.005</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">TMO-4C4</td> <td rowspan="1" colspan="1" class="alignTD">428</td> <td rowspan="1" colspan="1" class="alignTD">Intron</td> <td rowspan="1" colspan="1" class="alignTDchar">54</td> <td rowspan="1" colspan="1" class="alignTDchar">32</td> <td rowspan="1" colspan="1" class="alignTDchar">8.02</td> <td rowspan="1" colspan="1" class="alignTDchar">0.019</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">fgf6b</td> <td rowspan="1" colspan="1" class="alignTD">471</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">29</td> <td rowspan="1" colspan="1" class="alignTDchar">7</td> <td rowspan="1" colspan="1" class="alignTDchar">2.64</td> <td rowspan="1" colspan="1" class="alignTDchar">0.006</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">runx2</td> <td rowspan="1" colspan="1" class="alignTD">360</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron, UTR</td> <td rowspan="1" colspan="1" class="alignTDchar">16</td> <td rowspan="1" colspan="1" class="alignTDchar">5</td> <td rowspan="1" colspan="1" class="alignTDchar">2.06</td> <td rowspan="1" colspan="1" class="alignTDchar">0.006</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">furina</td> <td rowspan="1" colspan="1" class="alignTD">311</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">34</td> <td rowspan="1" colspan="1" class="alignTDchar">8</td> <td rowspan="1" colspan="1" class="alignTDchar">2.88</td> <td rowspan="1" colspan="1" class="alignTDchar">0.009</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">wnt7b</td> <td rowspan="1" colspan="1" class="alignTD">389</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">16</td> <td rowspan="1" colspan="1" class="alignTDchar">4</td> <td rowspan="1" colspan="1" class="alignTDchar">1.41</td> <td rowspan="1" colspan="1" class="alignTDchar">0.004</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">pax9</td> <td rowspan="1" colspan="1" class="alignTD">394</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">22</td> <td rowspan="1" colspan="1" class="alignTDchar">7</td> <td rowspan="1" colspan="1" class="alignTDchar">2.20</td> <td rowspan="1" colspan="1" class="alignTDchar">0.006</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">sox10b</td> <td rowspan="1" colspan="1" class="alignTD">378</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">40</td> <td rowspan="1" colspan="1" class="alignTDchar">15</td> <td rowspan="1" colspan="1" class="alignTDchar">4.43</td> <td rowspan="1" colspan="1" class="alignTDchar">0.012</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">otx2</td> <td rowspan="1" colspan="1" class="alignTD">412</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">19</td> <td rowspan="1" colspan="1" class="alignTDchar">7</td> <td rowspan="1" colspan="1" class="alignTDchar">1.89</td> <td rowspan="1" colspan="1" class="alignTDchar">0.005</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">otx1</td> <td rowspan="1" colspan="1" class="alignTD">356</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">15</td> <td rowspan="1" colspan="1" class="alignTDchar">9</td> <td rowspan="1" colspan="1" class="alignTDchar">1.86</td> <td rowspan="1" colspan="1" class="alignTDchar">0.005</td> <td rowspan="1" colspan="1" class="alignTD">5</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">dlx2a</td> <td rowspan="1" colspan="1" class="alignTD">497</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">83</td> <td rowspan="1" colspan="1" class="alignTDchar">27</td> <td rowspan="1" colspan="1" class="alignTDchar">6.94</td> <td rowspan="1" colspan="1" class="alignTDchar">0.015</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">dlx4b</td> <td rowspan="1" colspan="1" class="alignTD">356</td> <td rowspan="1" colspan="1" class="alignTD">UTR, exon</td> <td rowspan="1" colspan="1" class="alignTDchar">29</td> <td rowspan="1" colspan="1" class="alignTDchar">7</td> <td rowspan="1" colspan="1" class="alignTDchar">2.43</td> <td rowspan="1" colspan="1" class="alignTDchar">0.007</td> <td rowspan="1" colspan="1" class="alignTD">4</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">barx1</td> <td rowspan="1" colspan="1" class="alignTD">220</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">30</td> <td rowspan="1" colspan="1" class="alignTDchar">11</td> <td rowspan="1" colspan="1" class="alignTDchar">3.47</td> <td rowspan="1" colspan="1" class="alignTDchar">0.019</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">ednrb1a</td> <td rowspan="1" colspan="1" class="alignTD">438</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">59</td> <td rowspan="1" colspan="1" class="alignTDchar">28</td> <td rowspan="1" colspan="1" class="alignTDchar">6.82</td> <td rowspan="1" colspan="1" class="alignTDchar">0.016</td> <td rowspan="1" colspan="1" class="alignTD">6</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">mc1r</td> <td rowspan="1" colspan="1" class="alignTD">426</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">30</td> <td rowspan="1" colspan="1" class="alignTDchar">9</td> <td rowspan="1" colspan="1" class="alignTDchar">2.71</td> <td rowspan="1" colspan="1" class="alignTDchar">0.007</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">skia</td> <td rowspan="1" colspan="1" class="alignTD">453</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">38</td> <td rowspan="1" colspan="1" class="alignTDchar">11</td> <td rowspan="1" colspan="1" class="alignTDchar">2.67</td> <td rowspan="1" colspan="1" class="alignTDchar">0.006</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">kita</td> <td rowspan="1" colspan="1" class="alignTD">431</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">45</td> <td rowspan="1" colspan="1" class="alignTDchar">20</td> <td rowspan="1" colspan="1" class="alignTDchar">4.93</td> <td rowspan="1" colspan="1" class="alignTDchar">0.012</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">mitfa</td> <td rowspan="1" colspan="1" class="alignTD">434</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">57</td> <td rowspan="1" colspan="1" class="alignTDchar">21</td> <td rowspan="1" colspan="1" class="alignTDchar">6.41</td> <td rowspan="1" colspan="1" class="alignTDchar">0.016</td> <td rowspan="1" colspan="1" class="alignTD">6</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">tyr</td> <td rowspan="1" colspan="1" class="alignTD">525</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">72</td> <td rowspan="1" colspan="1" class="alignTDchar">26</td> <td rowspan="1" colspan="1" class="alignTDchar">8.47</td> <td rowspan="1" colspan="1" class="alignTDchar">0.019</td> <td rowspan="1" colspan="1" class="alignTD">3</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">hagoromo (fbxw4)</td> <td rowspan="1" colspan="1" class="alignTD">493</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">110</td> <td rowspan="1" colspan="1" class="alignTDchar">59</td> <td rowspan="1" colspan="1" class="alignTDchar">16.01</td> <td rowspan="1" colspan="1" class="alignTDchar">0.043</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">slc45a2 (aim)</td> <td rowspan="1" colspan="1" class="alignTD">286</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">38</td> <td rowspan="1" colspan="1" class="alignTDchar">16</td> <td rowspan="1" colspan="1" class="alignTDchar">4.55</td> <td rowspan="1" colspan="1" class="alignTDchar">0.016</td> <td rowspan="1" colspan="1" class="alignTD">3</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">rh1</td> <td rowspan="1" colspan="1" class="alignTD">404</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">43</td> <td rowspan="1" colspan="1" class="alignTDchar">32</td> <td rowspan="1" colspan="1" class="alignTDchar">9.59</td> <td rowspan="1" colspan="1" class="alignTDchar">0.024</td> <td rowspan="1" colspan="1" class="alignTD">6</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">opn1mw (lws)</td> <td rowspan="1" colspan="1" class="alignTD">420</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">53</td> <td rowspan="1" colspan="1" class="alignTDchar">22</td> <td rowspan="1" colspan="1" class="alignTDchar">6.65</td> <td rowspan="1" colspan="1" class="alignTDchar">0.017</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">opn1sw (sws)</td> <td rowspan="1" colspan="1" class="alignTD">450</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">80</td> <td rowspan="1" colspan="1" class="alignTDchar">36</td> <td rowspan="1" colspan="1" class="alignTDchar">10.01</td> <td rowspan="1" colspan="1" class="alignTDchar">0.024</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">ccng1</td> <td rowspan="1" colspan="1" class="alignTD">460</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">69</td> <td rowspan="1" colspan="1" class="alignTDchar">20</td> <td rowspan="1" colspan="1" class="alignTDchar">6.55</td> <td rowspan="1" colspan="1" class="alignTDchar">0.017</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">snx33</td> <td rowspan="1" colspan="1" class="alignTD">437</td> <td rowspan="1" colspan="1" class="alignTD">Exon</td> <td rowspan="1" colspan="1" class="alignTDchar">43</td> <td rowspan="1" colspan="1" class="alignTDchar">19</td> <td rowspan="1" colspan="1" class="alignTDchar">5.10</td> <td rowspan="1" colspan="1" class="alignTDchar">0.012</td> <td rowspan="1" colspan="1" class="alignTD">1</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">rpl13a</td> <td rowspan="1" colspan="1" class="alignTD">370</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">28</td> <td rowspan="1" colspan="1" class="alignTDchar">9</td> <td rowspan="1" colspan="1" class="alignTDchar">3.00</td> <td rowspan="1" colspan="1" class="alignTDchar">0.013</td> <td rowspan="1" colspan="1" class="alignTD">4</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">edar</td> <td rowspan="1" colspan="1" class="alignTD">372</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">41</td> <td rowspan="1" colspan="1" class="alignTDchar">13</td> <td rowspan="1" colspan="1" class="alignTDchar">3.29</td> <td rowspan="1" colspan="1" class="alignTDchar">0.009</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> <tr> <td rowspan="1" colspan="1" class="alignTD">csf1ra</td> <td rowspan="1" colspan="1" class="alignTD">366</td> <td rowspan="1" colspan="1" class="alignTD">Exon, intron</td> <td rowspan="1" colspan="1" class="alignTDchar">54</td> <td rowspan="1" colspan="1" class="alignTDchar">19</td> <td rowspan="1" colspan="1" class="alignTDchar">5.29</td> <td rowspan="1" colspan="1" class="alignTDchar">0.015</td> <td rowspan="1" colspan="1" class="alignTD">2</td> </tr> </tbody> </table> </dd> <dd class="fullsizeTable"><a data-url="/science?_ob=MiamiCaptionURL&amp;_method=retrieve&amp;_eid=1-s2.0-S1055790314003601&amp;_image=t0010&amp;_cid=272322&amp;_explode=defaultEXP_LIST&amp;_idxType=defaultREF_WORK_INDEX_TYPE&amp;_alpha=defaultALPHA&amp;_ba=&amp;_rdoc=1&amp;_fmt=FULL&amp;_issn=10557903&amp;_pii=S1055790314003601&amp;_isTablePopup=Y&amp;md5=4737877d858e4d513e4a2f4b976308e5" class="tableAnchor" title="Full-size table - Opens new window" href="#">Full-size table</a></dd> </dl> </div> <p class="svArticle section" id="p0050">The 42 nuclear markers were PCR amplified in several separate multiplex reactions in a final volume of 25 μL on a Veriti or 2720 thermal cycler (both Applied Biosystems, Rotkreuz, Switzerland). All PCR reactions contained the Multiplex PCR Kit (QIAGEN, Hombrechtikon, Switzerland) and a primer mix including eight to ten barcoded primer pairs (0.1 μM of each primer), water, and template DNA (5–20 ng/μL). We used barcoded fusion primers synthesized by Microsynth (Balgach, Switzerland). The PCR conditions were standardized for all reactions with an initial heat activation phase of 95 °C for 15 min, followed by 35 amplification cycles with denaturation steps at 94 °C for 30 s, annealing steps at 60–62 °C for 90 s and extension steps at 72 °C for 90 s; reactions were completed by a final extension phase at 72 °C for 10 min. </p> <p class="svArticle section" id="p0055">To remove small fragments, residual primers and primer-dimers, we applied the Agencourt AMPure XP magnetic bead system following the manufacturer’s protocol (Beckman Coulter, Nyon, Switzerland) and using a bead/DNA ratio of 1:1. Purification results were inspected with a 2100 Bioanalyzer (Agilent, Basel, Switzerland) using the DNA 1000 Kit. The amplification products of five individual PCR reactions with different primer combinations were then pooled (on the basis of the concentration measurements with the Bioanalyzer) to obtain the final libraries containing all 42 markers of one individual. In a second pooling step, 16 barcoded individuals were pooled for one 1/16th run on a 454 PicoTiterPlate. The subsequent library handling and sequencing was conducted by Microsynth (Balgach, Switzerland) with the GS FLX system (454 Sequencing, Roche). Sequencing was unidirectional starting at the forward primer, which also contained the barcodes. </p> <p class="svArticle section" id="p0060">Individual sequences (in both fasta and fastq format) were separated and extracted with Roche’s sffinfo tool (described in 454 Sequencing System Software Manual Version 2.6). Quality control was conducted with the software PRINSEQ (v0.20.3) (<span id="bb0355"><a href="#b0355" id="ancbb0355" class="intra_ref">Schmieder and Edwards, 2011</a></span>). We excluded individual reads that were shorter than 150 bp, that had an average Phred quality score below 15, or that contained more than 1% unidentified bases coded as "N". In a second step, we filtered out exact duplicates. The assembly to reference sequences from the <em>A. burtoni</em> genome ( <span id="bb0030"><a href="#b0030" id="ancbb0030" class="intra_ref">Brawand et al., 2014</a></span>) was performed with the software bwa and the BWA-SW algorithm (the Burrows-Wheeler Aligner’s Smith-Waterman Alignment) ( <span id="bb0220"><a href="#b0220" id="ancbb0220" class="intra_ref">Li and Durbin, 2010</a></span>). The resulting SAM files were imported into Geneious (v6.1.6–7.0.3, Biomatters Ltd, Auckland, New Zealand; available from <span id="ir0005" class="interref" data-locatortype="url" data-locatorkey="http://www.geneious.com">http://www.geneious.com</span>), visually inspected, if necessary reassembled, and further trimmed (we allowed a 0.05 error probability limit and a maximum of 10 low quality bases at the 3′ end). The final consensus sequences for each individual and marker were constructed with a 50% threshold, where bases were called "N" if the Phred score was below 20. Sequence data has been deposited on GenBank under the accession numbers KP129679-KP131427 (see <span id="bs0115"><a href="#s0115" id="ancbs0115" class="intra_ref">Table S2</a></span> for details) and KM263618-KM263752 ( <span id="bb0340"><a href="#b0340" id="ancbb0340" class="intra_ref">Santos et al., 2014</a></span>). </p> <h3 class="svArticle" id="s0025">2.3. Alignment and sequence characterization</h3> <p class="svArticle section" id="p0065">Sequences for each locus were aligned with the software MAFFT (v7.017) (<span id="bb0125"><a href="#b0125" id="ancbb0125" class="intra_ref">Katoh and Standley, 2013</a></span>), using the "—auto" option. Resulting alignments were visually inspected and manually improved when obvious sequencing artefacts (e.g. homopolymers) were observed or homology appeared questionable. </p> <p class="svArticle section" id="p0070">Overall mean distance for each locus was calculated with the software MEGA (v5.2.1) (<span id="bb0460"><a href="#b0460" id="ancbb0460" class="intra_ref">Tamura et al., 2011</a></span>) as the total number of differences and the <em>p</em>-distance. This was done for all ingroup taxa (i.e. excluding <em>Tylochromis polylepis</em>), with pairwise deletion for missing and ambiguous data. For the concatenated alignment the within group mean distance was also calculated for the three most species-rich lineages, the Haplochromini, the Lamprologini and the Ectodini. </p> <h3 class="svArticle" id="s0030">2.4. Gene tree discordance tests</h3> <p class="svArticle section" id="p0075">We first tested for topological incongruence between individual gene trees, using hierarchical likelihood ratio tests as implemented in the software CONCATERPILLAR (v1.7.2) (<span id="bb0205"><a href="#b0205" id="ancbb0205" class="intra_ref">Leigh et al., 2008</a></span>), with default settings and the assumption of linked branch lengths. As part of the CONCATERPILLAR analysis, tree inference was performed using RAxML (v7.2.8) (<span id="bb0415"><a href="#b0415" id="ancbb0415" class="intra_ref">Stamatakis, 2006</a></span>), assuming a single GTR substitution model for each sequence alignment. The two largest sets of markers identified by CONCATERPILLAR to have concordant histories (containing 13 and 14 markers, respectively) were each concatenated and subjected to phylogenetic analyses as described below. </p> <h3 class="svArticle" id="s0035">2.5. Phylogenetic analysis of concatenated datasets</h3> <p class="svArticle section" id="p0080">In brief, sequence alignments for sets of loci were concatenated according to different strategies (see below) and phylogenetic analyses were based on both maximum likelihood with GARLI-PART (v2.0.1019) (<span id="bb0510"><a href="#b0510" id="ancbb0510" class="intra_ref">Zwickl, 2006</a></span>) and RAxML (v7.7) (<span id="bb0415"><a href="#b0415" id="ancbb0415" class="intra_ref">Stamatakis, 2006</a></span>), and on Bayesian inference with MrBayes v3.2.1 (<span id="bb0300"><a href="#b0300" id="ancbb0300" class="intra_ref">Ronquist et al., 2012</a></span>). Prior to tree inference, sequence alignments were subdivided according to gene region (exons, introns and UTRs) and codon position, and the optimal substitution models and partitioning schemes for these subdivisions were selected with the greedy algorithm of PartitionFinder (v1.1.1) (<span id="bb0190"><a href="#b0190" id="ancbb0190" class="intra_ref">Lanfear et al., 2012</a></span>) applying the Bayesian information criterion (BIC), and always taking into account substitution models available in the respective tree inference software (<span id="bb0360"><a href="#b0360" id="ancbb0360" class="intra_ref">Schwarz, 1978</a></span>). Phylogenetic analyses were run locally or at the CIPRES Science Gateway (<span id="bb0255"><a href="#b0255" id="ancbb0255" class="intra_ref">Miller et al., 2010</a></span>) and at Bioportal (<span id="bb0185"><a href="#b0185" id="ancbb0185" class="intra_ref">Kumar et al., 2009</a></span>). </p> <p class="svArticle section" id="p0085">We first inferred the phylogeny for each of the two largest sets of loci with concordant histories according to CONCATERPILLAR. To this end, sequence alignments of all markers included in each set were concatenated. We then used concatenation of the full set of 42 loci to infer the phylogenetic history of LT cichlid fishes. This method assumes that all markers share a common evolutionary history and that discordant signals resulting from homoplasies can be counterbalanced by extensive and genome wide marker sampling (<span id="bb0295"><a href="#b0295" id="ancbb0295" class="intra_ref">Rokas et al., 2003</a></span>). While the assumption of a common evolutionary history seems to be violated at least for the analysis of the full marker set, concatenation may still lead to correct phylogenetic estimates when the true tree lies outside of the "anomaly zone" (<span id="bb0180"><a href="#b0180" id="ancbb0180" class="intra_ref">Kubatko and Degnan, 2007</a></span>). As there is no fully unlinked branch length option in GARLI, analyses were run with linked branch lengths (subsetspecificrates = 1, linkmodels = 0) and partitioning schemes and substitution models selected by PartitionFinder with respective settings (branchlengths = linked, models = all, resulting in 17 distinct partitions for the full-concatenated dataset). A total of 50 independent ML inferences were conducted in GARLI, with the termination condition set to at least 10,000 generations without any substantial (0.01) topological enhancement. Node support was assessed with 500 replicates of non-parametric bootstrapping with the same settings. Bootstrap values were mapped to the ML topology with SumTrees (v3.3.1), using the DendroPy Phylogenetic Computing Library (v3.12.0) (<span id="bb0440"><a href="#b0440" id="ancbb0440" class="intra_ref">Sukumaran and Holder, 2010</a></span>). </p> <p class="svArticle section" id="p0090">ML phylogenies with unlinked partition-specific branch lengths were estimated with RAxML, using the -M option and applying a partitioning scheme obtained by a PartitionFinder analysis (settings: branchlengths = unlinked, model = raxml, resulting in 2 partitions). For the ML inference, we used RAxML’s rapid hill-climbing algorithm and the GTR + GAMMA model in 50 alternative runs and with 500 bootstrap replicates each. </p> <p class="svArticle section" id="p0095">Likewise, MrBayes analyses were conducted with unlinked branch lengths (unlink brlens = (all), prset ratepr = fixed) and a partitioning scheme estimated by PartitionFinder (settings: branchlengths = unlinked, model = mrbayes, resulting in 2 partitions). Using the default prior probability distribution (exponential prior with a mean of 0.1) on branch lengths, two independent MrBayes runs were conducted with four chains for 10,000,000 MCMC generations, sampling every 100th generation, and discarding the first 25% as burn-in. All other settings were left at their defaults. Convergence of MCMC was assessed by MrBayes’ Potential Scale Reduction Factor (PSRF) reaching 1.0, and the average standard deviation of split frequencies falling below 0.01. We further evaluated effective sample sizes in Tracer (v1.5) (<span id="bb0290"><a href="#b0290" id="ancbb0290" class="intra_ref">Rambaut and Drummond, 2007</a></span>) and plotted posterior probabilities of splits over the MCMC run with AWTY online to test for convergence of runs (<span id="bb0275"><a href="#b0275" id="ancbb0275" class="intra_ref">Nylander et al., 2008</a></span>). </p> <p class="svArticle section" id="p0100">To examine the phylogenetic signal contained in length-mutational events and to evaluate the potential power of a combined analysis (alignment plus indel information), the indels from the concatenated alignment were translated into a presence/absence matrix. This was performed with the software SeqState v1.4.1 (<span id="bb0260"><a href="#b0260" id="ancbb0260" class="intra_ref">Müller, 2005</a></span>) using the simple indel coding procedure (SIC) (<span id="bb0390"><a href="#b0390" id="ancbb0390" class="intra_ref">Simmons and Ochoterena, 2000</a></span>). Phylogenetic inference for these two datasets was conducted with GARLI, applying the Mkv model of <span id="bb0215"><a href="#b0215" id="ancbb0215" class="intra_ref">Lewis (2001)</a></span>, and otherwise using default settings as described above. </p> <h3 class="svArticle" id="s0040">2.6. Gene tree summary statistics and Bayesian concordance analysis</h3> <p class="svArticle section" id="p0105">In order to visualize potentially conflicting signal contained in the 42 loci, gene trees for each individual marker were inferred using GARLI with settings as specified in Section <span id="bs0035"><a href="#s0035" id="ancbs0035" class="intra_ref">2.5</a></span>. The 50 best topologies from each run and from all 42 markers (a total of 2100 gene trees) were used to generate an average consensus tree in SplitsTree (v4.12.3) (<span id="bb0115"><a href="#b0115" id="ancbb0115" class="intra_ref">Huson and Bryant, 2006</a></span>). The implemented "average consensus tree" function constructs a neighbor-net using the average pairwise distances of the individual trees. </p> <p class="svArticle section" id="p0110">As a further approach to investigate the discordance among the sampled gene trees and to combine conflicting data in a primary concordance and a population tree, we applied a Bayesian concordance analysis (BCA) (<a href="#b0010" id="bb0010" class="intra_ref">Ane et al., 2007</a> and <a href="#b0020" id="bb0020" class="intra_ref">Baum, 2007</a>), as implemented in the software BUCKy v1.4.0 (<span id="bb0200"><a href="#b0200" id="ancbb0200" class="intra_ref">Larget et al., 2010</a></span>). Using samples of MrBayes’ posterior tree distribution as input, this analysis accounts for both uncertainty in individual gene trees and potential discordance among trees inferred from different loci. The primary concordance tree, as estimated by BUCKy, visualizes the most dominant history from several gene trees, along with concordance factors (CF) indicating the proportion of loci supporting a given clade (<span id="bb0020"><a href="#b0020" id="ancbb0020" class="intra_ref">Baum, 2007</a></span>). In addition, a population tree with coalescent units as branch lengths is generated by BUCKy, based on quartets of concordance factors. This population tree is known to be consistent in the presence of incomplete lineage sorting (<a href="#b0050" id="bb0050" class="intra_ref">Chung and Ané, 2011</a> and <a href="#b0200" id="bb0200" class="intra_ref">Larget et al., 2010</a>). </p> <p class="svArticle section" id="p0115">In order to apply BUCKy, MrBayes was used to infer gene trees from the individual loci, with substitution models and partitioning schemes selected by PartitionFinder (assuming linked branch lengths for all subdivisions of each locus). For each locus, we conducted two replicate MrBayes runs with six chains of 15 million generations, sampling every 100th generation. As reported by <span id="bb0490"><a href="#b0490" id="ancbb0490" class="intra_ref">Willis et al. (2013)</a></span>, we found that for most loci, all of the 150,000 sampled trees represented unique topologies, suggesting a lack of resolution in some parts of the tree. This could partly be due to polytomies, which would be displayed as multiple weakly supported topologies with very short branches in MrBayes, as this software only provides fully resolved trees. To reduce the large number of distinct tree topologies, we pruned our dataset to 14 taxa, keeping only one representative per tribe (as our primary interest was a tribal level phylogeny). This deletion was done with the pruning option in BUCKy. The BUCKy analysis was conducted with 4 runs, 10 chains and 500,000 generations per chain. The alpha prior, which represents the a priori expected level of discordance, was set to 1–100. </p> <h3 class="svArticle" id="s0045">2.7. Testing the strength of the phylogenetic signal as a function of dataset size</h3> <p class="svArticle section" id="p0120">In order to test whether our dataset contains a sufficiently large number of markers to recover the "true" phylogenetic history of LT cichlids, we randomly resampled and concatenated different numbers of markers, and produced ML phylogenies from these sets. We then measured the topological difference between the tree resulting from one set of randomly chosen markers and the tree resulting from the complete set including of all markers and between the trees resulting from two different and mutually exclusive marker sets. As our full dataset contained 42 markers, the first comparisons were done for 1–41 randomly chosen markers, whereas the latter was performed for 1–21 randomly chosen markers. For each number of markers between 1 and 41, we compiled 20 sets drawn at random from the full set of 42 markers. Then, for each of the sets containing at most 21 markers, a comparison set was produced containing the same number of markers so that the two sets did not share any marker. In order to take into account marker concordance according to the results of the CONCATERPILLAR analysis (see Section <span id="bs0030"><a href="#s0030" id="ancbs0030" class="intra_ref">2.4</a></span>.) we repeated the same procedure for 1–13 markers, again with 20 replications each. For the latter analysis, we always compiled two sets of markers, so that markers shared a concordant history within each set, but a discordant history between the two sets (according to CONCATERPILLAR). All generated marker sets were subjected to phylogenetic analysis with GARLI (see above, Section <span id="bs0035"><a href="#s0035" id="ancbs0035" class="intra_ref">2.5</a></span>.), using marker-specific partitions and substitution models as suggested by PartitionFinder. Topological differences between resulting ML trees were measured by means of their <em>K</em>-score ( <span id="bb0410"><a href="#b0410" id="ancbb0410" class="intra_ref">Soria-Carrasco et al., 2007</a></span>), as the <em>K-</em>score accounts for variable substitution rates between marker sets. </p> <p class="svArticle section" id="p0125">Then, <em>K-</em>scores of 20 replicate comparisons were plotted against the number of markers used in the datasets for which the respective ML trees had been inferred (see <a href="#b0035" id="bb0035" class="intra_ref">Camargo et al., 2012</a> and <a href="#b0490" id="bb0490" class="intra_ref">Willis et al., 2013</a>). We expected a general decrease of mean <em>K-</em>scores (i.e., fewer topological differences) with increasing marker number due to an increase in the phylogenetic signal for larger datasets. We further expected <em>K-</em>scores between a tree based on randomly drawn markers and the tree based on the full dataset of 42 markers to approach zero for marker numbers close to 42, as the alignments used for the reconstruction of the two trees would become increasingly similar. Nevertheless, we expected the degree to which <em>K-</em>scores decrease with increasing number of markers to inform about the minimum number of markers needed to reliably construct the relationships among cichlid tribes in LT. </p> <p class="svArticle section" id="p0130">As an additional measure of discordance, we tested for statistically significant topological differences between the tree based on all 42 markers, and trees based on smaller datasets, using the Shimodaira–Hasegawa (<span id="bb0385"><a href="#b0385" id="ancbb0385" class="intra_ref">Shimodaira and Hasegawa, 1999</a></span>) and Approximated Unbiased (<span id="bb0380"><a href="#b0380" id="ancbb0380" class="intra_ref">Shimodaira, 2002</a></span>) tests as implemented in PAUP<sup>∗</sup> (v.4.0a129) (<span id="bb0445"><a href="#b0445" id="ancbb0445" class="intra_ref">Swofford, 2003</a></span>). For each number of markers between 1 and 41, we plotted the number of tree replicates that fitted the full dataset significantly worse than the tree produced from all 42 markers. </p> <h2 class="svArticle" id="s0050">3. Results</h2> <h3 class="svArticle" id="s0055">3.1. Sequencing</h3> <p class="svArticle section" id="p0135">Amplicon sequencing was successful for most of the 42 markers for the 45 taxa. In total, we obtained 98.3% of the 1890 possible sequences. Of 789,525 bp in the final alignment, 26,854 bp (3.40%) consisted of gaps; 27,211 bp (3.45%) were undetermined ("N") and 476 bp (0.06%) were ambiguous ("WRYSMK" coded). </p> <h3 class="svArticle" id="s0060">3.2. Alignment and sequence characterization</h3> <p class="svArticle section" id="p0140">The concatenated alignment had a total length of 17,545 bp, of which 1932 positions (11.01%) were variable and 769 positions (4.38%) were parsimony informative (not considering the outgroup <em>Tylochromis polylepis</em>). The amount of variable sites per marker varied between 13 and 110 sites (average: 46, median: 43), the number of parsimony informative sites ranged between four and 59 (average: 18.3, median: 16) ( <span id="bt0010"><a href="#t0010" id="ancbt0010" class="intra_ref">Table 2</a></span>). The average sequence length for each marker was 417.7 bp (median: 423 bp), and the average total number of differences across all sequence pairs was 208.8 (uncorrected <em>p</em>-distance: 0.013). Within three of the major lineages, we found that the Ectodini showed the highest divergence (114.1 differences; uncorrected <em>p</em>-distance: 0.007), followed by the Lamprologini (110.4; 0.007) and the Haplochromini (all species included; 103.1; 0.006). Separate analyses of the within group mean distance of the haplochromines of the three lakes indicated a higher number of base differences between the four species of Lake Malawi (14.3; 0.0009) than the four species of Lake Victoria (6.8; 0.0004). The Tropheini (<em>Ctenochromis horei, Lobochilotes labiatus, Gnathochromis pfefferi, Tropheus moori</em>) included in this study showed a higher level of diversity (73.0; 0.004). </p> <h3 class="svArticle" id="s0065">3.3. Gene tree discordance tests</h3> <p class="svArticle section" id="p0145">We used CONCATERPILLAR to test for topological incongruence between markers and to identify concordant sets of markers. Based on hierarchical likelihood ratio tests, CONCATERPILLAR detected six sets of markers that were concordant internally, but exhibited significant levels of discordance (<em>p</em>-value &amp;lt; 0.001) between them. The three largest sets contained 14, 13, and 6 markers, respectively, whereas the remaining three sets included 3 markers each (the assignment of each marker to one of these subsets is indicated in <span id="bt0010"><a href="#t0010" id="ancbt0010" class="intra_ref">Table 2</a></span>). The six sets exhibited no obvious clustering of markers according to gene function, coding and non-coding parts, or variability. The two largest sets of markers were subjected to individual phylogenetic analysis. Subset 1 (14 markers) contained a total of 5872 concatenated bp, of which 10.30% were variable and 3.92% were parsimony informative. The average pairwise distance was 61.05 mutational steps, and the uncorrected <em>p</em>-distance was 0.012. Subset 2 (13 markers) had a length of 5507 bp, with 12.69% variable sites, and 5.25% parsimony informative sites. This marker-set showed a somewhat higher variability (average pairwise distance: 76.07; uncorrected <em>p</em>-distance: 0.015). </p> <h3 class="svArticle" id="s0070">3.4. Phylogenetic analysis of concatenated datasets</h3> <p class="svArticle section" id="p0150">Phylogenetic analysis of concatenated subsets revealed conflicting topologies between subset 1 and subset 2 (<span id="bf0010"><a href="#f0010" id="ancbf0010" class="intra_ref">Fig. 2</a></span>a and b). While the base of the resultant trees (i.e. the position of the Bathybatini, Boulengerochromini and Trematocarini) was highly similar, the topologies differed with respect to the relative placement of the Eretmodini, the Lamprologini, the Limnochromini and the Cyprichromini/Perissodini clade. For subset 1 the three inferred topologies from the different analyses were congruent. In these trees, the Lamprologini were nested within the mouthbrooding tribes of the ‘H-lineage’, of which the Cyprichromini/Perissodini clade branched off first. The Lamprologini were resolved as sister group to the Limnochromini in BI (BPP 0.81), and the same relationship was weakly supported in GARLI and RAxML inferences (BS 37 and 45). The Ectodini were placed as sister group to a clade formed by Cyphotilapiini, Eretmodini and the Haplochromini (GARLI BS 26, RAxML BS 35, BPP 0.89). </p> <div class="figTblUpiOuter svArticle" id="figure_f0010"> <dl class="figure" id="f0010" data-t="f"> <dt class="autoScroll" data-style="height:999px;width:728px;" /> <dd id="labelCaptionf0010"> <div class="caption">
 Fig. 2. <p id="sp0060">Results from the phylogenetic analyses based on the two largest subsets of markers identified with CONCATERPILLAR. (a) Maximum likelihood phylogeny of subset 1 (14 markers; see <span id="f0010bt0010"><a href="#t0010" id="ancf0010bt0010" class="intra_ref">Table 2</a></span>) inferred with GARLI. (b) Maximum likelihood phylogeny of subset 2 (13 markers; see <span id="f0010bt0010"><a href="#t0010" id="ancf0010bt0010" class="intra_ref">Table 2</a></span>) inferred with GARLI. Numbers above the branches represent maximum likelihood bootstrap support values (⩾50%) as obtained with GARLI, numbers below the branches indicate Bayesian posterior probabilities (⩾0.75) as revealed with MrBayes. The branch leading to the outgroup taxon, <em>Tylochromis polylepis</em>, is shortened by one third. The colors indicate the affiliation of each taxon to one of the cichlid tribes. </p> </div> </dd> </dl> </div> <p class="svArticle section" id="p0155">The phylogenetic analyses of subset 2 revealed a monophyletic group containing the ‘H-lineage’ taxa (BS 99, BPP 1.0), which were placed as sister taxon to the Lamprologini. The Eretmodini branched off first, and the Ectodini were consistently grouped together with a clade formed by Cyprichromini/Perissodini, the Cyphotilapiini and the Limnochromini (BS 32-34, BPP 0.90). In general, the interrelationships of tribes received only moderate support, which is likely a consequence of the comparatively small number of markers in this subset (see Section <span id="bs0080"><a href="#s0080" id="ancbs0080" class="intra_ref">3.6</a></span>). Excluding <em>Eretmodus cyanostictus</em> from these two phylogenetic analyses did not change the resulting tree topologies (data not shown). </p> <p class="svArticle section" id="p0160">The trees obtained with the entire concatenated dataset of 42 markers were highly congruent and most nodes were very well supported (mean GARLI BS 79.2; mean RAxML BS 78.1; mean BPP 0.941). <span id="bf0015"><a href="#f0015" id="ancbf0015" class="intra_ref">Fig. 3</a></span>b depicts the ML tree inferred with GARLI; the ML tree obtained with RAxML and the 50% majority rule consensus tree of our MrBayes analysis are shown in <span id="bs0115"><a href="#s0115" id="ancbs0115" class="intra_ref">Fig. S1</a></span>. In all three trees, <em>Oreochromis tanganicae</em> appeared as the sister to <em>Tilapia sparrmanii</em> and a strongly supported clade formed by the remaining tribes (GARLI BS 100, RAxML BS, 100, BPP 1.0). The monophyly of these tribes was strongly supported (BS 100, BPP 1.0 for all tribes of which more than two representatives have been included). Within this group <em>T. nigrifrons</em> and <em>B. graueri</em> appeared as sister taxa (BS 100, BPP 1.0) in all our analyses. The three tribes Boulengerochromini (represented by their only member, <em>B. microlepis</em>), Trematocarini (represented by <em>T. nigrifrons</em>), and Bathybatini (represented by <em>B. graueri</em>) appeared outside of a strongly supported clade (BS 100, BPP 1.0), in which the substrate spawning Lamprologini, the most species-rich tribe within LT, are clearly separated from the mouthbrooding tribes (i.e. Cyphotilapiini, Cyprichromini, Ectodini, Eretmodini, Haplochromini, Limnochromini, Perissodini; BS 73-75, BPP 1.0). </p> <div class="figTblUpiOuter svArticle" id="figure_f0015"> <dl class="figure" id="f0015" data-t="f"> <dt class="autoScroll" data-style="height:635px;width:786px;" /> <dd id="labelCaptionf0015"> <div class="caption">
 Fig. 3. <p id="sp0065">Tribal level phylogeny of the Lake Tanganyika cichlid fishes. (a) Map of the area showing the three East African Great Lakes. (b) Maximum likelihood tree based on the concatenated dataset (17,545 bp) as obtained from a partitioned analysis with GARLI. Numbers above the branches indicate maximum likelihood bootstrap support values (⩾50%) produced with GARLI, numbers below the branches represent Bayesian posterior probabilities (⩾0.75) as revealed with MrBayes. Alternative branching orders between the maximum likelihood analysis with GARLI (as shown here) and the maximum likelihood analysis with RAxML (<span id="f0015bs0115"><a href="#s0115" id="ancf0015bs0115" class="intra_ref">Fig. S1a</a></span>) and Bayesian inference with MrBayes (<span id="f0015bs0115"><a href="#s0115" id="ancf0015bs0115" class="intra_ref">Fig. S1b</a></span>) are indicated with dotted lines; the branch leading to <em>Tylochromis polylepis</em> was shortened by one third; colors indicate the tribal affiliation of each taxon. Sample origin other than LT are indicated with boxes on the right; R = riverine. Fish pictures were taken in the field, except for <em>P. nyererei</em> and <em>R. esox</em> (credit: E. Schraml), <em>P. rockkribensis</em> (credit: M. Negrini) and <em>L. sp.</em> ‘stone’ (credit: O. Seehausen). </p> </div> </dd> </dl> </div> <p class="svArticle section" id="p0165">The branching order within the mouthbrooding tribes of the ‘H-lineage’ received less support, and there was incongruence between the tree topologies resulting from the different analyses with respect to the placement of the Cyphotilapiini and the Limnochromini relative to each other, and regarding the first divergence events within the Haplochromini (indicated by dotted lines in <span id="bf0015"><a href="#f0015" id="ancbf0015" class="intra_ref">Fig. 3</a></span>b). The Cyprichromini were consistently resolved as the sister group of Perissodini (BS 100, BPP 1.0), and the clade formed by these two tribes represented the sister of all remaining tribes of the ‘H-lineage’ in all analyses of the full-concatenated dataset. The Limnochromini and the Cyphotilapiini formed a monophyletic group that was sister to a clade combining the Ectodini, the Eretmodini, and the Haplochromini (GARLI BS 65, RAxML BS 59, BPP 1.00) in the GARLI analysis, whereas the Cyphotilapiini appeared closer to this clade according to the RAxML and MrBayes analyses. Within this clade, the representative of the Eretmodini (<em>E. cyanostictus</em>) was consistently placed as sister group to the Haplochromini (GARLI BS 71, RAxML BS 50, BPP 0.88). Similarly, the species from Lake Victoria and Lake Malawi appeared reciprocally monophyletic (BS 100, BPP 1.0) within the Haplochromini. <em>Haplochromis cf. stappersii</em> from LT was resolved as sister taxon to the Lake Victoria cichlids (BS 100, BPP 1.0). The riverine species <em>Astatotilapia burtoni</em> was always placed outside of the species-flocks of the Lake Malawi and Victoria cichlids (BS 100, BPP 1.0). The haplochromines <em>Serranochromis macrocephalus</em> and <em>Pseudocrenilabrus philander</em> were either put into a separate clade (in RAxML and BI), or placed together with the LT haplochromines (Tropheini) (with GARLI). </p> <p class="svArticle section" id="p0170">Translating all indels of the 42 loci into a binary code resulted in a dataset comprising 167 positions, of which 70 were parsimony informative. A phylogenetic hypothesis obtained with this dataset with GARLI was, overall, concordant with the trees resulting from the concatenated dataset. However, while the monophyly of most tribes and the position of the Eretmodini as sister group to the Haplochromini was recovered, the respective support values were generally low and the position of most of the tribes relative to each other could not be recovered (see <span id="bs0115"><a href="#s0115" id="ancbs0115" class="intra_ref">Fig. S2</a></span>). </p> <h3 class="svArticle" id="s0075">3.5. Gene tree summary statistics and Bayesian concordance analysis</h3> <p class="svArticle section" id="p0175">Inferring single gene trees from 42 genes and 45 taxa with both GARLI and MrBayes (data not shown) resulted in 42 alternative topologies with some to numerous polytomies or low support values for certain branches, whereas other parts of the trees were well resolved. <span id="bf0020"><a href="#f0020" id="ancbf0020" class="intra_ref">Fig. 4</a></span> shows the average consensus network of 2100 trees with 168 splits representing the conflicting affinities within the individual gene trees at the base of the tribes. The tribes themselves seem clearly defined and show only few alternative splits. </p> <div class="figTblUpiOuter svArticle" id="figure_f0020"> <dl class="figure" id="f0020" data-t="f"> <dt class="autoScroll" data-style="height:494px;width:787px;" /> <dd id="labelCaptionf0020"> <div class="caption">
 Fig. 4. <p id="sp0070">Average consensus neighbor-net inferred with SplitsTree4 from average pairwise distances in the best gene trees obtained from 50 GARLI runs for each marker (2,100 trees). Note that in this consensus network each gene tree estimate contributed equally and that differences in alignment lengths, degrees of variation, and uncertainties (e.g. bootstrap values) among markers are not considered. The color code is the same is all other figures, the numbers refer to the different species (see <span id="f0020bs0115"><a href="#s0115" id="ancf0020bs0115" class="intra_ref">Table S1</a></span>). (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article.) </p> </div> </dd> </dl> </div> <p class="svArticle section" id="p0180">For the Bayesian concordance analysis with BUCKy, we pruned the dataset to one representative per tribe (<span id="bf0025"><a href="#f0025" id="ancbf0025" class="intra_ref">Fig. 5</a></span>). Changes in the alpha prior had no influence in the topology of both primary concordance and population tree. Its topology (with alpha default prior) is mostly consistent with the species tree inferred from the full-concatenated dataset (see above; <span id="bf0015"><a href="#f0015" id="ancbf0015" class="intra_ref">Fig. 3</a></span>). However, one topological disagreement was found regarding the position of <em>Boulengerochromis microlepis</em>, which was placed as a sister group to the clade composed of the Lamprologini and the representatives of the ‘H-lineage’ (including the Eretmodini) in the population tree, but clustered with the Trematocarini and the Bathybatini in the primary concordance tree. Within the population tree the Eretmodini were again resolved as sister group to the Haplochromini. This close relationship is also reflected in the concordance factors of splits within the primary concordance tree (see <span id="bs0115"><a href="#s0115" id="ancbs0115" class="intra_ref">Text S1</a></span>). </p> <div class="figTblUpiOuter svArticle" id="figure_f0025"> <dl class="figure" id="f0025" data-t="f"> <dt class="autoScroll" data-style="height:710px;width:600px;" /> <dd id="labelCaptionf0025"> <div class="caption">
 Fig. 5. <p id="sp0075">Population tree topology from the Bayesian concordance analysis (conducted with BUCKy) of 14 taxa representing the different cichlid tribes in LT. Numbers above the branches represent the averaged concordance factors, numbers below are coalescence units (see <span id="f0025bs0115"><a href="#s0115" id="ancf0025bs0115" class="intra_ref">Text S1</a></span> for further details). Fish pictures and color codes are the same as in <span id="f0025bf0020"><a href="#f0020" id="ancf0025bf0020" class="intra_ref">Fig. 4</a></span>. (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article.) </p> </div> </dd> </dl> </div> <h3 class="svArticle" id="s0080">3.6. Strength of the phylogenetic signal as a function of dataset size</h3> <p class="svArticle section" id="p0185">After 20 repetitions of random resampling and concatenation of 1–41 markers, we used GARLI to infer ML phylogenies from all replicate marker sets, and compared the resulting trees between each other and with the optimal tree based on the full concatenated dataset of 42 markers, in order to test the strength of the phylogenetic signal as a function of dataset size. We expected topological differences between two trees to decrease with increasing size of the respective marker sets as shown in <span id="bb0035"><a href="#b0035" id="ancbb0035" class="intra_ref">Camargo et al. (2012)</a></span>. Different types of comparisons were performed: Between one tree based on 1–41 markers and the tree resulting from the full marker set (<span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>a), between two trees produced from mutually exclusive sets containing 1–21 markers (<span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>b), and between two trees based on mutually exclusive sets of 1–13 markers found to be internally concordant but externally discordant in topology according to the CONCATERPILLAR analysis (<span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>c). </p> <div class="figTblUpiOuter svArticle" id="figure_f0030"> <dl class="figure" id="f0030" data-t="f"> <dt class="autoScroll" data-style="height:229px;width:791px;" /> <dd id="labelCaptionf0030"> <div class="caption">
 Fig. 6. <p id="sp0080">Topological differences between ML trees measured by their <em>K</em>-scores as a function of the number of randomly resampled and concatenated markers. (a) <em>K</em>-scores between trees based on randomly sampled and concatenated markers and the tree based on the full dataset of 42 markers. Light blue lines indicate the number of tree replicates (out of a total of 20 replicates) significantly different to the tree based on the full dataset, according to the Shimodaira–Hasegawa (SH) test (solid line), and the Approximately Unbiased (AU) test (dashed line). (b) <em>K</em>-scores between two trees that are both based on mutually exclusive randomly sampled marker sets of the given size. (c) As (b), but strictly grouping concordant markers in each set (according to CONCATERPILLAR, see text). Boxplots are based on 20 replicates of each comparison. Whiskers indicate the lowest <em>K</em>-score still within 1.5 inter-quartile range of the lower quartile, and the highest <em>K</em>-score still within 1.5 inter-quartile range of the upper quartile. Outliers are indicated with dots. (For interpretation of the references to color in this figure legend, the reader is referred to the web version of this article.) </p> </div> </dd> </dl> </div> <p class="svArticle section" id="p0190">As expected, topological differences between two trees, as measured by their <em>K</em>-score, generally decreased with increasing marker number; the steepest decrease was observed for marker numbers between 1 and 8–10. The median <em>K</em>-score between one tree based on a randomly compiled marker set of a given size and the tree based on the full set of 42 markers was always lower than median <em>K</em>-scores between two trees based on randomly compiled marker sets of the same size ( <span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>a versus b). Furthermore, topological comparisons involving the tree based on the full marker set generally resulted in a lower variance of <em>K</em>-scores than comparisons between two trees that were produced from randomly sampled mutually exclusive marker sets. In the latter case, the two trees represent independent phylogenetic estimates and are thus particularly useful to assess variance in discordance as a function of marker set size. For this type of comparisons, <em>K</em>-scores appear relatively constant for datasets combining between 11 and 21 markers. Nevertheless, <em>K</em>-scores between trees based on 21 markers (mean 0.0111) are significantly lower than those between trees constructed from sets of 16 markers (mean 0.0140, <em>t</em>-test <em>p</em>-value = 0.01613) or less (means ⩾ 0.0128, <em>t</em>-test <em>p</em>-values ⩽ 0.01704). For most marker set sizes, mean and median <em>K</em>-scores of two trees based on mutually exclusive marker sets were slightly lower when all markers with a set were concordant according to the CONCATERPILLAR analysis ( <span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>c) compared to when sets were composed of randomly sampled markers ( <span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>b). This reduction was significant for marker sets with eight markers or more (<em>t</em>-test <em>p</em>-values ⩽ 0.0295), with the exception of sets containing eleven markers (<em>t</em>-test <em>p</em>-value = 0.0881), which suggests that the discordance between the two largest marker sets identified by CONCATERPILLAR is lower than that between randomly compiled marker sets of the same size. </p> <p class="svArticle section" id="p0195">Similarly, the number of marker set replicates, for which ML trees differ significantly from the ML tree based on 42 markers, shows an overall decrease with increasing size of the respective marker sets. For concatenated sets of 1–5 markers, and for sets of 8 markers, phylogenies produced from all 20 replicate sets are significantly different to the full ML tree, according to both the SH and the AU tests. On the other hand, for concatenated sets of 34 or more markers, none of the phylogenies based on these sets differ significantly from the tree obtained with the full set of markers, according to either of the two tests. Between these extremes, we observe a general decrease in the number of rejected tree replicates with increasing number of markers, based on which these trees were produced (<span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>a). </p> <h2 class="svArticle" id="s0085">4. Discussion</h2> <p class="svArticle section" id="p0200">The present study is the most extensive phylogenetic analysis of cichlid fishes in East African Lake Tanganyika with respect to the number of nuclear DNA markers and the total length of the ncDNA sequences analyzed. The main goal of our work was to establish a robust phylogenetic hypothesis for the relationships among the cichlid tribes of LT, which has so far been inferred on the basis of mtDNA or relatively few nuclear markers only (<a href="#b0055" id="bb0055" class="intra_ref">Clabaut et al., 2005</a>, <a href="#b0065" id="bb0065" class="intra_ref">Day et al., 2008</a>, <a href="#b0085" id="bb0085" class="intra_ref">Friedman et al., 2013</a>, <a href="#b0175" id="bb0175" class="intra_ref">Kocher et al., 1995</a>, <a href="#b0265" id="bb0265" class="intra_ref">Muschick et al., 2012</a>, <a href="#b0320" id="bb0320" class="intra_ref">Salzburger et al., 2002a</a> and <a href="#b0425" id="bb0425" class="intra_ref">Sturmbauer and Meyer, 1993</a>). </p> <p class="svArticle section" id="p0205">The comparatively high information content provided by mtDNA sequences and the availability of universal primers were the main reasons for the utilization of mtDNA markers in earlier phylogenetic analyses aiming to resolve the relatively young and rapid radiation of cichlid fishes in LT. Among the many drawbacks of mtDNA markers are that only maternal inheritance patterns are captured and that past events of introgression and hybridization remain largely invisible (<span id="bb0015"><a href="#b0015" id="ancbb0015" class="intra_ref">Ballard and Whitlock, 2004</a></span>). In addition, a single locus (irrespective of being based on mtDNA or ncDNA) might not accurately reflect the species tree, as individual gene trees often differ from the true species tree (<span id="bb0280"><a href="#b0280" id="ancbb0280" class="intra_ref">Pamilo and Nei, 1988</a></span>). Nuclear DNA markers, on the other hand, usually contain fewer variable sites thus less phylogenetic signal. <span id="bb0055"><a href="#b0055" id="ancbb0055" class="intra_ref">Clabaut et al. (2005)</a></span> showed, for example, that in LT cichlids, ncDNA datasets would need to contain about ten times more sequence data to obtain the same quantity of phylogenetic information as provided by mtDNA markers – a task not reached by any previous study. </p> <p class="svArticle section" id="p0210">Here we took advantage of the 454 next-generation pyrosequencing technology and compiled a ncDNA dataset for LT cichlids containing 42 markers in well characterized genes and reaching a total alignment length of 17,545 bp. We chose a locus re-sequencing strategy with barcoded primers in order to obtain long enough sequence reads and to sample a large number of gene histories. Primers were chosen to bind in more conserved exons and to amplify (if possible) more variable intron regions (<span id="bb0250"><a href="#b0250" id="ancbb0250" class="intra_ref">Meyer and Salzburger, 2012</a></span>). </p> <h3 class="svArticle" id="s0090">4.1. Single gene-tree discordance and evaluation of the strength of the phylogenetic signal </h3> <p class="svArticle section" id="p0215">Not surprisingly, the individual single locus datasets did not contain enough phylogenetic information to accurately resolve the phylogenetic relationships among the cichlid tribes of LT. Most single locus trees were not very well resolved, the branch support values in these trees were generally rather low, and all 42 single locus topologies differed at least to some extent (in part because of the occurrence of polytomies; not shown). Overall, however, many of the single locus topologies follow a general trend as is illustrated in the average consensus network shown in <span id="bf0020"><a href="#f0020" id="ancbf0020" class="intra_ref">Fig. 4</a></span>. Many branches, and especially the monophyly of cichlid tribes, are well supported across the datasets. However, the consensus network indicates certain areas of uncertainties, which might result from hybridization and/or incomplete lineage sorting or simply reflect the low power of resolution in some of the individual markers (see below). </p> <p class="svArticle section" id="p0220">In order to estimate the strength of the phylogenetic signal as a function of dataset size and to evaluate whether our dataset contained enough phylogenetic information, we applied a strategy that compares tree topologies inferred from randomly chosen datasets with varying numbers of markers per alignment on the basis of their <em>K</em>-scores ( <span id="bb0035"><a href="#b0035" id="ancbb0035" class="intra_ref">Camargo et al., 2012</a></span>). More specifically, we compiled datasets from 1 to 41 randomly chosen markers (in 20 replications each) and compared the ML trees based on these marker sets to the tree produced in the same way from the full dataset containing all 42 concatenated markers. Obviously, and as expected, the topologies resulting from the randomly drawn marker sets become increasingly similar to the best tree obtained with 42 markers the more markers are included in each concatenated dataset ( <span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>a). Also, differences between equally large and mutually exclusive marker sets generally decrease with increases in the number of markers included in both sets ( <span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>b). The same decrease was observed when trees were produced from two sets of markers that were identified as topologically concordant within each set, but discordant between sets ( <span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>c). However, topological differences were generally slightly lower when marker sets were discordant to each other ( <span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>c). This was unexpected but could in part be explained if the phylogenetic histories of marker sets 3–6 (which are included in <span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>b, but excluded from <span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>c) are even more discordant than those of marker sets 1 and 2. </p> <p class="svArticle section" id="p0225">Importantly, while all tree topologies resulting from datasets of 1–5 markers were significantly distinct from the best tree according to both SH and AU tests, inferred trees become successively more similar with an increasing number of markers, and statistically indifferent from the best tree when more than 34 markers are included (light blue lines in <span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>a). These results suggest that our full dataset is large enough to reliably resolve the phylogenetic history of the LT cichlid fishes. Whether or not an extension of our marker set to even more than 42 markers would provide additional phylogenetic signal remains to be tested. </p> <h3 class="svArticle" id="s0095">4.2. A threefold strategy for phylogenetic analyses in LT cichlids</h3> <p class="svArticle section" id="p0230">In order to account for potential problems with dataset concatenation (see below), we opted to apply three strategies to analyze our data. In a first step, we performed ML and BI phylogenetic analyses with a concatenated dataset containing all 42 markers of all 45 species. These analyses were based of the naïve assumptions that all gene histories equally reflect the species tree, and that the ‘true’ phylogenetic signal should dominate over phylogenetic noise in a large enough dataset (<span id="bb0295"><a href="#b0295" id="ancbb0295" class="intra_ref">Rokas et al., 2003</a></span>). The usage of the concatenated dataset is further backed up by our phylogenetic analyses of randomly chosen subsets of varying numbers of markers, which demonstrate that the phylogenetic signal improves with increasing number of included markers (<span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>). </p> <p class="svArticle section" id="p0235">Although concatenation of multiple markers is often thought to improve accuracy (<a href="#b0025" id="bb0025" class="intra_ref">Bayzid and Warnow, 2013</a>, <a href="#b0040" id="bb0040" class="intra_ref">Chen and Li, 2001</a> and <a href="#b0295" id="bb0295" class="intra_ref">Rokas et al., 2003</a>; but see <span id="bb0305"><a href="#b0305" id="ancbb0305" class="intra_ref">Salichos and Rokas, 2013</a></span>), this approach assumes that genes share a common evolutionary history, and it has been shown that violation of this assumption can lead to strongly supported yet incorrect phylogenies (<a href="#b0070" id="bb0070" class="intra_ref">Degnan and Rosenberg, 2009</a>, <a href="#b0090" id="bb0090" class="intra_ref">Gadagkar et al., 2005</a>, <a href="#b0180" id="bb0180" class="intra_ref">Kubatko and Degnan, 2007</a> and <a href="#b0305" id="bb0305" class="intra_ref">Salichos and Rokas, 2013</a>). One situation, in which concatenation may lead to inconsistent species tree estimates, is incomplete lineage sorting (<a href="#b0070" id="bb0070" class="intra_ref">Degnan and Rosenberg, 2009</a>, <a href="#b0180" id="bb0180" class="intra_ref">Kubatko and Degnan, 2007</a> and <a href="#b0500" id="bb0500" class="intra_ref">Yang and Rannala, 2012</a>). We thus, in a second approach, applied a gene tree discordance test with CONCATERPILLAR to evaluate the incongruence between individual gene trees. This test suggested the existence of six sets of markers that were concordant within them, but discordant between each other. The two largest sets, containing 14 and 13 markers respectively, were then subjected to in-depth phylogenetic analysis. </p> <p class="svArticle section" id="p0240">As a third strategy, we performed a Bayesian concordance analysis with BUCKy, which accounts for uncertainty and variability in the individual locus phylogenies and has been shown to deal well with incomplete lineage sorting (<a href="#b0050" id="bb0050" class="intra_ref">Chung and Ané, 2011</a>, <a href="#b0140" id="bb0140" class="intra_ref">Knowles and Kubatko, 2011</a> and <a href="#b0505" id="bb0505" class="intra_ref">Yang and Warnow, 2011</a>). In this analysis, we pruned our dataset to one species per tribe. </p> <p class="svArticle section" id="p0245">Overall, the three strategies applied to analyze our multi-marker dataset resulted in congruent topologies. All analyses confirm the monophyly of the LT tribes (in cases where more than one representative was included; this does, hence, not apply to the BUCKy analysis with the reduced taxon set). In all analyses, the Tylochromini, Oreochromini and Tilapiini were resolved outside of all other included species. The representatives of the Trematocarini and the Bathybatini always formed a clade, and were, together with <em>B. microlepis</em> (Boulengerochromini), consistently placed as sister-group to the remaining cichlid tribes; the Cyprichromini and Perissodini always clustered together. Furthermore, in all analyses except in those based on subset 1 of CONCATERPILLAR, the Lamprologini were resolved as sister group to the ‘H-linage’ consisting of Cyphotilapiini, Limnochromini, Cyprichromini, Perissodini, Ectodini, Eretmodini and Haplochromini. In all analyses, the Eretmodini appear as a member of the ‘H-lineage’ and, with one exception (i.e. subset 2 of CONCATERPILLAR), appear as sister-group to the Haplochromini. </p> <p class="svArticle section" id="p0250">Within the ‘H-lineage’, the relationships of the cichlid tribes differed between the three approaches. Especially the analysis of subset 1 of CONCATERPILLAR revealed a rather different topology, whereas in subset 2 the relative position of the Eretmodini and Ectodini varied in comparison to the other approaches. Note, however, that the two largest subsets of markers identified by CONCATERPILLAR contain only 14 (subset 1) and 13 markers (subset 2), respectively. Our analyses have shown that sets with as many as 34 markers can still produce significantly different trees for the same set of taxa. The phylogenetic hypotheses resulting from these small marker sets (<span id="bf0010"><a href="#f0010" id="ancbf0010" class="intra_ref">Fig. 2</a></span>a and b) should thus be taken with caution. </p> <p class="svArticle section" id="p0255">Taken together, we believe that, in our case, the concatenation of all markers is a justified strategy (<span id="bf0015"><a href="#f0015" id="ancbf0015" class="intra_ref">Fig. 3</a></span>), as it leads to the best-supported tree topologies, which are backed-up by similar results in both the average consensus network (<span id="bf0020"><a href="#f0020" id="ancbf0020" class="intra_ref">Fig. 4</a></span>) and the Bayesian concordance analysis (<span id="bf0025"><a href="#f0025" id="ancbf0025" class="intra_ref">Fig. 5</a></span>). The concatenation strategy is further supported by our phylogenetic signal tests, which show that the largest datasets lead to significantly more robust topologies (<span id="bf0030"><a href="#f0030" id="ancbf0030" class="intra_ref">Fig. 6</a></span>), whereas the subsets suggested by CONCATERPILLAR may not contain enough phylogenetic information. At the same time, these tests indicate the presence of a sufficient phylogenetic signal in the concatenated dataset, so that remaining uncertainties in the resultant tree topologies (GARLI, RAxML and MrBayes analyses of concatenated dataset and subsets) should not be due to lacking power of resolution (‘soft polytomy’ problem). Instead, it appears that the remaining uncertainties in our trees, most notably the phylogenetic relationships among ‘H-lineage’ tribes (see <a href="#f0010" id="bf0010" class="intra_ref">Fig. 2</a>, <a href="#f0015" id="bf0015" class="intra_ref">Fig. 3</a> and <a href="#f0020" id="bf0020" class="intra_ref">Fig. 4</a>), are due to high speciation rates at the onset of radiation of the LT mouthbrooders (‘hard polytomy’ problem), past events of hybridization, and/or the persistence of ancestral polymorphisms. It has previously been recognized that it is notoriously difficult to resolve, with the available methodology, the phylogenetic relationships among lineages that emerged from adaptive radiation events (<span id="bb0105"><a href="#b0105" id="ancbb0105" class="intra_ref">Glor, 2010</a></span>), which is not least due to the fact that such tree topologies are expected to be ‘bottom-heavy’ (<span id="bb0095"><a href="#b0095" id="ancbb0095" class="intra_ref">Gavrilets and Vose, 2005</a></span>). </p> <h3 class="svArticle" id="s0100">4.3. Conclusions</h3> <p class="svArticle section" id="p0260">With this study, we present a novel hypothesis for the phylogenetic relationships among East African cichlid tribes, which is based on the largest set of ncDNA sequences so far, and which differs from all previous hypotheses (<span id="bf0005"><a href="#f0005" id="ancbf0005" class="intra_ref">Fig. 1</a></span>). Our analyses provide strong support for the monophyly of LT mouthbrooding cichlids (i.e. the ‘H-lineage’ of <span id="bb0270"><a href="#b0270" id="ancbb0270" class="intra_ref">Nishida, 1991</a></span>) as sister-group to the substrate spawning Lamprologini. We thus confirm the scenario that both lineages have radiated in parallel within LT (<span id="bb0315"><a href="#b0315" id="ancbb0315" class="intra_ref">Salzburger and Meyer, 2004</a></span>), leading to some intriguing cases of convergent evolution (<span id="bb0265"><a href="#b0265" id="ancbb0265" class="intra_ref">Muschick et al., 2012</a></span>). The clustering of the tribes within the ‘H-lineage’ generally reflects the life styles and habitat use of the respective tribes. The Cyprichromini and Perissodini, which are consistently put together (<a href="#f0010" id="bf0010" class="intra_ref">Fig. 2</a>, <a href="#f0015" id="bf0015" class="intra_ref">Fig. 3</a>, <a href="#f0020" id="bf0020" class="intra_ref">Fig. 4</a> and <a href="#f0025" id="bf0025" class="intra_ref">Fig. 5</a>), are both adapted to the open-water column; the Cyphotilapiini and Limnochromini, which cluster together in most analyses (<a href="#f0010" id="bf0010" class="intra_ref">Fig. 2</a>, <a href="#f0015" id="bf0015" class="intra_ref">Fig. 3</a>, <a href="#f0020" id="bf0020" class="intra_ref">Fig. 4</a> and <a href="#f0025" id="bf0025" class="intra_ref">Fig. 5</a>, excluding 2A), are restricted to deep-water habitats; and the Ectodini, Eretmodini and Haplochromini dominate (together with many lamprologine species) the shallow waters of LT. Our phylogenies thus reveal the general trend that the less species-rich cichlid tribes in LT (including the Bathybatini, Boulengerochromini and Trematocarini) occupy less-productive habitats such as the open-water column or deeper areas, whereas the generally more species-rich tribes of the ‘H-lineage’ dominate the more-productive and generally preferred shallow/rocky habitats (<span id="bb0265"><a href="#b0265" id="ancbb0265" class="intra_ref">Muschick et al., 2012</a></span>). </p> <p class="svArticle section" id="p0265">We further postulate a nested position of the Eretmodini within the ‘H-lineage’, as sister-group to the Haplochromini, which is in clear contrast to most of the studies relying on mtDNA markers (<a href="#b0055" id="bb0055" class="intra_ref">Clabaut et al., 2005</a>, <a href="#b0065" id="bb0065" class="intra_ref">Day et al., 2008</a>, <a href="#b0175" id="bb0175" class="intra_ref">Kocher et al., 1995</a> and <a href="#b0265" id="bb0265" class="intra_ref">Muschick et al., 2012</a>), yet in concordance to allozyme data (<span id="bb0270"><a href="#b0270" id="ancbb0270" class="intra_ref">Nishida, 1991</a></span>) and ncDNA phylogenies (<span id="bb0085"><a href="#b0085" id="ancbb0085" class="intra_ref">Friedman et al., 2013</a></span>). The obvious discordance between the Lamprologini-like mtDNA and Haplochromini-like ncDNA in the Eretmodini can either be explained by incomplete mtDNA lineage sorting, or, more likely, by an ancient hybridization event (<span id="bb0245"><a href="#b0245" id="ancbb0245" class="intra_ref">Meng and Kubatko, 2009</a></span>). The positions of the oldest tribes (Tylochromini, Oreochromini, Trematocarini, Bathybatini, Boulengerochromini) are largely in agreement with previous studies, as most studies suggested a sister-group relationship between the Bathybatini and Trematocarini (<a href="#b0055" id="bb0055" class="intra_ref">Clabaut et al., 2005</a>, <a href="#b0065" id="bb0065" class="intra_ref">Day et al., 2008</a> and <a href="#b0320" id="bb0320" class="intra_ref">Salzburger et al., 2002a</a>) and placed the Oreochromini outside of this group (<a href="#b0085" id="bb0085" class="intra_ref">Friedman et al., 2013</a>, <a href="#b0265" id="bb0265" class="intra_ref">Muschick et al., 2012</a> and <a href="#b0320" id="bb0320" class="intra_ref">Salzburger et al., 2002a</a>). The placement of the Boulengerochromini differed slightly between our analyses, but in all cases this monotypic tribe was resolved outside the clade formed by the Lamprologini and the ‘H-lineage’. </p> <h2 class="svArticle" id="s0105">5. Outlook</h2> <p class="svArticle section" id="p0270">With this study, we provide a strong phylogenetic hypothesis for the cichlid tribes in LT based on 42 ncDNA makers. Yet, we also identified remaining areas of uncertainties, especially with respect to the phylogenetic relationships of the mouthbrooding tribes within the ‘H-lineage’. Future analyses should focus on the amount and relative proportion of shared genes among the different cichlid lineages to allow further insights into stochastic processes such as incomplete lineage sorting or hybridization. To this end, we recommend the usage of much larger datasets such as whole transcriptomes or genomes. RAD-sequencing could also provide a large random sample of ncDNA loci, although the current read lengths render the phylogenetic inference based on individual loci problematic. Another important next step to understand the evolutionary history of LT cichlids and to establish a species tree would be to perform coalescent-based analysis with BEST and <sup>∗</sup>BEAST (<a href="#b0230" id="bb0230" class="intra_ref">Liu, 2008</a> and <a href="#b0110" id="bb0110" class="intra_ref">Heled and Drummond, 2010</a>), using phased alleles and more individuals per species. Finally, future analyses should increase taxon sampling, ultimately leading to a complete species tree for the cichlid species of LT. </p> <h2 class="svArticle" id="s0120">Data accessibility</h2> <p class="svArticle section" id="p0315">All sequences are accessible in Genbank KP129679-KP131427 and KM263618-KM263752.</p> <p class="svArticle section" id="p0320">Trees are deposited at Treebase (<span id="ir0025" class="interref" data-locatortype="url" data-locatorkey="http://purl.org/phylo/treebase/phylows/study/TB2:S16660">http://purl.org/phylo/treebase/phylows/study/TB2:S16660</span>). </p> <div class="articleText svArticle"> <h2 class="h2 secHeading" id="ack001">Acknowledgments</h2> <p class="svArticle section" id="p0310">We thank Moritz Muschick, Adrian Indermaur and Frauke Münzel for help with sampling, and Ole Seehausen for providing tissues of Lake Victoria cichlids. We are grateful to Brigitte Aeschbach, Nicolas Boileau as well as Astrid Böhne and Emilía M. Santos for assistance in the laboratory, and to Christof Wunderlin and Georges Wigger from Microsynth for help with library preparation and 454 sequencing. We thank Stuart Willis and Cecile Ané for discussing the BUCKy analysis, and Sebastian Höhna for help with MrBayes. Finally, we would like to thank the editor Guillermo Orti, two anonymous reviewers, Heinz Büscher, Adrian Indermaur, and Uli Schliewen for discussion and helpful suggestions on the manuscript. This study was supported by the <span id="gp005" class="interref" data-locatortype="" data-locatorkey="">University of Basel</span>, the <span id="gp010" class="interref" data-locatortype="" data-locatorkey="">Freiwillige Akademische Gesellschaft (FAG) Basel</span> and the <span id="gp015" class="interref" data-locatortype="" data-locatorkey="">Burckhardt-Bürgin Stiftung</span> (to BSM), the <span id="gp020" class="interref" data-locatortype="" data-locatorkey="">Swiss National Science Foundation</span> (Grant <a class="grantnum" href="#gp020">PBBSP3-138680</a> to MM and Grants <a class="grantnum" href="#gp020">3100A0_122458</a> and <a class="grantnum" href="#gp020">3100A0_138224</a> to WS) and the <span id="gp025" class="interref" data-locatortype="" data-locatorkey="">European Research Council</span> (ERC; Starting Grant "INTERGENADAPT" and Consolidator Grant "Cichlid∼X" to WS). </p> </div> <p /> <div id="s0115"> <h2 id="appd002">Appendix A. Supplementary material</h2> </div> <p class="section svArticle" id="p0280" /> <div class="figTblUpiOuter svArticle                          inlEcom                     "> <dl class="ecomponent" id="m0005" data-ecmpt="y"> <dd class="ecomponent" id="mmc1" data-t="e" data-label="Supplementary data 1" data-ext="txt" data-eid="1-s2.0-S1055790314003601-mmc1.txt"><span id="MMCvOTHERN70c80e90N737750b0" class="MMCvOTHER" mmcarticleimageurl="/science/page/static/science/mmc_txt.gif" mmcimageurl="/science/page/static/science/mmc_txt.gif" customthumbheight="50" customthumbwidth="50" /> </dd> <dd class="mmcCaption"> <p class="caption"><span class="MMCvLABEL_SRC label"><a href="#" class="videoclip">Supplementary data 1. </a> </span></p> <p id="sp0020">Extract of the concordance file from the Bayesian Concordance Analysis with BUCKy including more information about the concordance factors and coalescent units as well as the tree topologies of both the primary concordance tree and the population tree. </p> <p /> </dd> </dl> </div> <div class="figTblUpiOuter svArticle                          inlEcom                     "> <dl class="ecomponent" id="m0010" data-ecmpt="y"> <dd class="ecomponent" id="mmc2" data-t="e" data-label="Supplementary data 2" data-ext="pdf" data-eid="1-s2.0-S1055790314003601-mmc2.pdf"><span id="MMCvOTHERN70c80e90N737751dc" class="MMCvOTHER" mmcarticleimageurl="/science/page/static/science/mmc_pdf.gif" mmcimageurl="/science/page/static/science/mmc_pdf.gif" customthumbheight="50" customthumbwidth="50" /> </dd> <dd class="mmcCaption"> <p class="caption"><span class="MMCvLABEL_SRC label"><a href="#" class="videoclip">Supplementary data 2. </a> </span></p> <p id="sp0025">List of specimens used in this study. The species names, the classification into tribes, the sample ID’s, sampling date and location, and institute where the samples are deposited are indicated. Numbers ("ID for neighbor-net") refer to <span id="bf0020"><a href="#f0020" id="ancbf0020" class="intra_ref">Fig. 4</a></span>. </p> <p /> </dd> </dl> </div> <div class="figTblUpiOuter svArticle                          inlEcom                     "> <dl class="ecomponent" id="m0015" data-ecmpt="y"> <dd class="ecomponent" id="mmc3" data-t="e" data-label="Supplementary data 3" data-ext="pdf" data-eid="1-s2.0-S1055790314003601-mmc3.pdf"><span id="MMCvOTHERN70c80e90N737752cc" class="MMCvOTHER" mmcarticleimageurl="/science/page/static/science/mmc_pdf.gif" mmcimageurl="/science/page/static/science/mmc_pdf.gif" customthumbheight="50" customthumbwidth="50" /> </dd> <dd class="mmcCaption"> <p class="caption"><span class="MMCvLABEL_SRC label"><a href="#" class="videoclip">Supplementary data 3. </a> </span></p> <p id="sp0030">Sample ID’s and GenBank accession numbers for all sequences used in this study.</p> <p /> </dd> </dl> </div> <div class="figTblUpiOuter svArticle                          inlEcom                     "> <dl class="ecomponent" id="m0020" data-ecmpt="y"> <dd class="ecomponent" id="mmc4" data-t="e" data-label="Supplementary data 4" data-ext="pdf" data-eid="1-s2.0-S1055790314003601-mmc4.pdf"><span id="MMCvOTHERN70c80e90N737753bc" class="MMCvOTHER" mmcarticleimageurl="/science/page/static/science/mmc_pdf.gif" mmcimageurl="/science/page/static/science/mmc_pdf.gif" customthumbheight="50" customthumbwidth="50" /> </dd> <dd class="mmcCaption"> <p class="caption"><span class="MMCvLABEL_SRC label"><a href="#" class="videoclip">Supplementary data 4. </a> </span></p> <p id="sp0035">Tree topologies of the concatenated supermatrix inferred with MrBayes (a and b), RAxML (c), and GARLI (d). Note that A and B result from different partitions (as suggested by PartitionFinder) and unlinking branch-lengths. All support values are plotted. </p> <p /> </dd> </dl> </div> <div class="figTblUpiOuter svArticle                          inlEcom                     "> <dl class="ecomponent" id="m0025" data-ecmpt="y"> <dd class="ecomponent" id="mmc5" data-t="e" data-label="Supplementary data 5" data-ext="pdf" data-eid="1-s2.0-S1055790314003601-mmc5.pdf"><span id="MMCvOTHERN70c80e90N737754e8" class="MMCvOTHER" mmcarticleimageurl="/science/page/static/science/mmc_pdf.gif" mmcimageurl="/science/page/static/science/mmc_pdf.gif" customthumbheight="50" customthumbwidth="50" /> </dd> <dd class="mmcCaption"> <p class="caption"><span class="MMCvLABEL_SRC label"><a href="#" class="videoclip">Supplementary data 5. </a> </span></p> <p id="sp0040">Phylogram inferred using only the information provided by indels, which were transformed into a presence/absence matrix using the SIC coding procedure (<span id="bb0390"><a href="#b0390" id="ancbb0390" class="intra_ref">Simmons and Ochoterena, 2000</a></span>) and further processed with GARLI using the Mkv model. Bootstrap values (⩾50%) are provided above the branches. </p> <p /> </dd> </dl> </div> <div class="figTblUpiOuter svArticle                          inlEcom                     "> <dl class="ecomponent" id="m0030" data-ecmpt="y"> <dd class="ecomponent" id="mmc6" data-t="e" data-label="Supplementary data 6" data-ext="pdf" data-eid="1-s2.0-S1055790314003601-mmc6.pdf"><span id="MMCvOTHERN70c80e90N737755d8" class="MMCvOTHER" mmcarticleimageurl="/science/page/static/science/mmc_pdf.gif" mmcimageurl="/science/page/static/science/mmc_pdf.gif" customthumbheight="50" customthumbwidth="50" /> </dd> <dd class="mmcCaption"> <p class="caption"><span class="MMCvLABEL_SRC label"><a href="#" class="videoclip">Supplementary data 6. </a> </span></p> <p id="sp0045">Tree topologies of subset 1 from the CONCATERPILLAR analysis, inferred with GARLI, MrBayes and RAXML. All support values are plotted. The branch leading to <em>Tylochromis polylepis</em> was shortened by one third. </p> <p /> </dd> </dl> </div> <div class="figTblUpiOuter svArticle                          inlEcom                     "> <dl class="ecomponent" id="m0035" data-ecmpt="y"> <dd class="ecomponent" id="mmc7" data-t="e" data-label="Supplementary data 7" data-ext="pdf" data-eid="1-s2.0-S1055790314003601-mmc7.pdf"><span id="MMCvOTHERN70c80e90N737756c8" class="MMCvOTHER" mmcarticleimageurl="/science/page/static/science/mmc_pdf.gif" mmcimageurl="/science/page/static/science/mmc_pdf.gif" customthumbheight="50" customthumbwidth="50" /> </dd> <dd class="mmcCaption"> <p class="caption"><span class="MMCvLABEL_SRC label"><a href="#" class="videoclip">Supplementary data 7. </a> </span></p> <p id="sp0050">Tree topologies of subset 2 from the CONCATERPILLAR analysis, inferred with GARLI, MrBayes and RAXML. All support values are plotted. The branch leading to <em>Tylochromis polylepis</em> was shortened by one third. </p> <p /> </dd> </dl> </div> <p /> <p /> <h2 id="bi0005">References</h2> <ol class="references"> <li id="b0005" data-t="r"> <ul class="reference" id="bibsbref1"> <li class="referenceLabel"><a href="#b0005" class="label intra_ref">Albertson et al., 2003</a></li> <li class="author">R.C. Albertson, J.T. Streelman, T.D. Kocher</li> <li class="title"> <p>Directional selection has shaped the oral jaws of Lake Malawi cichlid fishes</p> </li> <li class="source"> <p>Proc. Natl. Acad. Sci. USA, 100 (2003), pp. 5252–5257</p> </li> <li class="external refPlaceHolder" id="bibsbref1" data-refres="N" /> </ul> </li> <li id="b0010" data-t="r"> <ul class="reference" id="bibsbref2"> <li class="referenceLabel"><a href="#b0010" class="label intra_ref">Ane et al., 2007</a></li> <li class="author">C. Ane, B. Larget, D.A. Baum, S.D. Smith, A. Rokas</li> <li class="title"> <p>Bayesian estimation of concordance among gene trees</p> </li> <li class="source"> <p>Mol. Biol. Evol., 24 (2007), pp. 412–426</p> </li> <li class="external refPlaceHolder" id="bibsbref2" data-refres="N" /> </ul> </li> <li id="b0015" data-t="r"> <ul class="reference" id="bibsbref3"> <li class="referenceLabel"><a href="#b0015" class="label intra_ref">Ballard and Whitlock, 2004</a></li> <li class="author">J.W.O. Ballard, M.C. Whitlock</li> <li class="title"> <p>The incomplete natural history of mitochondria</p> </li> <li class="source"> <p>Mol. Ecol., 13 (2004), pp. 729–744</p> </li> <li class="external refPlaceHolder" id="bibsbref3" data-refres="N" /> </ul> </li> <li id="b0020" data-t="r"> <ul class="reference" id="bibsbref4"> <li class="referenceLabel"><a href="#b0020" class="label intra_ref">Baum, 2007</a></li> <li class="author">D. Baum</li> <li class="title"> <p>Concordance trees, concordance factors, and the exploration of reticulate genealogy</p> </li> <li class="source"> <p>Taxon, 56 (2007), pp. 417–426</p> </li> <li class="external refPlaceHolder" id="bibsbref4" data-refres="N" /> </ul> </li> <li id="b0025" data-t="r"> <ul class="reference" id="bibsbref5"> <li class="referenceLabel"><a href="#b0025" class="label intra_ref">Bayzid and Warnow, 2013</a></li> <li class="author">M.S. Bayzid, T. Warnow</li> <li class="title"> <p>Naive binning improves phylogenomic analyses</p> </li> <li class="source"> <p>Bioinformatics, 29 (2013), pp. 2277–2284</p> </li> <li class="external refPlaceHolder" id="bibsbref5" data-refres="N" /> </ul> </li> <li id="b0030" data-t="r"> <ul class="reference" id="bibsbref6"> <li class="referenceLabel"><a href="#b0030" class="label intra_ref">Brawand et al., 2014</a></li> <li class="author">D. Brawand, <em>et al.</em></li> <li class="title"> <p>The genomic substrate for adaptive radiation in African cichlid fish</p> </li> <li class="source"> <p>Nature, 513 (2014), pp. 375–381</p> </li> <li class="external refPlaceHolder" id="bibsbref6" data-refres="N" /> </ul> </li> <li id="b0035" data-t="r"> <ul class="reference" id="bibsbref7"> <li class="referenceLabel"><a href="#b0035" class="label intra_ref">Camargo et al., 2012</a></li> <li class="author">A. Camargo, L.J. Avila, M. Morando, J.W. Sites</li> <li class="title"> <p>Accuracy and precision of species trees: effects of locus, individual, and base pair sampling on inference of species trees in lizards of the Liolaemus darwinii group (Squamata, Liolaemidae) </p> </li> <li class="source"> <p>Syst. Biol., 61 (2012), pp. 272–288</p> </li> <li class="external refPlaceHolder" id="bibsbref7" data-refres="N" /> </ul> </li> <li id="b0040" data-t="r"> <ul class="reference" id="bibsbref8"> <li class="referenceLabel"><a href="#b0040" class="label intra_ref">Chen and Li, 2001</a></li> <li class="author">F.-C. Chen, W.-H. Li</li> <li class="title"> <p>Genomic divergences between humans and other hominoids and the effective population size of the common ancestor of humans and chimpanzees </p> </li> <li class="source"> <p>Am. J. Human Genet., 68 (2001), pp. 444–456</p> </li> <li class="external refPlaceHolder" id="bibsbref8" data-refres="N" /> </ul> </li> <li id="b0045" data-t="r"> <ul class="reference" id="bibsbref9"> <li class="referenceLabel"><a href="#b0045" class="label intra_ref">Chow and Hazama, 1998</a></li> <li class="author">S. Chow, K. Hazama</li> <li class="title"> <p>Universal PCR primers for S7 ribosomal protein gene introns in fish</p> </li> <li class="source"> <p>Mol. Ecol., 7 (1998), pp. 1255–1256</p> </li> <li class="external refPlaceHolder" id="bibsbref9" data-refres="N" /> </ul> </li> <li id="b0050" data-t="r"> <ul class="reference" id="bibsbref10"> <li class="referenceLabel"><a href="#b0050" class="label intra_ref">Chung and Ané, 2011</a></li> <li class="author">Y. Chung, C. Ané</li> <li class="title"> <p>Comparing two bayesian methods for gene tree/species tree reconstruction: simulations with incomplete lineage sorting and horizontal gene transfer </p> </li> <li class="source"> <p>Syst. Biol., 60 (2011), pp. 261–275</p> </li> <li class="external refPlaceHolder" id="bibsbref10" data-refres="N" /> </ul> </li> <li id="b0055" data-t="r"> <ul class="reference" id="bibsbref11"> <li class="referenceLabel"><a href="#b0055" class="label intra_ref">Clabaut et al., 2005</a></li> <li class="author">C. Clabaut, W. Salzburger, A. Meyer</li> <li class="title"> <p>Comparative phylogenetic analyses of the adaptive radiation of Lake Tanganyika cichlid fish: nuclear sequences are less homoplasious but also less informative than mitochondrial DNA </p> </li> <li class="source"> <p>J. Mol. Evol., 61 (2005), pp. 666–681</p> </li> <li class="external refPlaceHolder" id="bibsbref11" data-refres="N" /> </ul> </li> <li id="b0060" data-t="r"> <ul class="reference" id="bibsbref12"> <li class="referenceLabel"><a href="#b0060" class="label intra_ref">Cohen et al., 1997</a></li> <li class="author">A.S. Cohen, K.E. Lezzar, J.J. Tiercelin, M. Soreghan</li> <li class="title"> <p>New palaeogeographic and lake-level reconstructions of Lake Tanganyika: implications for tectonic, climatic and biological evolution in a rift lake </p> </li> <li class="source"> <p>Basin Res., 9 (1997), pp. 107–132</p> </li> <li class="external refPlaceHolder" id="bibsbref12" data-refres="N" /> </ul> </li> <li id="b0065" data-t="r"> <ul class="reference" id="bibsbref13"> <li class="referenceLabel"><a href="#b0065" class="label intra_ref">Day et al., 2008</a></li> <li class="author">J.J. Day, J.A. Cotton, T.G. Barraclough</li> <li class="title"> <p>Tempo and mode of diversification of Lake Tanganyika cichlid fishes</p> </li> <li class="source"> <p>PLoS ONE, 3 (2008), p. e1730</p> </li> <li class="external refPlaceHolder" id="bibsbref13" data-refres="N" /> </ul> </li> <li id="b0070" data-t="r"> <ul class="reference" id="bibsbref14"> <li class="referenceLabel"><a href="#b0070" class="label intra_ref">Degnan and Rosenberg, 2009</a></li> <li class="author">J.H. Degnan, N.A. Rosenberg</li> <li class="title"> <p>Gene tree discordance, phylogenetic inference and the multispecies coalescent</p> </li> <li class="source"> <p>Trends Ecol. Evol., 24 (2009), pp. 332–340</p> </li> <li class="external refPlaceHolder" id="bibsbref14" data-refres="N" /> </ul> </li> <li id="b0075" data-t="r"> <ul class="reference" id="bibsbref15"> <li class="referenceLabel"><a href="#b0075" class="label intra_ref">Duftner et al., 2005</a></li> <li class="author">N. Duftner, S. Koblmuller, C. Sturmbauer</li> <li class="title"> <p>Evolutionary relationships of the Limnochromini, a tribe of benthic deepwater cichlid fish endemic to Lake Tanganyika, East Africa </p> </li> <li class="source"> <p>J. Mol. Evol., 60 (2005), pp. 277–289</p> </li> <li class="external refPlaceHolder" id="bibsbref15" data-refres="N" /> </ul> </li> <li id="b0080" data-t="r"> <ul class="reference" id="bibsbref16"> <li class="referenceLabel"><a href="#b0080" class="label intra_ref">Dunz and Schliewen, 2013</a></li> <li class="author">A.R. Dunz, U.K. Schliewen</li> <li class="title"> <p>Molecular phylogeny and revised classification of the haplotilapiine cichlid fishes formerly referred to as "<em>Tilapia</em>" </p> </li> <li class="source"> <p>Mol. Phylogenet. Evol., 68 (2013), pp. 64–80</p> </li> <li class="external refPlaceHolder" id="bibsbref16" data-refres="N" /> </ul> </li> <li id="b0085" data-t="r"> <ul class="reference" id="bibsbref17"> <li class="referenceLabel"><a href="#b0085" class="label intra_ref">Friedman et al., 2013</a></li> <li class="author">M. Friedman, B.P. Keck, A. Dornburg, R.I. Eytan, C.H. Martin, C.D. Hulsey, P.C. Wainwright, T.J. Near </li> <li class="title"> <p>Molecular and fossil evidence place the origin of cichlid fishes long after Gondwanan rifting </p> </li> <li class="source"> <p>Proc. R. Soc. Lond. B, 280 (2013), p. 20131733</p> </li> <li class="external refPlaceHolder" id="bibsbref17" data-refres="N" /> </ul> </li> <li id="b0090" data-t="r"> <ul class="reference" id="bibsbref18"> <li class="referenceLabel"><a href="#b0090" class="label intra_ref">Gadagkar et al., 2005</a></li> <li class="author">S.R. Gadagkar, M.S. Rosenberg, S. Kumar</li> <li class="title"> <p>Inferring species phylogenies from multiple genes: concatenated sequence tree versus consensus gene tree </p> </li> <li class="source"> <p>J. Exp. Zool. B. Mol. Dev. Evol., 304 (2005), pp. 64–74</p> </li> <li class="external refPlaceHolder" id="bibsbref18" data-refres="N" /> </ul> </li> <li id="b0095" data-t="r"> <ul class="reference" id="bibsbref19"> <li class="referenceLabel"><a href="#b0095" class="label intra_ref">Gavrilets and Vose, 2005</a></li> <li class="author">S. Gavrilets, A. Vose</li> <li class="title"> <p>Dynamic patterns of adaptive radiation</p> </li> <li class="source"> <p>Proc. Natl. Acad. Sci. USA, 102 (2005), pp. 18040–18045</p> </li> <li class="external refPlaceHolder" id="bibsbref19" data-refres="N" /> </ul> </li> <li id="b0100" data-t="r"> <ul class="reference" id="bibsbref20"> <li class="referenceLabel"><a href="#b0100" class="label intra_ref">Genner et al., 2007</a></li> <li class="author">M.J. Genner, O. Seehausen, D.H. Lunt, D.A. Joyce, P.W. Shaw, G.R. Carvalho, G.F. Turner</li> <li class="title"> <p>Age of cichlids: new dates for ancient lake fish radiations</p> </li> <li class="source"> <p>Mol. Biol. Evol., 24 (2007), pp. 1269–1282</p> </li> <li class="external refPlaceHolder" id="bibsbref20" data-refres="N" /> </ul> </li> <li id="b0105" data-t="r"> <ul class="reference" id="bibsbref21"> <li class="referenceLabel"><a href="#b0105" class="label intra_ref">Glor, 2010</a></li> <li class="author">R.E. Glor</li> <li class="title"> <p>Phylogenetic insights on adaptive radiation</p> </li> <li class="source"> <p>Ann. Rev. Ecol. Evol. Syst., 41 (2010), pp. 251–270</p> </li> <li class="external refPlaceHolder" id="bibsbref21" data-refres="N" /> </ul> </li> <li id="b0110" data-t="r"> <ul class="reference" id="bibsbref22"> <li class="referenceLabel"><a href="#b0110" class="label intra_ref">Heled and Drummond, 2010</a></li> <li class="author">J. Heled, A.J. Drummond</li> <li class="title"> <p>Bayesian inference of species trees from multilocus data</p> </li> <li class="source"> <p>Mol. Biol. Evol., 27 (2010), pp. 570–580</p> </li> <li class="external refPlaceHolder" id="bibsbref22" data-refres="N" /> </ul> </li> <li id="b0115" data-t="r"> <ul class="reference" id="bibsbref23"> <li class="referenceLabel"><a href="#b0115" class="label intra_ref">Huson and Bryant, 2006</a></li> <li class="author">D.H. Huson, D. Bryant</li> <li class="title"> <p>Application of phylogenetic networks in evolutionary studies</p> </li> <li class="source"> <p>Mol. Biol. Evol., 23 (2006), pp. 254–267</p> </li> <li class="external refPlaceHolder" id="bibsbref23" data-refres="N" /> </ul> </li> <li id="b0120" data-t="r"> <ul class="reference" id="bibsbref24"> <li class="referenceLabel"><a href="#b0120" class="label intra_ref">Joyce et al., 2011</a></li> <li class="author">D.A. Joyce, D.H. Lunt, M.J. Genner, G.F. Turner, R. Bills, O. Seehausen</li> <li class="title"> <p>Repeated colonization and hybridization in Lake Malawi cichlids</p> </li> <li class="source"> <p>Curr. Biol., 21 (2011), pp. R108–109</p> </li> <li class="external refPlaceHolder" id="bibsbref24" data-refres="N" /> </ul> </li> <li id="b0125" data-t="r"> <ul class="reference" id="bibsbref25"> <li class="referenceLabel"><a href="#b0125" class="label intra_ref">Katoh and Standley, 2013</a></li> <li class="author">K. Katoh, D.M. Standley</li> <li class="title"> <p>MAFFT multiple sequence alignment software version 7: improvements in performance and usability </p> </li> <li class="source"> <p>Mol. Biol. Evol., 30 (2013), pp. 772–780</p> </li> <li class="external refPlaceHolder" id="bibsbref25" data-refres="N" /> </ul> </li> <li id="b0130" data-t="r"> <ul class="reference" id="bibsbref26"> <li class="referenceLabel"><a href="#b0130" class="label intra_ref">Kirchberger et al., 2014</a></li> <li class="author">P.C. Kirchberger, K.M. Sefc, C. Sturmbauer, S. Koblmüller</li> <li class="title"> <p>Outgroup effects on root position and tree topology in the AFLP phylogeny of a rapidly radiating lineage of cichlid fish </p> </li> <li class="source"> <p>Mol. Phylogenet. Evol., 70 (2014), pp. 57–62</p> </li> <li class="external refPlaceHolder" id="bibsbref26" data-refres="N" /> </ul> </li> <li id="b0135" data-t="r"> <ul class="reference" id="bibsbref27"> <li class="referenceLabel"><a href="#b0135" class="label intra_ref">Klett and Meyer, 2002</a></li> <li class="author">V. Klett, A. Meyer</li> <li class="title"> <p>What, if anything, is a Tilapia? – mitochondrial ND2 phylogeny of tilapiines and the evolution of parental care systems in the African cichlid fishes </p> </li> <li class="source"> <p>Mol. Biol. Evol., 19 (2002), pp. 865–883</p> </li> <li class="external refPlaceHolder" id="bibsbref27" data-refres="N" /> </ul> </li> <li id="b0140" data-t="r"> <ul class="reference" id="bibsbref28"> <li class="referenceLabel"><a href="#b0140" class="label intra_ref">Knowles and Kubatko, 2011</a></li> <li class="author">L.L. Knowles, L.S. Kubatko</li> <li class="title"> <p>Estimating Speciest: Practical and Theoretical Aspects</p> </li> <li class="source"> <p>Wiley (2011)</p> </li> <li class="external refPlaceHolder" id="bibsbref28" data-refres="N" /> </ul> </li> <li id="b0145" data-t="r"> <ul class="reference" id="bibsbref29"> <li class="referenceLabel"><a href="#b0145" class="label intra_ref">Koblmüller et al., 2005</a></li> <li class="author">S. Koblmüller, N. Duftner, C. Katongo, H. Phiri, C. Sturmbauer</li> <li class="title"> <p>Ancient divergence in bathypelagic Lake Tanganyika deepwater cichlids: mitochondrial phylogeny of the tribe Bathybatini </p> </li> <li class="source"> <p>J. Mol. Evol., 60 (2005), pp. 297–314</p> </li> <li class="external refPlaceHolder" id="bibsbref29" data-refres="N" /> </ul> </li> <li id="b0150" data-t="r"> <ul class="reference" id="bibsbref30"> <li class="referenceLabel"><a href="#b0150" class="label intra_ref">Koblmüller et al., 2007</a></li> <li class="author">S. Koblmüller, N. Duftner, K.M. Sefc, M. Aibara, M. Stipacek, M. Blanc, B. Egger, C. Sturmbauer </li> <li class="title"> <p>Reticulate phylogeny of gastropod-shell-breeding cichlids from Lake Tanganyika – the result of repeated introgressive hybridization </p> </li> <li class="source"> <p>BMC Evol. Biol., 7 (2007), p. 7</p> </li> <li class="external refPlaceHolder" id="bibsbref30" data-refres="N" /> </ul> </li> <li id="b0155" data-t="r"> <ul class="reference" id="bibsbref31"> <li class="referenceLabel"><a href="#b0155" class="label intra_ref">Koblmüller et al., 2008a</a></li> <li class="author">S. Koblmüller, U.K. Schliewen, N. Duftner, K.M. Sefc, C. Katongo, C. Sturmbauer</li> <li class="title"> <p>Age and spread of the haplochromine cichlid fishes in Africa</p> </li> <li class="source"> <p>Mol. Phylogenet. Evol., 49 (2008), pp. 153–169</p> </li> <li class="external refPlaceHolder" id="bibsbref31" data-refres="N" /> </ul> </li> <li id="b0160" data-t="r"> <ul class="reference" id="bibsbref32"> <li class="referenceLabel"><a href="#b0160" class="label intra_ref">Koblmüller et al., 2008b</a></li> <li class="author">S. Koblmüller, K.M. Sefc, C. Sturmbauer</li> <li class="title"> <p>The Lake Tanganyika cichlid species assemblage: recent advances in molecular phylogenetics</p> </li> <li class="source"> <p>Hydrobiol., 615 (2008), pp. 5–20</p> </li> <li class="external refPlaceHolder" id="bibsbref32" data-refres="N" /> </ul> </li> <li id="b0165" data-t="r"> <ul class="reference" id="bibsbref33"> <li class="referenceLabel"><a href="#b0165" class="label intra_ref">Koch et al., 2007</a></li> <li class="author">M. Koch, S. Koblmüller, K. Sefc</li> <li class="title"> <p>Evolutionary history of the endemic Lake Tanganyika cichlid fish <em>Tylochromis polylepis</em>: a recent intruder to a mature adaptive radiation </p> </li> <li class="source"> <p>J. Zool. Syst. Evol. Res., 45 (2007), pp. 64–71</p> </li> <li class="external refPlaceHolder" id="bibsbref33" data-refres="N" /> </ul> </li> <li id="b0170" data-t="r"> <ul class="reference" id="bibsbref34"> <li class="referenceLabel"><a href="#b0170" class="label intra_ref">Kocher, 2004</a></li> <li class="author">T.D. Kocher</li> <li class="title"> <p>Adaptive evolution and explosive speciation: the cichlid fish model</p> </li> <li class="source"> <p>Nature Rev. Genet., 5 (2004), pp. 288–298</p> </li> <li class="external refPlaceHolder" id="bibsbref34" data-refres="N" /> </ul> </li> <li id="b0175" data-t="r"> <ul class="reference" id="bibsbref35"> <li class="referenceLabel"><a href="#b0175" class="label intra_ref">Kocher et al., 1995</a></li> <li class="author">T.D. Kocher, J.A. Conroy, K.R. McKaye, J.R. Stauffer, S.F. Lockwood</li> <li class="title"> <p>Evolution of NADH dehydrogenase subunit 2 in east African cichlid fish</p> </li> <li class="source"> <p>Mol. Phylogenet. Evol., 4 (1995), pp. 420–432</p> </li> <li class="external refPlaceHolder" id="bibsbref35" data-refres="N" /> </ul> </li> <li id="b0180" data-t="r"> <ul class="reference" id="bibsbref36"> <li class="referenceLabel"><a href="#b0180" class="label intra_ref">Kubatko and Degnan, 2007</a></li> <li class="author">L. Kubatko, J. Degnan</li> <li class="title"> <p>Inconsistency of phylogenetic estimates from concatenated data under coalescence</p> </li> <li class="source"> <p>Syst. Biol., 56 (2007), pp. 17–24</p> </li> <li class="external refPlaceHolder" id="bibsbref36" data-refres="N" /> </ul> </li> <li id="b0185" data-t="r"> <ul class="reference" id="bibsbref37"> <li class="referenceLabel"><a href="#b0185" class="label intra_ref">Kumar et al., 2009</a></li> <li class="author">S. Kumar, A. Skjaeveland, R.J. Orr, P. Enger, T. Ruden, B.H. Mevik, F. Burki, A. Botnen, K. Shalchian-Tabrizi </li> <li class="title"> <p>AIR: a batch-oriented web program package for construction of supermatrices ready for phylogenomic analyses </p> </li> <li class="source"> <p>BMC Bioinf., 10 (2009), p. 357</p> </li> <li class="external refPlaceHolder" id="bibsbref37" data-refres="N" /> </ul> </li> <li id="b0190" data-t="r"> <ul class="reference" id="bibsbref38"> <li class="referenceLabel"><a href="#b0190" class="label intra_ref">Lanfear et al., 2012</a></li> <li class="author">R. Lanfear, B. Calcott, S.Y. Ho, S. Guindon</li> <li class="title"> <p>Partitionfinder: combined selection of partitioning schemes and substitution models for phylogenetic analyses </p> </li> <li class="source"> <p>Mol. Biol. Evol., 29 (2012), pp. 1695–1701</p> </li> <li class="external refPlaceHolder" id="bibsbref38" data-refres="N" /> </ul> </li> <li id="b0195" data-t="r"> <ul class="reference" id="bibsbref39"> <li class="referenceLabel"><a href="#b0195" class="label intra_ref">Lang et al., 2006</a></li> <li class="author">M. Lang, T. Miyake, I. Braasch, D. Tinnemore, N. Siegel, W. Salzburger, C.T. Amemiya, A. Meyer </li> <li class="title"> <p>A BAC library of the East African haplochromine cichlid fish <em>Astatotilapia burtoni</em></p> </li> <li class="source"> <p>J. Exp. Zool. B. Mol. Dev. Evol., 306 (2006), pp. 35–44</p> </li> <li class="external refPlaceHolder" id="bibsbref39" data-refres="N" /> </ul> </li> <li id="b0200" data-t="r"> <ul class="reference" id="bibsbref40"> <li class="referenceLabel"><a href="#b0200" class="label intra_ref">Larget et al., 2010</a></li> <li class="author">B.R. Larget, S.K. Kotha, C.N. Dewey, C. Ane</li> <li class="title"> <p>BUCKy: gene tree/species tree reconciliation with Bayesian concordance analysis</p> </li> <li class="source"> <p>Bioinformatics, 26 (2010), pp. 2910–2911</p> </li> <li class="external refPlaceHolder" id="bibsbref40" data-refres="N" /> </ul> </li> <li id="b0205" data-t="r"> <ul class="reference" id="bibsbref41"> <li class="referenceLabel"><a href="#b0205" class="label intra_ref">Leigh et al., 2008</a></li> <li class="author">J.W. Leigh, E. Susko, M. Baumgartner, A.J. Roger</li> <li class="title"> <p>Testing congruence in phylogenomic analysis</p> </li> <li class="source"> <p>Syst. Biol., 57 (2008), pp. 104–115</p> </li> <li class="external refPlaceHolder" id="bibsbref41" data-refres="N" /> </ul> </li> <li id="b0210" data-t="r"> <ul class="reference" id="bibsbref42"> <li class="referenceLabel"><a href="#b0210" class="label intra_ref">Lessa, 1992</a></li> <li class="author">E.P. Lessa</li> <li class="title"> <p>Rapid surveying of DNA sequence variation in natural populations</p> </li> <li class="source"> <p>Mol. Biol. Evol., 9 (1992), p. 323</p> </li> <li class="external refPlaceHolder" id="bibsbref42" data-refres="N" /> </ul> </li> <li id="b0215" data-t="r"> <ul class="reference" id="bibsbref43"> <li class="referenceLabel"><a href="#b0215" class="label intra_ref">Lewis, 2001</a></li> <li class="author">P.O. Lewis</li> <li class="title"> <p>A likelihood approach to estimating phylogeny from discrete morphological character data </p> </li> <li class="source"> <p>Syst. Biol., 50 (2001), pp. 913–925</p> </li> <li class="external refPlaceHolder" id="bibsbref43" data-refres="N" /> </ul> </li> <li id="b0220" data-t="r"> <ul class="reference" id="bibsbref44"> <li class="referenceLabel"><a href="#b0220" class="label intra_ref">Li and Durbin, 2010</a></li> <li class="author">H. Li, R. Durbin</li> <li class="title"> <p>Fast and accurate long-read alignment with Burrows-Wheeler transform</p> </li> <li class="source"> <p>Bioinformatics, 26 (2010), pp. 589–595</p> </li> <li class="external refPlaceHolder" id="bibsbref44" data-refres="N" /> </ul> </li> <li id="b0225" data-t="r"> <ul class="reference" id="bibsbref45"> <li class="referenceLabel"><a href="#b0225" class="label intra_ref">Li et al., 2007</a></li> <li class="author">C. Li, G. Ortí, G. Zhang, G. Lu</li> <li class="title"> <p>A practical approach to phylogenomics: the phylogeny of ray-finned fish (Actinopterygii) as a case study </p> </li> <li class="source"> <p>BMC Evol. Biol., 7 (2007), p. 44</p> </li> <li class="external refPlaceHolder" id="bibsbref45" data-refres="N" /> </ul> </li> <li id="b0230" data-t="r"> <ul class="reference" id="bibsbref46"> <li class="referenceLabel"><a href="#b0230" class="label intra_ref">Liu, 2008</a></li> <li class="author">L. Liu</li> <li class="title"> <p>BEST: Bayesian estimation of species trees under the coalescent model</p> </li> <li class="source"> <p>Bioinformatics, 24 (2008), pp. 2542–2543</p> </li> <li class="external refPlaceHolder" id="bibsbref46" data-refres="N" /> </ul> </li> <li id="b0235" data-t="r"> <ul class="reference" id="bibsbref47"> <li class="referenceLabel"><a href="#b0235" class="label intra_ref">Maddison, 1989</a></li> <li class="author">W. Maddison</li> <li class="title"> <p>Reconstructing character evolution on polytomous cladograms</p> </li> <li class="source"> <p>Cladistics, 5 (1989), pp. 365–377</p> </li> <li class="external refPlaceHolder" id="bibsbref47" data-refres="N" /> </ul> </li> <li id="b0240" data-t="r"> <ul class="reference" id="bibsbref48"> <li class="referenceLabel"><a href="#b0240" class="label intra_ref">McCormack et al., 2013</a></li> <li class="author">J.E. McCormack, S.M. Hird, A.J. Zellmer, B.C. Carstens, R.T. Brumfield</li> <li class="title"> <p>Applications of next-generation sequencing to phylogeography and phylogenetics</p> </li> <li class="source"> <p>Mol. Phylogenet. Evol., 66 (2013), pp. 526–538</p> </li> <li class="external refPlaceHolder" id="bibsbref48" data-refres="N" /> </ul> </li> <li id="b0245" data-t="r"> <ul class="reference" id="bibsbref49"> <li class="referenceLabel"><a href="#b0245" class="label intra_ref">Meng and Kubatko, 2009</a></li> <li class="author">C. Meng, L.S. Kubatko</li> <li class="title"> <p>Detecting hybrid speciation in the presence of incomplete lineage sorting using gene tree incongruence: a model </p> </li> <li class="source"> <p>Theor. Popul. Biol., 75 (2009), pp. 35–45</p> </li> <li class="external refPlaceHolder" id="bibsbref49" data-refres="N" /> </ul> </li> <li id="b0250" data-t="r"> <ul class="reference" id="bibsbref50"> <li class="referenceLabel"><a href="#b0250" class="label intra_ref">Meyer and Salzburger, 2012</a></li> <li class="author">B.S. Meyer, W. Salzburger</li> <li class="title"> <p>A novel primer set for multilocus phylogenetic inference in East African cichlid fishes</p> </li> <li class="source"> <p>Mol. Ecol. Res., 12 (2012), pp. 1097–1104</p> </li> <li class="external refPlaceHolder" id="bibsbref50" data-refres="N" /> </ul> </li> <li id="b0255" data-t="r"> <ul class="reference" id="bibsbref51"> <li class="referenceLabel"><a href="#b0255" class="label intra_ref">Miller et al., 2010</a></li> <li class="author">M.A. Miller, W. Pfeiffer, T. Schwartz</li> <li class="title"> <p>Creating the CIPRES Science Gateway for inference of large phylogenetic trees</p> </li> <li class="source"> <p>Gateway Comput. Environ. Workshop (GCE), 2010 (2010), pp. 1–8</p> </li> <li class="external refPlaceHolder" id="bibsbref51" data-refres="N" /> </ul> </li> <li id="b0260" data-t="r"> <ul class="reference" id="bibsbref52"> <li class="referenceLabel"><a href="#b0260" class="label intra_ref">Müller, 2005</a></li> <li class="author">K. Müller</li> <li class="title"> <p>SeqState: primer design and sequence statistics for phylogenetic DNA datasets</p> </li> <li class="source"> <p>Appl. Bioinf., 4 (2005), pp. 65–69</p> </li> <li class="external refPlaceHolder" id="bibsbref52" data-refres="N" /> </ul> </li> <li id="b0265" data-t="r"> <ul class="reference" id="bibsbref53"> <li class="referenceLabel"><a href="#b0265" class="label intra_ref">Muschick et al., 2012</a></li> <li class="author">M. Muschick, A. Indermaur, W. Salzburger</li> <li class="title"> <p>Convergent evolution within an adaptive radiation of cichlid fishes</p> </li> <li class="source"> <p>Curr. Biol., 22 (2012), pp. 2362–2368</p> </li> <li class="external refPlaceHolder" id="bibsbref53" data-refres="N" /> </ul> </li> <li id="b0270" data-t="r"> <ul class="reference" id="bibsbref54"> <li class="referenceLabel"><a href="#b0270" class="label intra_ref">Nishida, 1991</a></li> <li class="author">M. Nishida</li> <li class="title"> <p>Lake Tanganyika as an evolutionary reservoir of old lineages of East African cichlid fishes: inferences from allozyme data </p> </li> <li class="source"> <p>Experientia, 47 (1991), pp. 974–979</p> </li> <li class="external refPlaceHolder" id="bibsbref54" data-refres="N" /> </ul> </li> <li id="b0275" data-t="r"> <ul class="reference" id="bibsbref55"> <li class="referenceLabel"><a href="#b0275" class="label intra_ref">Nylander et al., 2008</a></li> <li class="author">J.A.A. Nylander, J.C. Wilgenbusch, D.L. Warren, D.L. Swofford</li> <li class="title"> <p>AWTY (are we there yet?): a system for graphical exploration of MCMC convergence in Bayesian phylogenetics </p> </li> <li class="source"> <p>Bioinformatics, 24 (2008), pp. 581–583</p> </li> <li class="external refPlaceHolder" id="bibsbref55" data-refres="N" /> </ul> </li> <li id="b0280" data-t="r"> <ul class="reference" id="bibsbref56"> <li class="referenceLabel"><a href="#b0280" class="label intra_ref">Pamilo and Nei, 1988</a></li> <li class="author">P. Pamilo, M. Nei</li> <li class="title"> <p>Relationships between gene trees and species trees</p> </li> <li class="source"> <p>Mol. Biol. Evol., 5 (1988), pp. 568–583</p> </li> <li class="external refPlaceHolder" id="bibsbref56" data-refres="N" /> </ul> </li> <li id="b0285" data-t="r"> <ul class="reference" id="bibsbref57"> <li class="referenceLabel"><a href="#b0285" class="label intra_ref">Poll, 1986</a></li> <li class="author">M. Poll</li> <li class="title"> <p>Classification des Cichlidae du lac Tanganika. Tribus, genres et especes. Academie Royale de Belgique </p> </li> <li class="source"> <p>Classe des Sciences. Memoires, 45 (1986), pp. 1–163</p> </li> <li class="external refPlaceHolder" id="bibsbref57" data-refres="N" /> </ul> </li> <li id="b0290" data-t="r"> <ul class="reference" id="bibsbref58"> <li class="referenceLabel"><a href="#b0290" class="label intra_ref">Rambaut and Drummond, 2007</a></li> <li class="author">Rambaut, A., Drummond, A.J., 2007. Tracer v1.5 &amp;lt;<span id="ir0020" class="interref" data-locatortype="url" data-locatorkey="http://beast.bio.ed.ac.uk/Tracer">http://beast.bio.ed.ac.uk/Tracer</span>&amp;gt;. </li> <li class="external refPlaceHolder" id="bibsbref58" data-refres="N" /> </ul> </li> <li id="b0295" data-t="r"> <ul class="reference" id="bibsbref59"> <li class="referenceLabel"><a href="#b0295" class="label intra_ref">Rokas et al., 2003</a></li> <li class="author">A. Rokas, B.L. Williams, N. King, S.B. Carroll</li> <li class="title"> <p>Genome-scale approaches to resolving incongruence in molecular phylogenies</p> </li> <li class="source"> <p>Nature, 425 (2003), pp. 798–804</p> </li> <li class="external refPlaceHolder" id="bibsbref59" data-refres="N" /> </ul> </li> <li id="b0300" data-t="r"> <ul class="reference" id="bibsbref60"> <li class="referenceLabel"><a href="#b0300" class="label intra_ref">Ronquist et al., 2012</a></li> <li class="author">F. Ronquist, M. Teslenko, P. van der Mark, D.L. Ayres, A. Darling, S. Hohna, B. Larget, L. Liu, M.A. Suchard, J.P. Huelsenbeck </li> <li class="title"> <p>MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space </p> </li> <li class="source"> <p>Syst. Biol., 61 (2012), pp. 539–542</p> </li> <li class="external refPlaceHolder" id="bibsbref60" data-refres="N" /> </ul> </li> <li id="b0305" data-t="r"> <ul class="reference" id="bibsbref61"> <li class="referenceLabel"><a href="#b0305" class="label intra_ref">Salichos and Rokas, 2013</a></li> <li class="author">L. Salichos, A. Rokas</li> <li class="title"> <p>Inferring ancient divergences requires genes with strong phylogenetic signals</p> </li> <li class="source"> <p>Nature, 497 (2013), pp. 327–331</p> </li> <li class="external refPlaceHolder" id="bibsbref61" data-refres="N" /> </ul> </li> <li id="b0310" data-t="r"> <ul class="reference" id="bibsbref62"> <li class="referenceLabel"><a href="#b0310" class="label intra_ref">Salzburger, 2009</a></li> <li class="author">W. Salzburger</li> <li class="title"> <p>The interaction of sexually and naturally selected traits in the adaptive radiations of cichlid fishes </p> </li> <li class="source"> <p>Mol. Ecol., 18 (2009), pp. 169–185</p> </li> <li class="external refPlaceHolder" id="bibsbref62" data-refres="N" /> </ul> </li> <li id="b0315" data-t="r"> <ul class="reference" id="bibsbref63"> <li class="referenceLabel"><a href="#b0315" class="label intra_ref">Salzburger and Meyer, 2004</a></li> <li class="author">W. Salzburger, A. Meyer</li> <li class="title"> <p>The species flocks of East African cichlid fishes: recent advances in molecular phylogenetics and population genetics </p> </li> <li class="source"> <p>Naturwissenschaften, 91 (2004), pp. 277–290</p> </li> <li class="external refPlaceHolder" id="bibsbref63" data-refres="N" /> </ul> </li> <li id="b0320" data-t="r"> <ul class="reference" id="bibsbref64"> <li class="referenceLabel"><a href="#b0320" class="label intra_ref">Salzburger et al., 2002a</a></li> <li class="author">W. Salzburger, A. Meyer, S. Baric, E. Verheyen, C. Sturmbauer</li> <li class="title"> <p>Phylogeny of the Lake Tanganyika cichlid species flock and its relationship to the Central and East African haplochromine cichlid fish faunas </p> </li> <li class="source"> <p>Syst. Biol., 51 (2002), pp. 113–135</p> </li> <li class="external refPlaceHolder" id="bibsbref64" data-refres="N" /> </ul> </li> <li id="b0325" data-t="r"> <ul class="reference" id="bibsbref65"> <li class="referenceLabel"><a href="#b0325" class="label intra_ref">Salzburger et al., 2002b</a></li> <li class="author">W. Salzburger, S. Baric, C. Sturmbauer</li> <li class="title"> <p>Speciation via introgressive hybridization in East African cichlids?</p> </li> <li class="source"> <p>Mol. Ecol., 11 (2002), pp. 619–625</p> </li> <li class="external refPlaceHolder" id="bibsbref65" data-refres="N" /> </ul> </li> <li id="b0330" data-t="r"> <ul class="reference" id="bibsbref66"> <li class="referenceLabel"><a href="#b0330" class="label intra_ref">Salzburger et al., 2005</a></li> <li class="author">W. Salzburger, T. Mack, E. Verheyen, A. Meyer</li> <li class="title"> <p>Out of Tanganyika: genesis, explosive speciation, key-innovations and phylogeography of the haplochromine cichlid fishes </p> </li> <li class="source"> <p>BMC Evol. Biol., 5 (2005), p. 17</p> </li> <li class="external refPlaceHolder" id="bibsbref66" data-refres="N" /> </ul> </li> <li id="b9000" data-t="r"> <ul class="reference" id="bibsbref67"> <li class="referenceLabel"><a href="#b9000" class="label intra_ref">Salzburger et al., 2014</a></li> <li class="author">W. Salzburger, B. Van Bocxlaer, A.S. Cohen</li> <li class="title"> <p>Ecology and evolution of the African Great Lakes and their faunas</p> </li> <li class="source"> <p>Annu. Rev. Ecol. Evol. Syst., 45 (2014), pp. 519–545</p> </li> <li class="external refPlaceHolder" id="bibsbref67" data-refres="N" /> </ul> </li> <li id="b0335" data-t="r"> <ul class="reference" id="bibsbref68"> <li class="referenceLabel"><a href="#b0335" class="label intra_ref">Santos and Salzburger, 2012</a></li> <li class="author">M.E. Santos, W. Salzburger</li> <li class="title"> <p>Evolution. How cichlids diversify</p> </li> <li class="source"> <p>Science, 338 (2012), pp. 619–621</p> </li> <li class="external refPlaceHolder" id="bibsbref68" data-refres="N" /> </ul> </li> <li id="b0340" data-t="r"> <ul class="reference" id="bibsbref69"> <li class="referenceLabel"><a href="#b0340" class="label intra_ref">Santos et al., 2014</a></li> <li class="author">M.E. Santos, I. Braasch, N. Boileau, B.S. Meyer, L. Sauteur, A. Böhne, H.-G. Belting, M. Affolter, W. Salzburger </li> <li class="title"> <p>The evolution of cichlid fish egg-spots is linked with a cis-regulatory change</p> </li> <li class="source"> <p>Nat. Commun., 5 (2014) <a id="ddDoi" href="http://dx.doi.org/10.1038/ncomms6149" target="doilink" data-url="10.1038/ncomms6149">http://dx.doi.org/10.1038/ncomms6149</a></p> </li> <li class="external refPlaceHolder" id="bibsbref69" data-refres="N" /> </ul> </li> <li id="b0345" data-t="r"> <ul class="reference" id="bibsbref70"> <li class="referenceLabel"><a href="#b0345" class="label intra_ref">Schelly and Stiassny, 2004</a></li> <li class="author">R.C. Schelly, M.L.J. Stiassny</li> <li class="title"> <p>Revision of the Congo River <em>Lamprologus</em> Schilthuis, 1891 (Teleostei: Cichlidae), with descriptions of two new species </p> </li> <li class="source"> <p>Am. Mus. Nov., 3451 (2004), pp. 1–40</p> </li> <li class="external refPlaceHolder" id="bibsbref70" data-refres="N" /> </ul> </li> <li id="b0350" data-t="r"> <ul class="reference" id="bibsbref71"> <li class="referenceLabel"><a href="#b0350" class="label intra_ref">Schelly et al., 2003</a></li> <li class="author">R. Schelly, M.L. Stiassny, L. Seegers</li> <li class="title"> <p><em>Neolamprologus devosi</em> sp. n., a new riverine lamprologine cichlid (Teleostei, Cichlidae) from the lower Malagarasi River </p> </li> <li class="source"> <p>Tanzania. Zootaxa, 373 (2003), pp. 1–11</p> </li> <li class="external refPlaceHolder" id="bibsbref71" data-refres="N" /> </ul> </li> <li id="b0355" data-t="r"> <ul class="reference" id="bibsbref72"> <li class="referenceLabel"><a href="#b0355" class="label intra_ref">Schmieder and Edwards, 2011</a></li> <li class="author">R. Schmieder, R. Edwards</li> <li class="title"> <p>Quality control and preprocessing of metagenomic datasets</p> </li> <li class="source"> <p>Bioinformatics, 27 (2011), pp. 863–864</p> </li> <li class="external refPlaceHolder" id="bibsbref72" data-refres="N" /> </ul> </li> <li id="b0360" data-t="r"> <ul class="reference" id="bibsbref73"> <li class="referenceLabel"><a href="#b0360" class="label intra_ref">Schwarz, 1978</a></li> <li class="author">G. Schwarz</li> <li class="title"> <p>Estimating the dimension of a model</p> </li> <li class="source"> <p>Ann. Stat., 6 (1978), pp. 461–464</p> </li> <li class="external refPlaceHolder" id="bibsbref73" data-refres="N" /> </ul> </li> <li id="b0365" data-t="r"> <ul class="reference" id="bibsbref74"> <li class="referenceLabel"><a href="#b0365" class="label intra_ref">Schwarzer et al., 2012</a></li> <li class="author">J. Schwarzer, E.R. Swartz, E. Vreven, J. Snoeks, F.P.D. Cotterill, B. Misof, U.K. Schliewen </li> <li class="title"> <p>Repeated trans-watershed hybridization among haplochromine cichlids (Cichlidae) was triggered by Neogene landscape evolution </p> </li> <li class="source"> <p>Proc. R. Soc. Lond. B, 279 (2012), pp. 4389–4398</p> </li> <li class="external refPlaceHolder" id="bibsbref74" data-refres="N" /> </ul> </li> <li id="b0370" data-t="r"> <ul class="reference" id="bibsbref75"> <li class="referenceLabel"><a href="#b0370" class="label intra_ref">Seehausen, 2004</a></li> <li class="author">O. Seehausen</li> <li class="title"> <p>Hybridization and adaptive radiation</p> </li> <li class="source"> <p>Trends Ecol. Evol., 19 (2004), pp. 198–207</p> </li> <li class="external refPlaceHolder" id="bibsbref75" data-refres="N" /> </ul> </li> <li id="b0375" data-t="r"> <ul class="reference" id="bibsbref76"> <li class="referenceLabel"><a href="#b0375" class="label intra_ref">Seehausen, 2006</a></li> <li class="author">O. Seehausen</li> <li class="title"> <p>African cichlid fish: a model system in adaptive radiation research</p> </li> <li class="source"> <p>Proc. R. Soc. Lond. B, 273 (2006), pp. 1987–1998</p> </li> <li class="external refPlaceHolder" id="bibsbref76" data-refres="N" /> </ul> </li> <li id="b0380" data-t="r"> <ul class="reference" id="bibsbref77"> <li class="referenceLabel"><a href="#b0380" class="label intra_ref">Shimodaira, 2002</a></li> <li class="author">H. Shimodaira</li> <li class="title"> <p>An approximately unbiased test of phylogenetic tree selection</p> </li> <li class="source"> <p>Syst. Biol., 51 (2002), pp. 492–508</p> </li> <li class="external refPlaceHolder" id="bibsbref77" data-refres="N" /> </ul> </li> <li id="b0385" data-t="r"> <ul class="reference" id="bibsbref78"> <li class="referenceLabel"><a href="#b0385" class="label intra_ref">Shimodaira and Hasegawa, 1999</a></li> <li class="author">H. Shimodaira, M. Hasegawa</li> <li class="title"> <p>Multiple comparisons of log-likelihoods with applications to phylogenetic inference</p> </li> <li class="source"> <p>Mol. Biol. Evol., 16 (1999), pp. 1114–1116</p> </li> <li class="external refPlaceHolder" id="bibsbref78" data-refres="N" /> </ul> </li> <li id="b0390" data-t="r"> <ul class="reference" id="bibsbref79"> <li class="referenceLabel"><a href="#b0390" class="label intra_ref">Simmons and Ochoterena, 2000</a></li> <li class="author">M.P. Simmons, H. Ochoterena</li> <li class="title"> <p>Gaps as characters in sequence-based phylogenetic analyses</p> </li> <li class="source"> <p>Syst. Biol., 49 (2000), pp. 369–381</p> </li> <li class="external refPlaceHolder" id="bibsbref79" data-refres="N" /> </ul> </li> <li id="b0395" data-t="r"> <ul class="reference" id="bibsbref80"> <li class="referenceLabel"><a href="#b0395" class="label intra_ref">Slade et al., 1993</a></li> <li class="author">R.W. Slade, C. Moritz, A. Heideman, P.T. Hale</li> <li class="title"> <p>Rapid assessment of single-copy nuclear DNA variation in diverse species</p> </li> <li class="source"> <p>Mol. Ecol., 2 (1993), pp. 359–373</p> </li> <li class="external refPlaceHolder" id="bibsbref80" data-refres="N" /> </ul> </li> <li id="b0400" data-t="r"> <ul class="reference" id="bibsbref81"> <li class="referenceLabel"><a href="#b0400" class="label intra_ref">Slowinski, 2001</a></li> <li class="author">J.B. Slowinski</li> <li class="title"> <p>Molecular polytomies</p> </li> <li class="source"> <p>Mol. Phylogenet. Evol., 19 (2001), pp. 114–120</p> </li> <li class="external refPlaceHolder" id="bibsbref81" data-refres="N" /> </ul> </li> <li id="b0405" data-t="r"> <ul class="reference" id="bibsbref82"> <li class="referenceLabel"><a href="#b0405" class="label intra_ref">Snoeks, 2000</a></li> <li class="author">J. Snoeks</li> <li class="title"> <p>How well known is the ichthyodiversity of the large East African lakes?</p> </li> <li class="source"> <p>,in: A. Rossiter, H. Kawanabe (Eds.), Advances in Ecological Research, 9780120139316, 0065-250431, Academic Press (2000), pp. 17–38 <a id="ddDoi" href="http://dx.doi.org/10.1016/S0065-2504(00)31005-4" target="doilink" data-url="10.1016/S0065-2504(00)31005-4">http://dx.doi.org/10.1016/S0065-2504(00)31005-4</a></p> </li> <li class="external refPlaceHolder" id="bibsbref82" data-refres="N" /> </ul> </li> <li id="b0410" data-t="r"> <ul class="reference" id="bibsbref83"> <li class="referenceLabel"><a href="#b0410" class="label intra_ref">Soria-Carrasco et al., 2007</a></li> <li class="author">V. Soria-Carrasco, G. Talavera, J. Igea, J. Castresana</li> <li class="title"> <p>The K tree score: quantification of differences in the relative branch length and topology of phylogenetic trees </p> </li> <li class="source"> <p>Bioinformatics, 23 (2007), pp. 2954–2956</p> </li> <li class="external refPlaceHolder" id="bibsbref83" data-refres="N" /> </ul> </li> <li id="b0415" data-t="r"> <ul class="reference" id="bibsbref84"> <li class="referenceLabel"><a href="#b0415" class="label intra_ref">Stamatakis, 2006</a></li> <li class="author">A. Stamatakis</li> <li class="title"> <p>RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models </p> </li> <li class="source"> <p>Bioinformatics, 22 (2006), pp. 2688–2690</p> </li> <li class="external refPlaceHolder" id="bibsbref84" data-refres="N" /> </ul> </li> <li id="b0420" data-t="r"> <ul class="reference" id="bibsbref85"> <li class="referenceLabel"><a href="#b0420" class="label intra_ref">Stiassny, 1990</a></li> <li class="author">M. Stiassny</li> <li class="title"> <p>Tylochromis, relationships and the phylogenetic status of the African Cichlidae</p> </li> <li class="source"> <p>Am. Mus. Nov., 2993 (1990), pp. 1–14</p> </li> <li class="external refPlaceHolder" id="bibsbref85" data-refres="N" /> </ul> </li> <li id="b0425" data-t="r"> <ul class="reference" id="bibsbref86"> <li class="referenceLabel"><a href="#b0425" class="label intra_ref">Sturmbauer and Meyer, 1993</a></li> <li class="author">C. Sturmbauer, A. Meyer</li> <li class="title"> <p>Mitochondrial phylogeny of the endemic mouthbrooding lineages of cichlid fishes from Lake Tanganyika in eastern Africa </p> </li> <li class="source"> <p>Mol. Biol. Evol., 10 (1993), pp. 751–768</p> </li> <li class="external refPlaceHolder" id="bibsbref86" data-refres="N" /> </ul> </li> <li id="b0430" data-t="r"> <ul class="reference" id="bibsbref87"> <li class="referenceLabel"><a href="#b0430" class="label intra_ref">Sturmbauer et al., 2003</a></li> <li class="author">C. Sturmbauer, U. Hainz, S. Baric, E. Verheyen, W. Salzburger</li> <li class="title"> <p>Evolution of the tribe Tropheini from Lake Tanganyika: synchronized explosive speciation producing multiple evolutionary parallelism </p> </li> <li class="source"> <p>Hydrobiologia, 500 (2003), pp. 51–64</p> </li> <li class="external refPlaceHolder" id="bibsbref87" data-refres="N" /> </ul> </li> <li id="b0435" data-t="r"> <ul class="reference" id="bibsbref88"> <li class="referenceLabel"><a href="#b0435" class="label intra_ref">Sturmbauer et al., 2010</a></li> <li class="author">C. Sturmbauer, W. Salzburger, N. Duftner, R. Schelly, S. Koblmuller</li> <li class="title"> <p>Evolutionary history of the Lake Tanganyika cichlid tribe Lamprologini (Teleostei: Perciformes) derived from mitochondrial and nuclear DNA data </p> </li> <li class="source"> <p>Mol. Phylogenet. Evol., 57 (2010), pp. 266–284</p> </li> <li class="external refPlaceHolder" id="bibsbref88" data-refres="N" /> </ul> </li> <li id="b0440" data-t="r"> <ul class="reference" id="bibsbref89"> <li class="referenceLabel"><a href="#b0440" class="label intra_ref">Sukumaran and Holder, 2010</a></li> <li class="author">J. Sukumaran, M.T. Holder</li> <li class="title"> <p>DendroPy: a Python library for phylogenetic computing</p> </li> <li class="source"> <p>Bioinformatics, 26 (2010), pp. 1569–1571</p> </li> <li class="external refPlaceHolder" id="bibsbref89" data-refres="N" /> </ul> </li> <li id="b0445" data-t="r"> <ul class="reference" id="bibsbref90"> <li class="referenceLabel"><a href="#b0445" class="label intra_ref">Swofford, 2003</a></li> <li class="author">Swofford, D.L., 2003. PAUP<sup>∗</sup>. Phylogenetic Analysis Using Parsimony (∗and Other Methods). Sinauer Associates, Sunderland, Massachusetts. </li> <li class="external refPlaceHolder" id="bibsbref90" data-refres="N" /> </ul> </li> <li id="b0450" data-t="r"> <ul class="reference" id="bibsbref91"> <li class="referenceLabel"><a href="#b0450" class="label intra_ref">Takahashi, 2003</a></li> <li class="author">T. Takahashi</li> <li class="title"> <p>Systematics of Tanganyikan cichlid fishes (Teleostei: Perciformes)</p> </li> <li class="source"> <p>Ichthyol. Res., 50 (2003), pp. 367–382</p> </li> <li class="external refPlaceHolder" id="bibsbref91" data-refres="N" /> </ul> </li> <li id="b0455" data-t="r"> <ul class="reference" id="bibsbref92"> <li class="referenceLabel"><a href="#b0455" class="label intra_ref">Takahashi et al., 2001</a></li> <li class="author">K. Takahashi, Y. Terai, M. Nishida, N. Okada</li> <li class="title"> <p>Phylogenetic relationships and ancient incomplete lineage sorting among cichlid fishes in Lake Tanganyika as revealed by analysis of the insertion of retroposons </p> </li> <li class="source"> <p>Mol. Biol. Evol., 18 (2001), pp. 2057–2066</p> </li> <li class="external refPlaceHolder" id="bibsbref92" data-refres="N" /> </ul> </li> <li id="b0460" data-t="r"> <ul class="reference" id="bibsbref93"> <li class="referenceLabel"><a href="#b0460" class="label intra_ref">Tamura et al., 2011</a></li> <li class="author">K. Tamura, D. Peterson, N. Peterson, G. Stecher, M. Nei, S. Kumar</li> <li class="title"> <p>MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods </p> </li> <li class="source"> <p>Mol. Biol. Evol., 28 (2011), pp. 2731–2739</p> </li> <li class="external refPlaceHolder" id="bibsbref93" data-refres="N" /> </ul> </li> <li id="b0465" data-t="r"> <ul class="reference" id="bibsbref94"> <li class="referenceLabel"><a href="#b0465" class="label intra_ref">Turner et al., 2001</a></li> <li class="author">G.F. Turner, O. Seehausen, M.E. Knight, C.J. Allender, R.L. Robinson</li> <li class="title"> <p>How many species of cichlid fishes are there in African lakes?</p> </li> <li class="source"> <p>Mol. Ecol., 10 (2001), pp. 793–806</p> </li> <li class="external refPlaceHolder" id="bibsbref94" data-refres="N" /> </ul> </li> <li id="b0470" data-t="r"> <ul class="reference" id="bibsbref95"> <li class="referenceLabel"><a href="#b0470" class="label intra_ref">Verheyen et al., 2003</a></li> <li class="author">Verheyen, E., Salzburger, W., Snoeks, J., Meyer, A., 2003. Origin of the superflock of cichlid fishes from Lake Victoria, East Africa. Science. </li> <li class="external refPlaceHolder" id="bibsbref95" data-refres="N" /> </ul> </li> <li id="b0475" data-t="r"> <ul class="reference" id="bibsbref96"> <li class="referenceLabel"><a href="#b0475" class="label intra_ref">Wagner et al., 2012</a></li> <li class="author">C.E. Wagner, L.J. Harmon, O. Seehausen</li> <li class="title"> <p>Ecological opportunity and sexual selection together predict adaptive radiation</p> </li> <li class="source"> <p>Nature, 487 (2012), pp. 366–369</p> </li> <li class="external refPlaceHolder" id="bibsbref96" data-refres="N" /> </ul> </li> <li id="b0480" data-t="r"> <ul class="reference" id="bibsbref97"> <li class="referenceLabel"><a href="#b0480" class="label intra_ref">Walsh et al., 1999</a></li> <li class="author">H.E. Walsh, M.G. Kidd, T. Moum, V.L. Friesen</li> <li class="title"> <p>Polytomies and the power of phylogenetic inference</p> </li> <li class="source"> <p>Evolution, 53 (1999), pp. 932–937</p> </li> <li class="external refPlaceHolder" id="bibsbref97" data-refres="N" /> </ul> </li> <li id="b0485" data-t="r"> <ul class="reference" id="bibsbref98"> <li class="referenceLabel"><a href="#b0485" class="label intra_ref">Whitfield and Lockhart, 2007</a></li> <li class="author">J.B. Whitfield, P.J. Lockhart</li> <li class="title"> <p>Deciphering ancient rapid radiations</p> </li> <li class="source"> <p>Trends Ecol. Evol., 22 (2007), pp. 258–265</p> </li> <li class="external refPlaceHolder" id="bibsbref98" data-refres="N" /> </ul> </li> <li id="b0490" data-t="r"> <ul class="reference" id="bibsbref99"> <li class="referenceLabel"><a href="#b0490" class="label intra_ref">Willis et al., 2013</a></li> <li class="author">S.C. Willis, I.P. Farias, G. Orti</li> <li class="title"> <p>Multi-locus species tree for the Amazonian peacock basses (Cichlidae: <em>Cichla</em>): Emergent phylogenetic signal despite limited nuclear variation </p> </li> <li class="source"> <p>Mol. Phylogenet. Evol., 69 (2013), pp. 479–490</p> </li> <li class="external refPlaceHolder" id="bibsbref99" data-refres="N" /> </ul> </li> <li id="b0495" data-t="r"> <ul class="reference" id="bibsbref100"> <li class="referenceLabel"><a href="#b0495" class="label intra_ref">Won et al., 2005</a></li> <li class="author">Y.-J. Won, A. Sivasundar, Y. Wang, J. Hey</li> <li class="title"> <p>On the origin of Lake Malawi cichlid species: a population genetic analysis of divergence</p> </li> <li class="source"> <p>Proc. Natl. Acad. Sci. USA, 102 (Suppl. 1) (2005), pp. 6581–6586</p> </li> <li class="external refPlaceHolder" id="bibsbref100" data-refres="N" /> </ul> </li> <li id="b0500" data-t="r"> <ul class="reference" id="bibsbref101"> <li class="referenceLabel"><a href="#b0500" class="label intra_ref">Yang and Rannala, 2012</a></li> <li class="author">Z. Yang, B. Rannala</li> <li class="title"> <p>Molecular phylogenetics: principles and practice</p> </li> <li class="source"> <p>Nature Rev. Genet. (2012), pp. 303–314</p> </li> <li class="external refPlaceHolder" id="bibsbref101" data-refres="N" /> </ul> </li> <li id="b0505" data-t="r"> <ul class="reference" id="bibsbref102"> <li class="referenceLabel"><a href="#b0505" class="label intra_ref">Yang and Warnow, 2011</a></li> <li class="author">J. Yang, T. Warnow</li> <li class="title"> <p>Fast and accurate methods for phylogenomic analyses</p> </li> <li class="source"> <p>BMC Bioinf., 12 (Suppl 9) (2011), p. S4</p> </li> <li class="external refPlaceHolder" id="bibsbref102" data-refres="N" /> </ul> </li> <li id="b0510" data-t="r"> <ul class="reference" id="bibsbref103"> <li class="referenceLabel"><a href="#b0510" class="label intra_ref">Zwickl, 2006</a></li> <li class="author">Zwickl, D.J., 2006. Genetic Algorithm Approaches for the Phylogenetic Analysis of Large Biological Sequence Datasets Under the Maximum Likelihood Criterion. PhD Thesis. The University of Texas at Austin. </li> <li class="external refPlaceHolder" id="bibsbref103" data-refres="N" /> </ul> </li> </ol> <div class="artFooterContent"> <dl class="correspondence" id="cor1"> <dt class="label topPadd" /> <dd>
 Corresponding authors at: Evolutionary Ecology of Marine Fishes, GEOMAR Helmholtz Centre for Ocean Research Kiel, Düsternbrooker Weg 20, 24105 Kiel, Germany (B.S. Meyer). Zoological Institute, University of Basel, Vesalgasse 1, 4051 Basel, Switzerland (W. Salzburger). </dd> </dl> </div> <p class="copyright">Copyright © 2014 The Authors. Published by Elsevier Inc.</p> </div> <div id="footer-area-dummy" /> <div id="footer-area" aria-label="product support" role="contentinfo"> <div id="sdFoot"> <div id="footer" aria-label="Footer" role="navigation"> <div class="bottomArticle" /> </div> </div> </div> <div style="clear:both;" /> <div id="suggestedPdfList" style="display:none;"> </div> <div id="sdpa" /> <div id="appBackground" /> <div id="appDialog" class="files"> <div id="contentBlk" class="view"> <div id="filesTemplate" /> <div id="namingTemplate" /> </div> <div class="footer"> </div> </div>  </body></html>