<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html>
<html lang="en" dir="ltr" version="HTML+RDFa+MathML 1.1"
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:dc="http://purl.org/dc/terms/"
  xmlns:foaf="http://xmlns.com/foaf/0.1/"
  xmlns:og="http://ogp.me/ns#"
  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
  xmlns:sioc="http://rdfs.org/sioc/ns#"
  xmlns:sioct="http://rdfs.org/sioc/types#"
  xmlns:skos="http://www.w3.org/2004/02/skos/core#"
  xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="description" content="eLife - Open access to the most promising advances in science" />
<link rel="canonical" href="http://elifesciences.org/content/3/e04407" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=10, minimum-scale=1, user-scalable=yes" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=10, minimum-scale=1, user-scalable=yes" />
<link rel="shortlink" href="/node/22851" />
<meta http-equiv="x-dns-prefetch-control" content="on" />
<link rel="dns-prefetch" href="//dex3165296d6d.cloudfront.net" />
<meta name="HW.identifier" content="/elife/3/e04407.atom" />
<meta name="HW.pisa" content="elife;3/0/e04407" />
<meta name="keywords" content="neuroblast lineage, central complex, olfactory projection neurons, functional imaging, orthodenticle, D. melanogaster" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genetic transformation of structural and functional circuitry rewires the Drosophila brain" />
<meta name="DC.Identifier" content="10.7554/eLife.04407" />
<meta name="DC.Date" content="2015-01-28" />
<meta name="DC.Publisher" content="eLife Sciences Publications Limited" />
<meta name="DC.Rights" content="© 2014, Sen et al. This article is distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use and redistribution provided that the original author and source are credited." />
<meta name="DC.Description" content="The cells in the brain—including the neurons that transmit information—work together in groups called neural circuits. These cells develop from precursor cells called neuroblasts. Each neuroblast can produce many cells, and it is likely that cells that develop from the same neuroblast work together in the adult brain in the same neural circuit. How the adult cells develop into their final form plays an important role in creating a neural circuit, but this process is not fully understood.

In many animals, the complexity of their brain makes it difficult to follow how each individual neuroblast develops. However, all of the neuroblasts in the relatively simple brain of the fruit fly Drosophila have been identified. Furthermore, the genes responsible for establishing the initial identity of each neuroblast in the Drosophila brain are known. These genes may also determine which adult neurons develop from the neuroblast, and when each type of neuron is produced. However, the extent to which a single gene can influence the identity of neurons is unclear.

Sen et al. focused on two types of neuroblasts, each of which, although found next to each other in the developing Drosophila brain, produces neurons for different neural circuits. One of the neuroblasts generates the olfactory neurons responsible for detecting smells; the other innervates the ‘central complex’ that has a number of roles, including controlling the fly's movements. A gene called orthodenticle is expressed by the central complex neuroblast, but not by the olfactory neuroblast, and helps to separate the two neural circuits into different regions of the fly brain.

Sen et al. found that deleting the orthodenticle gene from the central complex neuroblast causes it to develop into olfactory neurons instead of central complex neurons. Tests showed that the modified neurons are completely transformed; they not only work like olfactory neurons, but they also have the same structure and molecular properties. Sen et al. have therefore demonstrated that it is possible to drastically alter the circuitry of the fruit fly brain by changing how one gene is expressed in one neuroblast. This suggests that new neural circuits can form relatively easily, and so could help us to understand how different brain structures and neural circuits evolved.

DOI: [http://dx.doi.org/10.7554/eLife.04407.002][1]

 [1]: /lookup/doi/10.7554/eLife.04407.002" />
<meta name="DC.Contributor" content="Sonia Sen" />
<meta name="DC.Contributor" content="Deshou Cao" />
<meta name="DC.Contributor" content="Ramveer Choudhary" />
<meta name="DC.Contributor" content="Silvia Biagini" />
<meta name="DC.Contributor" content="Jing W Wang" />
<meta name="DC.Contributor" content="Heinrich Reichert" />
<meta name="DC.Contributor" content="K VijayRaghavan" />
<meta name="DC.Contributor" content="Graeme W Davis" />
<meta name="og:type" content="article" />
<meta name="article:published_time" content="2015-01-28" />
<meta name="article:section" content="Developmental biology and stem cells" />
<meta name="citation_journal_title" content="eLife" />
<meta name="citation_publisher" content="eLife Sciences Publications Limited" />
<meta name="citation_title" content="Genetic transformation of structural and functional circuitry rewires the Drosophila brain" />
<meta name="citation_publication_date" content="2015/01/28" />
<meta name="citation_mjid" content="elife;3/0/e04407" />
<meta name="citation_id" content="3/0/e04407" />
<meta name="citation_public_url" content="http://elifesciences.org/content/3/e04407" />
<meta name="citation_abstract_html_url" content="http://elifesciences.org/content/3/e04407.abstract" />
<meta name="citation_full_html_url" content="http://elifesciences.org/content/3/e04407.full" />
<meta name="citation_pdf_url" content="http://elifesciences.org/content/elife/3/e04407.full.pdf" />
<meta name="citation_issn" content="2050-084X" />
<meta name="citation_journal_abbrev" content="eLife Sciences" />
<meta name="citation_doi" content="10.7554/eLife.04407" />
<meta name="citation_volume" content="3" />
<meta name="citation_section" content="Developmental biology and stem cells" />
<meta name="citation_firstpage" content="e04407" />
<meta name="citation_author" content="Sonia Sen" />
<meta name="citation_author_institution" content="Department of Developmental Biology and Genetics" />
<meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute for Fundamental Research" />
<meta name="citation_author_institution" content="Manipal University" />
<meta name="citation_author" content="Deshou Cao" />
<meta name="citation_author_institution" content="Division of Biological Sciences" />
<meta name="citation_author_institution" content="University of California, San Diego" />
<meta name="citation_author" content="Ramveer Choudhary" />
<meta name="citation_author_institution" content="Department of Developmental Biology and Genetics" />
<meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute for Fundamental Research" />
<meta name="citation_author_institution" content="Institute for Molecular Oncology" />
<meta name="citation_author" content="Silvia Biagini" />
<meta name="citation_author_institution" content="Department of Developmental Biology and Genetics" />
<meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute for Fundamental Research" />
<meta name="citation_author_institution" content="Institute for Molecular Oncology" />
<meta name="citation_author" content="Jing W Wang" />
<meta name="citation_author_institution" content="Division of Biological Sciences" />
<meta name="citation_author_institution" content="University of California, San Diego" />
<meta name="citation_author" content="Heinrich Reichert" />
<meta name="citation_author_institution" content="Department of Biozentrum" />
<meta name="citation_author_institution" content="University of Basel" />
<meta name="citation_author" content="K VijayRaghavan" />
<meta name="citation_author_institution" content="Department of Developmental Biology and Genetics" />
<meta name="citation_author_institution" content="National Centre for Biological Sciences, Tata Institute for Fundamental Research" />
<meta name="citation_author_email" content="vijay@ncbs.res.in" />
<meta name="citation_editor" content="Graeme W Davis" />
<meta name="citation_editor_institution" content="University of California, San Francisco" />
<meta name="citation_reference" content="citation_journal_title=Journal of Neuroscience;citation_journal_abbrev=J. Neurosci.;citation_author=A. Cardona;citation_author=S. Saalfeld;citation_author=I. Arganda;citation_author=W. Pereanu;citation_author=J. Schindelin;citation_author=V. Hartenstein;citation_title=Identifying Neuronal Lineages of Drosophila by Sequence Analysis of Axon Tracts;citation_pages=7538-7553;citation_volume=30;citation_year=2010;citation_issue=22;citation_issn=0270-6474;citation_pmid=20519528;citation_doi=10.1523/JNEUROSCI.0186-10.2010" />
<meta name="citation_reference" content="citation_journal_title=Nature neuroscience;citation_journal_abbrev=Nat Neurosci;citation_author=YH. Chou;citation_author=ML. Spletter;citation_author=E. Yaksi;citation_author=JC. Leong;citation_author=RI. Wilson;citation_author=L. Luo;citation_title=Diversity and wiring variability of olfactory local interneurons in the Drosophila antennal lobe.;citation_pages=439-449;citation_volume=13;citation_year=2010;citation_issue=4;citation_pmid=20139975;citation_doi=10.1038/nn.2489" />
<meta name="citation_reference" content="citation_journal_title=Nature Reviews Neuroscience;citation_author=L. Custo Greig;citation_author=M. Woodworth;citation_author=M. Galazo;citation_author=H. Padmanabhan;citation_author=J. Macklis;citation_title=Molecular logic of neocortical projection neuron specification, development and diversity;citation_pages=755-769;citation_volume=14;citation_year=2013;citation_pmid=24105342;citation_doi=10.1038/nrn3586" />
<meta name="citation_reference" content="citation_journal_title=Journal of neurogenetics;citation_journal_abbrev=J Neurogenet;citation_author=AM. Dacks;citation_author=DS. Green;citation_author=CM. Root;citation_author=AJ. Nighorn;citation_author=JW. Wang;citation_title=Serotonin modulates olfactory processing in the antennal lobe of Drosophila.;citation_pages=366-377;citation_volume=23;citation_year=2009;citation_issue=4;citation_pmid=19863268;citation_doi=10.3109/01677060903085722" />
<meta name="citation_reference" content="citation_journal_abbrev=Neural Syst Circuits;citation_author=A. Das;citation_author=A. Chiang;citation_author=S. Davla;citation_author=R. Priya;citation_author=H. Reichert;citation_author=K. Vijayraghavan;citation_author=V. Rodrigues;citation_title=Identification and analysis of a glutamatergic local interneuron lineage in the adult Drosophila olfactory system.;citation_pages=4-4;citation_volume=1;citation_year=2011;citation_issue=1;citation_pmid=22330097;citation_doi=10.1186/2042-1001-1-4" />
<meta name="citation_reference" content="citation_journal_title=Developmental Biology;citation_author=A. Das;citation_author=T. Gupta;citation_author=S. Davla;citation_author=LL. Prieto-Godino;citation_author=S. Diegelmann;citation_author=OV. Reddy;citation_author=KV. Raghavan;citation_author=H. Reichert;citation_author=J. Lovick;citation_author=V. Hartenstein;citation_title=Neuroblast lineage-specific origin of the neurons of the Drosophila larval olfactory system;citation_pages=322-337;citation_volume=373;citation_year=2013;citation_pmid=23149077;citation_doi=10.1016/j.ydbio.2012.11.003" />
<meta name="citation_reference" content="citation_journal_title=Neural Development;citation_author=A. Das;citation_author=S. Sen;citation_author=R. Lichtneckert;citation_author=R. Okada;citation_author=K. Ito;citation_author=V. Rodrigues;citation_author=H. Reichert;citation_title=Drosophila olfactory local interneurons and projection neurons derive from a common neuroblast lineage specified by the empty spiracles gene;citation_pages=33;citation_volume=3;citation_year=2008;citation_pmid=19055770;citation_doi=10.1186/1749-8104-3-33" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=N. Deshpande;citation_author=R. Dittrich;citation_author=GM. Technau;citation_author=J. Urban;citation_title=Successive specification of Drosophila neuroblasts NB 6-4 and NB 7-3 depends on interaction of the segment polarity genes wingless, gooseberry and naked cuticle;citation_pages=3253-3261;citation_volume=128;citation_year=2001;citation_issue=17;citation_issn=0950-1991;citation_pmid=11546742" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=SJ. Franco;citation_author=U. Muller;citation_title=Shaping our Minds: stem and progenitor cell diversity in the Mammalian neocortex;citation_pages=19-34;citation_volume=77;citation_year=2013;citation_pmid=23312513;citation_doi=10.1016/j.neuron.2012.12.022" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=P. Gao;citation_author=KT. Sultan;citation_author=XJ. Zhang;citation_author=SH. Shi;citation_title=Lineage-dependent circuit assembly in the neocortex;citation_pages=2645-2655;citation_volume=140;citation_year=2013;citation_issue=13;citation_issn=0950-1991;citation_pmid=23757410;citation_doi=10.1242/dev.087668" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=K. Ito;citation_author=W. Awano;citation_author=K. Suzuki;citation_author=Y. Hiromi;citation_author=D. Yamamoto;citation_title=The Drosophila mushroom body is a quadruple structure of clonal units each of which contains a virtually identical set of neurones and glial cells;citation_pages=761-771;citation_volume=124;citation_year=1997;citation_issue=4;citation_issn=0950-1991;citation_pmid=9043058" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=K. Ito;citation_author=Y. Hotta;citation_title=Proliferation pattern of postembryonic neuroblasts in the brain of Drosophila melanogaster.;citation_pages=134-148;citation_volume=149;citation_year=1992;citation_issue=1;citation_pmid=1728583;citation_doi=10.1016/0012-1606(92)90270-Q" />
<meta name="citation_reference" content="citation_journal_title=Current Biology;citation_author=M. Ito;citation_author=N. Masuda;citation_author=K. Shinomiya;citation_author=K. Endo;citation_author=K. Ito;citation_title=Systematic analysis of neural projections reveals clonal composition of the Drosophila brain;citation_pages=644-655;citation_volume=23;citation_year=2013;citation_pmid=23541729;citation_doi=10.1016/j.cub.2013.03.015" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=GS. Jefferis;citation_author=EC. Marin;citation_author=RF. Stocker;citation_author=L. Luo;citation_title=Target neuron prespecification in the olfactory map of Drosophila.;citation_pages=204-208;citation_volume=414;citation_year=2001;citation_issue=6860;citation_issn=0028-0836;citation_pmid=11719930;citation_doi=10.1038/35102574" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=GS. Jefferis;citation_author=RM. Vyas;citation_author=D. Berdnik;citation_author=A. Ramaekers;citation_author=RF. Stocker;citation_author=NK. Tanaka;citation_author=K. Ito;citation_author=L. Luo;citation_title=Developmental origin of wiring specificity in the olfactory system of Drosophila;citation_volume=131;citation_year=2004;citation_pmid=14645123;citation_doi=10.1242/dev.00896" />
<meta name="citation_reference" content="citation_journal_title=Nature Reviews Neuroscience;citation_author=M. Kohwi;citation_author=CQ. Doe;citation_title=Temporal fate specification and neural progenitor competence during development;citation_pages=823-838;citation_volume=14;citation_year=2013;citation_pmid=24400340;citation_doi=10.1038/nrn3618" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=SL. Lai;citation_author=T. Awasaki;citation_author=K. Ito;citation_author=T. Lee;citation_title=Clonal analysis of Drosophila antennal lobe neurons: diverse neuronal architectures in the lateral neuroblast lineage;citation_volume=135;citation_year=2008;citation_pmid=18653555;citation_doi=10.1242/dev.024380" />
<meta name="citation_reference" content="citation_journal_title=Nature neuroscience;citation_journal_abbrev=Nat Neurosci;citation_author=SL. Lai;citation_author=T. Lee;citation_title=Genetic mosaic with dual binary transcriptional systems in Drosophila.;citation_pages=703-709;citation_volume=9;citation_year=2006;citation_issue=5;citation_pmid=16582903;citation_doi=10.1038/nn1681" />
<meta name="citation_reference" content="citation_journal_title=Trends in neurosciences;citation_journal_abbrev=Trends Neurosci;citation_author=T. Lee;citation_author=L. Luo;citation_title=Mosaic analysis with a repressible cell marker (MARCM) for Drosophila neural development.;citation_pages=251-254;citation_volume=24;citation_year=2001;citation_issue=5;citation_pmid=11311363;citation_doi=10.1016/S0166-2236(00)01791-4" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=Y. Li;citation_author=H. Lu;citation_author=P. Cheng;citation_author=S. Ge;citation_author=H. Xu;citation_author=SH. Shi;citation_author=Y. Dan;citation_title=Clonally related visual cortical neurons show similar stimulus feature selectivity.;citation_pages=118-121;citation_volume=486;citation_year=2012;citation_issue=7401;citation_issn=0028-0836;citation_pmid=22678292;citation_doi=10.1038/nature11110" />
<meta name="citation_reference" content="citation_journal_title=Advances in experimental medicine and biology;citation_journal_abbrev=Adv Exp Med Biol;citation_author=R. Lichtneckert;citation_author=H. Reichert;citation_title=Anteroposterior regionalization of the brain: genetic and comparative aspects.;citation_pages=32-41;citation_volume=628;citation_year=2008;citation_pmid=18683636;citation_doi=10.1007/978-0-387-78261-4_2" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_journal_abbrev=Heredity;citation_author=R. Lichtneckert;citation_author=H. Reichert;citation_title=Insights into the urbilaterian brain: conserved genetic patterning mechanisms in insect and vertebrate brain development.;citation_pages=465-477;citation_volume=94;citation_year=2005;citation_issue=5;citation_pmid=15770230;citation_doi=10.1038/sj.hdy.6800664" />
<meta name="citation_reference" content="citation_journal_title=Developmental dynamics :  an official publication of the American Association of Anatomists;citation_journal_abbrev=Dev Dyn;citation_author=S. Lin;citation_author=T. Lee;citation_title=Generating neuronal diversity in the Drosophila central nervous system.;citation_pages=57-68;citation_volume=241;citation_year=2012;citation_issue=1;citation_pmid=21932323;citation_doi=10.1002/dvdy.22739" />
<meta name="citation_reference" content="citation_journal_title=Developmental Biology;citation_author=JK. Lovick;citation_author=KT. Ngo;citation_author=JJ. Omoto;citation_author=DC. Wong;citation_author=JD. Nguyen;citation_author=V. Hartenstein;citation_title=Postembryonic lineages of the Drosophila brain: I. Development of the lineage-associated fiber tracts;citation_pages=228-257;citation_volume=384;citation_year=2013;citation_pmid=23880429;citation_doi=10.1016/j.ydbio.2013.07.008" />
<meta name="citation_reference" content="citation_journal_abbrev=Neuron;citation_author=G. Ohtsuki;citation_author=M. Nishiyama;citation_author=T. Yoshida;citation_author=T. Murakami;citation_author=M. Histed;citation_author=C. Lois;citation_author=K. Ohki;citation_title=Similarity of visual selectivity among clonally related neurons in visual cortex.;citation_pages=65-72;citation_volume=75;citation_year=2012;citation_issue=1;citation_pmid=22794261;citation_doi=10.1016/j.neuron.2012.05.023" />
<meta name="citation_reference" content="citation_journal_title=Annual review of cell and developmental biology;citation_journal_abbrev=Annu Rev Cell Dev Biol;citation_author=BJ. Pearson;citation_author=CQ. Doe;citation_title=Specification of temporal identity in the developing nervous system;citation_pages=619-647;citation_volume=20;citation_year=2004;citation_issn=1081-0706;citation_pmid=15473854;citation_doi=10.1146/annurev.cellbio.19.111301.115142" />
<meta name="citation_reference" content="citation_journal_title=Journal of Neuroscience;citation_journal_abbrev=J. Neurosci.;citation_author=W. Pereanu;citation_author=V. Hartenstein;citation_title=Neural lineages of the Drosophila brain: a three-dimensional digital atlas of the pattern of lineage location and projection at the late larval stage.;citation_pages=5534-5553;citation_volume=26;citation_year=2006;citation_issue=20;citation_issn=0270-6474;citation_pmid=16707805;citation_doi=10.1523/JNEUROSCI.4708-05.2006" />
<meta name="citation_reference" content="citation_journal_title=The Journal of comparative neurology;citation_journal_abbrev=J Comp Neurol;citation_author=W. Pereanu;citation_author=A. Kumar;citation_author=A. Jennett;citation_author=H. Reichert;citation_author=V. Hartenstein;citation_title=Development-based compartmentalization of the Drosophila central brain.;citation_pages=2996-3023;citation_volume=518;citation_year=2010;citation_issue=15;citation_pmid=20533357;citation_doi=10.1002/cne.22376" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=CJ. Potter;citation_author=B. Tasic;citation_author=EV. Russler;citation_author=L. Liang;citation_author=L. Luo;citation_title=The Q system: a repressible binary system for transgene expression, lineage tracing, and mosaic analysis.;citation_pages=536-548;citation_volume=141;citation_year=2010;citation_issue=3;citation_pmid=20434990;citation_doi=10.1016/j.cell.2010.02.025" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Rakic;citation_title=Specification of cerebral cortical areas;citation_pages=170-176;citation_volume=241;citation_year=1988;citation_issue=4862;citation_issn=0036-8075;citation_pmid=3291116;citation_doi=10.1126/science.3291116" />
<meta name="citation_reference" content="citation_journal_title=Biology Letters;citation_journal_abbrev=Biol Lett;citation_author=H. Reichert;citation_title=Evolutionary conservation of mechanisms for neural regionalization, proliferation and interconnection in brain development;citation_pages=112-116;citation_volume=5;citation_year=2009;citation_issue=1;citation_issn=1744-9561;citation_pmid=18755655;citation_doi=10.1098/rsbl.2008.0337" />
<meta name="citation_reference" content="citation_journal_title=Advances in experimental medicine and biology;citation_journal_abbrev=Adv Exp Med Biol;citation_author=H. Reichert;citation_author=B. Bello;citation_title=Hox genes and brain development in Drosophila.;citation_pages=145-153;citation_volume=689;citation_year=2010;citation_pmid=20795329;citation_doi=10.1007/978-1-4419-6673-5_11" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=H. Reichert;citation_author=A. Simeone;citation_title=Developmental genetic evidence for a monophyletic origin of the bilaterian brain;citation_pages=1533-1544;citation_volume=356;citation_year=2001;citation_issue=1414;citation_issn=0080-4622;citation_pmid=11604121;citation_doi=10.1098/rstb.2001.0972" />
<meta name="citation_reference" content="citation_journal_abbrev=Neuron;citation_author=CM. Root;citation_author=K. Masuyama;citation_author=DS. Green;citation_author=LE. Enell;citation_author=DR. Nässel;citation_author=CH. Lee;citation_author=JW. Wang;citation_title=A presynaptic gain control mechanism fine-tunes olfactory behavior.;citation_pages=311-321;citation_volume=59;citation_year=2008;citation_issue=2;citation_pmid=18667158;citation_doi=10.1016/j.neuron.2008.07.003" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=JL. Semmelhack;citation_author=JW. Wang;citation_title=Select Drosophila glomeruli mediate innate olfactory attraction and aversion.;citation_pages=218-223;citation_volume=459;citation_year=2009;citation_issue=7244;citation_issn=0028-0836;citation_pmid=19396157;citation_doi=10.1038/nature07983" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=AP. Singh;citation_author=K. VijayRaghavan;citation_author=V. Rodrigues;citation_title=Dendritic refinement of an identified neuron in the Drosophila CNS is regulated by neuronal activity and Wnt signaling;citation_volume=137;citation_year=2010;citation_pmid=20223760;citation_doi=10.1242/dev.044131" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in neurobiology;citation_journal_abbrev=Curr Opin Neurobiol;citation_author=JB. Skeath;citation_author=S. Thor;citation_title=Genetic control of Drosophila nerve cord development.;citation_pages=8-15;citation_volume=13;citation_year=2003;citation_issue=1;citation_pmid=12593977;citation_doi=10.1016/S0959-4388(03)00007-2" />
<meta name="citation_reference" content="citation_journal_title=Neural Development;citation_author=SR. Spindler;citation_author=V. Hartenstein;citation_title=Bazooka mediates secondary axon morphology in Drosophila brain lineages;citation_pages=16;citation_volume=6;citation_year=2011;citation_pmid=21524279;citation_doi=10.1186/1749-8104-6-16" />
<meta name="citation_reference" content="citation_journal_title=Journal of neurobiology;citation_journal_abbrev=J Neurobiol;citation_author=RF. Stocker;citation_author=G. Heimbeck;citation_author=N. Gendre;citation_author=JS. de Belle;citation_title=Neuroblast ablation in Drosophila P[GAL4] lines reveals origins of olfactory interneurons.;citation_pages=443-456;citation_volume=32;citation_year=1997;citation_issue=5;citation_pmid=9110257;citation_doi=10.1002/(SICI)1097-4695(199705)32:5<443::AID-NEU1>3.0.CO;2-5" />
<meta name="citation_reference" content="citation_journal_title=Developmental dynamics :  an official publication of the American Association of Anatomists;citation_journal_abbrev=Dev Dyn;citation_author=GM. Technau;citation_author=C. Berger;citation_author=R. Urbach;citation_title=Generation of cell diversity and segmental pattern in the embryonic central nervous system of Drosophila.;citation_pages=861-869;citation_volume=235;citation_year=2006;citation_issue=4;citation_pmid=16222713;citation_doi=10.1002/dvdy.20566" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=M. Tripodi;citation_author=JF. Evers;citation_author=A. Mauss;citation_author=M. Bate;citation_author=M. Landgraf;citation_title=Structural homeostasis: compensatory adjustments of dendritic arbor geometry in response to variations of synaptic input.;citation_pages=e260-e260;citation_volume=6;citation_year=2008;citation_issue=10;citation_pmid=18959482;citation_doi=10.1371/journal.pbio.0060260" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=JW. Truman;citation_author=M. Bate;citation_title=Spatial and temporal patterns of neurogenesis in the central nervous system of Drosophila melanogaster.;citation_pages=145-157;citation_volume=125;citation_year=1988;citation_issue=1;citation_pmid=3119399;citation_doi=10.1016/0012-1606(88)90067-X" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=R. Urbach;citation_author=R. Schnabel;citation_author=GM. Technau;citation_title=The pattern of neuroblast formation, mitotic domains and proneural gene expression during early brain development in Drosophila;citation_pages=3589-3606;citation_volume=130;citation_year=2003;citation_issue=16;citation_issn=0950-1991;citation_pmid=12835378;citation_doi=10.1242/dev.00528" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=R. Urbach;citation_author=GM. Technau;citation_title=Molecular markers for identified neuroblasts in the developing brain of Drosophila;citation_pages=3621-3637;citation_volume=130;citation_year=2003a;citation_issue=16;citation_issn=0950-1991;citation_pmid=12835380;citation_doi=10.1242/dev.00533" />
<meta name="citation_reference" content="citation_journal_title=BioEssays :  news and reviews in molecular, cellular and developmental biology;citation_journal_abbrev=Bioessays;citation_author=R. Urbach;citation_author=GM. Technau;citation_title=Neuroblast formation and patterning during early brain development in Drosophila.;citation_pages=739-751;citation_volume=26;citation_year=2004;citation_issue=7;citation_pmid=15221856;citation_doi=10.1002/bies.20062" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=R. Urbach;citation_author=GM. Technau;citation_title=Segment polarity and DV patterning gene expression reveals segmental organization of the Drosophila brain;citation_pages=3607-3620;citation_volume=130;citation_year=2003b;citation_issue=16;citation_issn=0950-1991;citation_pmid=12835379;citation_doi=10.1242/dev.00532" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=JW. Wang;citation_author=AM. Wong;citation_author=J. Flores;citation_author=LB. Vosshall;citation_author=R. Axel;citation_title=Two-photon calcium imaging reveals an odor-evoked map of activity in the fly brain.;citation_pages=271-282;citation_volume=112;citation_year=2003;citation_issue=2;citation_pmid=12553914;citation_doi=10.1016/S0092-8674(03)00004-7" />
<meta name="citation_reference" content="citation_journal_title=Developmental Biology;citation_author=DC. Wong;citation_author=JK. Lovick;citation_author=KT. Ngo;citation_author=W. Borisuthirattana;citation_author=JJ. Omoto;citation_author=V. Hartenstein;citation_title=Postembryonic lineages of the Drosophila brain: II. Identification of lineage projection patterns based on MARCM clones;citation_pages=258-289;citation_volume=384;citation_year=2013;citation_pmid=23872236;citation_doi=10.1016/j.ydbio.2013.07.009" />
<meta name="citation_reference" content="citation_journal_title=Current Biology;citation_author=HH. Yu;citation_author=T. Awasaki;citation_author=MD. Schroeder;citation_author=F. Long;citation_author=JS. Yang;citation_author=Y. He;citation_author=P. Ding;citation_author=JC. Kao;citation_author=GY. Wu;citation_author=H. Peng;citation_author=G. Myers;citation_author=T. Lee;citation_title=Clonal development and organization of the adult Drosophila central brain;citation_pages=633-643;citation_volume=23;citation_year=2013;citation_pmid=23541733;citation_doi=10.1016/j.cub.2013.02.057" />
<meta name="citation_reference" content="citation_journal_title=Nature neuroscience;citation_journal_abbrev=Nat Neurosci;citation_author=HH. Yu;citation_author=CH. Chen;citation_author=L. Shi;citation_author=Y. Huang;citation_author=T. Lee;citation_title=Twin-spot MARCM to reveal the developmental origin and identity of neurons.;citation_pages=947-953;citation_volume=12;citation_year=2009a;citation_issue=7;citation_pmid=19525942;citation_doi=10.1038/nn.2345" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=YC. Yu;citation_author=RS. Bultje;citation_author=X. Wang;citation_author=SH. Shi;citation_title=Specific synapses develop preferentially among sister excitatory neurons in the neocortex.;citation_pages=501-504;citation_volume=458;citation_year=2009b;citation_issue=7237;citation_issn=0028-0836;citation_pmid=19204731;citation_doi=10.1038/nature07722" />
<meta name="citation_xml_url" content="http://elifesciences.org/content/3/e04407.source.xml" />
<link rel="shortcut icon" href="http://dex3165296d6d.cloudfront.net/sites/default/themes/elife/favicon.ico" type="image/vnd.microsoft.icon" />
<meta property="fb:app_id" content="1471000156467630" />
<meta property="og:site_name" content="eLife" />
<meta property="og:url" content="http://elifesciences.org/content/3/e04407" />
<meta property="og:title" content="Genetic transformation of structural and functional circuitry rewires the Drosophila brain" />
<meta property="og:image" content="http://cdn.elifesciences.org/images/elife_final_logo_rgb.jpg" />
  <title>Genetic transformation of structural and functional circuitry rewires the Drosophila brain | eLife</title>  
  <link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_A7kboaogQd3fVnBM_G5lMjEN_plX-dG2OJz-m-ANFjs.css" media="all" />
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_kxeHOJ-ytzvGoq5w_qkD0a-NvtNsggo0Z2yOqjAqb6U.css" media="all" />
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_UxgGRFGNssKnw337GNDuWgjzGbw8dPdjHNeNSxbOhUE.css" media="all" />
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_biLdE05qSjK7yzYbeU1Z5Q6WnYjeGroLzaPvA5Suwlg.css" media="all" />
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_RDcMSV7IP1zyN3I_Sg8JaeIiRE_2rv_M1tLiaESUnf4_highwire.style.highwire.css.css" media="all" />
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_T_okTHlmgVRGEv7Dj3fUlushX-Yik03aH05ooFcTftE.css" media="all" />
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_ed6qtHpAgH_x4rNm5QfZSbCdlUXgimB3e_VuBV5DvsQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_n5Shh2o6KrGg6jphztyg4yq-x5PCuqAptfQ7r6e5qtA.css" media="all" />
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_yl78d1Daj-EkC_2FeylUtLTs76l5BGVUYEE9viB9zd8.css" media="print" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_7MewLKwwGqnoEzpneKtoXii-auHzFF5vVqOIQ7X_sXc.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_7MewLKwwGqnoEzpneKtoXii-auHzFF5vVqOIQ7X_sXc.css" media="all and (min-width: 768px)" />
<!--<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_3xUNx5yMqQEotwXjTyabe23ACl7kwNxF3uom9W65qtE.css" media="all and (min-width: 1220px)" />
<!--<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_j7mT2jalZMxeW19givVBhzQc3eb1f4mrFT4OHVYkbl0.css" media="all" />
<!--<![endif]-->

<!--[if lte IE 8]>
<link type="text/css" rel="stylesheet" href="http://dex3165296d6d.cloudfront.net/sites/default/files/cdn/css/http/css_GMVBfrApeoA3SvSFV-DupjLopqcmz22oGLzswmBJ0Zc.css" media="all" />
<![endif]-->
  <link rel="stylesheet" type="text/css" href="http://fast.fonts.com/cssapi/c10f70b9-7e7a-4f33-9361-d43022ae343e.css"/>
  <script type="text/javascript" src="http://elifesciences.org/sites/all/libraries/modernizr/modernizr.min.js?niyg83"></script>
<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
window.jQuery || document.write("<script src='/sites/all/modules/contrib/jquery_update/replace/jquery/1.8/jquery.min.js'>\x3C/script>")
//--><!]]>
</script>
<script type="text/javascript" src="http://elifesciences.org/sites/default/files/js/js_IkgOrrVZK1UTp2HktvFi5vo1g7b25Muyr5GoySikuic.js"></script>
<script type="text/javascript" src="http://elifesciences.org/sites/default/files/js/js_itu2PgFdrjV-docKmLK8Jn5oXe_05RgvQh73eOhI_mE.js"></script>
<script type="text/javascript" src="http://elifesciences.org/sites/default/files/js/js_q7gIZiIqyq1o1DD7DRKmLcPN4LdHS3SyJLvTE8_cCl0.js"></script>
<script type="text/javascript" src="http://elifesciences.org/sites/default/files/js/js_A22bgNPNzzgX8DOkniwaXlKxPGGKFizdrD8kslJOC9s.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-30290146-1", {"cookieDomain":"auto"});ga("send", "pageview");ga('create', 'UA-189672-49', 'auto', {'name': 'hwTracker'});
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="http://elifesciences.org/sites/default/files/js/js_F6IdQunGTRbTqybdjqsS7JEOVhimNbVXQ2Tq1MtP0zA.js"></script>
<script type="text/javascript" src="http://elifesciences.org/sites/default/files/js/js_TtMTaJ-2yjsITIbgfQReoLIgiBw0vUys0v7DtKvTgo0.js"></script>
<script type="text/javascript" src="http://elifesciences.org/sites/default/files/js/js_uwrfBOJCFVq42muTMjrq2bD-RUh8NNq_fZRTb6dsNb4.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"elife","theme_token":"dTu9o69IPWNwNqM2-jQn98mL4sAyxUic0MLttqHk4Is","js":{"sites\/default\/modules\/elife\/elife_header\/js\/elife_header.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"https:\/\/ajax.googleapis.com\/ajax\/libs\/jquery\/1.8.2\/jquery.min.js":1,"0":1,"misc\/jquery.once.js":1,"misc\/drupal.js":1,"sites\/all\/libraries\/cluetip\/jquery.cluetip.js":1,"sites\/all\/libraries\/cluetip\/lib\/jquery.hoverIntent.js":1,"sites\/all\/libraries\/cluetip\/lib\/jquery.bgiframe.min.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/libraries\/colorbox\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/default\/modules\/elife\/elife_article\/js\/elife.javascript.disqus.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/default\/modules\/elife\/elife_article\/js\/elife.javascript.title.js":1,"sites\/default\/modules\/elife\/elife_article\/js\/elife.javascript.article_authors.js":1,"sites\/default\/modules\/elife\/elife_article\/js\/elife.javascript.article_reflinks_sort.js":1,"sites\/default\/modules\/contrib\/disqus\/disqus.js":1,"sites\/default\/modules\/elife\/elife_article\/js\/elife.javascript.article_citation.js":1,"sites\/all\/modules\/contrib\/ctools\/js\/collapsible-div.js":1,"sites\/default\/modules\/elife\/elife_article\/js\/elife.javascript.podcast-jumpto.js":1,"sites\/default\/modules\/elife\/elife_article\/js\/elife.javascript.jumpto.js":1,"sites\/default\/modules\/elife\/elife_article\/js\/elife.javascript.share-jumpto.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"1":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"sites\/default\/modules\/elife\/elife_article\/plugins\/highwire_markup_process\/elife_figures.js":1,"sites\/default\/modules\/elife\/elife_article\/js\/jquery.scrollTo\/jquery.scrollTo.min.js":1,"sites\/default\/modules\/elife\/elife_article\/plugins\/highwire_markup_process\/elife_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/default\/themes\/elife\/js\/theme-scripts.js":1,"sites\/default\/themes\/elife\/js\/css_browser_selector.js":1,"sites\/default\/themes\/elife\/js\/jquery.formalize.js":1,"sites\/default\/themes\/elife\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1},"css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"sites\/default\/modules\/elife\/elife_header\/css\/header.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/default\/modules\/elife\/elife_footer\/css\/footer.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"modules\/node\/node.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"sites\/all\/libraries\/cluetip\/jquery.cluetip.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/collapsible-div.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/default\/themes\/elife\/css\/colors.css":1,"sites\/default\/themes\/elife\/css\/alpha-reset.css":1,"sites\/default\/themes\/elife\/css\/alpha-alpha.css":1,"sites\/default\/themes\/elife\/css\/formalize.css":1,"sites\/default\/themes\/elife\/css\/omega-text.css":1,"sites\/default\/themes\/elife\/css\/omega-branding.css":1,"sites\/default\/themes\/elife\/css\/omega-menu.css":1,"sites\/default\/themes\/elife\/css\/omega-forms.css":1,"sites\/default\/themes\/elife\/css\/omega-visuals.css":1,"sites\/default\/themes\/elife\/css\/font-awesome.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-visuals.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/default\/themes\/elife\/css\/elife-colorbox.css":1,"sites\/default\/themes\/elife\/css\/global.css":1,"sites\/default\/themes\/elife\/css\/elife-text.css":1,"sites\/default\/themes\/elife\/css\/css3pie.css":1,"sites\/default\/themes\/elife\/css\/print.css":1,"ie::normal::sites\/default\/themes\/elife\/css\/elife-default.css":1,"ie::normal::sites\/default\/themes\/elife\/css\/elife-elife-default.css":1,"ie::normal::sites\/default\/themes\/elife\/css\/elife-elife-default-normal.css":1,"ie::normal::sites\/default\/themes\/elife\/css\/grid\/elife_default\/normal\/elife-default-normal-32.css":1,"normal::sites\/default\/themes\/elife\/css\/elife-default.css":1,"normal::sites\/default\/themes\/elife\/css\/elife-elife-default-normal.css":1,"normal::sites\/default\/themes\/elife\/css\/grid\/elife_default\/normal\/elife-default-normal-32.css":1,"wide::sites\/default\/themes\/elife\/css\/elife-default.css":1,"wide::sites\/default\/themes\/elife\/css\/elife-elife-default-wide.css":1,"wide::sites\/default\/themes\/elife\/css\/grid\/elife_default\/wide\/elife-default-wide-32.css":1,"normal::sites\/default\/themes\/elife\/css\/elife-elife-default.css":1,"sites\/default\/themes\/elife\/css\/grid\/elife_default\/normal\/elife-default-normal-32.css":1,"wide::sites\/default\/themes\/elife\/css\/elife-elife-default.css":1,"sites\/default\/themes\/elife\/css\/grid\/elife_default\/wide\/elife-default-wide-32.css":1,"sites\/default\/themes\/elife\/css\/lte-ie8.css":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true},"highwire":{"nid":"22851","apath":"\/elife\/3\/e04407.atom","pisa":"elife;3\/0\/e04407","markup":[{"requested":"full-text","variant":"full-text","view":"full","pisa":"elife;3\/0\/e04407"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"elife;3\/0\/e04407"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"elife;3\/0\/e04407"},{"requested":"abstract","variant":"abstract","view":"abstract","pisa":"elife;3\/0\/e04407"},{"requested":"executive-summary","variant":"executive-summary","view":"summary","pisa":"elife;3\/0\/e04407"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG1"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG1"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG2"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG2"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG3"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG3"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG4"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG4"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG4S1"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG4S1"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG4S2"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG4S2"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG5"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG5"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG6"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG6"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG7"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG7"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG8"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG8"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG9"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG9"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG10"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG10"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG11"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG11"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG12"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG12"},{"requested":"long","variant":"full-text","view":"full","pisa":"elife;3\/0\/e04407"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"elife;3\/0\/e04407"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"elife;3\/0\/e04407"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"elife;3\/0\/e04407"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"elife;3\/0\/e04407"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG13"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG13"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG14"},{"requested":"expansion","variant":"expansion","view":"full","pisa":"elife;3\/0\/e04407\/FIG14"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"elife;3\/0\/e04407"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"elife;3\/0\/e04407"}],"ac":{"elife;3\/0\/e04407":{"access":{"abstract":true,"summary":true,"full":true},"pisa_id":"elife;3\/0\/e04407","jcode":"elife","reason":{"full":"ac_disabled","abstract":"ac_disabled"},"ac_override":true}}},"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"disqusComments":"hwelifesciences","disqus":{"domain":"hwelifesciences","url":"http:\/\/elifesciences.org\/content\/3\/e04407","title":"Genetic transformation of structural and functional circuitry rewires the Drosophila brain","identifier":"node\/22851"},"elife_article_toolbox":{"url":"url=http:\/\/elifesciences.org\/content\/3\/e04407\u0026doi=10.7554\/eLife.04407"},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":0,"breakpoints":{"normal":"all and (min-width: 768px)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip"},"omega":{"layouts":{"primary":"normal","order":["normal","wide","normal","wide"],"queries":{"normal":"all and (min-width: 768px)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
  <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
</head>
<body class="html not-front not-logged-in page-node page-node- page-node-22851 node-type-highwire-article context-content">
  <div id="skip-link">
    <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
  </div>
  <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
    <div class="page_header" role="banner">
  <a href="/" class="header__identity"><img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_header/images/elife-identity-header.jpg" alt="eLife" /></a>  <nav class="header__nav">
    <ol class="header__list header__list--about header__list--careers"><li class="header__list_item first"><a href="/about" class="header__list_link header__list_link--about">About</a></li>
<li class="header__list_item"><a href="/careers" class="header__list_link header__list_link--careers">Careers</a></li>
<li class="header__list_item"><a href="http://submit.elifesciences.org/" class="header__list_link header__list_link--submit"><div>Submit</div></a></li>
<li class="header__list_item last"><a href="/elife/search" class="header__list_link header__list_link--search">Search</a></li>
</ol>        <ul>
      <li class="header__list_secondary">
        <ol class="header__list_secondary--section"><li class="header__list_secondary_item first"><a href="/category/research-article" class="header__list_secondary_item_link header__list_secondary_item_link--section research">Research articles</a></li>
<li class="header__list_secondary_item"><a href="/category/short-report" class="header__list_secondary_item_link header__list_secondary_item_link--section short">Short reports</a></li>
<li class="header__list_secondary_item"><a href="/category/research-advance" class="header__list_secondary_item_link header__list_secondary_item_link--section advance">Research advances</a></li>
<li class="header__list_secondary_item"><a href="/collections/reproducibility-project-cancer-biology" class="header__list_secondary_item_link header__list_secondary_item_link--section reproducibility-project">Reproducibility Project</a></li>
<li class="header__list_secondary_item"><a href="/category/editorial" class="header__list_secondary_item_link header__list_secondary_item_link--section editorials">Editorials</a></li>
<li class="header__list_secondary_item"><a href="/category/insight" class="header__list_secondary_item_link header__list_secondary_item_link--section insights">Insights</a></li>
<li class="header__list_secondary_item"><a href="/category/feature-article" class="header__list_secondary_item_link header__list_secondary_item_link--section features">Features</a></li>
<li class="header__list_secondary_item"><a href="/podcast" class="header__list_secondary_item_link header__list_secondary_item_link--section podcasts">Podcasts</a></li>
<li class="header__list_secondary_item last"><a href="/elife-news" class="header__list_secondary_item_link header__list_secondary_item_link--section news">eLife News</a></li>
</ol>        <ol class="header__list_secondary--category"><li class="header__list_secondary_item first"><a href="/major-subject-areas" class="header__list_secondary_item_link header__list_secondary_item_link--category">All categories</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/biochemistry" class="header__list_secondary_item_link header__list_secondary_item_link--category">Biochemistry</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/biophysics-and-structural-biology" class="header__list_secondary_item_link header__list_secondary_item_link--category">Biophysics and structural biology</a></li>
<li class="header__list_secondary_item hidden"><a href="/collections/reproducibility-project-cancer-biology" class="header__list_secondary_item_link header__list_secondary_item_link--category">Cancer biology</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/cell-biology" class="header__list_secondary_item_link header__list_secondary_item_link--category">Cell biology</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/computational-and-systems-biology" class="header__list_secondary_item_link header__list_secondary_item_link--category">Computational and systems biology</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/developmental-biology-and-stem-cells" class="header__list_secondary_item_link header__list_secondary_item_link--category">Developmental biology and stem cells</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/ecology" class="header__list_secondary_item_link header__list_secondary_item_link--category">Ecology</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/epidemiology-and-global-health" class="header__list_secondary_item_link header__list_secondary_item_link--category">Epidemiology and global health</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/genes-and-chromosomes" class="header__list_secondary_item_link header__list_secondary_item_link--category">Genes and chromosomes</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/genomics-and-evolutionary-biology" class="header__list_secondary_item_link header__list_secondary_item_link--category">Genomics and evolutionary biology</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/human-biology-and-medicine" class="header__list_secondary_item_link header__list_secondary_item_link--category">Human biology and medicine</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/immunology" class="header__list_secondary_item_link header__list_secondary_item_link--category">Immunology</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/microbiology-and-infectious-disease" class="header__list_secondary_item_link header__list_secondary_item_link--category">Microbiology and infectious disease</a></li>
<li class="header__list_secondary_item hidden"><a href="/category/neuroscience" class="header__list_secondary_item_link header__list_secondary_item_link--category">Neuroscience</a></li>
<li class="header__list_secondary_item hidden last"><a href="/category/plant-biology" class="header__list_secondary_item_link header__list_secondary_item_link--category">Plant biology</a></li>
</ol>      </li> <!-- /.header__list_secondary -->
    </ul>
      </nav>
</div>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-WVM8KG"
height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
'//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-WVM8KG');</script>
<!-- End Google Tag Manager -->
  </div>
</div>  <div class="page clearfix panels-page panels-layout-elife_article" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-elife-bar-wrapper" class="zone-wrapper zone-elife-bar-wrapper clearfix hidden-small">  
  <div id="zone-elife-bar" class="zone zone-elife-bar clearfix container-32">
    <div class="grid-32 region region-elife-bar" id="region-elife-bar">
  <div class="region-inner region-elife-bar-inner">
      </div>
</div>  </div>
</div><div id="zone-menu-wrapper" class="zone-wrapper zone-menu-wrapper clearfix">  
  <div id="zone-menu" class="zone zone-menu clearfix container-32">
    <div class="grid-2 region region-menu" id="region-menu">
  <div class="region-inner region-menu-inner">
          </div>
</div>
  </div>
</div></header>    
      <section id="section-content" class="section section-content">
  <div id="zone-header-wrapper" class="zone-wrapper zone-header-wrapper clearfix">  
  <div id="zone-header" class="zone zone-header clearfix container-32">
    <div class="grid-7 region region-header-second push-25" id="region-header-second">
  <div class="region-inner region-header-second-inner">
      </div>
</div><div class="grid-22 prefix-1 region region-header-first pull-5" id="region-header-first">
  <div class="region-inner region-header-first-inner">
  	 
      </div>
</div><div class="grid-31 prefix-2 region region-responsive-header" id="region-responsive-header">
  <div class="region-inner region-responsive-header-inner">
    <div class="block block-panels-mini block-jnl-elife-responsive-bar block-panels-mini-jnl-elife-responsive-bar odd block-without-title" id="block-panels-mini-jnl-elife-responsive-bar">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_elife_responsive_bar">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-page-breadcrumb float-me-left" >
  
      
  
  <div class="pane-content">
    <ul class="breadcrumbs clearfix"><li class="crumb breadcrumb-0 first"><span class="wrapper"><a href="/">eLife</a></span></li><li class="crumb breadcrumb-1"><span class="wrapper"><a href="/category/research-article">Research article</a></span></li><li class="crumb breadcrumb-2 last"><span class="wrapper"><a href="/category/developmental-biology-and-stem-cells">Developmental biology and stem cells</a> / <a href="/category/neuroscience">Neuroscience</a></span></li></ul>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-mobile-menu float-me-right visible-small" >
  
        <h2 class="pane-title">Mobile Menu</h2>
    
  
  <div class="pane-content">
    <ul class="menu"><li class="first leaf menu-862 menu-home"><a href="/" title="Home">Home</a></li>
<li class="last leaf menu-865 menu-browse"><a href="/browse" title="Browse">Browse</a></li>
</ul>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div><div id="zone-content-wrapper" class="zone-wrapper zone-content-wrapper clearfix">  
  <div id="zone-content" class="zone zone-content clearfix container-32">    
		    <div class="grid-30 region region-content push-2" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
            <div class="element-invisible">    <h1 class="title" id="page-title">Genetic transformation of structural and functional circuitry rewires the Drosophila brain</h1>
    </div>                    <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout elife-article-layout" >
  <div class="panel-row-wrapper panel-row-first clearfix">
  
    	<div class="top-wrapper grid-21 suffix-9 alpha omega">
  				  		<div class="panel-panel panel-region-content-top">
	  			<div class="inside"><div class="panel-pane pane-panels-ajax-tab-tabs hidden-small"  id="article">
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/e04407" class="panels-ajax-tab-tab" data-panel-name="jnl_elife_article_article" data-target-id="elife-research-article-tabs" data-entity-context="node:22851" data-trigger="" data-url-enabled="0">Article</a><a href="/panels_ajax_tab/jnl_elife_article_article/node:22851/0" rel="nofollow" style="display:none" class="js-crawler-link"></a></li>
<li><a href="/content/3/e04407/article-data" class="panels-ajax-tab-tab" data-panel-name="jnl_elife_article_figdata" data-target-id="elife-research-article-tabs" data-entity-context="node:22851" data-trigger="article-data" data-url-enabled="1">Figures & data</a><a href="/panels_ajax_tab/jnl_elife_article_figdata/node:22851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li>
<li><a href="/content/3/e04407/article-metrics" class="panels-ajax-tab-tab" data-panel-name="jnl_elife_article_metrics" data-target-id="elife-research-article-tabs" data-entity-context="node:22851" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/jnl_elife_article_metrics/node:22851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li>
<li class="last"><a href="/content/3/e04407/article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_elife_article_author" data-target-id="elife-research-article-tabs" data-entity-context="node:22851" data-trigger="article-info" data-url-enabled="1">Article & author info</a><a href="/panels_ajax_tab/jnl_elife_article_author/node:22851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li>
</ul></div>  </div>

  
  </div>
</div>
	  		</div>
  		  	</div>
  
  </div>
  
  <div class="panel-row-wrapper panel-row-last clearfix">
		
			  	<div class="main-wrapper grid-21 alpha">
	  							<div class="panel-panel panel-region-main-content">
				  <div class="inside"><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category visible-small category-color-code elife-cat-research-article" >
  
      
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/research-article">Research article</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-elife-article-title" >
  
      
  
  <div class="pane-content">
    <div class="elife-article-indicators clearfix"><a href="http://en.wikipedia.org/wiki/Open_access"><img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_article/images/oa.png" alt="Open access" title="Open access" /></a><a href="http://creativecommons.org/licenses/by/4.0/"><img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_article/images/cc.png" alt="Copyright info" title="Copyright info" /></a><a href="/content/3/e04407.full.pdf" title="PDF" class="pdf-download" type="application/pdf" rel="alternative"><img class="visible-small" src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_article/images/pdf.png" width="22" alt="Download PDF" title="Download PDF" /></a><a href="/content/3/e04407#related-content" title="Related content" rel="#related-content" class="js-trigger-cluetip related-content active"><img class="visible-small" src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_article/images/related.gif" width="20" alt="Related content" title="Related content" /></a><a href="http://lens.elifesciences.org/04407/index.html"><img class="visible-small" src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_article/images/lens-icon.png" alt="eLife Lens" title="View in eLife Lens" /></a></div><h1 class="page-title"><div xmlns="http://www.w3.org/1999/xhtml">Genetic transformation of structural and functional circuitry rewires the <em>Drosophila</em> brain</div></h1>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-elife-article-authors" >
  
      
  
  <div class="pane-content">
    <div class="author-list"><ul><span class='elife-article-author-item ' data-tooltip-content='|&lt;div class=&quot;author-tooltip&quot;&gt;&lt;div class=&quot;author-tooltip-name&quot;&gt;&lt;span class=&quot;nlm-given-names&quot;&gt;Sonia&lt;/span&gt; &lt;span class=&quot;nlm-surname&quot;&gt;Sen&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-affiliation&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;span class=&quot;nlm-aff&quot;&gt;&lt;span class=&quot;nlm-institution&quot; data-content-type=&quot;dept&quot; data-hwp-id=&quot;institution-1&quot;&gt;Department of Developmental Biology and Genetics&lt;/span&gt;, &lt;span class=&quot;nlm-institution&quot; data-hwp-id=&quot;institution-2&quot;&gt;National Centre for Biological Sciences, Tata Institute for Fundamental Research&lt;/span&gt;, &lt;span class=&quot;nlm-addr-line&quot;&gt;&lt;span class=&quot;nlm-named-content&quot; data-content-type=&quot;city&quot; data-hwp-id=&quot;named-content-1&quot;&gt;Bangalore&lt;/span&gt;&lt;/span&gt;, &lt;span class=&quot;nlm-country&quot;&gt;India&lt;/span&gt;&lt;/span&gt;; &lt;span class=&quot;nlm-aff&quot;&gt;&lt;span class=&quot;nlm-institution&quot; data-hwp-id=&quot;institution-3&quot;&gt;Manipal University&lt;/span&gt;, &lt;span class=&quot;nlm-addr-line&quot;&gt;&lt;span class=&quot;nlm-named-content&quot; data-content-type=&quot;city&quot; data-hwp-id=&quot;named-content-2&quot;&gt;Manipal&lt;/span&gt;&lt;/span&gt;, &lt;span class=&quot;nlm-country&quot;&gt;India&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-contrib&quot;&gt;&lt;span class=&quot;author-tooltip-label&quot;&gt;Contribution: &lt;/span&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;Conception and design, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-conflict&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;i&gt;No competing interests declared&lt;/i&gt;&lt;/span&gt;&lt;/div&gt;&lt;/div&gt;' data-author-inst='NationalCentreforBiologicalSciencesTataInstituteforFundamentalResearchIndia|ManipalUniversityIndia'><span class="nlm-given-names">Sonia</span> <span class="nlm-surname">Sen</span></span>, <span class='elife-article-author-item ' data-tooltip-content='|&lt;div class=&quot;author-tooltip&quot;&gt;&lt;div class=&quot;author-tooltip-name&quot;&gt;&lt;span class=&quot;nlm-given-names&quot;&gt;Deshou&lt;/span&gt; &lt;span class=&quot;nlm-surname&quot;&gt;Cao&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-affiliation&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;span class=&quot;nlm-aff&quot;&gt;&lt;span class=&quot;nlm-institution&quot; data-content-type=&quot;dept&quot; data-hwp-id=&quot;institution-4&quot;&gt;Division of Biological Sciences&lt;/span&gt;, &lt;span class=&quot;nlm-institution&quot; data-hwp-id=&quot;institution-5&quot;&gt;University of California, San Diego&lt;/span&gt;, &lt;span class=&quot;nlm-addr-line&quot;&gt;&lt;span class=&quot;nlm-named-content&quot; data-content-type=&quot;city&quot; data-hwp-id=&quot;named-content-3&quot;&gt;San Diego&lt;/span&gt;&lt;/span&gt;, &lt;span class=&quot;nlm-country&quot;&gt;United States&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-contrib&quot;&gt;&lt;span class=&quot;author-tooltip-label&quot;&gt;Contribution: &lt;/span&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;Conception and design, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-conflict&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;i&gt;No competing interests declared&lt;/i&gt;&lt;/span&gt;&lt;/div&gt;&lt;/div&gt;' data-author-inst='UniversityofCaliforniaSanDiegoUnitedStates'><span class="nlm-given-names">Deshou</span> <span class="nlm-surname">Cao</span></span>, <span class='elife-article-author-item ' data-tooltip-content='|&lt;div class=&quot;author-tooltip&quot;&gt;&lt;div class=&quot;author-tooltip-name&quot;&gt;&lt;span class=&quot;nlm-given-names&quot;&gt;Ramveer&lt;/span&gt; &lt;span class=&quot;nlm-surname&quot;&gt;Choudhary&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-affiliation&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;span class=&quot;nlm-aff&quot;&gt;&lt;span class=&quot;nlm-institution&quot; data-content-type=&quot;dept&quot; data-hwp-id=&quot;institution-1&quot;&gt;Department of Developmental Biology and Genetics&lt;/span&gt;, &lt;span class=&quot;nlm-institution&quot; data-hwp-id=&quot;institution-2&quot;&gt;National Centre for Biological Sciences, Tata Institute for Fundamental Research&lt;/span&gt;, &lt;span class=&quot;nlm-addr-line&quot;&gt;&lt;span class=&quot;nlm-named-content&quot; data-content-type=&quot;city&quot; data-hwp-id=&quot;named-content-1&quot;&gt;Bangalore&lt;/span&gt;&lt;/span&gt;, &lt;span class=&quot;nlm-country&quot;&gt;India&lt;/span&gt;&lt;/span&gt;; &lt;span class=&quot;nlm-aff&quot;&gt;&lt;span class=&quot;nlm-institution&quot; data-hwp-id=&quot;institution-6&quot;&gt;Institute for Molecular Oncology&lt;/span&gt;, &lt;span class=&quot;nlm-addr-line&quot;&gt;&lt;span class=&quot;nlm-named-content&quot; data-content-type=&quot;city&quot; data-hwp-id=&quot;named-content-4&quot;&gt;Milan&lt;/span&gt;&lt;/span&gt;, &lt;span class=&quot;nlm-country&quot;&gt;Italy&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-contrib&quot;&gt;&lt;span class=&quot;author-tooltip-label&quot;&gt;Contribution: &lt;/span&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;Conception and design, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-conflict&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;i&gt;No competing interests declared&lt;/i&gt;&lt;/span&gt;&lt;/div&gt;&lt;/div&gt;' data-author-inst='NationalCentreforBiologicalSciencesTataInstituteforFundamentalResearchIndia|InstituteforMolecularOncologyItaly'><span class="nlm-given-names">Ramveer</span> <span class="nlm-surname">Choudhary</span></span>, <span class='elife-article-author-item ' data-tooltip-content='|&lt;div class=&quot;author-tooltip&quot;&gt;&lt;div class=&quot;author-tooltip-name&quot;&gt;&lt;span class=&quot;nlm-given-names&quot;&gt;Silvia&lt;/span&gt; &lt;span class=&quot;nlm-surname&quot;&gt;Biagini&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-affiliation&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;span class=&quot;nlm-aff&quot;&gt;&lt;span class=&quot;nlm-institution&quot; data-content-type=&quot;dept&quot; data-hwp-id=&quot;institution-1&quot;&gt;Department of Developmental Biology and Genetics&lt;/span&gt;, &lt;span class=&quot;nlm-institution&quot; data-hwp-id=&quot;institution-2&quot;&gt;National Centre for Biological Sciences, Tata Institute for Fundamental Research&lt;/span&gt;, &lt;span class=&quot;nlm-addr-line&quot;&gt;&lt;span class=&quot;nlm-named-content&quot; data-content-type=&quot;city&quot; data-hwp-id=&quot;named-content-1&quot;&gt;Bangalore&lt;/span&gt;&lt;/span&gt;, &lt;span class=&quot;nlm-country&quot;&gt;India&lt;/span&gt;&lt;/span&gt;; &lt;span class=&quot;nlm-aff&quot;&gt;&lt;span class=&quot;nlm-institution&quot; data-hwp-id=&quot;institution-6&quot;&gt;Institute for Molecular Oncology&lt;/span&gt;, &lt;span class=&quot;nlm-addr-line&quot;&gt;&lt;span class=&quot;nlm-named-content&quot; data-content-type=&quot;city&quot; data-hwp-id=&quot;named-content-4&quot;&gt;Milan&lt;/span&gt;&lt;/span&gt;, &lt;span class=&quot;nlm-country&quot;&gt;Italy&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-contrib&quot;&gt;&lt;span class=&quot;author-tooltip-label&quot;&gt;Contribution: &lt;/span&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;Conception and design, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-conflict&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;i&gt;No competing interests declared&lt;/i&gt;&lt;/span&gt;&lt;/div&gt;&lt;/div&gt;' data-author-inst='NationalCentreforBiologicalSciencesTataInstituteforFundamentalResearchIndia|InstituteforMolecularOncologyItaly'><span class="nlm-given-names">Silvia</span> <span class="nlm-surname">Biagini</span></span>, <span class='elife-article-author-item ' data-tooltip-content='|&lt;div class=&quot;author-tooltip&quot;&gt;&lt;div class=&quot;author-tooltip-name&quot;&gt;&lt;span class=&quot;nlm-given-names&quot;&gt;Jing W&lt;/span&gt; &lt;span class=&quot;nlm-surname&quot;&gt;Wang&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-affiliation&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;span class=&quot;nlm-aff&quot;&gt;&lt;span class=&quot;nlm-institution&quot; data-content-type=&quot;dept&quot; data-hwp-id=&quot;institution-4&quot;&gt;Division of Biological Sciences&lt;/span&gt;, &lt;span class=&quot;nlm-institution&quot; data-hwp-id=&quot;institution-5&quot;&gt;University of California, San Diego&lt;/span&gt;, &lt;span class=&quot;nlm-addr-line&quot;&gt;&lt;span class=&quot;nlm-named-content&quot; data-content-type=&quot;city&quot; data-hwp-id=&quot;named-content-3&quot;&gt;San Diego&lt;/span&gt;&lt;/span&gt;, &lt;span class=&quot;nlm-country&quot;&gt;United States&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-contrib&quot;&gt;&lt;span class=&quot;author-tooltip-label&quot;&gt;Contribution: &lt;/span&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;Conception and design, Analysis and interpretation of data, Drafting or revising the article&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-conflict&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;i&gt;No competing interests declared&lt;/i&gt;&lt;/span&gt;&lt;/div&gt;&lt;/div&gt;' data-author-inst='UniversityofCaliforniaSanDiegoUnitedStates'><span class="nlm-given-names">Jing W</span> <span class="nlm-surname">Wang</span></span>, <span class='elife-article-author-item ' data-tooltip-content='|&lt;div class=&quot;author-tooltip&quot;&gt;&lt;div class=&quot;author-tooltip-name&quot;&gt;&lt;span class=&quot;nlm-given-names&quot;&gt;Heinrich&lt;/span&gt; &lt;span class=&quot;nlm-surname&quot;&gt;Reichert&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-affiliation&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;span class=&quot;nlm-aff&quot;&gt;&lt;span class=&quot;nlm-institution&quot; data-content-type=&quot;dept&quot; data-hwp-id=&quot;institution-7&quot;&gt;Department of Biozentrum&lt;/span&gt;, &lt;span class=&quot;nlm-institution&quot; data-hwp-id=&quot;institution-8&quot;&gt;University of Basel&lt;/span&gt;, &lt;span class=&quot;nlm-addr-line&quot;&gt;&lt;span class=&quot;nlm-named-content&quot; data-content-type=&quot;city&quot; data-hwp-id=&quot;named-content-5&quot;&gt;Basel&lt;/span&gt;&lt;/span&gt;, &lt;span class=&quot;nlm-country&quot;&gt;Switzerland&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-contrib&quot;&gt;&lt;span class=&quot;author-tooltip-label&quot;&gt;Contribution: &lt;/span&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;Conception and design, Analysis and interpretation of data, Drafting or revising the article&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-conflict&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;i&gt;No competing interests declared&lt;/i&gt;&lt;/span&gt;&lt;/div&gt;&lt;/div&gt;' data-author-inst='UniversityofBaselSwitzerland'><span class="nlm-given-names">Heinrich</span> <span class="nlm-surname">Reichert</span></span>, <span class='elife-article-author-item corresp' data-tooltip-content='|&lt;div class=&quot;author-tooltip&quot;&gt;&lt;div class=&quot;author-tooltip-name&quot;&gt;&lt;span class=&quot;nlm-given-names&quot;&gt;K&lt;/span&gt; &lt;span class=&quot;nlm-surname&quot;&gt;VijayRaghavan&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-affiliation&quot;&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;span class=&quot;nlm-aff&quot;&gt;&lt;span class=&quot;nlm-institution&quot; data-content-type=&quot;dept&quot; data-hwp-id=&quot;institution-1&quot;&gt;Department of Developmental Biology and Genetics&lt;/span&gt;, &lt;span class=&quot;nlm-institution&quot; data-hwp-id=&quot;institution-2&quot;&gt;National Centre for Biological Sciences, Tata Institute for Fundamental Research&lt;/span&gt;, &lt;span class=&quot;nlm-addr-line&quot;&gt;&lt;span class=&quot;nlm-named-content&quot; data-content-type=&quot;city&quot; data-hwp-id=&quot;named-content-1&quot;&gt;Bangalore&lt;/span&gt;&lt;/span&gt;, &lt;span class=&quot;nlm-country&quot;&gt;India&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-corresp&quot;&gt;&lt;span class=&quot;author-tooltip-label&quot;&gt;For correspondence: &lt;/span&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;a href=&quot;mailto:vijay@ncbs.res.in&quot;&gt;vijay@ncbs.res.in&lt;/a&gt;&lt;/span&gt;&lt;/div&gt;&lt;div class=&quot;author-tooltip-conflict&quot;&gt;&lt;span class=&quot;author-tooltip-label&quot;&gt;Competing Interests: &lt;/span&gt;&lt;span class=&quot;author-tooltip-text&quot;&gt;&lt;span class=&quot;nlm-p&quot; data-hwp-id=&quot;p-104&quot;&gt;KVR: Senior editor, &lt;span class=&quot;nlm-italic&quot; data-toggle=&quot;yes&quot;&gt;eLife.&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;&lt;/div&gt;&lt;/div&gt;' data-author-inst='NationalCentreforBiologicalSciencesTataInstituteforFundamentalResearchIndia'><span class="nlm-given-names">K</span> <span class="nlm-surname">VijayRaghavan</span><a href="mailto:vijay@ncbs.res.in"><img class="corresp-icon" src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_article/images/corresp.png" alt="Corresponding Author" /></a></span></ul></div><div class="elife-institutions-list"><span class="elife-institution" id="NationalCentreforBiologicalSciencesTataInstituteforFundamentalResearchIndia"><span class="nlm-institution" data-hwp-id="institution-2">National Centre for Biological Sciences, Tata Institute for Fundamental Research</span>, <span class="nlm-country">India</span></span>; <span class="elife-institution" id="ManipalUniversityIndia"><span class="nlm-institution" data-hwp-id="institution-3">Manipal University</span>, <span class="nlm-country">India</span></span>; <span class="elife-institution" id="UniversityofCaliforniaSanDiegoUnitedStates"><span class="nlm-institution" data-hwp-id="institution-5">University of California, San Diego</span>, <span class="nlm-country">United States</span></span>; <span class="elife-institution" id="InstituteforMolecularOncologyItaly"><span class="nlm-institution" data-hwp-id="institution-6">Institute for Molecular Oncology</span>, <span class="nlm-country">Italy</span></span>; <span class="elife-institution" id="UniversityofBaselSwitzerland"><span class="nlm-institution" data-hwp-id="institution-8">University of Basel</span>, <span class="nlm-country">Switzerland</span></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="ctools-collapsible-container ctools-collapsed compact-collapsible visible-small hide-separator-after"><h2 class="pane-title ctools-collapsible-handle">Affiliation details</h2><div class="ctools-collapsible-content"><div class="author-list"><ul><div class='elife-article-author-affiliation-item corresp'><li class="author-detail"><h2 class="author-detail-name"><span class="nlm-given-names">Sonia</span> <span class="nlm-surname">Sen</span></h2><div class="author-detail-affiliation"><span class="author-detail-text"><span class="nlm-aff"><span class="nlm-institution" data-content-type="dept" data-hwp-id="institution-1">Department of Developmental Biology and Genetics</span>, <span class="nlm-institution" data-hwp-id="institution-2">National Centre for Biological Sciences, Tata Institute for Fundamental Research</span>, <span class="nlm-addr-line"><span class="nlm-named-content" data-content-type="city" data-hwp-id="named-content-1">Bangalore</span></span>, <span class="nlm-country">India</span></span>; <span class="nlm-aff"><span class="nlm-institution" data-hwp-id="institution-3">Manipal University</span>, <span class="nlm-addr-line"><span class="nlm-named-content" data-content-type="city" data-hwp-id="named-content-2">Manipal</span></span>, <span class="nlm-country">India</span></span></span></div><div class="author-detail-contrib"><span class="author-detail-label">Contribution: </span><span class="author-detail-text"> Conception and design, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article</span></span></span></div><div class="author-detail-conflict"><span class="author-detail-text"><i>No competing interests declared</i></span></div></li><li class="author-detail"><h2 class="author-detail-name"><span class="nlm-given-names">Deshou</span> <span class="nlm-surname">Cao</span></h2><div class="author-detail-affiliation"><span class="author-detail-text"><span class="nlm-aff"><span class="nlm-institution" data-content-type="dept" data-hwp-id="institution-4">Division of Biological Sciences</span>, <span class="nlm-institution" data-hwp-id="institution-5">University of California, San Diego</span>, <span class="nlm-addr-line"><span class="nlm-named-content" data-content-type="city" data-hwp-id="named-content-3">San Diego</span></span>, <span class="nlm-country">United States</span></span></span></div><div class="author-detail-contrib"><span class="author-detail-label">Contribution: </span><span class="author-detail-text"> Conception and design, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article</span></span></span></div><div class="author-detail-conflict"><span class="author-detail-text"><i>No competing interests declared</i></span></div></li><li class="author-detail"><h2 class="author-detail-name"><span class="nlm-given-names">Ramveer</span> <span class="nlm-surname">Choudhary</span></h2><div class="author-detail-affiliation"><span class="author-detail-text"><span class="nlm-aff"><span class="nlm-institution" data-content-type="dept" data-hwp-id="institution-1">Department of Developmental Biology and Genetics</span>, <span class="nlm-institution" data-hwp-id="institution-2">National Centre for Biological Sciences, Tata Institute for Fundamental Research</span>, <span class="nlm-addr-line"><span class="nlm-named-content" data-content-type="city" data-hwp-id="named-content-1">Bangalore</span></span>, <span class="nlm-country">India</span></span>; <span class="nlm-aff"><span class="nlm-institution" data-hwp-id="institution-6">Institute for Molecular Oncology</span>, <span class="nlm-addr-line"><span class="nlm-named-content" data-content-type="city" data-hwp-id="named-content-4">Milan</span></span>, <span class="nlm-country">Italy</span></span></span></div><div class="author-detail-contrib"><span class="author-detail-label">Contribution: </span><span class="author-detail-text"> Conception and design, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article</span></span></span></div><div class="author-detail-conflict"><span class="author-detail-text"><i>No competing interests declared</i></span></div></li><li class="author-detail"><h2 class="author-detail-name"><span class="nlm-given-names">Silvia</span> <span class="nlm-surname">Biagini</span></h2><div class="author-detail-affiliation"><span class="author-detail-text"><span class="nlm-aff"><span class="nlm-institution" data-content-type="dept" data-hwp-id="institution-1">Department of Developmental Biology and Genetics</span>, <span class="nlm-institution" data-hwp-id="institution-2">National Centre for Biological Sciences, Tata Institute for Fundamental Research</span>, <span class="nlm-addr-line"><span class="nlm-named-content" data-content-type="city" data-hwp-id="named-content-1">Bangalore</span></span>, <span class="nlm-country">India</span></span>; <span class="nlm-aff"><span class="nlm-institution" data-hwp-id="institution-6">Institute for Molecular Oncology</span>, <span class="nlm-addr-line"><span class="nlm-named-content" data-content-type="city" data-hwp-id="named-content-4">Milan</span></span>, <span class="nlm-country">Italy</span></span></span></div><div class="author-detail-contrib"><span class="author-detail-label">Contribution: </span><span class="author-detail-text"> Conception and design, Acquisition of data, Analysis and interpretation of data, Drafting or revising the article</span></span></span></div><div class="author-detail-conflict"><span class="author-detail-text"><i>No competing interests declared</i></span></div></li><li class="author-detail"><h2 class="author-detail-name"><span class="nlm-given-names">Jing W</span> <span class="nlm-surname">Wang</span></h2><div class="author-detail-affiliation"><span class="author-detail-text"><span class="nlm-aff"><span class="nlm-institution" data-content-type="dept" data-hwp-id="institution-4">Division of Biological Sciences</span>, <span class="nlm-institution" data-hwp-id="institution-5">University of California, San Diego</span>, <span class="nlm-addr-line"><span class="nlm-named-content" data-content-type="city" data-hwp-id="named-content-3">San Diego</span></span>, <span class="nlm-country">United States</span></span></span></div><div class="author-detail-contrib"><span class="author-detail-label">Contribution: </span><span class="author-detail-text"> Conception and design, Analysis and interpretation of data, Drafting or revising the article</span></span></span></div><div class="author-detail-conflict"><span class="author-detail-text"><i>No competing interests declared</i></span></div></li><li class="author-detail"><h2 class="author-detail-name"><span class="nlm-given-names">Heinrich</span> <span class="nlm-surname">Reichert</span></h2><div class="author-detail-affiliation"><span class="author-detail-text"><span class="nlm-aff"><span class="nlm-institution" data-content-type="dept" data-hwp-id="institution-7">Department of Biozentrum</span>, <span class="nlm-institution" data-hwp-id="institution-8">University of Basel</span>, <span class="nlm-addr-line"><span class="nlm-named-content" data-content-type="city" data-hwp-id="named-content-5">Basel</span></span>, <span class="nlm-country">Switzerland</span></span></span></div><div class="author-detail-contrib"><span class="author-detail-label">Contribution: </span><span class="author-detail-text"> Conception and design, Analysis and interpretation of data, Drafting or revising the article</span></span></span></div><div class="author-detail-conflict"><span class="author-detail-text"><i>No competing interests declared</i></span></div></li><li class="author-detail"><h2 class="author-detail-name"><span class="nlm-given-names">K</span> <span class="nlm-surname">VijayRaghavan</span></h2><div class="author-detail-affiliation"><span class="author-detail-text"><span class="nlm-aff"><span class="nlm-institution" data-content-type="dept" data-hwp-id="institution-1">Department of Developmental Biology and Genetics</span>, <span class="nlm-institution" data-hwp-id="institution-2">National Centre for Biological Sciences, Tata Institute for Fundamental Research</span>, <span class="nlm-addr-line"><span class="nlm-named-content" data-content-type="city" data-hwp-id="named-content-1">Bangalore</span></span>, <span class="nlm-country">India</span></span></span></div><div class="author-detail-contrib"><span class="author-detail-label">Contribution: </span><span class="author-detail-text"> Conception and design, Analysis and interpretation of data, Drafting or revising the article</span></span></span></div><div class="author-detail-corresp"><span class="author-detail-label">For correspondence: </span><span class="author-detail-text"><a href="mailto:vijay@ncbs.res.in">vijay@ncbs.res.in</a></span></div><div class="author-detail-conflict"><span class="author-detail-label">Competing Interests: </span><span class="author-detail-text"><span class="nlm-p" data-hwp-id="p-104">KVR: Senior editor, <span class="nlm-italic" data-toggle="yes">eLife.</span></span></span></div></li></div></ul></div><div class="elife-institutions-list"></div></div></div><div class="panel-separator"></div><div class="panel-pane pane-elife-article-doi" >
  
      
  
  <div class="pane-content">
    <div class="pane-highwire-doi"><span class="highwire-doi-doi"><span class="highwire-doi-pre-label label">DOI: </span><span class="elife-doi-doi"><a href="http://dx.doi.org/10.7554/eLife.04407">http://dx.doi.org/10.7554/eLife.04407</a></span></span><span class="highwire-doi-epubdate"><span class="highwire-doi-epubdate-label label">Published</span> <span class="highwire-doi-epubdate-data">December 29, 2014</span></span><span class="highwire-doi-cite-as">
          <span class="highwire-doi-cite-as-label label">Cite as </span>
          <span class="highwire-doi-cite-as-data">eLife 2014;3:e04407</span>
        </span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="ctools-collapsible-container ctools-collapsed compact-collapsible visible-small"><h2 class="pane-title ctools-collapsible-handle">Categories and tags</h2><div class="ctools-collapsible-content"><div class="item-list"><ul class="elife-article-categories"><li class="first"><a href="/category/research-article" class="category-display-channel">Research article</a></li>
<li><a href="/category/developmental-biology-and-stem-cells" class="category-heading">Developmental biology and stem cells</a></li>
<li><a href="/category/neuroscience" class="category-heading">Neuroscience</a></li>
<li><a href="/browse?keys=%22neuroblast%20lineage%22" class="keyword">neuroblast lineage</a></li>
<li><a href="/browse?keys=%22central%20complex%22" class="keyword">central complex</a></li>
<li><a href="/browse?keys=%22olfactory%20projection%20neurons%22" class="keyword">olfactory projection neurons</a></li>
<li><a href="/browse?keys=%22functional%20imaging%22" class="keyword">functional imaging</a></li>
<li><a href="/browse?keys=%22orthodenticle%22" class="keyword">orthodenticle</a></li>
<li class="last"><a href="/browse?keys=%22D.%20melanogaster%22" class="keyword">D. melanogaster</a></li>
</ul></div></div></div><div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-tab-tabs visible-small" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/3/e04407" class="panels-ajax-tab-tab" data-panel-name="jnl_elife_article_article" data-target-id="elife-research-article-tabs" data-entity-context="node:22851" data-trigger="" data-url-enabled="0">Article</a><a href="/panels_ajax_tab/jnl_elife_article_article/node:22851/0" rel="nofollow" style="display:none" class="js-crawler-link"></a></li>
<li><a href="/content/3/e04407/article-data" class="panels-ajax-tab-tab" data-panel-name="jnl_elife_article_figdata" data-target-id="elife-research-article-tabs" data-entity-context="node:22851" data-trigger="article-data" data-url-enabled="1">Figures & data</a><a href="/panels_ajax_tab/jnl_elife_article_figdata/node:22851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li>
<li><a href="/content/3/e04407/article-metrics" class="panels-ajax-tab-tab" data-panel-name="jnl_elife_article_metrics" data-target-id="elife-research-article-tabs" data-entity-context="node:22851" data-trigger="article-metrics" data-url-enabled="1">Metrics</a><a href="/panels_ajax_tab/jnl_elife_article_metrics/node:22851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li>
<li class="last"><a href="/content/3/e04407/article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_elife_article_author" data-target-id="elife-research-article-tabs" data-entity-context="node:22851" data-trigger="article-info" data-url-enabled="1">Article info</a><a href="/panels_ajax_tab/jnl_elife_article_author/node:22851/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li>
</ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-tab-container" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_elife_article_article" id="panels-ajax-tab-container-elife-research-article-tabs" class="panels-ajax-tab-container"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div id="abstract" class="ctools-collapsible-container"><h2 class="pane-title ctools-collapsible-handle">Abstract</h2><div class="ctools-collapsible-content"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" class="section abstract" id="abstract-1" xmlns:xhtml="http://www.w3.org/1999/xhtml"><p id="p-2">Acquisition of distinct neuronal identities during development is critical for the assembly of diverse functional neural circuits in the brain. In both vertebrates and invertebrates, intrinsic determinants are thought to act in neural progenitors to specify their identity and the identity of their neuronal progeny. However, the extent to which individual factors can contribute to this is poorly understood. We investigate the role of <em>orthodenticle</em> in the specification of an identified neuroblast (neuronal progenitor) lineage in the <em>Drosophila</em> brain. Loss of <em>orthodenticle</em> from this neuroblast affects molecular properties, neuroanatomical features, and functional inputs of progeny neurons, such that an entire central complex lineage transforms into a functional olfactory projection neuron lineage. This ability to change functional macrocircuitry of the brain through changes in gene expression in a single neuroblast reveals a surprising capacity for novel circuit formation in the brain and provides a paradigm for large-scale evolutionary modification of circuitry.</p><p id="p-3"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.001">http://dx.doi.org/10.7554/eLife.04407.001</a></p></div><a href="http://elifesciences.org/content/3/e04407" class="hw-link hw-link-article-full-text">View Full Text</a></div></div></div><div class="panel-separator"></div><div id="elife-digest" class="ctools-collapsible-container ctools-collapsed"><h2 class="pane-title ctools-collapsible-handle">eLife digest</h2><div class="ctools-collapsible-content"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" class="section executive-summary" id="abstract-2" xmlns:xhtml="http://www.w3.org/1999/xhtml"><p id="p-4">The cells in the brain—including the neurons that transmit information—work together in groups called neural circuits. These cells develop from precursor cells called neuroblasts. Each neuroblast can produce many cells, and it is likely that cells that develop from the same neuroblast work together in the adult brain in the same neural circuit. How the adult cells develop into their final form plays an important role in creating a neural circuit, but this process is not fully understood.</p><p id="p-5">In many animals, the complexity of their brain makes it difficult to follow how each individual neuroblast develops. However, all of the neuroblasts in the relatively simple brain of the fruit fly <em>Drosophila</em> have been identified. Furthermore, the genes responsible for establishing the initial identity of each neuroblast in the <em>Drosophila</em> brain are known. These genes may also determine which adult neurons develop from the neuroblast, and when each type of neuron is produced. However, the extent to which a single gene can influence the identity of neurons is unclear.</p><p id="p-6">Sen et al. focused on two types of neuroblasts, each of which, although found next to each other in the developing <em>Drosophila</em> brain, produces neurons for different neural circuits. One of the neuroblasts generates the olfactory neurons responsible for detecting smells; the other innervates the ‘central complex’ that has a number of roles, including controlling the fly's movements. A gene called <em>orthodenticle</em> is expressed by the central complex neuroblast, but not by the olfactory neuroblast, and helps to separate the two neural circuits into different regions of the fly brain.</p><p id="p-7">Sen et al. found that deleting the <em>orthodenticle</em> gene from the central complex neuroblast causes it to develop into olfactory neurons instead of central complex neurons. Tests showed that the modified neurons are completely transformed; they not only work like olfactory neurons, but they also have the same structure and molecular properties. Sen et al. have therefore demonstrated that it is possible to drastically alter the circuitry of the fruit fly brain by changing how one gene is expressed in one neuroblast. This suggests that new neural circuits can form relatively easily, and so could help us to understand how different brain structures and neural circuits evolved.</p><p id="p-8"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.002">http://dx.doi.org/10.7554/eLife.04407.002</a></p></div></div></div></div><div class="panel-separator"></div><div id="main-text" class="ctools-collapsible-container"><h2 class="pane-title ctools-collapsible-handle">Main text</h2><div class="ctools-collapsible-content"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section intro" id="sec-1"><h2 class="">Introduction</h2><p id="p-9">Animals display a wide repertoire of complex and adaptable behaviours executed by equally complex nervous systems. Understanding how the vast number of diverse cell types is assembled into functional neural circuits in complex brains during development is a major challenge. Studies of lineage tracing and circuit mapping reveal that heterogeneous pools of neural progenitors sequentially generate series of neuronal progeny, and that such lineally related neurons with shared developmental histories often share functional connectivity in the brain. In consequence, neural lineages can be considered to form neuroanatomical units of projection that represent the developmental basis of the functional circuitry of the brain (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Pearson and Doe, 2004</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Cardona et al., 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Pereanu et al., 2010</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Custo Greig et al., 2013</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Franco and Muller, 2013</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gao et al., 2013</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kohwi and Doe, 2013</a>).</p><p id="p-10">This is exemplified in <em>Drosophila</em> where the tens of thousands of neurons that comprise the adult brain are generated during development by a set of approximately 100 pairs of individually identifiable neural stem cells called neuroblasts (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Truman and Bate, 1988</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Urbach et al., 2003</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Urbach and Technau, 2003a</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">2003b</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Technau et al., 2006</a>). Each neuroblast gives rise to a specific, invariant lineage of post-mitotic neural cells in a highly stereotyped manner and in many cases, lineally related neurons share functional connectivity and many neuroanatomical features such as innervation of common neuropiles in the brain and common axon tract projection patterns (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Pereanu and Hartenstein, 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ito et al., 2013</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wong et al., 2013</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Yu et al., 2013</a>). Examples of this are the four neuroblast lineages that give rise to the intrinsic cells of the mushroom body, a neuropile compartment involved in learning and memory, or the five neuroblast lineages that innervate the antennal lobe, the primary olfactory processing centre in the fly brain (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Ito and Hotta, 1992</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Ito et al., 1997</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Stocker et al., 1997</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jefferis et al., 2001</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Das et al., 2008</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">2011</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">2013</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Lai et al., 2008</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chou et al., 2010</a>). Thus, neuroblast lineages are considered to form neuroanatomical units of projection that represent the developmental basis of the functional macrocircuitry of the fly brain (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Pereanu and Hartenstein, 2006</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Cardona et al., 2010</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Wong et al., 2013</a>).</p><p id="p-11">Comparable principles are manifested in the developing cerebral cortex of vertebrates, which consists of diverse neurons organized into six distinct layers, each of which is laid in place sequentially during development. Neural progenitors in the cortex are known to be multipotent, capable of generating neurons that populate each of the layers. Lineage tracing experiments in mice suggest that lineally related neurons occupy columns spanning across the layers of the cerebral cortex, as proposed in the ‘radial unit hypothesis’ (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rakic, 1988</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Yu et al., 2009b</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li et al., 2012</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Ohtsuki et al., 2012</a>). Furthermore, lineally related neurons also show a propensity to interconnect and have functional similarity, for example, similar orientation preferences in the visual cortex (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Li et al., 2012</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Ohtsuki et al., 2012</a>). Thus in vertebrates as in invertebrates, developmental history and lineage relationships govern the assembly of functional circuits.</p><p id="p-12">In order to understand how lineally specified circuits develop in the brain, it is critical to understand the molecular mechanisms that confer unique identities to neural progenitors and their lineages. Studies on the molecular genetics of brain development indicate that neural progenitors emerge from the embryonic neuroectoderm where unique spatial information represented by unique combinations of gene expression specifies unique identities to the progenitors. For example, in <em>Drosophila</em>, the embryonic neuroectoderm becomes spatially regionalized due to the action of embryonic patterning genes, which define the anterior-posterior and dorsal-ventral body axes. Their combined expression creates a Cartesian coordinate-like gene expression system in the neuroectoderm, resulting in unique domains of expression of developmental control genes along the neuroectoderm (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Skeath and Thor, 2003</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Urbach and Technau, 2004</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Technau et al., 2006</a>). Changes in the combinatorial expression pattern of these genes in specific domains can lead to changes in the identities of the neuroblasts that delaminate from the neuroectoderm during embryogenesis (e.g., <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Deshpande et al., 2001</a>). The process of spatial regionalization of the embryonic neuroectoderm is very similar in vertebrates. Homologous embryonic patterning genes result in unique domains of combinatorial gene expression along the vertebrate neuroectoderm (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Reichert and Simeone, 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Lichtneckert and Reichert, 2005</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Reichert, 2009</a>). Thus, in both vertebrates and invertebrates, the cells of the neuroectoderm acquire unique spatial information in the form of a combinatorial code of gene expression, which is conferred by embryonic patterning genes. It is noteworthy that neural progenitors also use temporal information (typically a series of sequentially expressed transcription factors) to generate neuronal diversity within lineages (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Pearson and Doe, 2004</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lin and Lee, 2012</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Kohwi and Doe, 2013</a>). While spatial cues convert a homogenous pool of progenitors into heterogeneous populations, temporal cues result in the ordered production of different neural subtypes from each progenitor.</p><p id="p-13">Given that spatial information in the neuroectoderm, in the form of embryonic patterning genes, imparts heterogeneity to neural progenitor populations, it is likely that these genes might also act as intrinsic determinants in the progenitor to give lineages their unique identities and hence determine their place in neural circuitry. This implies that spatially encoded intrinsic factors determine the identity of the progenitor and, as a consequence, the unique circuit features of its neural lineage. According to this, removal or addition of one or more of these genes could lead to a change in neuroblast identity resulting in transformation of the neuronal lineage and the lineal circuitry that derives from it. However, the extent to which individual transcription factors can contribute to this specification of neuroblast identity is not well understood.</p><p id="p-14">In order to test this, it is important to be able to uniquely identify individual neural progenitors and their lineages in the brain. The complexity of the vertebrate brain makes it difficult to conduct such an analysis at the resolution of single progenitors and single lineages. However, each of the neuroblasts in the <em>Drosophila</em> brain has been identified and their lineages characterized in the larval and adult brains (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Ito et al., 2013</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Wong et al., 2013</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Yu et al., 2013</a>). Furthermore, each of these neuroblasts has also been characterized by the expression of a specific combination of spatial genes, which could act as cell intrinsic determinants in the specification of unique neuroblast identity and hence control lineage-specific neuronal cell fate (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Skeath and Thor, 2003</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Urbach and Technau, 2004</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Technau et al., 2006</a>). This allows an investigation of the role of putative intrinsic determinants by changing their expression in identified stem cells and assessing its effect at the lineage level in an otherwise normal brain.</p><p id="p-15">Here, we focus on two identified neuroblast lineages in the <em>Drosophila</em> brain, LALv1 and ALad1, which develop in close spatial proximity to each other in the larval brain but become spatially segregated in the adult brain. While the ALad1 neuroblast generates olfactory projection interneurons that innervate the antennal lobe, the LALv1 neuroblast generates wide-field interneurons that innervate the central complex. We show that <em>orthodenticle</em> (<em>otd</em>), an embryonic patterning gene involved in specifying the anterior-most regions of the neuroectoderm and embryonic brain (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Lichtneckert and Reichert, 2008</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Reichert and Bello, 2010</a>), is expressed during development in the LALv1 neuroblast lineage but not in the ALad1 neuroblast lineage. Remarkably, loss of <em>otd</em> from the LALv1 neuroblast results in a complete transformation in the identity of the neurons that derive from this lineage. The <em>otd</em> null LALv1 neurons transform into antennal lobe projection interneurons similar to the ALad1 lineage, and this transformation includes a complete change in the neuroanatomy of the neurons, a change in their molecular properties as well as in their functional connectivity. This remarkably complete respecification of a neuroblast lineage upon the mutation of a single gene in the brain demonstrates that intrinsic determinants acting in the neuroblast during development specify the identity of its neural progeny and the macrocircuitry that these progeny establish. This large-scale modification of functional circuits in the brain by a single transcription factor in a single stem cell is unprecedented and reveals a surprising capacity for novel neural circuit formation in the developing brain, which may provide a paradigm for large-scale evolutionary modification of brain connectivity.</p></div><div class="section results" id="sec-2"><h2 class="">Results</h2><div id="sec-3" class="subsection"><h3>Development, morphogenesis, and differential expression of Otd in two identified central brain neuroblast lineages, LALv1 and ALad1</h3><p id="p-16">We focused our analysis on two identified neuroblast lineages referred to as LALv1 and ALad1 (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Pereanu and Hartenstein, 2006</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>) (see ‘Materials and methods’ for lineage nomenclature). During postembryonic development in the larval brain, the adult-specific (postembryonically generated) neural progeny of these lineages have their cell bodies clustered close to each other, dorsal to the larval antennal lobe (<a id="xref-fig-1-1" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1A,B</a>). Although their cell body clusters are closely apposed, the two lineages can be easily identified based on their distinct and unique axon tracts that project to different brain regions (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Pereanu et al., 2010</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Das et al., 2013</a>; <a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>).</p><div id="F1" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22864"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F1"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F1.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22864" title="Figure 1." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 1.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Development, morphogenesis, and differential Otd expression in two identified central brain neuroblast lineages, LALv1 and ALad1.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-17&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;) show anterior and lateral views of 3D reconstructions of the LALv1 (green) and the ALad1 (magenta) lineages in the larval brain. (&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) shows that the cell bodies are closely apposed to each other and lie above the larval antennal lobe (AL, yellow), (&amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;) shows their tracts diverge&#x2014;the ALad1 tract (magenta) projects dorsally and the LALv1 tract (green) projects posteriorly behind the AL and splits. (&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) show WT MARCM clones of the larval ALad1 and LALv1 lineages, respectively. Their cell bodies are outlined by white dotted lines and the white arrows trace their tracts. Insets in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt; show that while LALv1 cells (cyan arrow in &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) express Otd, ALad1 cells (cyan arrow in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) do not. (&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt;) is a third larval instar brain (CS) immunolabelled with neurotactin (green, to identify lineages), Otd (red) and TOPRO-3 (to label nuclei). The LALv1 lineage is documented in &amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt;, and the ALad1 lineage is documented in &amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt;. The neuroblasts are marked with yellow dotted lines and the lineages are marked with white dotted lines. The LALv1 neuroblast expresses Otd (yellow arrow in &amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt;) and the ALad1 neuroblast does not (yellow arrow in &amp;lt;strong&amp;gt;J&amp;lt;/strong&amp;gt;). (&amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;N&amp;lt;/strong&amp;gt;) show anterior and lateral views of 3D reconstructions of the LALv1 (green) and the ALad1 (magenta) lineages in the adult brain. Note that the adult antennal lobe (AL, yellow in &amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;N&amp;lt;/strong&amp;gt;) is situated between the ALad1 lineage (antero-dorsal to AL) and the LALv1 lineage (ventral to AL) and the cell bodies of these lineages are not closely apposed anymore. The arrows in &amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;N&amp;lt;/strong&amp;gt; indicate the ALad1 tract (magenta), which projects dorsally towards the protocerebrum and the LALv1 tract (green), which projects posterior to the AL. (&amp;lt;strong&amp;gt;O&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;P&amp;lt;/strong&amp;gt;) show WT clones of the adult LALv1 and ALad1 lineages, respectively. Their cell bodies are outlined by white dotted lines and the AL is outlined by yellow dotted lines. White arrows trace the tracts of these lineages. The LALv1 lineage innervates the lateral accessory lobe (LAL) and the central complex (CC). The ALad1 lineage innervates the calyx of the mushroom body (MB) and lateral horn (LH). The midline is represented by a yellow line in all images. Scale bars in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt; (applicable to &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) and in &amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt; (applicable to &amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt;) are 20 &#xB5;m. Scale bar in &amp;lt;strong&amp;gt;P&amp;lt;/strong&amp;gt; (applicable to &amp;lt;strong&amp;gt;O&amp;lt;/strong&amp;gt;) is 50 &#xB5;m. Genotypes in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; Tub-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;O&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;P&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;.&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-18&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.003&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.003&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22864"><img data-img="/content/elife/3/e04407/F1.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F1.medium.gif" alt="Figure 1."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F1.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F1.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22864">Download powerpoint</a></span></span><span class="fig-label">Figure 1.</span> <span class="caption-title">Development, morphogenesis, and differential Otd expression in two identified central brain neuroblast lineages, LALv1 and ALad1.</span><p id="p-17">(<strong>A</strong> and <strong>B</strong>) show anterior and lateral views of 3D reconstructions of the LALv1 (green) and the ALad1 (magenta) lineages in the larval brain. (<strong>A</strong>) shows that the cell bodies are closely apposed to each other and lie above the larval antennal lobe (AL, yellow), (<strong>B</strong>) shows their tracts diverge—the ALad1 tract (magenta) projects dorsally and the LALv1 tract (green) projects posteriorly behind the AL and splits. (<strong>C</strong> and <strong>D</strong>) show WT MARCM clones of the larval ALad1 and LALv1 lineages, respectively. Their cell bodies are outlined by white dotted lines and the white arrows trace their tracts. Insets in <strong>C</strong> and <strong>D</strong> show that while LALv1 cells (cyan arrow in <strong>D</strong>) express Otd, ALad1 cells (cyan arrow in <strong>C</strong>) do not. (<strong>E</strong>–<strong>L</strong>) is a third larval instar brain (CS) immunolabelled with neurotactin (green, to identify lineages), Otd (red) and TOPRO-3 (to label nuclei). The LALv1 lineage is documented in <strong>E</strong>–<strong>H</strong>, and the ALad1 lineage is documented in <strong>I</strong>–<strong>L</strong>. The neuroblasts are marked with yellow dotted lines and the lineages are marked with white dotted lines. The LALv1 neuroblast expresses Otd (yellow arrow in <strong>F</strong>) and the ALad1 neuroblast does not (yellow arrow in <strong>J</strong>). (<strong>M</strong> and <strong>N</strong>) show anterior and lateral views of 3D reconstructions of the LALv1 (green) and the ALad1 (magenta) lineages in the adult brain. Note that the adult antennal lobe (AL, yellow in <strong>M</strong> and <strong>N</strong>) is situated between the ALad1 lineage (antero-dorsal to AL) and the LALv1 lineage (ventral to AL) and the cell bodies of these lineages are not closely apposed anymore. The arrows in <strong>M</strong> and <strong>N</strong> indicate the ALad1 tract (magenta), which projects dorsally towards the protocerebrum and the LALv1 tract (green), which projects posterior to the AL. (<strong>O</strong> and <strong>P</strong>) show WT clones of the adult LALv1 and ALad1 lineages, respectively. Their cell bodies are outlined by white dotted lines and the AL is outlined by yellow dotted lines. White arrows trace the tracts of these lineages. The LALv1 lineage innervates the lateral accessory lobe (LAL) and the central complex (CC). The ALad1 lineage innervates the calyx of the mushroom body (MB) and lateral horn (LH). The midline is represented by a yellow line in all images. Scale bars in <strong>C</strong> (applicable to <strong>D</strong>) and in <strong>L</strong> (applicable to <strong>E</strong>–<strong>L</strong>) are 20 µm. Scale bar in <strong>P</strong> (applicable to <strong>O</strong>) is 50 µm. Genotypes in <strong>C</strong> and <strong>D</strong>: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; Tub-Gal4,UAS-mCD8::GFP/+</em>. Genotype in <strong>O</strong>: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/+</em>. Genotype in <strong>P</strong>: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+</em>.</p><p id="p-18"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.003">http://dx.doi.org/10.7554/eLife.04407.003</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><p id="p-19">The anatomical features of these two wild-type lineages can be visualized by MARCM clonal labelling (randomly induced neuroblast clones; ubiquitous <em>Tub-Gal4</em> driver). The ALad1 lineage initially projects its axon tract medially, dorsal to the larval antennal lobe, then turns posteriorly and projects towards the protocerebrum via the medial antennal lobe tract (<a id="xref-fig-1-2" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1B,C</a>) (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Das et al., 2013</a>; <a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>). The LALv1 lineage initially projects its axon tract ventro-medially, posterior to the larval antennal lobe, then loops dorsally and splits into two secondary axon tracts (<a id="xref-fig-1-3" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1B,D</a>) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Spindler and Hartenstein, 2011</a>; <a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>).</p><p id="p-20">In addition to the differences in axonal trajectories, we found that these two lineages also differed in their expression of the transcription factor Otd. Co-immunolabelling for the homeodomain transcription factor Otd and for Neurotactin (to identify lineage-specific axon tracts) shows that the LALv1 neuroblast (<a id="xref-fig-1-4" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1F</a>) and all of its lineal progeny (white dotted lines in <a id="xref-fig-1-5" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1F</a> and inset in <a id="xref-fig-1-6" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1D</a>) express Otd. In contrast, neither the ALad1 neuroblast nor its lineal progeny are found to express Otd (<a id="xref-fig-1-7" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1I–L</a> and inset in <a id="xref-fig-1-8" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1C</a>).</p><p id="p-21">In the adult brain, the neural progeny of the ALad1 lineage are olfactory projection neurons, which innervate the glomeruli of the antennal lobe and the neural progeny of the LALv1 lineage are wide-field interneurons that innervate the central complex, a sensorimotor integration centre in the fly brain (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Ito et al., 2013</a>; <a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Wong et al., 2013</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Yu et al., 2013</a>). To study the neuroanatomical features of the two lineages in the mature brain, we took advantage of the fact that they are differentially labelled by four enhancer-Gal4 driver lines in the adult brain. Thus, the adult LALv1 lineage is labelled by the <em>OK371-</em>Gal4 (a glutamatergic neuron label) and <em>Per</em>-Gal4 driver lines (<a id="xref-fig-1-9" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1O</a>, <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>), while the ALad1 lineage is not. Conversely, the adult ALad1 lineage is labelled by the <em>Cha7.4-</em>Gal4 (a cholinergic neuron label) and <em>GH146</em>-Gal4 lines, while the adult LALv1 lineage is not (<a id="xref-fig-1-10" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1P</a>, <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline"><div class="callout"><span>View this table:</span><ul class="callout-links"><li><a href="/highwire/markup/22876/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=elife_inline_linked_media%2Chighwire_math" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables">View popup</a></li><li><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/22876/expansion?postprocessors=elife_inline_linked_media%2Chighwire_math">View inline</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1.</span> <p id="p-22" class="first-child">Summary of the specific molecular changes in the LALv1 and Alad1 lineages</p><p id="p-23"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.004">http://dx.doi.org/10.7554/eLife.04407.004</a></p><div class="sb-div caption-clear"></div></div></div><p id="p-25">MARCM clonal labelling of the neurons in the adult ALad1 lineage using <em>GH146-Gal4</em> or <em>Cha7.4</em>-Gal4 drivers shows that their cell bodies are positioned dorsal to the adult antennal lobe, their dendrites innervate the antennal lobe glomeruli and their axons exit the lobe via the medial antennal lobe tract (<a id="xref-fig-1-11" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1M,N,P</a>). The axons then project dorso-posteriorly to innervate the calyx of the mushroom body and the lateral horn (<a id="xref-media-1-1" class="xref-other" href="#media-1">Video 1</a>) (<a id="xref-ref-13-4" class="xref-bibr" href="#ref-13">Ito et al., 2013</a>; <a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Wong et al., 2013</a>; <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Yu et al., 2013</a>).<div class="media video-content" id="media-1"><div class="media-inline video-inline"><div class="elife-inline-video"><video controls="controls" poster="http://static-movie-usa.glencoesoftware.com/jpg/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v001.jpg" preload="none" style="background-color:black"><source src="http://static-movie-usa.glencoesoftware.com/mp4/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v001.mp4" type="video/mp4; codecs=&quot;avc1.42E01E, mp4a.40.2&quot;"></source><source src="http://static-movie-usa.glencoesoftware.com/ogv/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v001.ogv" type="video/ogg; codecs=&quot;theora, vorbis&quot;"></source><img src="http://static-movie-usa.glencoesoftware.com/jpg/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v001.jpg"></img>Your browser does not appear to support HTML5 video. Please upgrade your browser.</video><div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="/elife/download-video/http%253A%252F%252Fstatic-movie-usa.glencoesoftware.com%252Fmp4%252F10.7554%252F118%252Fd679b2c58bf1802d1982180c8c3644d322d4b265%252Felife04407v001.mp4">Download Video </a></span></div></div></div><div class="media-caption"><span class="media-label">Video 1.</span> <span class="caption-title">Projection pattern of the wild-type ALad1 lineage.</span><p id="p-26" class="first-child"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.014">http://dx.doi.org/10.7554/eLife.04407.014</a></p></div></div></p><p id="p-27">MARCM clonal labelling of the neurons in the adult LALv1 lineage using <em>Per-Gal4</em> or <em>OK371-Gal4</em> drivers shows that their cell bodies are positioned ventral to the adult antennal lobe and their axons project into the loVM tract, which courses posteriorly behind the adult antennal lobe, then loops dorsally creating the prominent LEp fascicle, which innervates the central complex neuropiles and the lateral accessory lobe (<a id="xref-media-2-1" class="xref-other" href="#media-2">Video 2</a>) (<a id="xref-fig-1-12" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1O</a>) (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Spindler and Hartenstein, 2011</a>; <a id="xref-ref-13-5" class="xref-bibr" href="#ref-13">Ito et al., 2013</a>; <a id="xref-ref-24-8" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>; <a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Wong et al., 2013</a>; <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Yu et al., 2013</a>). As in the corresponding larval lineages, the adult LALv1 neuroblast lineage expresses Otd while the adult ALad1 neuroblast lineage does not (insets in <a id="xref-fig-1-13" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1O,P</a>). It is noteworthy that in the adult brain the cell bodies of the LALv1 neurons are located ventral to the adult antennal lobe, whereas in the larval brain the position of the cell bodies is dorsal to the larval antennal lobe (compare <a id="xref-fig-1-14" class="xref-fig" href="#F1" data-fragment-nid="22864">Figure 1A,M</a>). This change in cell body position of the LALv1 lineage occurs as a consequence of the morphogenetic changes associated with the de novo development of the adult antennal lobe (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Jefferis et al., 2004</a>; <a id="xref-ref-38-3" class="xref-bibr" href="#ref-38">Spindler and Hartenstein, 2011</a>; <a id="xref-ref-24-9" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>; <a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">Wong et al., 2013</a>).<div class="media video-content" id="media-2"><div class="media-inline video-inline"><div class="elife-inline-video"><video controls="controls" poster="http://static-movie-usa.glencoesoftware.com/jpg/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v002.jpg" preload="none" style="background-color:black"><source src="http://static-movie-usa.glencoesoftware.com/mp4/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v002.mp4" type="video/mp4; codecs=&quot;avc1.42E01E, mp4a.40.2&quot;"></source><source src="http://static-movie-usa.glencoesoftware.com/ogv/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v002.ogv" type="video/ogg; codecs=&quot;theora, vorbis&quot;"></source><img src="http://static-movie-usa.glencoesoftware.com/jpg/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v002.jpg"></img>Your browser does not appear to support HTML5 video. Please upgrade your browser.</video><div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="/elife/download-video/http%253A%252F%252Fstatic-movie-usa.glencoesoftware.com%252Fmp4%252F10.7554%252F118%252Fd679b2c58bf1802d1982180c8c3644d322d4b265%252Felife04407v002.mp4">Download Video </a></span></div></div></div><div class="media-caption"><span class="media-label">Video 2.</span> <span class="caption-title">Projection pattern of the wild-type LAlv1 lineage.</span><p id="p-28" class="first-child"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.013">http://dx.doi.org/10.7554/eLife.04407.013</a></p></div></div></p><p id="p-29">In summary, the LALv1 neuroblast and its progeny, which innervate the central complex, express <em>otd</em> throughout brain development as well as in the adult brain, while the ALad1 neuroblast and its progeny neurons, which innervate the antennal lobe, do not.</p></div><div id="sec-4" class="subsection"><h3>Loss of Otd from the LALv1 neuroblast results in lineage identity transformation</h3><p id="p-30">As <em>otd</em> is expressed in all of the cells of the central complex lineage—the neuroblast and the postmitotic neurons—we tested its possible function in both these cell types. In order to do this, we used MARCM-based clonal mutational methods to generate GFP-labelled <em>otd</em><sup><em>−/−</em></sup> clones in the LALv1 lineage in an otherwise heterozygous background (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Lee and Luo, 2001</a>). Using this technique, it is possible to genetically inactivate <em>otd</em> in the postmitotic neurons, the GMC, or the neuroblast, thus allowing us to assess its role in each of these cell types (see schematic in <a id="xref-fig-2-1" class="xref-fig" href="#F2" data-fragment-nid="22870">Figure 2A</a>). We generated such <em>otd</em><sup><em>−/−</em></sup> clones early in larval development and analyzed them in the adult brain.</p><div id="F2" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22870"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F2"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F2.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22870" title="Figure 2." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 2.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;&amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; postmitotic clones of the LALv1 lineage.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-31&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) shows a schematic of clone generation by the MARCM method. (&amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) document two wild-type single cell MARCM clones of the LALv1 lineage. These neurons skirt around the antennal lobe (white dotted lines) and innervate the LAL. (&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt;) show that these wild-type single cell clones express Otd. (&amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt;) document two single cell &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; MARCM clones of the LALv1 lineage. (&amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;K&amp;lt;/strong&amp;gt;) shows one such cell (&amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;), which does not express Otd (&amp;lt;strong&amp;gt;J&amp;lt;/strong&amp;gt;). Note that the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; single cell clones of the LALv1 lineage shown in &amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt; skirt around the antennal lobe and innervate the LAL and one of the central complex nodule (yellow arrowhead in &amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt;), like the wild-type single cell clones shown in &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;. Genotypes in &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4 or OK371-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Genotypes in &amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;K&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A,oc&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;2&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; Per-Gal4 or OK371-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Scale bar is 50 &#xB5;m. Yellow line represents the midline.&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-32&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.005&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.005&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22870"><img data-img="/content/elife/3/e04407/F2.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F2.medium.gif" alt="Figure 2."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F2.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F2.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22870">Download powerpoint</a></span></span><span class="fig-label">Figure 2.</span> <span class="caption-title"><em>otd</em><sup><em>−/−</em></sup> postmitotic clones of the LALv1 lineage.</span><p id="p-31">(<strong>A</strong>) shows a schematic of clone generation by the MARCM method. (<strong>B</strong> and <strong>C</strong>) document two wild-type single cell MARCM clones of the LALv1 lineage. These neurons skirt around the antennal lobe (white dotted lines) and innervate the LAL. (<strong>D</strong>–<strong>F</strong>) show that these wild-type single cell clones express Otd. (<strong>G</strong> and <strong>H</strong>) document two single cell <em>otd</em><sup><em>−/−</em></sup> MARCM clones of the LALv1 lineage. (<strong>I</strong>–<strong>K</strong>) shows one such cell (<strong>I</strong>), which does not express Otd (<strong>J</strong>). Note that the <em>otd</em><sup><em>−/−</em></sup> single cell clones of the LALv1 lineage shown in <strong>G</strong> and <strong>H</strong> skirt around the antennal lobe and innervate the LAL and one of the central complex nodule (yellow arrowhead in <strong>G</strong> and <strong>H</strong>), like the wild-type single cell clones shown in <strong>B</strong> and <strong>C</strong>. Genotypes in <strong>B</strong>–<strong>F</strong>: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4 or OK371-Gal4,UAS-mCD8::GFP/+</em>. Genotypes in <strong>G</strong>–<strong>K</strong>: <em>FRT19A,oc</em><sup><em>2</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; Per-Gal4 or OK371-Gal4,UAS-mCD8::GFP/+</em>. Scale bar is 50 µm. Yellow line represents the midline.</p><p id="p-32"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.005">http://dx.doi.org/10.7554/eLife.04407.005</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><p id="p-33">Using this technique, we first investigated a possible requirement of <em>otd</em> in the postmitotic neurons of the central complex lineage. In these experiments, in which we used the <em>OK371-Gal4</em> and <em>Per-Gal4</em> driver lines to label the MARCM clones, we obtained a total of seven wild-type single cell clones and 11 <em>otd</em><sup><em>−/−</em></sup> single cell clones. Although we have not dated the birth of these clones precisely (matched the time of clone generation), the wild-type single cell clones that we obtained in our experiments were very similar to those described previously (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Yu et al., 2009a</a>). Six of these single cell wild-type clones consisted of neurons that innervated both the lateral accessory lobe as well as one of the noduli of the central complex (<a id="xref-fig-2-2" class="xref-fig" href="#F2" data-fragment-nid="22870">Figure 2B</a>) and one clone only innervated the lateral accessory lobe (<a id="xref-fig-2-3" class="xref-fig" href="#F2" data-fragment-nid="22870">Figure 2C</a>). All of the 11 <em>otd</em><sup><em>−/−</em></sup> single cell clones we recovered also displayed a similar neuroanatomy. Their cell bodies were located ventral to the antennal lobe, their axons coursed through the loVM and LEp tracts and they all innervated the lateral accessory lobe and one of the noduli of the central complex (<a id="xref-fig-2-4" class="xref-fig" href="#F2" data-fragment-nid="22870">Figure 2G,H</a>). Thus, loss of <em>otd</em> function from the postmitotic neurons did not result in any gross defects in the neuroanatomy of these neurons. It is however possible, that there are fine-scale changes in the arborisation of these neurons within the lateral accessory lobe and the central complex that we were unable to identify. It is also possible that <em>otd</em> function is required in the GMC for the targeting of the postmitotic neurons (see schematic in <a id="xref-fig-2-5" class="xref-fig" href="#F2" data-fragment-nid="22870">Figure 2A</a>). However, in our experiments, we never obtained two cell GMC clones in order to be able to address this possibility.</p><p id="p-34">We then asked if <em>otd</em> might be required in the neuroblast of the LALv1 lineage for its proper development. In order to test this, we inactivated <em>otd</em> in the neuroblast during early larval development and analyzed the neuroanatomy of the resultant labelled wild-type and <em>otd</em><sup><em>−/−</em></sup> mutant neurons in the adult brain. In these experiments, in which we used the <em>OK371-Gal4</em> and <em>Per-Gal4 driver</em> lines to label the MARCM clones, we recovered 6 and 14 wild-type clones, respectively. As expected, all the wild-type neuroblast clones displayed the neuroanatomy of the central complex lineage as described above (ventrally position cell bodies, axon projection via the loVM and LEp tracts and innervations in the lateral accessory lobe and central complex. <a id="xref-fig-3-1" class="xref-fig" href="#F3" data-fragment-nid="22871">Figure 3B–E,J–M</a>). However, when we generated <em>otd</em><sup><em>−/−</em></sup> neuroblast clones in the LALv1 lineage (identifiable by the loss of Otd staining in the corresponding cell cluster ventral to the antennal lobe; white arrowhead in <a id="xref-fig-3-2" class="xref-fig" href="#F3" data-fragment-nid="22871">Figure 3G,O</a>), neither of these drivers labelled the mutant LALv1 lineage (<a id="xref-fig-3-3" class="xref-fig" href="#F3" data-fragment-nid="22871">Figure 3F–I,N–Q</a>).</p><div id="F3" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22871"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F3"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F3.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22871" title="Figure 3." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 3.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Loss of &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt; from the LALv1 lineage results in the suppression of the OK371 and Per enhancers.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-35&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) schematises the experimental logic. In neuroblast clones of the LALv1 lineage-specific enhancer Gal4s label the wild-type LALv1 neurons because they are active in the lineage (for example, OK371 and Per). Inability to label the LALv1 neurons in &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; neuroblast clones of the LALv1 lineage (detectable by absence of Otd immunolabelling) will suggest that the enhancers become suppressed in the mutant neurons. The brains in &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;J&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt; show wild-type MARCM clones of the LALv1 lineage labelled by &amp;lt;em&amp;gt;OK371&amp;lt;/em&amp;gt;-&amp;lt;em&amp;gt;Gal4&amp;lt;/em&amp;gt; and &amp;lt;em&amp;gt;Per&amp;lt;/em&amp;gt;-&amp;lt;em&amp;gt;Gal4&amp;lt;/em&amp;gt;, respectively. The insets in &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;J&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt; show that these cells express Otd. The brains in &amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;N&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;Q&amp;lt;/strong&amp;gt; show &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; clones of the LALv1 lineage. Otd expression is lost in one hemisphere in these brains (white arrowhead in &amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;O&amp;lt;/strong&amp;gt;; compare with the Otd expression within white dotted lines in the other brain hemisphere). Neither the &amp;lt;em&amp;gt;OK371&amp;lt;/em&amp;gt;-&amp;lt;em&amp;gt;Gal4&amp;lt;/em&amp;gt; (&amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt;) nor the &amp;lt;em&amp;gt;Per&amp;lt;/em&amp;gt;-&amp;lt;em&amp;gt;Gal4&amp;lt;/em&amp;gt; (&amp;lt;strong&amp;gt;N&amp;lt;/strong&amp;gt;) enhancers drive the expression of UAS-&amp;lt;em&amp;gt;mCD8&amp;lt;/em&amp;gt;::&amp;lt;em&amp;gt;GFP&amp;lt;/em&amp;gt; in these cells and the transformed lineage is not labelled in these experiments. Genotype in &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; OK371-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A, otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; OK371-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;J&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;N&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;Q&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A, otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Midline is represented by a yellow line. Scale bar is 50 &#xB5;m.&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-36&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.006&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.006&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22871"><img data-img="/content/elife/3/e04407/F3.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F3.medium.gif" alt="Figure 3."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F3.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F3.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22871">Download powerpoint</a></span></span><span class="fig-label">Figure 3.</span> <span class="caption-title">Loss of <em>otd</em> from the LALv1 lineage results in the suppression of the OK371 and Per enhancers.</span><p id="p-35">(<strong>A</strong>) schematises the experimental logic. In neuroblast clones of the LALv1 lineage-specific enhancer Gal4s label the wild-type LALv1 neurons because they are active in the lineage (for example, OK371 and Per). Inability to label the LALv1 neurons in <em>otd</em><sup><em>−/−</em></sup> neuroblast clones of the LALv1 lineage (detectable by absence of Otd immunolabelling) will suggest that the enhancers become suppressed in the mutant neurons. The brains in <strong>B</strong>–<strong>E</strong> and <strong>J</strong>–<strong>M</strong> show wild-type MARCM clones of the LALv1 lineage labelled by <em>OK371</em>-<em>Gal4</em> and <em>Per</em>-<em>Gal4</em>, respectively. The insets in <strong>B</strong>–<strong>E</strong> and <strong>J</strong>–<strong>M</strong> show that these cells express Otd. The brains in <strong>F</strong>–<strong>I</strong> and <strong>N</strong>–<strong>Q</strong> show <em>otd</em><sup><em>−/−</em></sup> clones of the LALv1 lineage. Otd expression is lost in one hemisphere in these brains (white arrowhead in <strong>G</strong> and <strong>O</strong>; compare with the Otd expression within white dotted lines in the other brain hemisphere). Neither the <em>OK371</em>-<em>Gal4</em> (<strong>F</strong>) nor the <em>Per</em>-<em>Gal4</em> (<strong>N</strong>) enhancers drive the expression of UAS-<em>mCD8</em>::<em>GFP</em> in these cells and the transformed lineage is not labelled in these experiments. Genotype in <strong>B</strong>–<strong>E</strong>: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; OK371-Gal4,UAS-mCD8::GFP/+</em>. Genotype in <strong>F</strong>–<strong>I</strong>: <em>FRT19A, otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; OK371-Gal4,UAS-mCD8::GFP/+</em>. Genotype in <strong>J</strong>–<strong>M</strong>: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/+</em>. Genotype in <strong>N</strong>–<strong>Q</strong>: <em>FRT19A, otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/+</em>. Midline is represented by a yellow line. Scale bar is 50 µm.</p><p id="p-36"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.006">http://dx.doi.org/10.7554/eLife.04407.006</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><p id="p-37">In order to investigate the neuroanatomy of the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage further, we utilized the ubiquitously expressed <em>Tub-Gal4</em> driver to label neuroblast clones and recovered 19 WT and 37 <em>otd</em><sup><em>−/−</em></sup> neuroblast MARCM clones in the LALv1 lineage. While the wild-type neurons displayed all the features of the LALv1 lineage described above (<a id="xref-fig-5-1" class="xref-fig" href="#F4" data-fragment-nid="22884">Figure 4—figure supplement 1A</a>), the <em>otd</em><sup><em>−/−</em></sup> LALv1 neuroblast clones had dramatic changes in its neuroanatomy (<a id="xref-fig-5-2" class="xref-fig" href="#F4" data-fragment-nid="22884">Figure 4—figure supplement 1B–D</a>). Mutant neurons no longer innervated the central complex or lateral accessory lobe neuropiles; instead they innervated the antennal lobe neuropile (asterisk in <a id="xref-fig-5-3" class="xref-fig" href="#F4" data-fragment-nid="22884">Figure 4—figure supplement 1B–D</a>) and sent projections via the medial antennal lobe tract towards the protocerebrum (arrowhead in <a id="xref-fig-5-4" class="xref-fig" href="#F4" data-fragment-nid="22884">Figure 4—figure supplement 1B–D</a>). These changes in dendritic and axonal innervation patterns were reversed by targeted expression of the full-length <em>otd</em> coding sequence in mutant neuroblast clones using the <em>Tub-Gal4</em> (<a id="xref-fig-6-1" class="xref-fig" href="#F4" data-fragment-nid="22885">Figure 4—figure supplement 2</a>).</p><p id="p-38">Taken together, these findings indicate that the mutant LALv1 neurons have acquired a transformed identity. Moreover these data suggest that this transformed identity has features characteristic of antennal lobe projection neurons.</p></div><div id="sec-5" class="subsection"><h3>Determining the lineage identity of the transformed neurons</h3><p id="p-39">As the neuroanatomy of the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage shows such a dramatic transformation, we wanted to confirm that the transformed neurons did indeed belong to the LALv1 lineage. We used three different approaches to determine that it was indeed the LALv1 lineage that transformed into an antennal lobe-like lineage upon the loss of <em>otd</em> from its neuroblast. First, we showed that the appearance of the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage corresponds to the appearance of an extra antennal lobe lineage. Second, we showed that the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage results in the corresponding loss of the LEp tract specific to the wild-type LALv1 lineage. Third, we used an independent molecular marker to unambiguously identify the wild-type and <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage.</p><div id="sec-6" class="subsection"><h4>An extra antennal lobe lineage</h4><p id="p-40">In the wild-type adult brain, the <em>GH146-Gal4</em> driver specifically labels a subset of the antennal lobe projection interneurons that derive from three identified neuroblast lineages, ALad1, ALl1, and ALv1 (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Stocker et al., 1997</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Jefferis et al., 2001</a>). If the transformed identity of the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage does indeed correspond to that of an antennal lobe lineage, it should also express the antennal lobe specific enhancer Gal4 driver line, <em>GH146.</em> To investigate this, we generated wild-type and <em>otd</em><sup><em>−/−</em></sup> MARCM clones in the LALv1 lineage and used the <em>GH146-Gal4</em> enhancer line to label the recovered clones. As expected, the only wild-type neuroblast clones recovered corresponded to the ALad1, ALl1, and ALv1 lineages; <em>GH146-Gal4</em> labelled wild-type neuroblast clones of the LALv1 lineage were never recovered in these experiments. However, when <em>otd</em><sup><em>−/−</em></sup> neuroblast clones were induced in the LALv1 lineage, we recovered 39 examples of a fourth type of <em>GH146-Gal4</em>-labelled neuroblast clone. As the expression of <em>GH146-Gal4</em> is much more restricted that <em>Tubulin-Gal4</em>, this also provided us with the opportunity to describe the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage in more detail. The neurons in this type of mutant clone all exhibited extensive dendritic innervation of the antennal lobe (39/39; <a id="xref-fig-4-1" class="xref-fig" href="#F4" data-fragment-nid="22872">Figure 4D–G</a>). This innervation was largely multiglomerular (<a id="xref-fig-4-2" class="xref-fig" href="#F4" data-fragment-nid="22872">Figure 4D–G</a>), although not all glomeruli were always innervated (yellow arrowhead in <a id="xref-fig-4-3" class="xref-fig" href="#F4" data-fragment-nid="22872">Figure 4E</a>) and innervation tended to be more intense in the posterior parts of antennal lobe (<a id="xref-fig-4-4" class="xref-fig" href="#F4" data-fragment-nid="22872">Figure 4G</a>). Furthermore, the axons of this clone projected via the medial antennal lobe tract towards the protocerebrum (37/39; white arrow in <a id="xref-fig-4-5" class="xref-fig" href="#F4" data-fragment-nid="22872">Figure 4B,G,H,K</a>), where they innervated the calyx of the mushroom bodies (39/39; <a id="xref-fig-4-6" class="xref-fig" href="#F4" data-fragment-nid="22872">Figure 4I,L</a>) and then turned laterally to innervate the lateral horn (39/39; <a id="xref-fig-4-7" class="xref-fig" href="#F4" data-fragment-nid="22872">Figure 4J,M</a>).</p><div class="fig-group" id="fig-group-1"><div id="F4" class="fig-inline-img-set fig-inline-img-set-carousel"><div class="elife-fig-slider-wrapper"><div class="elife-fig-slider"><div class="elife-fig-slider-img elife-fig-slider-primary"><img data-fragment-nid="22872" class="figure-icon-fragment-nid-22872" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F4.medium.gif" alt="Figure 4."></img></div><div class="figure-carousel-inner-wrapper"><div class="figure-carousel figure-carousel-22872"><div class="elife-fig-slider-img elife-fig-slider-secondary"><img data-fragment-nid="22884" class="figure-icon-fragment-nid-22884" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F4/F5.small.gif" alt="Figure 4&#x2014;figure supplement 1."></img></div><div class="elife-fig-slider-img elife-fig-slider-secondary"><img data-fragment-nid="22885" class="figure-icon-fragment-nid-22885" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F4/F6.small.gif" alt="Figure 4&#x2014;figure supplement 2."></img></div></div></div></div></div><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22872"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F4"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F4.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22872" title="Figure 4." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 4.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Loss of &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt; from LALv1 lineage results in activation of the antennal lobe-specific enhancer, GH146-Gal4, thus allowing detailed study of the neuroanatomy of the transformed &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;-&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-41&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;) show WT LALv1 and ALad1 lineages, respectively, and (&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) shows an &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage. Note that the neuroanatomy of the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage (&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) is completely unlike the WT LALv1 lineage (&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) and is strikingly similar to the WT ALad1 lineage (&amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;). (&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt;) document different brains that contain the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage. They have multiglomerular dendritic innervations in the AL (&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;), but not all glomeruli are always innervated (yellow arrowhead in &amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;). There are more innervations in the more posterior parts of the antennal lobe (&amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;). Their axon tracts project via the medial antennal lobe tract (arrowhead in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;K&amp;lt;/strong&amp;gt;) to innervate the calyx of the mushroom body (MB; &amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt;) and the lateral horn (LH; &amp;lt;strong&amp;gt;J&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt;). Genotype in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A,oc&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;2&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; &amp;lt;em&amp;gt;or FRT19A,otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Scale bars are 50 &#xB5;m (scale bar in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt; is applicable to &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;; the one in &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt; is applicable through to &amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;; the one in &amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt; is applicable to &amp;lt;strong&amp;gt;K&amp;lt;/strong&amp;gt;; the one in &amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt; is applicable to &amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;J&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt;). Yellow line represents the midline.&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-42&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.007&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.007&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22872"><img data-img="/content/elife/3/e04407/F4.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F4.medium.gif" alt="Figure 4."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F4.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F4.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22872">Download powerpoint</a></span></span><span class="fig-label">Figure 4.</span> <span class="caption-title">Loss of <em>otd</em> from LALv1 lineage results in activation of the antennal lobe-specific enhancer, GH146-Gal4, thus allowing detailed study of the neuroanatomy of the transformed <em>otd</em><sup><em>-−/−</em></sup> LALv1 lineage.</span><p id="p-41">(<strong>A</strong> and <strong>B</strong>) show WT LALv1 and ALad1 lineages, respectively, and (<strong>C</strong>) shows an <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage. Note that the neuroanatomy of the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage (<strong>C</strong>) is completely unlike the WT LALv1 lineage (<strong>A</strong>) and is strikingly similar to the WT ALad1 lineage (<strong>B</strong>). (<strong>D</strong>–<strong>M</strong>) document different brains that contain the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage. They have multiglomerular dendritic innervations in the AL (<strong>D</strong>–<strong>G</strong>), but not all glomeruli are always innervated (yellow arrowhead in <strong>E</strong>). There are more innervations in the more posterior parts of the antennal lobe (<strong>F</strong> and <strong>G</strong>). Their axon tracts project via the medial antennal lobe tract (arrowhead in <strong>C</strong>, <strong>G</strong>, <strong>H</strong>, <strong>K</strong>) to innervate the calyx of the mushroom body (MB; <strong>I</strong> and <strong>L</strong>) and the lateral horn (LH; <strong>J</strong> and <strong>M</strong>). Genotype in <strong>A</strong>: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/+</em>. Genotype in <strong>B</strong>: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+</em>. Genotype in <strong>C</strong>–<strong>M</strong>: <em>FRT19A,oc</em><sup><em>2</em></sup> <em>or FRT19A,otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+</em>. Scale bars are 50 µm (scale bar in <strong>A</strong> is applicable to <strong>B</strong> and <strong>C</strong>; the one in <strong>D</strong> is applicable through to <strong>G</strong>; the one in <strong>H</strong> is applicable to <strong>K</strong>; the one in <strong>I</strong> is applicable to <strong>L</strong>, <strong>J</strong>, <strong>M</strong>). Yellow line represents the midline.</p><p id="p-42"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.007">http://dx.doi.org/10.7554/eLife.04407.007</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22884"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F5"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F4/F5.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22884" title="Figure 4&#x2014;figure supplement 1." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 4&#x2014;figure supplement 1.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Loss of &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt; from the LALv1 neuroblast lineage, transforms it into an antennal lobe PN lineage.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-43&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;) are Tubulin-Gal4 labelled adult, wild-type clones of the LALv1 lineage. (&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) shows the anterior sections and (&amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;) shows more posterior sections of the same clone. (&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) The cell bodies of LALv1 lie ventral to the antennal lobe (AL) and express the transcription factor otd (inset in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;). The axon tract of this lineage skirts around the AL (magenta arrows in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;) and innervates the lateral accessory lobe (LAL, magenta dotted lines in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) and the central complex (CC, magenta dotted lines in &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;). Note that there are no innervation from this lineage in the AL. (&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) are tubulin labelled adult, otd null clones of the LALv1 NB lineage. (&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) shows the anterior sections, and (&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) shows more posterior sections of the same clone. The otd&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage innervates the AL (magenta dotted lines in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) and sends its axon tracts towards the protocerebrum (magenta arrows in &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;).&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-44&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.008&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.008&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22884"><img data-img="/content/elife/3/e04407/F4/F5.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F4/F5.medium.gif" alt="Figure 4&#x2014;figure supplement 1."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F4/F5.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F4/F5.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22884">Download powerpoint</a></span></span><span class="fig-label">Figure 4—figure supplement 1.</span> <span class="caption-title">Loss of <em>otd</em> from the LALv1 neuroblast lineage, transforms it into an antennal lobe PN lineage.</span><p id="p-43">(<strong>A</strong> and <strong>B</strong>) are Tubulin-Gal4 labelled adult, wild-type clones of the LALv1 lineage. (<strong>A</strong>) shows the anterior sections and (<strong>B</strong>) shows more posterior sections of the same clone. (<strong>A</strong>) The cell bodies of LALv1 lie ventral to the antennal lobe (AL) and express the transcription factor otd (inset in <strong>A</strong>). The axon tract of this lineage skirts around the AL (magenta arrows in <strong>A</strong> and <strong>B</strong>) and innervates the lateral accessory lobe (LAL, magenta dotted lines in <strong>A</strong>) and the central complex (CC, magenta dotted lines in <strong>B</strong>). Note that there are no innervation from this lineage in the AL. (<strong>C</strong> and <strong>D</strong>) are tubulin labelled adult, otd null clones of the LALv1 NB lineage. (<strong>C</strong>) shows the anterior sections, and (<strong>D</strong>) shows more posterior sections of the same clone. The otd<sup><em>−/−</em></sup> LALv1 lineage innervates the AL (magenta dotted lines in <strong>C</strong> and <strong>D</strong>) and sends its axon tracts towards the protocerebrum (magenta arrows in <strong>D</strong>).</p><p id="p-44"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.008">http://dx.doi.org/10.7554/eLife.04407.008</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22885"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F6"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F4/F6.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22885" title="Figure 4&#x2014;figure supplement 2." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 4&#x2014;figure supplement 2.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Expressing full-length &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt; in the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage rescues the transformation phenotype.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-45&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) is a MARCM labelled &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage that also expresses &amp;lt;em&amp;gt;UAS&amp;lt;/em&amp;gt;-&amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;. The cell bodies of the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage shown in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt; (magenta dotted lines) express the transgenic &amp;lt;em&amp;gt;UAS-otd&amp;lt;/em&amp;gt;, shown in &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt; (magenta dotted lines). The neuroanatomy of this rescued lineage resembles that of the WT LALv1 lineage, which skirts around the antennal lobe (yellow dotted lines) and innervates the lateral accessory lobe (white arrowheads). Genotype: &amp;lt;em&amp;gt;FRT19A,otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; &amp;lt;em&amp;gt;or oc&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;2&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; Tub-Gal4,UAS-mCD8::GFP/UAS-otd&amp;lt;/em&amp;gt;).&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-46&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.009&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.009&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22885"><img data-img="/content/elife/3/e04407/F4/F6.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F4/F6.medium.gif" alt="Figure 4&#x2014;figure supplement 2."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F4/F6.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F4/F6.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22885">Download powerpoint</a></span></span><span class="fig-label">Figure 4—figure supplement 2.</span> <span class="caption-title">Expressing full-length <em>otd</em> in the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage rescues the transformation phenotype.</span><p id="p-45">(<strong>A</strong>–<strong>D</strong>) is a MARCM labelled <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage that also expresses <em>UAS</em>-<em>otd</em>. The cell bodies of the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage shown in <strong>A</strong> (magenta dotted lines) express the transgenic <em>UAS-otd</em>, shown in <strong>B</strong> (magenta dotted lines). The neuroanatomy of this rescued lineage resembles that of the WT LALv1 lineage, which skirts around the antennal lobe (yellow dotted lines) and innervates the lateral accessory lobe (white arrowheads). Genotype: <em>FRT19A,otd</em><sup><em>YH13</em></sup> <em>or oc</em><sup><em>2</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; Tub-Gal4,UAS-mCD8::GFP/UAS-otd</em>).</p><p id="p-46"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.009">http://dx.doi.org/10.7554/eLife.04407.009</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div></div><p id="p-47">The cell body position of this type of <em>otd</em><sup><em>−/−</em></sup> clone as well as the entry point of its tract into the antennal lobe (both ventro-medial) did not correspond to any of the known <em>GH146-Gal4</em> labelled antennal lobe lineages (<a id="xref-ref-13-6" class="xref-bibr" href="#ref-13">Ito et al., 2013</a>; <a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Yu et al., 2013</a>). Importantly, this is also true for the ALv1 lineage, whose cell bodies are also located ventral to the antennal lobe; despite the ventral cell body position of the LALv1 and ALv1 lineages, their overall neuroanatomy is very different from each other. The neurites of the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage enter the lobe medially (magenta asterisk in <a id="xref-fig-7-1" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5A,D,H</a>) while the neurites of the ALv1 (as well as the <em>otd</em><sup><em>−/−</em></sup> ALv1) enter it laterally (yellow asterisk in <a id="xref-fig-7-2" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5A,D,H</a>). Moreover, while the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage uses the medial antennal lobe tract (magenta arrow in <a id="xref-fig-7-3" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5A,D,H</a>), the ALv1 (as well as the <em>otd</em><sup><em>−/−</em></sup> ALv1) uses the mediolateral antennal lobe tract (yellow arrow in <a id="xref-fig-7-4" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5A,C</a>). Finally, while the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage first innervates the calyx of the mushroom body and then the lateral horn, the GH146-labelled neurons of ALv1 (as well as the <em>otd</em><sup><em>−/−</em></sup> ALv1) largely innervate only the lateral horn. The <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage was also recovered along with each of the three antennal lobe lineages, and in one case all three known <em>GH146-Gal4</em> labelled antennal lobe lineages (ALad1, ALl1 and ALv1) were recovered along with it, resulting in four distinct <em>GH146-Gal4</em> labelled lineages in the brain (<a id="xref-fig-7-5" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5A–C</a>).</p><div id="F7" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22873"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F7"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F7.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22873" title="Figure 5." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 5.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Loss of &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt; from the LALv1 lineage results in an extra, fourth antennal lobe lineage labelled by the &amp;lt;em&amp;gt;GH146&amp;lt;/em&amp;gt; enhancer.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-48&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) document a MARCM clonal brain, in which the clones null for &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt; function and are labelled by the &amp;lt;em&amp;gt;GH146-Gal4&amp;lt;/em&amp;gt;. In this brain, the three known antennal lobe lineages normally labelled by &amp;lt;em&amp;gt;GH146-Gal4&amp;lt;/em&amp;gt; (ALad1, ALl1, and Alv1) have been recovered, along with an additional fourth neuroblast lineage, LALv1. (&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;) document a control brain where Otd is not down regulated in the LALv1 lineage (yellow dotted lines in &amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;). In this brain, there are three clusters of antennal lobe projection neurons labelled by &amp;lt;em&amp;gt;GH146-QF&amp;lt;/em&amp;gt; corresponding to the ALad1, ALl1 and ALv1 lineages, magenta dotted lines in &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;). (&amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;K&amp;lt;/strong&amp;gt;) show a brain in which there has been efficient knock down of Otd in the LALv1 lineage (note loss of Otd immunolabelling ventral to the antennal lobe; yellow dotted lines in &amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;). In this brain, apart from the ALad1, ALl1, and ALv1 projection neurons, an additional, fourth cluster of cells is seen innervating the antennal lobe (&#x2018;4&#x2019;, LALv1). The yellow asterisks in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt; indicate the point of entry of the ALv1 lineage into the antennal lobe, and the yellow arrow indicates its axon tract. The magenta asterisks in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt; indicate the point of entry of the LALv1 lineage into the antennal lobe, and the magenta arrow indicates its axon tract. Note that these are distinct from each other. Genotype in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A,otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;K&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;UASdicer/+;Insc-Gal4/UAS-miRNA-otd-1;GH146-QF,QUAS-mtdTomato-HA/+&amp;lt;/em&amp;gt;). Scale bars are 50 &#xB5;m. The one in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt; is applicable to &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt; and the one in in &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt; is applicable to &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;K&amp;lt;/strong&amp;gt;. Yellow line represents the midline.&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-49&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.010&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.010&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22873"><img data-img="/content/elife/3/e04407/F7.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F7.medium.gif" alt="Figure 5."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F7.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F7.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22873">Download powerpoint</a></span></span><span class="fig-label">Figure 5.</span> <span class="caption-title">Loss of <em>otd</em> from the LALv1 lineage results in an extra, fourth antennal lobe lineage labelled by the <em>GH146</em> enhancer.</span><p id="p-48">(<strong>A</strong>–<strong>C</strong>) document a MARCM clonal brain, in which the clones null for <em>otd</em> function and are labelled by the <em>GH146-Gal4</em>. In this brain, the three known antennal lobe lineages normally labelled by <em>GH146-Gal4</em> (ALad1, ALl1, and Alv1) have been recovered, along with an additional fourth neuroblast lineage, LALv1. (<strong>D</strong>–<strong>G</strong>) document a control brain where Otd is not down regulated in the LALv1 lineage (yellow dotted lines in <strong>E</strong>). In this brain, there are three clusters of antennal lobe projection neurons labelled by <em>GH146-QF</em> corresponding to the ALad1, ALl1 and ALv1 lineages, magenta dotted lines in <strong>D</strong> and <strong>G</strong>). (<strong>H</strong>–<strong>K</strong>) show a brain in which there has been efficient knock down of Otd in the LALv1 lineage (note loss of Otd immunolabelling ventral to the antennal lobe; yellow dotted lines in <strong>I</strong>). In this brain, apart from the ALad1, ALl1, and ALv1 projection neurons, an additional, fourth cluster of cells is seen innervating the antennal lobe (‘4’, LALv1). The yellow asterisks in <strong>A</strong>, <strong>D</strong>, <strong>H</strong> indicate the point of entry of the ALv1 lineage into the antennal lobe, and the yellow arrow indicates its axon tract. The magenta asterisks in <strong>A</strong>, <strong>D</strong>, <strong>H</strong> indicate the point of entry of the LALv1 lineage into the antennal lobe, and the magenta arrow indicates its axon tract. Note that these are distinct from each other. Genotype in <strong>A</strong>–<strong>C</strong>: <em>FRT19A,otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+</em>. Genotype in <strong>D</strong>–<strong>K</strong>: <em>UASdicer/+;Insc-Gal4/UAS-miRNA-otd-1;GH146-QF,QUAS-mtdTomato-HA/+</em>). Scale bars are 50 µm. The one in <strong>A</strong> is applicable to <strong>B</strong> and <strong>C</strong> and the one in in <strong>D</strong> is applicable to <strong>D</strong>–<strong>K</strong>. Yellow line represents the midline.</p><p id="p-49"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.010">http://dx.doi.org/10.7554/eLife.04407.010</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><p id="p-50">We further confirmed this using the <em>GH146-QF</em> driver, which, like the <em>GH146-Gal4</em> driver labels the ALad1, ALl1, and the ALv1 lineages (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Potter et al., 2010</a>). In this background, we used the pan-neuroblast-specific <em>Insc-Gal4</em> driver to down regulate Otd expression early in all neuroblasts. As expected, in control brains, the <em>GH146</em>-<em>QF</em> driver labelled a total of three antennal lobe lineages ALad1, ALl1, and ALv1 (magenta dotted lines in <a id="xref-fig-7-6" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5D–G</a>). In contrast, in brains where <em>otd</em> was efficiently down regulated in all neuroblasts, the <em>GH146-QF</em> driver labelled an additional fourth projection interneuron lineage in addition to the three clusters normally seen (<a id="xref-fig-7-7" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5H–K</a>). This confirms that the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage is distinct from the other antennal lobe lineages and results in the addition of an extra projection interneuron lineage in the antennal lobe. Taken together, these data suggest that the loss of <em>otd</em> from the LALv1 neuroblast results in the addition of an extra antennal lobe lineage.</p></div><div id="sec-7" class="subsection"><h4>Loss of the LEp tract</h4><p id="p-51">If the loss of <em>otd</em> from the neuroblast of the LALv1 lineage does indeed result in its neuroanatomical transformation into a lineage of a different fate, then this should correspond to the loss of the LALv1-specific axon tract (LEp) in the brain. To investigate this, we first characterized the axon tract of the wild-type LALv1 lineage, which is readily identifiable in the adult brain based on Neuroglian immunolabelling patterns (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Pereanu et al., 2010</a>). In wild-type brains, Neuroglian immunolabelling shows the loVM tract (cyan arrow, left hemisphere in <a id="xref-fig-8-1" class="xref-fig" href="#F8" data-fragment-nid="22874">Figure 6C,D</a>) and the characteristic LEp tract of the LALv1 lineage, which projects around the antennal lobes and towards the central complex (cyan arrowhead, left hemisphere in <a id="xref-fig-8-2" class="xref-fig" href="#F8" data-fragment-nid="22874">Figure 6C,D</a>). In brains in which one LALv1 neuroblast clone is mutant, the brain hemisphere that contains the <em>otd</em><sup><em>−/−</em></sup> LALv1 neuroblast clone (identified by the loss of Otd immunolabelling; yellow dotted lines in <a id="xref-fig-8-3" class="xref-fig" href="#F8" data-fragment-nid="22874">Figure 6B′</a>), still shows the loVM tract (cyan arrow, right hemisphere in <a id="xref-fig-8-4" class="xref-fig" href="#F8" data-fragment-nid="22874">Figure 6C,D</a>), which is shared by other lineages. However, the LALv1-specific LEp tract, which projects towards the central complex is entirely missing (cyan arrowhead, right hemisphere in <a id="xref-fig-8-5" class="xref-fig" href="#F8" data-fragment-nid="22874">Figure 6C,D</a>). This shows that loss of <em>otd</em> from the LALv1 neuroblast corresponds to the loss of the LEp tract of the wild-type LALv1 lineage, providing support that loss of <em>otd</em> from the LALv1 neuroblast transforms it into a lineage of different identity.</p><div id="F8" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22874"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F8"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F8.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22874" title="Figure 6." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 6.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Loss of &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt; from the LALv1 lineage results in the absence of the LEp tract.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-52&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) and (&amp;lt;strong&amp;gt;A&#x2032;&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;D&#x2032;&amp;lt;/strong&amp;gt;) document a brain with a single MARCM labelled &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage. (&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) show more posterior sections that contain the tracts of the LALv1 lineage; (&amp;lt;strong&amp;gt;A&#x2032;&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;D&#x2032;&amp;lt;/strong&amp;gt;) show more anterior sections of the same brain that documents its cell bodies. The right brain hemisphere contains an &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage as seen by the loss of Otd immunolabelling ventral to the antennal lobe (yellow dotted lines in &amp;lt;strong&amp;gt;B&#x2032;&amp;lt;/strong&amp;gt;), while the left hemisphere contains a wild-type LALv1 lineage (hence not labelled by MARCM) as seen by the presence of Otd immunolabelling (magenta dotted lines in &amp;lt;strong&amp;gt;B&#x2032;&amp;lt;/strong&amp;gt;). In the left brain hemisphere, which contains the wild-type LALv1 lineage, the loVM (cyan arrow on the left in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) and the LALv1 specific LEp tracts (cyan arrowhead on the left in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) that are identifiable by Neuroglian immunolabelling (highlighted in magenta). In the right brain hemisphere, which contains the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage, the loVM tract (taken by other lineages) is still present (cyan arrow on the right in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;). The LALv1 specific LEp tract (cyan arrowhead on the right in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) that is exclusively made by the LALv1 lineage, is entirely missing in the right brain hemisphere, which contains the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage (cyan arrow on the right in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;). The yellow arrowheads in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;, &amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt; point to the new tract of the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage innervating the antennal lobe (magenta asterisk). Genotype: &amp;lt;em&amp;gt;FRT19A, otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. The midline is represented by a yellow line in all images.&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-53&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.011&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.011&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22874"><img data-img="/content/elife/3/e04407/F8.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F8.medium.gif" alt="Figure 6."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F8.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F8.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22874">Download powerpoint</a></span></span><span class="fig-label">Figure 6.</span> <span class="caption-title">Loss of <em>otd</em> from the LALv1 lineage results in the absence of the LEp tract.</span><p id="p-52">(<strong>A</strong>–<strong>D</strong>) and (<strong>A′</strong>–<strong>D′</strong>) document a brain with a single MARCM labelled <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage. (<strong>A</strong>–<strong>D</strong>) show more posterior sections that contain the tracts of the LALv1 lineage; (<strong>A′</strong>–<strong>D′</strong>) show more anterior sections of the same brain that documents its cell bodies. The right brain hemisphere contains an <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage as seen by the loss of Otd immunolabelling ventral to the antennal lobe (yellow dotted lines in <strong>B′</strong>), while the left hemisphere contains a wild-type LALv1 lineage (hence not labelled by MARCM) as seen by the presence of Otd immunolabelling (magenta dotted lines in <strong>B′</strong>). In the left brain hemisphere, which contains the wild-type LALv1 lineage, the loVM (cyan arrow on the left in <strong>C</strong>) and the LALv1 specific LEp tracts (cyan arrowhead on the left in <strong>C</strong>) that are identifiable by Neuroglian immunolabelling (highlighted in magenta). In the right brain hemisphere, which contains the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage, the loVM tract (taken by other lineages) is still present (cyan arrow on the right in <strong>C</strong>). The LALv1 specific LEp tract (cyan arrowhead on the right in <strong>C</strong>) that is exclusively made by the LALv1 lineage, is entirely missing in the right brain hemisphere, which contains the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage (cyan arrow on the right in <strong>C</strong>). The yellow arrowheads in <strong>A</strong>, <strong>C</strong>, <strong>D</strong> point to the new tract of the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage innervating the antennal lobe (magenta asterisk). Genotype: <em>FRT19A, otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+</em>. The midline is represented by a yellow line in all images.</p><p id="p-53"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.011">http://dx.doi.org/10.7554/eLife.04407.011</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div></div><div id="sec-8" class="subsection"><h4>An independent molecular marker</h4><p id="p-54">To investigate the identity of the mutant LALv1 lineage further, we identified Acj6 as a molecular marker that could unambiguously identify the LALv1 lineage in wild-type and <em>otd</em> mutants. Acj6 is a POU transcription factor that is known to be expressed in the ALad1 lineage and in a subset of the ALl1-derived projection interneurons of the wild-type brain (<a id="xref-fig-9-1" class="xref-fig" href="#F9" data-fragment-nid="22875">Figure 7B</a>). In addition to these two cell clusters, we observed a third Acj6 positive cell cluster ventral to the antennal lobe of the wild-type brain (cyan arrowhead in <a id="xref-fig-9-2" class="xref-fig" href="#F9" data-fragment-nid="22875">Figure 7B</a>). MARCM clonal labelling using the <em>Per-Gal4</em> enhancer line together with anti-Acj6 and anti-Otd antibodies unambiguously identified this cluster as the LALv1 lineage (cyan arrowhead in <a id="xref-fig-9-3" class="xref-fig" href="#F9" data-fragment-nid="22875">Figure 7A–D</a>). Importantly, this cell cluster continues to express Acj6 immunoreactivity following mutational inactivation of <em>otd</em> in the LALv1 lineage (<a id="xref-fig-9-4" class="xref-fig" href="#F9" data-fragment-nid="22875">Figure 7F,G</a>). Thus, Acj6 provides a molecular marker for the identification of the LALv1 lineage independent of Otd expression in wild-type and mutant clones. The analysis of <em>otd</em><sup><em>−/−</em></sup> LALv1 MARCM clones identified by Acj6 immunolabelling and co-labelled by <em>GH146-Gal4</em> shows that the fourth <em>GH146-</em>positive neuroblast clone described above does indeed correspond to the mutant LALv1 lineage (<a id="xref-fig-9-5" class="xref-fig" href="#F9" data-fragment-nid="22875">Figure 7E,H</a>). This confirms that upon the loss of <em>otd</em> from the neuroblast, the neural progeny of the LALv1 lineage transform into an antennal lobe fate.</p><div id="F9" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22875"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F9"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F9.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22875" title="Figure 7." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 7.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;The Acj6-positive cell cluster ventral to the AL is the LALv1 lineage and when null for &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;, innervates the AL.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-55&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) document a WT MARCM clone of the LALv1 lineage labelled with &amp;lt;em&amp;gt;Per-Gal4&amp;lt;/em&amp;gt;. This brain has been immunolabelled with Acj6 (&amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;) and Otd (&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;). In &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;, only three clusters of Acj6 positive cells (yellow dotted lines) are seen around each of the ALs (yellow dotted lines). The clusters that are ventral to AL (cyan arrowhead) belong to the LALv1 lineage as seen by the WT MARCM clone shown in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;. As expected, these cells are also Otd positive (cyan arrowhead in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;). (&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt;) show an &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 MARCM clone. This brain is also immunolabelled with Acj6 (&amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt;) and Otd (&amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;). Note that the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 cells, which have a transformed neuroanatomy (cyan arrowhead in &amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;), are null for &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt; (cyan arrowhead in &amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;) but are identifiable as the ventral cluster of Acj6 positive cells (cyan arrowhead in &amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt;). Genotype in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A, otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Scale bar is 50 &#xB5;m.&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-56&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.012&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.012&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22875"><img data-img="/content/elife/3/e04407/F9.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F9.medium.gif" alt="Figure 7."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F9.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F9.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22875">Download powerpoint</a></span></span><span class="fig-label">Figure 7.</span> <span class="caption-title">The Acj6-positive cell cluster ventral to the AL is the LALv1 lineage and when null for <em>otd</em>, innervates the AL.</span><p id="p-55">(<strong>A</strong>–<strong>D</strong>) document a WT MARCM clone of the LALv1 lineage labelled with <em>Per-Gal4</em>. This brain has been immunolabelled with Acj6 (<strong>B</strong>) and Otd (<strong>C</strong>). In <strong>B</strong>, only three clusters of Acj6 positive cells (yellow dotted lines) are seen around each of the ALs (yellow dotted lines). The clusters that are ventral to AL (cyan arrowhead) belong to the LALv1 lineage as seen by the WT MARCM clone shown in <strong>A</strong>. As expected, these cells are also Otd positive (cyan arrowhead in <strong>C</strong>). (<strong>E</strong>–<strong>H</strong>) show an <em>otd</em><sup><em>−/−</em></sup> LALv1 MARCM clone. This brain is also immunolabelled with Acj6 (<strong>F</strong>) and Otd (<strong>G</strong>). Note that the <em>otd</em><sup><em>−/−</em></sup> LALv1 cells, which have a transformed neuroanatomy (cyan arrowhead in <strong>E</strong>), are null for <em>otd</em> (cyan arrowhead in <strong>G</strong>) but are identifiable as the ventral cluster of Acj6 positive cells (cyan arrowhead in <strong>F</strong>). Genotype in <strong>A</strong>–<strong>D</strong>: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/+</em>. Genotype in <strong>E</strong>–<strong>H</strong>: <em>FRT19A, otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+</em>. Scale bar is 50 µm.</p><p id="p-56"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.012">http://dx.doi.org/10.7554/eLife.04407.012</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><p id="p-57">Interestingly, the cell body position of the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons varied somewhat in the 76 <em>otd</em><sup><em>−/−</em></sup> neuroblast clones obtained (from both the <em>Tub-Gal4</em> and <em>GH146-Gal4</em> experiments). In most cases (54/76) the cell body position of the <em>otd</em><sup><em>−/−</em></sup> LALv1 neuroblast clones remained ventral to the antennal lobe, similar to the cell body position of the wild-type central complex lineage. Thus, in most cases, in terms of cell body position the neuroanatomy did not transform towards the antennal lobe lineage position (antero-dorsal to the adult antennal lobe). Occasionally, however, the cell bodies were shifted closer to the midline (17/76), and in a few rare cases the <em>otd</em><sup><em>−/−</em></sup> LALv1 neuronal cell bodies were located antero-dorsal to the antennal lobe (5/76), a position similar to the wild-type antennal lobe lineage (see <a id="xref-media-3-1" class="xref-other" href="#media-3">Video 3</a>). This suggests that loss of <em>otd</em> in LALv1 neurons not only consistently transform their axonal and dendritic terminals towards the antennal lobe lineage neuroanatomy but also relocate their cell bodies in some cases to resemble the ALad1 antennal lobe projection neuron lineage.<div class="media video-content" id="media-3"><div class="media-inline video-inline"><div class="elife-inline-video"><video controls="controls" poster="http://static-movie-usa.glencoesoftware.com/jpg/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v003.jpg" preload="none" style="background-color:black"><source src="http://static-movie-usa.glencoesoftware.com/mp4/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v003.mp4" type="video/mp4; codecs=&quot;avc1.42E01E, mp4a.40.2&quot;"></source><source src="http://static-movie-usa.glencoesoftware.com/ogv/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v003.ogv" type="video/ogg; codecs=&quot;theora, vorbis&quot;"></source><img src="http://static-movie-usa.glencoesoftware.com/jpg/10.7554/118/d679b2c58bf1802d1982180c8c3644d322d4b265/elife04407v003.jpg"></img>Your browser does not appear to support HTML5 video. Please upgrade your browser.</video><div class="elife-video-links"><span class="elife-video-link elife-video-link-download"><a href="/elife/download-video/http%253A%252F%252Fstatic-movie-usa.glencoesoftware.com%252Fmp4%252F10.7554%252F118%252Fd679b2c58bf1802d1982180c8c3644d322d4b265%252Felife04407v003.mp4">Download Video </a></span></div></div></div><div class="media-caption"><span class="media-label">Video 3.</span> <span class="caption-title">Projection pattern of the otd<sup>−/−</sup> LAlv1 lineage.</span><p id="p-58" class="first-child"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.015">http://dx.doi.org/10.7554/eLife.04407.015</a></p></div></div></p><p id="p-59">In summary, loss of <em>otd</em> from the LALv1 neuroblast results in a transformation of its progeny neurons, from a wild-type central complex identity to an antennal lobe projection neuron identity.</p></div></div><div id="sec-9" class="subsection"><h3>Overexpression of Otd in the antennal lobe lineage results in a partial reciprocal transformation</h3><p id="p-60">We next asked whether <em>otd</em> gain-of-function in the antennal lobe lineage, ALad1, might result in a reciprocal anatomical transformation of this lineage into one resembling the wild-type central complex lineage, LALv1. We used the MARCM system to misexpress the full-length <em>otd</em> coding sequence in the antennal lobe neuroblast clones using a <em>Tub-Gal4</em> driver. In all Otd misexpression clones of the antennal lobe lineage (15/15), we found a partial transformation of this lineage towards the central complex identity (<a id="xref-fig-10-1" class="xref-fig" href="#F10" data-fragment-nid="22865">Figure 8</a>). All 15 clones comprised a few cells that retained neuroanatomical features of the wild-type antennal lobe lineage such as antero-dorsal cell body position, innervation of the antennal lobe, and axonal projections via the medial antennal lobe tract (yellow asterisk and arrowhead in <a id="xref-fig-10-2" class="xref-fig" href="#F10" data-fragment-nid="22865">Figure 8A–C</a>). However, most of the cells in the clones displayed neuroanatomical features of the central complex lineage. These cell bodies were positioned ventral to the adult antennal lobe, they projected their axons via the loVM and LEp tracts and they innervated the lateral accessory lobe (magenta asterisk and white arrowheads in <a id="xref-fig-10-3" class="xref-fig" href="#F10" data-fragment-nid="22865">Figure 8A–C</a>). Thus, <em>otd</em> gain-of-function was able to cause a partial, albeit incomplete transformation, of the antennal lobe lineage into a central complex-like lineage.</p><div id="F10" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22865"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F10"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F10.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22865" title="Figure 8." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 8.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Overexpression of Otd in the ALad1 lineage results in a partial reciprocal transformation.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-61&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) ALad1 neuroblast clone misexpressing Otd. (&amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) are the anterior and lateral views of the 3D reconstructions of the clone in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;. Few of the Otd misexpressing cells (yellow asterisk in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) retain the wild type neuroanatomy of the ALad1 lineage (magenta cells in the reconstructions)&#x2014;they have innervations in the AL glomeruli (yellow arrowheads in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) and project via the antennal-cerebral tract (yellow arrowhead in &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;). Most of the Otd misexpressing ALad1 neurons are seen ventral to the AL (magenta asterisk in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;; green cells in the reconstructions in &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;). They do not innervate the AL and instead project towards the LAL (white arrowheads in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;). Genotype in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; Tub-Gal4,UAS-mCD8::GFP/UAS-otd&amp;lt;/em&amp;gt;. Midline is represented by a yellow line. Scale bar is 50 &#xB5;m.&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-62&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.016&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.016&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22865"><img data-img="/content/elife/3/e04407/F10.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F10.medium.gif" alt="Figure 8."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F10.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F10.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22865">Download powerpoint</a></span></span><span class="fig-label">Figure 8.</span> <span class="caption-title">Overexpression of Otd in the ALad1 lineage results in a partial reciprocal transformation.</span><p id="p-61">(<strong>A</strong>–<strong>C</strong>) ALad1 neuroblast clone misexpressing Otd. (<strong>B</strong> and <strong>C</strong>) are the anterior and lateral views of the 3D reconstructions of the clone in <strong>A</strong>. Few of the Otd misexpressing cells (yellow asterisk in <strong>A</strong>) retain the wild type neuroanatomy of the ALad1 lineage (magenta cells in the reconstructions)—they have innervations in the AL glomeruli (yellow arrowheads in <strong>A</strong>–<strong>C</strong>) and project via the antennal-cerebral tract (yellow arrowhead in <strong>C</strong>). Most of the Otd misexpressing ALad1 neurons are seen ventral to the AL (magenta asterisk in <strong>A</strong>; green cells in the reconstructions in <strong>B</strong> and <strong>C</strong>). They do not innervate the AL and instead project towards the LAL (white arrowheads in <strong>A</strong>–<strong>C</strong>). Genotype in <strong>A</strong>–<strong>C</strong>: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; Tub-Gal4,UAS-mCD8::GFP/UAS-otd</em>. Midline is represented by a yellow line. Scale bar is 50 µm.</p><p id="p-62"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.016">http://dx.doi.org/10.7554/eLife.04407.016</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div></div><div id="sec-10" class="subsection"><h3>Specific molecular changes occur in the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage</h3><p id="p-63">The innervation pattern of the <em>otd</em><sup><em>−/−</em></sup> LALv1 neuroblast lineage strikingly resembled that of the antennal lobe lineage, ALad1. Furthermore, the projection neuron-specific <em>GH146</em> driver, which does not label the wild-type central complex lineage, labelled the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons. This ectopic activation of the <em>GH146</em> driver in the <em>otd</em><sup><em>−/−</em></sup> neurons indicates that the anatomical transformation of the mutant neurons might be accompanied by corresponding molecular transformations that reflect a transformation to the antennal lobe lineage identity. To test if this is indeed the case, we analyzed the activity of select molecular markers (the Gal4 driver lines described above) in the central complex lineage (summarized in <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). In order to test the activity of these enhancers in the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage, we generated <em>otd</em><sup><em>−/−</em></sup> MARCM clones and used these selected <em>Gal4</em> lines to label the mutant lineage (see schematic in <a id="xref-fig-3-4" class="xref-fig" href="#F3" data-fragment-nid="22871">Figure 3A</a>). Finally, we also tested the expression of the homeodomain transcription factor LIM1, which is expressed in the wild-type LALv1 lineage and is not expressed in the wild-type ALad1 lineage.</p><p id="p-64">As described above, when we generated wild-type MARCM clones and used either <em>OK371</em>-<em>Gal4</em> or <em>Per</em>-<em>Gal4</em> driver lines to label the clones, we found that both lines were able to drive reporter expression in the wild-type LALv1 lineage (<a id="xref-fig-3-5" class="xref-fig" href="#F3" data-fragment-nid="22871">Figure 3B–E,J–M</a>). In contrast, neither of these driver lines was able to drive reporter expression in the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage (<a id="xref-fig-3-6" class="xref-fig" href="#F3" data-fragment-nid="22871">Figure 3F–I,N–Q</a>). This suggests that the <em>OK371</em>-<em>Gal4</em> and <em>Per</em>-<em>Gal4</em> driver lines are suppressed in the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons.</p><p id="p-65">However, in these experiments, the transformed neurons were not labelled at all because the activity of the drivers was suppressed in the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage. In order to confirm this finding, we decided to positively label the <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons and in this background assay the activity of the <em>Gal4</em> drivers. In order to do this, we utilized the dual MARCM method (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Lai and Lee, 2006</a>), which uses two independent binary expression systems (Gal4-UAS and LexA–LexA operator) to label the MARCM clones. In these experiments, we used the <em>GH146-LexA</em> driver to label the <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons positively and combined it with <em>OK371</em>-<em>Gal4</em> and <em>Per</em>-<em>Gal4</em> driver lines to assay their activities. We first tested if the <em>GH146-LexA</em> driver, like the <em>GH146-Gal4</em> and the <em>GH146-QF</em> driver, was active in the <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons. Thus, in the first set of dual MARCM experiments, we combined the <em>GH146-LexA</em> with <em>Tubulin-Gal4</em> (<a id="xref-fig-11-1" class="xref-fig" href="#F11" data-fragment-nid="22866">Figure 9A–D</a>). Under these conditions, when we generated <em>otd</em><sup><em>−/−</em></sup> neuroblast clones in the LALv1 lineage, we found that the transformed neurons were labelled by both the <em>Tubulin-Gal4</em> (magenta dotted lines and inset in <a id="xref-fig-11-2" class="xref-fig" href="#F11" data-fragment-nid="22866">Figure 9A</a>) and <em>GH146-LexA</em> (magenta dotted lines and inset in <a id="xref-fig-11-3" class="xref-fig" href="#F11" data-fragment-nid="22866">Figure 9B</a>) drivers, confirming that <em>GH146-LexA</em>, like <em>GH146-Gal4</em> and the <em>GH146-QF</em>, is active in the <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons and thus able to label it. In the following dual MARCM experiments, we used the <em>GH146-LexA</em> to positively label the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons and combined it with either <em>OK371</em>-<em>Gal4</em> or <em>Per</em>-<em>Gal4</em> driver lines to assay for their activity in the transformed neurons. In both cases, while the <em>GH146-LexA</em> positively labelled the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons (magenta dotted lines and insets in <a id="xref-fig-11-4" class="xref-fig" href="#F11" data-fragment-nid="22866">Figure 9E,I</a>), neither of the Gal4 driver lines was able to drive reporter expression in these mutant neurons (magenta dotted lines and insets in <a id="xref-fig-11-5" class="xref-fig" href="#F11" data-fragment-nid="22866">Figure 9F,J</a>). This indicates that enhancers that are normally active in the wild-type central complex lineage and inactive in the antennal lobe lineage become suppressed in the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage.</p><div id="F11" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22866"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F11"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F11.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22866" title="Figure 9." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 9.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Specific molecular changes occur in the transformed &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-66&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) documents an &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 neuroblast clone that has been dual labelled with &amp;lt;em&amp;gt;Tubulin-Gal4&amp;lt;/em&amp;gt; (&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) and &amp;lt;em&amp;gt;GH146-LexA&amp;lt;/em&amp;gt; (&amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;) using the dual MARCM technique. (&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt;) document &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 neuroblast clones that have been dual labelled with &amp;lt;em&amp;gt;GH146-LexA&amp;lt;/em&amp;gt; (&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;) to positively label the mutant neurons and either &amp;lt;em&amp;gt;OK371-Gal4&amp;lt;/em&amp;gt; (&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt;) or Per-Gal4 (&amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt;) using the dual MARCM technique. In these clones, while the &amp;lt;em&amp;gt;GH146-LexA&amp;lt;/em&amp;gt; driver labels the transformed &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 neurons (&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;), neither the &amp;lt;em&amp;gt;OK371-Gal4&amp;lt;/em&amp;gt; (&amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt;) nor the &amp;lt;em&amp;gt;Per-Gal4&amp;lt;/em&amp;gt; (&amp;lt;strong&amp;gt;J&amp;lt;/strong&amp;gt;) do. (&amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;P&amp;lt;/strong&amp;gt;) documents a transformed &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage (inset and magenta dotted lines in &amp;lt;strong&amp;gt;N&amp;lt;/strong&amp;gt;) labelled with &amp;lt;em&amp;gt;Cha7.4-Gal4&amp;lt;/em&amp;gt;. While &amp;lt;em&amp;gt;Cha7.4-Gal4&amp;lt;/em&amp;gt; is not active in the wild-type LALv1 lineage (not shown), it becomes activated in the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage (inset and magenta dotted lines in &amp;lt;strong&amp;gt;M&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;P&amp;lt;/strong&amp;gt;). Genotypes in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A, otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; Tubulin-Gal4,UAS-mCD8::GFP/FRTG13,GH146-LexA::GAD,LexAop::rCD2::GFP&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A,otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; OK371-Gal4,UAS-mCD8::GFP/FRTG13,GH146-LexA::GAD,LexAop::rCD2::GFP&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A,otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP;Per-Gal4,UAS-mCD8::GFP/FRTG13,GH146-LexA::GAD,LexAop::rCD2::GFP&amp;lt;/em&amp;gt;. Genotype in &amp;lt;strong&amp;gt;H&amp;lt;/strong&amp;gt;: &amp;lt;em&amp;gt;FRT19A,otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; Cha7.4-Gal4,UAS-mCD8::GFP/+&amp;lt;/em&amp;gt;. Midline is represented by a yellow line.&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-67&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.017&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.017&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22866"><img data-img="/content/elife/3/e04407/F11.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F11.medium.gif" alt="Figure 9."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F11.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F11.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22866">Download powerpoint</a></span></span><span class="fig-label">Figure 9.</span> <span class="caption-title">Specific molecular changes occur in the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage.</span><p id="p-66">(<strong>A</strong>–<strong>D</strong>) documents an <em>otd</em><sup><em>−/−</em></sup> LALv1 neuroblast clone that has been dual labelled with <em>Tubulin-Gal4</em> (<strong>A</strong>) and <em>GH146-LexA</em> (<strong>B</strong>) using the dual MARCM technique. (<strong>E</strong>–<strong>H</strong> and <strong>I</strong>–<strong>L</strong>) document <em>otd</em><sup><em>−/−</em></sup> LALv1 neuroblast clones that have been dual labelled with <em>GH146-LexA</em> (<strong>E</strong> and <strong>I</strong>) to positively label the mutant neurons and either <em>OK371-Gal4</em> (<strong>E</strong>–<strong>H</strong>) or Per-Gal4 (<strong>I</strong>–<strong>L</strong>) using the dual MARCM technique. In these clones, while the <em>GH146-LexA</em> driver labels the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons (<strong>E</strong> and <strong>I</strong>), neither the <em>OK371-Gal4</em> (<strong>F</strong>) nor the <em>Per-Gal4</em> (<strong>J</strong>) do. (<strong>M</strong>–<strong>P</strong>) documents a transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage (inset and magenta dotted lines in <strong>N</strong>) labelled with <em>Cha7.4-Gal4</em>. While <em>Cha7.4-Gal4</em> is not active in the wild-type LALv1 lineage (not shown), it becomes activated in the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage (inset and magenta dotted lines in <strong>M</strong>–<strong>P</strong>). Genotypes in <strong>A</strong>–<strong>D</strong>: <em>FRT19A, otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; Tubulin-Gal4,UAS-mCD8::GFP/FRTG13,GH146-LexA::GAD,LexAop::rCD2::GFP</em>. Genotype in <strong>E</strong>–<strong>H</strong>: <em>FRT19A,otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; OK371-Gal4,UAS-mCD8::GFP/FRTG13,GH146-LexA::GAD,LexAop::rCD2::GFP</em>. Genotype in <strong>I</strong>–<strong>L</strong>: <em>FRT19A,otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP;Per-Gal4,UAS-mCD8::GFP/FRTG13,GH146-LexA::GAD,LexAop::rCD2::GFP</em>. Genotype in <strong>H</strong>: <em>FRT19A,otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; Cha7.4-Gal4,UAS-mCD8::GFP/+</em>. Midline is represented by a yellow line.</p><p id="p-67"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.017">http://dx.doi.org/10.7554/eLife.04407.017</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><p id="p-68">Might the converse be true; do enhancers that are normally inactive in the wild-type central complex lineage and active in the antennal lobe lineage become activated in the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage? To test this, we used the <em>Cha7</em>.<em>4-Gal4</em> in MARCM clonal experiments. As expected, we never recovered <em>Cha7</em>.<em>4-Gal4</em> labelled LALv1 neuroblast clones in wild-type MARCM experiments (data not shown). In contrast, when we generated <em>otd</em><sup><em>−/−</em></sup> neuroblast clones in the LALv1 lineage (identifiable by the loss of Otd staining ventral to the antennal lobe; magenta dotted lines in <a id="xref-fig-11-6" class="xref-fig" href="#F11" data-fragment-nid="22866">Figure 9N</a>) the <em>Cha7.4-Gal4</em> driver robustly drove reporter expression in the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage (magenta dotted lines and insets in <a id="xref-fig-11-7" class="xref-fig" href="#F11" data-fragment-nid="22866">Figure 9M–P</a>). This suggests that the <em>Cha7.4-Gal4</em> becomes activated in the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage. Furthermore, the concomitant loss of the <em>OK371</em>-<em>Gal4</em> driver (a putative glutamatergic label) and ectopic activation of the <em>Cha7.4-Gal4</em> driver (a putative cholinergic label) suggest that there might be a change in neurotransmitter identity of the LALv1 lineage from wild-type glutamatergic to transformed cholinergic identities, similar to the cholinergic identity of wild-type antennal lobe neurons.</p><p id="p-69">Finally, immunolabelling with an anti-LIM antibody indicates that the wild-type central complex lineage expresses the homeodomain transcription factor LIM (<a id="xref-fig-12-1" class="xref-fig" href="#F12" data-fragment-nid="22867">Figure 10G–I</a>) while the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage, like the wild-type antennal lobe lineage, does not (<a id="xref-fig-12-2" class="xref-fig" href="#F12" data-fragment-nid="22867">Figure 10J–L</a>). Interestingly, the <em>LN1-Gal4</em> driver, which is inactive in both the wild-type LALv1 and ALad1 lineages, remains inactive in the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage (data not shown).</p><div id="F12" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22867"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F12"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F12.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22867" title="Figure 10." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 10.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Specific molecular changes occur in the transformed &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-70&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;) document neuronal cell bodies of the wild-type LALv1 lineage immunolabelled with Acj6 (&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) and Lim1 (&amp;lt;strong&amp;gt;G&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;I&amp;lt;/strong&amp;gt;). The WT LALv1 neurons express both these molecular markers. (&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt; and &amp;lt;strong&amp;gt;J&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt;) document neuronal cell bodies of the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage immunolabelled with Acj6 (&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;F&amp;lt;/strong&amp;gt;) and Lim1 (&amp;lt;strong&amp;gt;J&amp;lt;/strong&amp;gt;&#x2013;&amp;lt;strong&amp;gt;L&amp;lt;/strong&amp;gt;). While &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 neurons continue to express Acj6 (&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;), they downregulate Lim1 expression (&amp;lt;strong&amp;gt;K&amp;lt;/strong&amp;gt;).&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-71&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.018&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.018&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22867"><img data-img="/content/elife/3/e04407/F12.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F12.medium.gif" alt="Figure 10."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F12.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F12.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22867">Download powerpoint</a></span></span><span class="fig-label">Figure 10.</span> <span class="caption-title">Specific molecular changes occur in the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage.</span><p id="p-70">(<strong>A</strong>–<strong>C</strong> and <strong>G</strong>–<strong>I</strong>) document neuronal cell bodies of the wild-type LALv1 lineage immunolabelled with Acj6 (<strong>A</strong>–<strong>C</strong>) and Lim1 (<strong>G</strong>–<strong>I</strong>). The WT LALv1 neurons express both these molecular markers. (<strong>D</strong>–<strong>F</strong> and <strong>J</strong>–<strong>L</strong>) document neuronal cell bodies of the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage immunolabelled with Acj6 (<strong>D</strong>–<strong>F</strong>) and Lim1 (<strong>J</strong>–<strong>L</strong>). While <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons continue to express Acj6 (<strong>E</strong>), they downregulate Lim1 expression (<strong>K</strong>).</p><p id="p-71"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.018">http://dx.doi.org/10.7554/eLife.04407.018</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><p id="p-72">Taken together, these findings indicate that the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage acquires the molecular signature of a wild-type antennal lobe lineage (see <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>) implying that <em>otd</em> loss-of-function in the LALv1 neuroblast lineage results in a molecular as well as an anatomical transformation of this lineage into one resembling the ALad1 lineage.</p></div><div id="sec-11" class="subsection"><h3>The <em>otd</em><sup><em>−/−</em></sup> transformed LALv1 lineage establishes functional connectivity in the antennal lobe and can respond to odour stimulation</h3><p id="p-73">Given the extent of the anatomical and molecular transformations seen in the <em>otd</em><sup><em>−/−</em></sup> LALv1 neuroblast lineage, might the neurons in the transformed lineage receive functional input from olfactory sensory neurons? To address this question, we specifically expressed the calcium sensor G-CaMP3 in the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage by MARCM clonal labeling (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wang et al., 2003</a>) and used two-photon microscopy to monitor calcium activity in the dendrites of these transformed neurons in the antennal lobe.</p><p id="p-74">We first tested if the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons established functional connectivity with other neurons in the antennal lobe. Typically, olfactory sensory neurons bring odour information to the antennal lobe via the antennal nerve. Here, they make synaptic connections with projection neurons and local interneurons; projection neurons, take the odour information to higher brain centres (mushroom body and lateral horn) and local interneurons process the odour information locally in the antennal lobe. We reasoned that if the transformed neurons did made functional connections within the antennal lobe they would be postsynaptic to the olfactory sensory neurons and would be activated upon the activation of the antennal nerve. We therefore electrically stimulated the antennal nerve while simultaneously monitoring calcium activity from the transformed neurons. We found that electrical stimulation of the antennal nerve, which contains the axons of the olfactory sensory neurons, resulted in an increase in calcium activity in the dendrites of the <em>otd</em><sup><em>−/−</em></sup> LALv1 lineage. Moreover, a greater number of electrical stimulus pulses applied to the antennal nerve resulted in a corresponding increase in the amplitude of the calcium signal recorded in the mutant transformed neurons (<a id="xref-fig-13-1" class="xref-fig" href="#F13" data-fragment-nid="22868">Figure 11A–C</a>). These results demonstrate that the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons were able to make functional connections in the antennal lobe and were able to receive functional input from sensory afferents.</p><div id="F13" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22868"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F13"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F13.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22868" title="Figure 11." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 11.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;The &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; transformed LALv1 lineage has functional synapses in the AL and can respond to odour stimulation.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-75&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) Two-photon calcium imaging from the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 neurons in response to electrical stimulation of the antennal nerve. Gray-scale image shows average pre-stimulation fluorescence of one transformed lineage. Colour images show fluorescence changes in response to different numbers of electrical stimulation (duration, 1 ms; amplitude, 10 V; frequency, 100 Hz). Graph in &amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt; shows peak amplitude of &#x394;F/F against the number of stimulations. Error bars represent S.E.M. n = 5. (&amp;lt;strong&amp;gt;C&amp;lt;/strong&amp;gt;) Average traces that plot &#x394;F/F over time at different stimulus intensities. Shaded region in each trace represents S.E.M. (&amp;lt;strong&amp;gt;D&amp;lt;/strong&amp;gt;) Two-photon calcium imaging from the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;&#x2212;/&#x2212;&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 neurons in response to odour stimulation of the antenna. Gray-scale image shows the structure of the transformed lineage and three identifiable glomeruli. The colour images show glomerular activation patterns evoked by isoamyl acetate (IAA), ethyl butyrate (EB), 3-octanol (3-O) and 3-heptanol (3-H). IAA activated VM2, DM2 and DM3. EB activated both VM2 and DM2. 3-O and 3-H activated DM2 and VM2, respectively. (&amp;lt;strong&amp;gt;E&amp;lt;/strong&amp;gt;) Average traces that plot &#x394;F/F over time in VM2, DM2 and DM3. The shaded region in each trace represents S.E.M. n = 3&#x2013;5. The false colour scales (&#x394;F) and scale bars are shown at the right of each panel. Genotype: &amp;lt;em&amp;gt;FRT19A, otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP;GH146-Gal4,UAS-GCaMP3/GH146-Gal4,UAS-GCaMP3&amp;lt;/em&amp;gt;.&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-76&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.019&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.019&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22868"><img data-img="/content/elife/3/e04407/F13.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F13.medium.gif" alt="Figure 11."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F13.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F13.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22868">Download powerpoint</a></span></span><span class="fig-label">Figure 11.</span> <span class="caption-title">The <em>otd</em><sup><em>−/−</em></sup> transformed LALv1 lineage has functional synapses in the AL and can respond to odour stimulation.</span><p id="p-75">(<strong>A</strong>) Two-photon calcium imaging from the <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons in response to electrical stimulation of the antennal nerve. Gray-scale image shows average pre-stimulation fluorescence of one transformed lineage. Colour images show fluorescence changes in response to different numbers of electrical stimulation (duration, 1 ms; amplitude, 10 V; frequency, 100 Hz). Graph in <strong>B</strong> shows peak amplitude of ΔF/F against the number of stimulations. Error bars represent S.E.M. n = 5. (<strong>C</strong>) Average traces that plot ΔF/F over time at different stimulus intensities. Shaded region in each trace represents S.E.M. (<strong>D</strong>) Two-photon calcium imaging from the <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons in response to odour stimulation of the antenna. Gray-scale image shows the structure of the transformed lineage and three identifiable glomeruli. The colour images show glomerular activation patterns evoked by isoamyl acetate (IAA), ethyl butyrate (EB), 3-octanol (3-O) and 3-heptanol (3-H). IAA activated VM2, DM2 and DM3. EB activated both VM2 and DM2. 3-O and 3-H activated DM2 and VM2, respectively. (<strong>E</strong>) Average traces that plot ΔF/F over time in VM2, DM2 and DM3. The shaded region in each trace represents S.E.M. n = 3–5. The false colour scales (ΔF) and scale bars are shown at the right of each panel. Genotype: <em>FRT19A, otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP;GH146-Gal4,UAS-GCaMP3/GH146-Gal4,UAS-GCaMP3</em>.</p><p id="p-76"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.019">http://dx.doi.org/10.7554/eLife.04407.019</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><p id="p-77">We further investigated if the transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons could respond to specific odour stimuli. To do this, we performed calcium imaging experiments similar to those described above but replaced the electrical stimulation of the antennal nerve with odour stimulation of the intact antenna (olfactory sensory neurons). Four different odorants (isoamyl acetate, ethyl butyrate, 3-octonal, 3-heptanol) were selected based on their ability to excite all or some of the VM2, DM2 and DM3 glomeruli (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Dacks et al., 2009</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Semmelhack and Wang, 2009</a>) in the antennal lobe, which are innervated by the <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons. Imaging calcium activity from the dendrites of the <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons in these glomeruli in response to the selected odours show that each of the four odorants evoked a unique pattern of glomerular activity. Isoamyl acetate excited all three glomeruli, whereas ethyl butyrate excited only the VM2 and DM2 glomeruli. 3-octanol and 3-heptanol, however, excited just the DM2 and VM2 glomeruli, respectively (<a id="xref-fig-13-2" class="xref-fig" href="#F13" data-fragment-nid="22868">Figure 11D,E</a>). These patterns of glomerular activation in the <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons are strikingly similar to that of wild-type antennal lobe olfactory projection neurons (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Wang et al., 2003</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Dacks et al., 2009</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Semmelhack and Wang, 2009</a>).</p><p id="p-78">Taken together, these functional studies indicate that the <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons receive specific input from olfactory sensory neurons that results in glomerulus-specific activation patterns to different odorants. This in turn implies that <em>otd</em> loss-of-function in a single neuroblast leads to a remarkably extensive reconfiguration of the macrocircuitry in the brain, which includes anatomical, molecular as well as functional transformation of neurons in the central complex lineage into neurons with properties of olfactory projection neurons.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-79">During neuronal development in both vertebrates and invertebrates neural progenitors use spatial and temporal information to generate diverse neuronal subtypes. For example, in <em>Drosophila</em>, unique spatial information imparts heterogeneity to the neuroblast pool and then temporal cues acting in the neuroblasts generate further diversity. In this way diverse neuronal subtypes are produced by the neuroblast lineages, which consequently create the diverse functional macrocircuitry of the brain. In addition, the neuroblasts in the central brain of <em>Drosophila</em> are characterized by the expression of a specific combination of cell intrinsic determinants (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Urbach and Technau, 2003a</a>) that are thought to act in the specification of unique neuroblast identity and hence control lineage-specific neuronal cell fate.</p><p id="p-80">In this study, we show that such intrinsic determinants present in the neuroblast are essential for the proper specification of the entire lineage that derives from the neuroblast. Our data demonstrate that a remarkable rewiring of the functional macrocircuitry of the brain occurs due to the manipulation of one intrinsic factor, <em>otd</em>, acting in an identified neuroblast during development. This transformation affects molecular properties, anatomical projection patterns (dendritic and axonal), and functional inputs in all of the neurons in the lineage (summarized in <a id="xref-fig-14-1" class="xref-fig" href="#F14" data-fragment-nid="22869">Figure 12</a>) such that a central complex lineage is transformed into a functional olfactory projection neuron lineage. This <em>otd</em>-dependent, lineage-specific respecification of interneurons has implications for our understanding of the development and evolution of the circuitry in the brain.</p><div id="F14" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xmlns:xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22869"><div xmlns="http://www.w3.org/1999/xhtml" xmlns:xhtml="http://www.w3.org/1999/xhtml" id="content-block-markup"><div class="fig-expansion " id="F14"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/F14.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22869" title="Figure 12." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Figure 12.&amp;lt;/span&amp;gt; &amp;lt;span class=&amp;quot;caption-title&amp;quot;&amp;gt;Transformation of the LALv1 lineage.&amp;lt;/span&amp;gt;&amp;lt;p id=&amp;quot;p-81&amp;quot;&amp;gt;(&amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) The WT LALv1 is a wide-field central complex lineage that expresses Otd (orange). (&amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;) The loss of &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt; function from the LALv1 neuroblast transforms these neurons into antennal lobe projection neurons. This neuroanatomical transformation is accompanied by molecular changes&#x2014;markers that are active in the WT LALv1 (green box in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;) are suppressed in the mutant (&amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;; with the exception of Acj6); those that are suppressed in the WT (red box in &amp;lt;strong&amp;gt;A&amp;lt;/strong&amp;gt;), are activated in the mutant (&amp;lt;strong&amp;gt;B&amp;lt;/strong&amp;gt;; with the exception of LN1-Gal4). The exceptions are in agreement with a transformation to antennal lobe projection neuron fate (see &amp;lt;span id=&amp;quot;xref-table-wrap-1-5&amp;quot; class=&amp;quot;xref-table&amp;quot;&amp;gt;Table 1&amp;lt;/span&amp;gt;).&amp;lt;/p&amp;gt;&amp;lt;p id=&amp;quot;p-82&amp;quot;&amp;gt;&amp;lt;strong&amp;gt;DOI:&amp;lt;/strong&amp;gt; &amp;lt;a href=&amp;quot;/lookup/doi/10.7554/eLife.04407.020&amp;quot;&amp;gt;http://dx.doi.org/10.7554/eLife.04407.020&amp;lt;/a&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22869"><img data-img="/content/elife/3/e04407/F14.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/F14.medium.gif" alt="Figure 12."></img></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/F14.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/F14.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22869">Download powerpoint</a></span></span><span class="fig-label">Figure 12.</span> <span class="caption-title">Transformation of the LALv1 lineage.</span><p id="p-81">(<strong>A</strong>) The WT LALv1 is a wide-field central complex lineage that expresses Otd (orange). (<strong>B</strong>) The loss of <em>otd</em> function from the LALv1 neuroblast transforms these neurons into antennal lobe projection neurons. This neuroanatomical transformation is accompanied by molecular changes—markers that are active in the WT LALv1 (green box in <strong>A</strong>) are suppressed in the mutant (<strong>B</strong>; with the exception of Acj6); those that are suppressed in the WT (red box in <strong>A</strong>), are activated in the mutant (<strong>B</strong>; with the exception of LN1-Gal4). The exceptions are in agreement with a transformation to antennal lobe projection neuron fate (see <span id="xref-table-wrap-1-5" class="xref-table">Table 1</span>).</p><p id="p-82"><strong>DOI:</strong> <a href="/lookup/doi/10.7554/eLife.04407.020">http://dx.doi.org/10.7554/eLife.04407.020</a></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><p id="p-83">The ability of a neuroblast lineage to transform completely into another upon the loss of a single intrinsic determinant suggests that many of the other putative members of a potential neural identity code might be shared between these lineages. The observation that the neuroblasts of the central complex and antennal lobe lineages develop in such close spatial proximity to each other during early development suggests that these two neuroblasts may experience similar spatial cues as they develop on the neuroectoderm. If this is the case, then by manipulating a single differentially expressed factor, <em>otd</em>, we might have been able to uncover the underlying similarity in the intrinsic spatial code between the two neuroblast lineages. Importantly, this neuroblast-specific transformation of lineage identity resulted in an alteration of the brain's circuitry such that an entire neuroanatomical unit of projection to the central complex was lacking while a novel and functional ectopic unit of projection was added to the antennal lobe.</p><p id="p-84">Implicit in these findings are the notions of ‘coded’ and ‘soft’ properties of circuit assembly. On the one hand, the neuroanatomical and molecular transformation described above demonstrates that circuitry in the brain is ‘hard-wired’ or ‘coded’ by the spatially encoded intrinsic factors—the presence or absence of <em>otd</em> from the central complex neuroblast determines the identity of the resultant neurons. On the other hand, the resulting functional transformation suggests that circuit assembly involves substantial ‘soft-wiring’—the olfactory sensory neurons and interneurons indigenous to the antennal lobe are able to make functional connections with the extraneous transformed <em>otd</em><sup><em>−/−</em></sup> LALv1 neurons, which they are normally not ‘hard-wired’ to connect with. Thus, while genetically encoded properties might ‘lock’ lineages into particular circuit states (central complex or antennal lobe) it is their ‘soft’ properties (developmental plasticity) that allow circuits to functionally incorporate changes as dramatic as extraneous neurons. As both these wiring strategies operate simultaneously in the brain, it bestows upon the brain the huge potential of evolvability of functional circuits.</p><p id="p-85">Many interesting questions emerge as a result of our findings. How does the developmental plasticity of a functional circuit support these large-scale rearrangements? Do developing circuits acquire a propensity for exuberant connectivities, or do they try and maintain a homeostasis in their connections and therefore make compensatory changes in the number of synapses with their normal partners? It has been shown in some cases that neuronal activity can mediate such ‘soft’ properties of synaptic connections (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Tripodi et al., 2008</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Singh et al., 2010</a>). It will be interesting to test if this is also the case for the transformed neurons and the olfactory circuit. Finally, do all parts of the brain display such striking developmental plasticity such that they can be remodelled to this extent and incorporate extraneous neurons into existing circuitry?</p><p id="p-86">The ability to change the functional macrocircuitry of the brain through changes in the expression of a single transcription factor in a single neuroblast lineage may provide a simple paradigm for large-scale modification of brain connectivity during evolution. The <em>otd</em><sup><em>−/−</em></sup>transformed LALv1 lineage functionally integrates into the antennal lobe circuitry and participates in olfactory information processing. This suggests that a functional rewiring of the olfactory circuitry can occur due to the addition of an entire neuroblast lineage to the normal olfactory circuit. In more general terms, this type of lineage-specific rewiring might fuel the evolutionary modification of neural circuitry in the brain. It provides an elegant and simple solution to the evolution of complex circuitry in that a ‘microevolutionary’ molecular change (changing the expression of one gene in one cell) can have ‘macroevolutionary’ consequences on the brain's circuitry (changing an entire macrocircuit or an information processing module). This simple strategy suggests that large-scale changes in the brain's wiring do not need to come about through many minor, sequentially accumulating changes at the cellular level. Instead, large-scale wiring changes can occur in response to remarkably simple changes in gene expression in single cells. In this context, our data may have provided evidence for the mechanistic ease with which circuitry in the brain can evolve.</p></div><div class="section materials-methods" id="sec-13"><h2 class="">Materials and methods</h2><div id="sec-14" class="subsection"><h3>Lineage nomenclature</h3><p id="p-87">The brain lineages have been named by various groups in the past. Here, we list the various names by which the two lineages, which are the focus of our study, have been called. ALad1 (<a id="xref-ref-13-7" class="xref-bibr" href="#ref-13">Ito et al., 2013</a>; <a id="xref-ref-49-7" class="xref-bibr" href="#ref-49">Yu et al., 2013</a>)/BAmv3 (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Pereanu and Hartenstein, 2006</a>; <a id="xref-ref-24-10" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>; <a id="xref-ref-48-8" class="xref-bibr" href="#ref-48">Wong et al., 2013</a>)/adNB (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Jefferis et al., 2001</a>). LALv1 (<a id="xref-ref-13-8" class="xref-bibr" href="#ref-13">Ito et al., 2013</a>; <a id="xref-ref-49-8" class="xref-bibr" href="#ref-49">Yu et al., 2013</a>)/BAmv1 (<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Pereanu and Hartenstein, 2006</a>; <a id="xref-ref-24-11" class="xref-bibr" href="#ref-24">Lovick et al., 2013</a>; <a id="xref-ref-48-9" class="xref-bibr" href="#ref-48">Wong et al., 2013</a>).</p></div><div id="sec-15" class="subsection"><h3>Fly strains and genetics</h3><p id="p-88">Fly stocks were obtained from the Bloomington Stock Centre (IN, USA) and, unless otherwise stated, were grown on cornmeal media, at 25°C. UAS-miRNA otd-1 was kindly provided by Henry Sun, Taiwan.</p><p id="p-89">To generate WT MARCM clones females of the genotype <em>FRT19A</em> or <em>FRT19A;UAS-mCD8::GFP</em> were crossed to males of the following genotypes: <em>FR19A,Tub-Gal80,hsFLP; Tub-Gal4,UAS-mCD8::GFP/CyO</em> or <em>FR19A,Tub-Gal80,hsFLP; OK371-Gal4,UAS-mCD8::GFP/CyO</em> or <em>FR19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/CyO</em> or <em>FR19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/CyO</em> or <em>FR19A,Tub-Gal80,hsFLP; LN1-Gal4,UAS-mCD8::GFP/CyO</em> or <em>FR19A,Tub-Gal80,hsFLP; Cha7.4-Gal4,UAS-GFP/CyO.</em> To generate <em>otd</em><sup><em>−/−</em></sup> MARCM clones, females of the genotype <em>FRT19A,otd</em><sup><em>YH13</em></sup><em>/FM7c</em> or <em>FRT19A,oc</em><sup><em>2</em></sup><em>/FM7c</em> or <em>FRT19A, otd</em><sup><em>YH13</em></sup><em>;UAS-mCD8::GFP</em> or <em>FRT19A,oc</em><sup><em>2</em></sup><em>;UAS-mCD8::GFP</em> were crossed to males of the above-mentioned genotypes.</p><p id="p-90">To knockdown <em>otd</em> in neuroblasts using RNAi and simultaneously visualise GH146-labelled cells, females of the genotype <em>UAS-dicer; Insc-Gal4/CyO</em> were crossed to males of the genotype <em>UAS-miRNA otd-1/CyO; GH146-QF,QUAS-mtdTomato-HA/TM6B</em> and the crosses were grown at 25°C.</p><p id="p-91">To generate Dual MARCM clones, females of the genotype <em>FRT19A,otd</em><sup><em>YH13</em></sup><em>/FM7a; FRTG13, GH146-LexA::GAD,LexAop-rCD2::GFP/CyO</em> were crossed to males of the following genotypes: <em>FRT19A,Tub-Gal80,hsFLP; Tub-Gal4,UAS-mCD8::GFP/CyO or FRT19A,Tub-Gal80,hsFLP; OK371-Gal4,UAS-mCD8::GFP/CyO</em> or <em>FRT19A,Tub-Gal80,hsFLP; Per-Gal4,UAS-mCD8::GFP/CyO.</em></p><p id="p-92">To generate Otd overexpression clones, females of the genotype <em>FRT19A</em>;<em>UAS-otd</em> were crossed to males of the genotype <em>FR19A,Tub-Gal80,hsFLP; Tub-Gal4,UAS-mCD8::GFP/CyO.</em> To generate rescue clones, females of the genotype <em>FRT19A otd</em><sup><em>YH13</em></sup><em>/FM7a</em>;<em>UAS-otd</em> were crossed to males of the genotype <em>FR19A,Tub-Gal80,hsFLP; Tub-Gal4,UAS-mCD8::GFP/CyO.</em></p><p id="p-93">To generate clones for functional imaging, females of the genotype <em>FRT19A,otd</em><sup><em>YH13</em></sup><em>/FM7a;GH146-Gal4,UAS-GCaMP3/CyO</em> were crossed to males of the genotype <em>FR19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-GCaMP3/CyO.</em></p><p id="p-94">Embryos were collected from the progeny of all the clonal crosses and these were given a heat shock for 1 hr at 37°C at either the embryonic stage, 0–4 hr after larval hatching or 24 hr after larval hatching.</p><p id="p-95">To knock down Otd in neuroblasts using RNA interference, females of the genotype <em>UASdicer; Insc-Gal4</em> were crossed to males of the genotype <em>UAS-miRNA-otd-1;GH146-QF,QUAS-mtdTomato-HA/TM6B</em> and grown at 25°C.</p><div id="sec-16" class="subsection"><h4>Generation of UAS-Otd</h4><p id="p-96">A full-length <em>otd</em>-cDNA for expression under the control of the UAS promoter was made in the KVR lab by cloning a full-length <em>otd</em> cDNA including the 5′ and 3′ UTRs into the pJRFC-MUH vector. The full-length o<em>td</em> cDNA clone RE-10280 (pBluscript backbone) was purchased from DGRC. Confirmed plasmid DNA (pJFRC-10xUAS-Otd FL with an attB integration site) was microinjected (2 μg/μl in 1× microinjection buffer) into <em>Drosophila</em> embryos that contain PhiC31 integrase and selected attP docking site on the second chromosome. Further crossing of G0 flies, screening of the transformants and balancing of insertions performed at the Transgenic Fly Facility at C-CAMP facility at NCBS campus, Bangalore, India.</p></div></div><div id="sec-17" class="subsection"><h3>Immunohistochemistry and imaging</h3><p id="p-97">Brains were dissected in 1× PBS and fixed in freshly prepared 4% PFA for 30 min at room temperature. The fixative was removed, and the brains were washed with blocking solution (1× PBS with 0.3% TritonX and 0.1% BSA). Primary antibodies were diluted in blocking solution. The samples were incubated in a moist chamber on horizontal shaker at 4°C for 24 hr. Samples were then washed with 0.3% PTX (1× PBS with 0.3% TritonX) and secondary antibody diluted in 0.3% PTX was added. The samples were incubated in this at 4°C in a moist chamber on horizontal shaker overnight, after which they were washed and mounted in vectashield on glass slides.</p><p id="p-98">Primary antibodies used were: rabbit anti-GFP (1:10,000; Molecular Probes, Invitrogen, Delhi, India); chick anti-GFP (1:10,000, AbCam, Cambridge, UK); mouse anti-Bruchpilot (mAbnc82, 1:20; DSHB, Iowa, USA); Rb anti-Otd (1:1,500, gift from Henry Sun, IMB, Sinica, Taiwan); guinea pig anti-Otd (1:7,500, gift from Tiffany Cook, Cincinnati, OH); mouse anti-Acj6 (1:25, DSHB, Iowa, USA); rabbit anti-dLim1 (1:1000; a gift from J Botas); rat anti-mCD8 (1:100, Caltag, Burlingname, Californina); mouse anti-rCD2 (AbD Serotec, Raleigh, NC, USA); mouse anti-Neurotactin (BP106, 1:10, DSHB, Iowa, USA); mouse anti-Neuroglian (BP102, 1:10, DSHB, Iowa, USA); rabbit anti-HA (1:100, AbCam). Secondary antibodies used were Alexa Fluor-488-, Alexa Fluor-568- and Alexa Fluor-647-coupled antibodies generated in goat (1:400; Molecular Probes, Invitrogen, Delhi, India) and TO-PRO -3 iodide-661 (Thermo Fischer Scientific, MA,USA).</p><p id="p-99">All samples were imaged on Olympus Fluoview (FV1000) laser scanning confocal microscope. Optical sections were acquired at 1-µm intervals with a picture size of 512 × 512 pixels. Images were digitally processed using Adobe Photoshop CS3. 3-D reconstructions were made using Amira.</p></div><div id="sec-18" class="subsection"><h3>Functional imaging</h3><p id="p-100">Brains of clonal animals were dissected in Ca<sup>2+</sup>-free AHL saline, which contains 108 mM NaCl, 5 mM KCl, 4 mM NaHCO<sub>3</sub>, 1 mM NaH<sub>2</sub>PO<sub>4</sub>, 8.2 mM MgCl<sub>2</sub>, 5 mM HEPES, 5 mM trehalose, and 10 mM sucrose, with pH adjusted to 7.4. Live brains that contain only the <em>otd</em> null transformed neurons were selected for imaging experiments.</p><div id="sec-19" class="subsection"><h4>Two-photon calcium imaging and antennal nerve stimulation (electrical)</h4><p id="p-101">Two-photon calcium imaging was performed as described previously (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Root et al., 2008</a>). The antennal nerves were cut from the base of the antennae. The brain preparation was then pinned down on a Sylgard-coated petri dish with AHL saline containing 2 mM CaCl<sub>2</sub>. The antennal nerve was stimulated electrically with a glass suction electrode, at 1 ms in duration, 10 V in amplitude and 100 Hz in frequency (S48 Grass stimulator). The response of the transformed neurons was monitored by a custom-built two-photon microscope. Excitation wavelength was 930 nm. Images were captured at 4 frames/s.</p></div><div id="sec-20" class="subsection"><h4>Two-photon calcium imaging and odour stimulation</h4><p id="p-102">For odour stimulation, the fly brain was dissected leaving the antennae intact and embedded in agarose containing AHL saline with 2 mM CaCl<sub>2</sub>. The agarose gel was removed from the antennae and a piece of Kimwipes was used to dry the antennae. A glass coverslip was placed on top of the brain preparation for imaging. Odour delivery was controlled by solenoid valves described previously (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Root et al., 2008</a>). Odour vapour was obtained by placing a filter paper containing 10 μl of an odorant in a 100-ml bottle. Mixing an air stream with an odour stream at different flow ratios was used to deliver odorants at a specific concentration. Isoamyl acetate and ethyl butyrate were delivered at 1% (odorant at 10 ml/min and air at 990 ml/min), whereas 3-heptanol and 3-octanol were delivered at 2.5% saturated vapor pressure. Each odorant was applied for a duration of 2 s. Images were acquired for 20 s at a rate of 4 frames/s and a resolution of 128 × 128 pixels. At the end of each experiment, an image stack was collected at a resolution of 512 × 512 pixels for glomerulus identification. The data were analyzed and plotted using Igor Pro 6.2 (Wavemetrics). The peak response of stimulation (ΔF/F) was shown as mean ± S.E.M.</p></div></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="http://elifesciences.org/content/3/e04407.abstract" class="hw-link hw-link-article-abstract">View Abstract</a></div></div></div><div class="panel-separator"></div><div id="references" class="ctools-collapsible-container"><h2 class="pane-title ctools-collapsible-handle">References</h2><div class="ctools-collapsible-content"><div class="elife-reflinks-sortby"><div class="elife-reflinks-sortby-label">Sort by:</div><div class="item-list"><ul class="tabs inline elife-reflinks-sort-tabs"><li class="first"><span class="elife-reflinks-sort-tab selected"><a href="#elife-reflinks" data-ref="original" data-type="int" data-sort="asc" class="active">original order</a></span></li>
<li><span class="elife-reflinks-sort-tab not-selected"><a href="#elife-reflinks" data-ref="author" data-type="string" data-sort="asc">author</a></span></li>
<li><span class="elife-reflinks-sort-tab not-selected"><a href="#elife-reflinks" data-ref="title" data-type="string" data-sort="asc">title</a></span></li>
<li><span class="elife-reflinks-sort-tab not-selected"><a href="#elife-reflinks" data-ref="date" data-type="date" data-sort="asc">year</a></span></li>
<li class="last"><span class="elife-reflinks-sort-tab not-selected"><a href="#elife-reflinks" data-ref="cited" data-type="int" data-sort="asc">cited in paper</a></span></li>
</ul></div></div><div class="elife-reflinks-links"><article class="elife-reflinks-reflink" id="ref-1" data-original="1" data-author="cardona, albert" data-title="identifying neuronal lineages of drosophilby sequence analysis of axon tracts" data-date="2010-06-02" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="1">1</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=jneuro&amp;resid=30/22/7538" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-2">Identifying neuronal lineages of <span class="nlm-italic" data-toggle="yes">Drosophila</span> by sequence analysis of axon tracts</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Albert Cardona&quot;">Albert Cardona</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Stephan Saalfeld&quot;">Stephan Saalfeld</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Ignacio Arganda&quot;">Ignacio Arganda</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Wayne Pereanu&quot;">Wayne Pereanu</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Johannes Schindelin&quot;">Johannes Schindelin</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Volker Hartenstein&quot;">Volker Hartenstein</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-1">The Journal of Neuroscience</span></span>, <span class="elife-reflink-details-volume">30</span>, <span class="elife-reflink-details-pages">7538-7553</span>, <span class="elife-reflink-details-year">2010</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1523/JNEUROSCI.0186-10.2010">http://dx.doi.org/10.1523/JNEUROSCI.0186-10.2010</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=jneuro&amp;resid=30/22/7538" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1523/JNEUROSCI.0186-10.2010&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=20519528&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-2" data-original="2" data-author="chou, yh" data-title="diversity and wiring variability of olfactory local interneurons in drosophilantennal lobe" data-date="2010" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="2">2</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/nn.2489&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-3">Diversity and wiring variability of olfactory local interneurons in the <span class="nlm-italic" data-toggle="yes">Drosophila</span> antennal lobe</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:YH Chou&quot;">YH Chou</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:ML Spletter&quot;">ML Spletter</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:E Yaksi&quot;">E Yaksi</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JC Leong&quot;">JC Leong</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:RI Wilson&quot;">RI Wilson</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:L Luo&quot;">L Luo</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-2">Nature Neuroscience</span></span>, <span class="elife-reflink-details-volume">13</span>, <span class="elife-reflink-details-pages">439-449</span>, <span class="elife-reflink-details-year">2010</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1038/nn.2489">http://dx.doi.org/10.1038/nn.2489</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1038/nn.2489&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=20139975&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-3" data-original="3" data-author="custo greig, l" data-title="molecular logic of neocortical projection neuron specification, development and diversity" data-date="2013" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="3">3</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/nrn3586&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-4">Molecular logic of neocortical projection neuron specification, development and diversity</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:L Custo Greig&quot;">L Custo Greig</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Woodworth&quot;">M Woodworth</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Galazo&quot;">M Galazo</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Padmanabhan&quot;">H Padmanabhan</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:J Macklis&quot;">J Macklis</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-3">Nature Reviews Neuroscience</span></span>, <span class="elife-reflink-details-volume">14</span>, <span class="elife-reflink-details-pages">755-769</span>, <span class="elife-reflink-details-year">2013</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1038/nrn3586">http://dx.doi.org/10.1038/nrn3586</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1038/nrn3586&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=24105342&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-4" data-original="4" data-author="dacks, am" data-title="serotonin modulates olfactory processing in antennal lobe of drosophila" data-date="2009" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="4">4</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.3109/01677060903085722&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-5">Serotonin modulates olfactory processing in the antennal lobe of <span class="nlm-italic" data-toggle="yes">Drosophila</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:AM Dacks&quot;">AM Dacks</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:DS Green&quot;">DS Green</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:CM Root&quot;">CM Root</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:AJ Nighorn&quot;">AJ Nighorn</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JW Wang&quot;">JW Wang</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-4">Journal of Neurogenetics</span></span>, <span class="elife-reflink-details-volume">23</span>, <span class="elife-reflink-details-pages">366-377</span>, <span class="elife-reflink-details-year">2009</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.3109/01677060903085722">http://dx.doi.org/10.3109/01677060903085722</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.3109/01677060903085722&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=19863268&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-5" data-original="5" data-author="das, a" data-title="identification and analysis of glutamatergic local interneuron lineage in adult drosophilolfactory system" data-date="2011" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="5">5</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1186/2042-1001-1-4&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-6">Identification and analysis of a glutamatergic local interneuron lineage in the adult <span class="nlm-italic" data-toggle="yes">Drosophila</span> olfactory system</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:A Das&quot;">A Das</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:A Chiang&quot;">A Chiang</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:S Davla&quot;">S Davla</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:R Priya&quot;">R Priya</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Reichert&quot;">H Reichert</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K Vijayraghavan&quot;">K Vijayraghavan</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:V Rodrigues&quot;">V Rodrigues</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-5">Neural Systems &amp; Circuits</span></span>, <span class="elife-reflink-details-volume">1</span>, <span class="elife-reflink-details-pages">4</span>, <span class="elife-reflink-details-year">2011</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1186/2042-1001-1-4">http://dx.doi.org/10.1186/2042-1001-1-4</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1186/2042-1001-1-4&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=22330097&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-6" data-original="6" data-author="das, a" data-title="neuroblast lineage-specific origin of neurons of drosophillarval olfactory system" data-date="2013" data-cited="3"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="6">6</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/j.ydbio.2012.11.003&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-7">Neuroblast lineage-specific origin of the neurons of the <span class="nlm-italic" data-toggle="yes">Drosophila</span> larval olfactory system</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:A Das&quot;">A Das</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:T Gupta&quot;">T Gupta</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:S Davla&quot;">S Davla</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:LL Prieto-Godino&quot;">LL Prieto-Godino</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:S Diegelmann&quot;">S Diegelmann</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:OV Reddy&quot;">OV Reddy</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:KV Raghavan&quot;">KV Raghavan</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Reichert&quot;">H Reichert</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:J Lovick&quot;">J Lovick</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:V Hartenstein&quot;">V Hartenstein</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-6">Developmental Biology</span></span>, <span class="elife-reflink-details-volume">373</span>, <span class="elife-reflink-details-pages">322-337</span>, <span class="elife-reflink-details-year">2013</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/j.ydbio.2012.11.003">http://dx.doi.org/10.1016/j.ydbio.2012.11.003</a></span>  — <span class="elife-reflink-details-cited">cited 3 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/j.ydbio.2012.11.003&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=23149077&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-7" data-original="7" data-author="das, a" data-title="drosophilolfactory local interneurons and projection neurons derive from common neuroblast lineage specified by empty spiracles gene" data-date="2008" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="7">7</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1186/1749-8104-3-33&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-8">
               <span class="nlm-italic" data-toggle="yes">Drosophila</span> olfactory local interneurons and projection neurons derive from a common neuroblast lineage specified by the empty spiracles gene</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:A Das&quot;">A Das</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:S Sen&quot;">S Sen</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:R Lichtneckert&quot;">R Lichtneckert</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:R Okada&quot;">R Okada</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K Ito&quot;">K Ito</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:V Rodrigues&quot;">V Rodrigues</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Reichert&quot;">H Reichert</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-7">Neural Development</span></span>, <span class="elife-reflink-details-volume">3</span>, <span class="elife-reflink-details-pages">33</span>, <span class="elife-reflink-details-year">2008</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1186/1749-8104-3-33">http://dx.doi.org/10.1186/1749-8104-3-33</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1186/1749-8104-3-33&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=19055770&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-8" data-original="8" data-author="deshpande, nirupama" data-title="successive specification of drosophilneuroblasts nb 6-4 and nb 7-3 depends on interaction of segment polarity genes wingless, gooseberry and naked cuticle" data-date="2001-09-01" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="8">8</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=128/17/3253" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-9">Successive specification of <span class="nlm-italic" data-toggle="yes">Drosophila</span> neuroblasts NB 6-4 and NB 7-3 depends on interaction of the segment polarity genes wingless, gooseberry and naked cuticle</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Nirupama Deshpande&quot;">Nirupama Deshpande</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Rainer Dittrich&quot;">Rainer Dittrich</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Gerhard M. Technau&quot;">Gerhard M. Technau</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Joachim Urban&quot;">Joachim Urban</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-8">Development</span></span>, <span class="elife-reflink-details-volume">128</span>, <span class="elife-reflink-details-pages">3253-3261</span>, <span class="elife-reflink-details-year">2001</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=128/17/3253" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=11546742&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-9" data-original="9" data-author="franco, sj" data-title="shaping our minds: stem and progenitor cell diversity in mammalineocortex" data-date="2013" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="9">9</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/j.neuron.2012.12.022&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-10">Shaping our Minds: stem and progenitor cell diversity in the Mammalian neocortex</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:SJ Franco&quot;">SJ Franco</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:U Muller&quot;">U Muller</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-9">Neuron</span></span>, <span class="elife-reflink-details-volume">77</span>, <span class="elife-reflink-details-pages">19-34</span>, <span class="elife-reflink-details-year">2013</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/j.neuron.2012.12.022">http://dx.doi.org/10.1016/j.neuron.2012.12.022</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/j.neuron.2012.12.022&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=23312513&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-10" data-original="10" data-author="gao, peng" data-title="lineage-dependent circuit assembly in neocortex" data-date="2013-07-01" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="10">10</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=140/13/2645" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-11">Lineage-dependent circuit assembly in the neocortex</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Peng Gao&quot;">Peng Gao</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Khadeejah T. Sultan&quot;">Khadeejah T. Sultan</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Xin-Jun Zhang&quot;">Xin-Jun Zhang</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Song-Hai Shi&quot;">Song-Hai Shi</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-10">Development</span></span>, <span class="elife-reflink-details-volume">140</span>, <span class="elife-reflink-details-pages">2645-2655</span>, <span class="elife-reflink-details-year">2013</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1242/dev.087668">http://dx.doi.org/10.1242/dev.087668</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=140/13/2645" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1242/dev.087668&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=23757410&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-11" data-original="11" data-author="ito, k." data-title="drosophilmushroom body is quadruple structure of clonal units each of which contains virtually identical set of neurones and glial cells" data-date="1997-02-15" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="11">11</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=124/4/761" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-12">The <span class="nlm-italic" data-toggle="yes">Drosophila</span> mushroom body is a quadruple structure of clonal units each of which contains a virtually identical set of neurones and glial cells</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K. Ito&quot;">K. Ito</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:W. Awano&quot;">W. Awano</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K. Suzuki&quot;">K. Suzuki</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Y. Hiromi&quot;">Y. Hiromi</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:D. Yamamoto&quot;">D. Yamamoto</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-11">Development</span></span>, <span class="elife-reflink-details-volume">124</span>, <span class="elife-reflink-details-pages">761-771</span>, <span class="elife-reflink-details-year">1997</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=124/4/761" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=9043058&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-12" data-original="12" data-author="ito, k" data-title="proliferation pattern of postembryonic neuroblasts in brain of drosophilmelanogaster" data-date="1992" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="12">12</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/0012-1606(92)90270-Q&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-13">Proliferation pattern of postembryonic neuroblasts in the brain of <span class="nlm-italic" data-toggle="yes">Drosophila melanogaster</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K Ito&quot;">K Ito</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Y Hotta&quot;">Y Hotta</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-12">Developmental Biology</span></span>, <span class="elife-reflink-details-volume">149</span>, <span class="elife-reflink-details-pages">134-148</span>, <span class="elife-reflink-details-year">1992</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/0012-1606(92)90270-Q">http://dx.doi.org/10.1016/0012-1606(92)90270-Q</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/0012-1606(92)90270-Q&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=1728583&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-13" data-original="13" data-author="ito, m" data-title="systematic analysis of neural projections reveals clonal composition of drosophilbrain" data-date="2013" data-cited="8"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="13">13</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/j.cub.2013.03.015&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-14">Systematic analysis of neural projections reveals clonal composition of the <span class="nlm-italic" data-toggle="yes">Drosophila</span> brain</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Ito&quot;">M Ito</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:N Masuda&quot;">N Masuda</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K Shinomiya&quot;">K Shinomiya</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K Endo&quot;">K Endo</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K Ito&quot;">K Ito</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-13">Current Biology</span></span>, <span class="elife-reflink-details-volume">23</span>, <span class="elife-reflink-details-pages">644-655</span>, <span class="elife-reflink-details-year">2013</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/j.cub.2013.03.015">http://dx.doi.org/10.1016/j.cub.2013.03.015</a></span>  — <span class="elife-reflink-details-cited">cited 8 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/j.cub.2013.03.015&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=23541729&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-14" data-original="14" data-author="jefferis, gs" data-title="target neuron prespecification in olfactory map of drosophila" data-date="2001" data-cited="3"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="14">14</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/35102574&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-15">Target neuron prespecification in the olfactory map of <span class="nlm-italic" data-toggle="yes">Drosophila</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:GS Jefferis&quot;">GS Jefferis</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:EC Marin&quot;">EC Marin</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:RF Stocker&quot;">RF Stocker</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:L Luo&quot;">L Luo</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-14">Nature</span></span>, <span class="elife-reflink-details-volume">414</span>, <span class="elife-reflink-details-pages">204-208</span>, <span class="elife-reflink-details-year">2001</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1038/35102574">http://dx.doi.org/10.1038/35102574</a></span>  — <span class="elife-reflink-details-cited">cited 3 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1038/35102574&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=11719930&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-15" data-original="15" data-author="jefferis, gregory s. x. e." data-title="developmental origin of wiring specificity in olfactory system of drosophila" data-date="2004-01-01" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="15">15</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=131/1/117" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-16">Developmental origin of wiring specificity in the olfactory system of <span class="nlm-italic" data-toggle="yes">Drosophila</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Gregory S. X. E. Jefferis&quot;">Gregory S. X. E. Jefferis</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Raj M. Vyas&quot;">Raj M. Vyas</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Daniela Berdnik&quot;">Daniela Berdnik</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Ariane Ramaekers&quot;">Ariane Ramaekers</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Reinhard F. Stocker&quot;">Reinhard F. Stocker</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Nobuaki K. Tanaka&quot;">Nobuaki K. Tanaka</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Kei Ito&quot;">Kei Ito</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Liqun Luo &quot;">Liqun Luo </a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-15">Development</span></span>, <span class="elife-reflink-details-volume">131</span>, <span class="elife-reflink-details-pages">117-130</span>, <span class="elife-reflink-details-year">2004</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1242/dev.00896">http://dx.doi.org/10.1242/dev.00896</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=131/1/117" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1242/dev.00896&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=14645123&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-16" data-original="16" data-author="kohwi, m" data-title="temporal fate specification and neural progenitor competence during development" data-date="2013" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="16">16</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/nrn3618&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-17">Temporal fate specification and neural progenitor competence during development</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Kohwi&quot;">M Kohwi</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:CQ Doe&quot;">CQ Doe</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-16">Nature Reviews Neuroscience</span></span>, <span class="elife-reflink-details-volume">14</span>, <span class="elife-reflink-details-pages">823-838</span>, <span class="elife-reflink-details-year">2013</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1038/nrn3618">http://dx.doi.org/10.1038/nrn3618</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1038/nrn3618&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=24400340&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-17" data-original="17" data-author="lai, sen-lin" data-title="clonal analysis of drosophilantennal lobe neurons: diverse neuronal architectures in lateral neuroblast lineage" data-date="2008-09-01" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="17">17</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=135/17/2883" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-18">Clonal analysis of <span class="nlm-italic" data-toggle="yes">Drosophila</span> antennal lobe neurons: diverse neuronal architectures in the lateral neuroblast lineage</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Sen-Lin Lai&quot;">Sen-Lin Lai</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Takeshi Awasaki&quot;">Takeshi Awasaki</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Kei Ito&quot;">Kei Ito</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Tzumin Lee&quot;">Tzumin Lee</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-17">Development</span></span>, <span class="elife-reflink-details-volume">135</span>, <span class="elife-reflink-details-pages">2883-2893</span>, <span class="elife-reflink-details-year">2008</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1242/dev.024380">http://dx.doi.org/10.1242/dev.024380</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=135/17/2883" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1242/dev.024380&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=18653555&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-18" data-original="18" data-author="lai, sl" data-title="genetic mosaic with dual binary transcriptional systems in drosophila" data-date="2006" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="18">18</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/nn1681&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-19">Genetic mosaic with dual binary transcriptional systems in <span class="nlm-italic" data-toggle="yes">Drosophila</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:SL Lai&quot;">SL Lai</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:T Lee&quot;">T Lee</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-18">Nature Neuroscience</span></span>, <span class="elife-reflink-details-volume">9</span>, <span class="elife-reflink-details-pages">703-709</span>, <span class="elife-reflink-details-year">2006</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1038/nn1681">http://dx.doi.org/10.1038/nn1681</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1038/nn1681&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=16582903&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-19" data-original="19" data-author="lee, t" data-title="mosaic analysis with repressible cell marker (marcm) for drosophilneural development" data-date="2001" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="19">19</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/S0166-2236(00)01791-4&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-20">Mosaic analysis with a repressible cell marker (MARCM) for <span class="nlm-italic" data-toggle="yes">Drosophila</span> neural development</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:T Lee&quot;">T Lee</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:L Luo&quot;">L Luo</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-19">Trends in Neurosciences</span></span>, <span class="elife-reflink-details-volume">24</span>, <span class="elife-reflink-details-pages">251-254</span>, <span class="elife-reflink-details-year">2001</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/S0166-2236(00)01791-4">http://dx.doi.org/10.1016/S0166-2236(00)01791-4</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/S0166-2236(00)01791-4&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=11311363&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-20" data-original="20" data-author="li, y" data-title="clonally related visual cortical neurons show similar stimulus feature selectivity" data-date="2012" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="20">20</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/nature11110&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-21">Clonally related visual cortical neurons show similar stimulus feature selectivity</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Y Li&quot;">Y Li</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Lu&quot;">H Lu</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:P Cheng&quot;">P Cheng</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:S Ge&quot;">S Ge</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Xu&quot;">H Xu</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:SH Shi&quot;">SH Shi</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Y Dan&quot;">Y Dan</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-20">Nature</span></span>, <span class="elife-reflink-details-volume">486</span>, <span class="elife-reflink-details-pages">118-121</span>, <span class="elife-reflink-details-year">2012</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1038/nature11110">http://dx.doi.org/10.1038/nature11110</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1038/nature11110&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=22678292&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-21" data-original="21" data-author="lichtneckert, r" data-title="anteroposterior regionalization of brain: genetic and comparative aspects" data-date="2008" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="21">21</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1007/978-0-387-78261-4_2&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-22">Anteroposterior regionalization of the brain: genetic and comparative aspects</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:R Lichtneckert&quot;">R Lichtneckert</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Reichert&quot;">H Reichert</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-21">Advances in Experimental Medicine and Biology</span></span>, <span class="elife-reflink-details-volume">628</span>, <span class="elife-reflink-details-pages">32-41</span>, <span class="elife-reflink-details-year">2008</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1007/978-0-387-78261-4_2">http://dx.doi.org/10.1007/978-0-387-78261-4_2</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1007/978-0-387-78261-4_2&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=18683636&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-22" data-original="22" data-author="lichtneckert, r" data-title="insights into urbilateribrain: conserved genetic patterning mechanisms in insect and vertebrate brain development" data-date="2005" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="22">22</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/sj.hdy.6800664&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-23">Insights into the urbilaterian brain: conserved genetic patterning mechanisms in insect and vertebrate brain development</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:R Lichtneckert&quot;">R Lichtneckert</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Reichert&quot;">H Reichert</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-22">Heredity</span></span>, <span class="elife-reflink-details-volume">94</span>, <span class="elife-reflink-details-pages">465-477</span>, <span class="elife-reflink-details-year">2005</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1038/sj.hdy.6800664">http://dx.doi.org/10.1038/sj.hdy.6800664</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1038/sj.hdy.6800664&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=15770230&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-23" data-original="23" data-author="lin, s" data-title="generating neuronal diversity in drosophilcentral nervous system" data-date="2012" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="23">23</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1002/dvdy.22739&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-24">Generating neuronal diversity in the <span class="nlm-italic" data-toggle="yes">Drosophila</span> central nervous system</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:S Lin&quot;">S Lin</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:T Lee&quot;">T Lee</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-23">Developmental Dynamics</span></span>, <span class="elife-reflink-details-volume">241</span>, <span class="elife-reflink-details-pages">57-68</span>, <span class="elife-reflink-details-year">2012</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1002/dvdy.22739">http://dx.doi.org/10.1002/dvdy.22739</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1002/dvdy.22739&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=21932323&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-24" data-original="24" data-author="lovick, jk" data-title="postembryonic lineages of drosophilbrain: i. development of lineage-associated fiber tracts" data-date="2013" data-cited="11"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="24">24</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/j.ydbio.2013.07.008&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-25">Postembryonic lineages of the <span class="nlm-italic" data-toggle="yes">Drosophila</span> brain: I. Development of the lineage-associated fiber tracts</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JK Lovick&quot;">JK Lovick</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:KT Ngo&quot;">KT Ngo</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JJ Omoto&quot;">JJ Omoto</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:DC Wong&quot;">DC Wong</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JD Nguyen&quot;">JD Nguyen</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:V Hartenstein&quot;">V Hartenstein</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-24">Developmental Biology</span></span>, <span class="elife-reflink-details-volume">384</span>, <span class="elife-reflink-details-pages">228-257</span>, <span class="elife-reflink-details-year">2013</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/j.ydbio.2013.07.008">http://dx.doi.org/10.1016/j.ydbio.2013.07.008</a></span>  — <span class="elife-reflink-details-cited">cited 11 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/j.ydbio.2013.07.008&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=23880429&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-25" data-original="25" data-author="ohtsuki, g" data-title="similarity of visual selectivity among clonally related neurons in visual cortex" data-date="2012" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="25">25</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/j.neuron.2012.05.023&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-26">Similarity of visual selectivity among clonally related neurons in visual cortex</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:G Ohtsuki&quot;">G Ohtsuki</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Nishiyama&quot;">M Nishiyama</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:T Yoshida&quot;">T Yoshida</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:T Murakami&quot;">T Murakami</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Histed&quot;">M Histed</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:C Lois&quot;">C Lois</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K Ohki&quot;">K Ohki</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-25">Neuron</span></span>, <span class="elife-reflink-details-volume">75</span>, <span class="elife-reflink-details-pages">65-72</span>, <span class="elife-reflink-details-year">2012</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/j.neuron.2012.05.023">http://dx.doi.org/10.1016/j.neuron.2012.05.023</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/j.neuron.2012.05.023&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=22794261&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-26" data-original="26" data-author="pearson, bj" data-title="specification of temporal identity in developing nervous system" data-date="2004" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="26">26</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1146/annurev.cellbio.19.111301.115142&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-27">Specification of temporal identity in the developing nervous system</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:BJ Pearson&quot;">BJ Pearson</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:CQ Doe&quot;">CQ Doe</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-26">Annual Review of Cell and Developmental Biology</span></span>, <span class="elife-reflink-details-volume">20</span>, <span class="elife-reflink-details-pages">619-647</span>, <span class="elife-reflink-details-year">2004</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1146/annurev.cellbio.19.111301.115142">http://dx.doi.org/10.1146/annurev.cellbio.19.111301.115142</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1146/annurev.cellbio.19.111301.115142&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=15473854&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-27" data-original="27" data-author="pereanu, wayne" data-title="neural lineages of drosophilbrain: three-dimensional digital atlas of pattern of lineage location and projection at late larval stage" data-date="2006-05-17" data-cited="5"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="27">27</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=jneuro&amp;resid=26/20/5534" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-28">Neural lineages of the <span class="nlm-italic" data-toggle="yes">Drosophila</span> brain: a three-dimensional digital atlas of the pattern of lineage location and projection at the late larval stage</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Wayne Pereanu&quot;">Wayne Pereanu</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Volker Hartenstein&quot;">Volker Hartenstein</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-27">The Journal of Neuroscience</span></span>, <span class="elife-reflink-details-volume">26</span>, <span class="elife-reflink-details-pages">5534-5553</span>, <span class="elife-reflink-details-year">2006</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1523/JNEUROSCI.4708-05.2006">http://dx.doi.org/10.1523/JNEUROSCI.4708-05.2006</a></span>  — <span class="elife-reflink-details-cited">cited 5 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=jneuro&amp;resid=26/20/5534" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1523/JNEUROSCI.4708-05.2006&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=16707805&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-28" data-original="28" data-author="pereanu, w" data-title="development-based compartmentalization of drosophilcentral brain" data-date="2010" data-cited="3"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="28">28</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1002/cne.22376&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-29">Development-based compartmentalization of the <span class="nlm-italic" data-toggle="yes">Drosophila</span> central brain</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:W Pereanu&quot;">W Pereanu</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:A Kumar&quot;">A Kumar</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:A Jennett&quot;">A Jennett</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Reichert&quot;">H Reichert</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:V Hartenstein&quot;">V Hartenstein</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-28">The Journal of Comparative Neurology</span></span>, <span class="elife-reflink-details-volume">518</span>, <span class="elife-reflink-details-pages">2996-3023</span>, <span class="elife-reflink-details-year">2010</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1002/cne.22376">http://dx.doi.org/10.1002/cne.22376</a></span>  — <span class="elife-reflink-details-cited">cited 3 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1002/cne.22376&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=20533357&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-29" data-original="29" data-author="potter, cj" data-title="q system: repressible binary system for transgene expression, lineage tracing and mosaic analysis" data-date="2010" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="29">29</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/j.cell.2010.02.025&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-30">The Q system: a repressible binary system for transgene expression, lineage tracing and mosaic analysis</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:CJ Potter&quot;">CJ Potter</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:B Tasic&quot;">B Tasic</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:EV Russler&quot;">EV Russler</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:L Liang&quot;">L Liang</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:L Luo&quot;">L Luo</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-29">Cell</span></span>, <span class="elife-reflink-details-volume">141</span>, <span class="elife-reflink-details-pages">536-548</span>, <span class="elife-reflink-details-year">2010</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.02.025">http://dx.doi.org/10.1016/j.cell.2010.02.025</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/j.cell.2010.02.025&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=20434990&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-30" data-original="30" data-author="rakic, p" data-title="specification of cerebral cortical areas" data-date="1988-07-08" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="30">30</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=241/4862/170" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-31">Specification of cerebral cortical areas</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:P Rakic&quot;">P Rakic</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-30">Science</span></span>, <span class="elife-reflink-details-volume">241</span>, <span class="elife-reflink-details-pages">170-176</span>, <span class="elife-reflink-details-year">1988</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1126/science.3291116">http://dx.doi.org/10.1126/science.3291116</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=241/4862/170" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1126/science.3291116&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=3291116&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-31" data-original="31" data-author="reichert, heinrich" data-title="evolutionary conservation of mechanisms for neural regionalization, proliferation and interconnection in brain development" data-date="2009-02-23" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="31">31</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=roybiolett&amp;resid=5/1/112" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-32">Evolutionary conservation of mechanisms for neural regionalization, proliferation and interconnection in brain development</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Heinrich Reichert&quot;">Heinrich Reichert</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-31">Biology Letters</span></span>, <span class="elife-reflink-details-volume">5</span>, <span class="elife-reflink-details-pages">112-116</span>, <span class="elife-reflink-details-year">2009</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1098/rsbl.2008.0337">http://dx.doi.org/10.1098/rsbl.2008.0337</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=roybiolett&amp;resid=5/1/112" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1098/rsbl.2008.0337&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=18755655&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-32" data-original="32" data-author="reichert, h" data-title="hox genes and brain development in drosophila" data-date="2010" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="32">32</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1007/978-1-4419-6673-5_11&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-33">Hox genes and brain development in <span class="nlm-italic" data-toggle="yes">Drosophila</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Reichert&quot;">H Reichert</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:B Bello&quot;">B Bello</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-32">Advances in Experimental Medicine and Biology</span></span>, <span class="elife-reflink-details-volume">689</span>, <span class="elife-reflink-details-pages">145-153</span>, <span class="elife-reflink-details-year">2010</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1007/978-1-4419-6673-5_11">http://dx.doi.org/10.1007/978-1-4419-6673-5_11</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1007/978-1-4419-6673-5_11&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=20795329&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-33" data-original="33" data-author="reichert, heinrich" data-title="developmental genetic evidence for monophyletic origin of bilateribrain" data-date="2001-10-29" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="33">33</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=royptb&amp;resid=356/1414/1533" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-34">Developmental genetic evidence for a monophyletic origin of the bilaterian brain</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Heinrich Reichert&quot;">Heinrich Reichert</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Antonio Simeone&quot;">Antonio Simeone</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-33">Philosophical Transactions of the Royal Society of London. Series B, Biological Sciences</span></span>, <span class="elife-reflink-details-volume">356</span>, <span class="elife-reflink-details-pages">1533-1544</span>, <span class="elife-reflink-details-year">2001</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1098/rstb.2001.0972">http://dx.doi.org/10.1098/rstb.2001.0972</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=royptb&amp;resid=356/1414/1533" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1098/rstb.2001.0972&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=11604121&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-34" data-original="34" data-author="root, cm" data-title="presynaptic gain control mechanism fine-tunes olfactory behavior" data-date="2008" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="34">34</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/j.neuron.2008.07.003&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-35">A presynaptic gain control mechanism fine-tunes olfactory behavior</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:CM Root&quot;">CM Root</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K Masuyama&quot;">K Masuyama</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:DS Green&quot;">DS Green</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:LE Enell&quot;">LE Enell</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:DR Nässel&quot;">DR Nässel</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:CH Lee&quot;">CH Lee</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JW Wang&quot;">JW Wang</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-34">Neuron</span></span>, <span class="elife-reflink-details-volume">59</span>, <span class="elife-reflink-details-pages">311-321</span>, <span class="elife-reflink-details-year">2008</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/j.neuron.2008.07.003">http://dx.doi.org/10.1016/j.neuron.2008.07.003</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/j.neuron.2008.07.003&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=18667158&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-35" data-original="35" data-author="semmelhack, jl" data-title="select drosophilglomeruli mediate innate olfactory attraction and aversion" data-date="2009" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="35">35</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/nature07983&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-36">Select <span class="nlm-italic" data-toggle="yes">Drosophila</span> glomeruli mediate innate olfactory attraction and aversion</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JL Semmelhack&quot;">JL Semmelhack</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JW Wang&quot;">JW Wang</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-35">Nature</span></span>, <span class="elife-reflink-details-volume">459</span>, <span class="elife-reflink-details-pages">218-223</span>, <span class="elife-reflink-details-year">2009</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1038/nature07983">http://dx.doi.org/10.1038/nature07983</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1038/nature07983&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=19396157&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-36" data-original="36" data-author="singh, ajeet pratap" data-title="dendritic refinement of identified neuron in drosophilcns is regulated by neuronal activity and wnt signaling" data-date="2010-04-15" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="36">36</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=137/8/1351" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-37">Dendritic refinement of an identified neuron in the <span class="nlm-italic" data-toggle="yes">Drosophila</span> CNS is regulated by neuronal activity and Wnt signaling</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Ajeet Pratap Singh&quot;">Ajeet Pratap Singh</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:K. VijayRaghavan&quot;">K. VijayRaghavan</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Veronica Rodrigues&quot;">Veronica Rodrigues</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-36">Development</span></span>, <span class="elife-reflink-details-volume">137</span>, <span class="elife-reflink-details-pages">1351-1360</span>, <span class="elife-reflink-details-year">2010</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1242/dev.044131">http://dx.doi.org/10.1242/dev.044131</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=137/8/1351" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1242/dev.044131&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=20223760&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-37" data-original="37" data-author="skeath, jb" data-title="genetic control of drosophilnerve cord development" data-date="2003" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="37">37</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/S0959-4388(03)00007-2&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-38">Genetic control of <span class="nlm-italic" data-toggle="yes">Drosophila</span> nerve cord development</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JB Skeath&quot;">JB Skeath</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:S Thor&quot;">S Thor</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-37">Current Opinion in Neurobiology</span></span>, <span class="elife-reflink-details-volume">13</span>, <span class="elife-reflink-details-pages">8-15</span>, <span class="elife-reflink-details-year">2003</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/S0959-4388(03)00007-2">http://dx.doi.org/10.1016/S0959-4388(03)00007-2</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/S0959-4388(03)00007-2&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=12593977&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-38" data-original="38" data-author="spindler, sr" data-title="bazookmediates secondary axon morphology in drosophilbrain lineages" data-date="2011" data-cited="3"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="38">38</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1186/1749-8104-6-16&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-39">Bazooka mediates secondary axon morphology in <span class="nlm-italic" data-toggle="yes">Drosophila</span> brain lineages</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:SR Spindler&quot;">SR Spindler</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:V Hartenstein&quot;">V Hartenstein</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-38">Neural Development</span></span>, <span class="elife-reflink-details-volume">6</span>, <span class="elife-reflink-details-pages">16</span>, <span class="elife-reflink-details-year">2011</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1186/1749-8104-6-16">http://dx.doi.org/10.1186/1749-8104-6-16</a></span>  — <span class="elife-reflink-details-cited">cited 3 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1186/1749-8104-6-16&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=21524279&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-39" data-original="39" data-author="stocker, rf" data-title="neuroblast ablation in drosophilp[gal4] lines reveals origins of olfactory interneurons" data-date="1997" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="39">39</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1002/(SICI)1097-4695(199705)32:5&lt;443::AID-NEU1&gt;3.0.CO;2-5&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-40">Neuroblast ablation in <span class="nlm-italic" data-toggle="yes">Drosophila</span> P[GAL4] lines reveals origins of olfactory interneurons</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:RF Stocker&quot;">RF Stocker</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:G Heimbeck&quot;">G Heimbeck</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:N Gendre&quot;">N Gendre</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JS de Belle&quot;">JS de Belle</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-39">Journal of Neurobiology</span></span>, <span class="elife-reflink-details-volume">32</span>, <span class="elife-reflink-details-pages">443-456</span>, <span class="elife-reflink-details-year">1997</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1002/(SICI)1097-4695(199705)32:5&lt;443::AID-NEU1&gt;3.0.CO;2-5">http://dx.doi.org/10.1002/(SICI)1097-4695(199705)32:5&lt;443::AID-NEU1&gt;3.0.CO;2-5</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1002/(SICI)1097-4695(199705)32:5&lt;443::AID-NEU1&gt;3.0.CO;2-5&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=9110257&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-40" data-original="40" data-author="technau, gm" data-title="generation of cell diversity and segmental pattern in embryonic central nervous system of drosophila" data-date="2006" data-cited="3"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="40">40</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1002/dvdy.20566&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-41">Generation of cell diversity and segmental pattern in the embryonic central nervous system of <span class="nlm-italic" data-toggle="yes">Drosophila</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:GM Technau&quot;">GM Technau</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:C Berger&quot;">C Berger</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:R Urbach&quot;">R Urbach</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-40">Developmental Dynamics</span></span>, <span class="elife-reflink-details-volume">235</span>, <span class="elife-reflink-details-pages">861-869</span>, <span class="elife-reflink-details-year">2006</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1002/dvdy.20566">http://dx.doi.org/10.1002/dvdy.20566</a></span>  — <span class="elife-reflink-details-cited">cited 3 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1002/dvdy.20566&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=16222713&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-41" data-original="41" data-author="tripodi, m" data-title="structural homeostasis: compensatory adjustments of dendritic arbor geometry in response to variations of synaptic input" data-date="2008" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="41">41</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1371/journal.pbio.0060260&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-42">Structural homeostasis: compensatory adjustments of dendritic arbor geometry in response to variations of synaptic input</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Tripodi&quot;">M Tripodi</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JF Evers&quot;">JF Evers</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:A Mauss&quot;">A Mauss</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Bate&quot;">M Bate</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Landgraf&quot;">M Landgraf</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-41">PLOS Biology</span></span>, <span class="elife-reflink-details-volume">6</span>, <span class="elife-reflink-details-pages">e260</span>, <span class="elife-reflink-details-year">2008</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0060260">http://dx.doi.org/10.1371/journal.pbio.0060260</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1371/journal.pbio.0060260&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=18959482&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-42" data-original="42" data-author="truman, jw" data-title="spatial and temporal patterns of neurogenesis in central nervous system of drosophilmelanogaster" data-date="1988" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="42">42</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/0012-1606(88)90067-X&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-43">Spatial and temporal patterns of neurogenesis in the central nervous system of <span class="nlm-italic" data-toggle="yes">Drosophila melanogaster</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JW Truman&quot;">JW Truman</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:M Bate&quot;">M Bate</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-42">Developmental Biology</span></span>, <span class="elife-reflink-details-volume">125</span>, <span class="elife-reflink-details-pages">145-157</span>, <span class="elife-reflink-details-year">1988</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/0012-1606(88)90067-X">http://dx.doi.org/10.1016/0012-1606(88)90067-X</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/0012-1606(88)90067-X&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=3119399&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-43" data-original="43" data-author="urbach, rolf" data-title="pattern of neuroblast formation, mitotic domains and proneural gene expression during early brain development in drosophila" data-date="2003-08-15" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="43">43</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=130/16/3589" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-44">The pattern of neuroblast formation, mitotic domains and proneural gene expression during early brain development in <span class="nlm-italic" data-toggle="yes">Drosophila</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Rolf Urbach&quot;">Rolf Urbach</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Ralf Schnabel&quot;">Ralf Schnabel</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Gerhard M. Technau&quot;">Gerhard M. Technau</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-43">Development</span></span>, <span class="elife-reflink-details-volume">130</span>, <span class="elife-reflink-details-pages">3589-3606</span>, <span class="elife-reflink-details-year">2003</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1242/dev.00528">http://dx.doi.org/10.1242/dev.00528</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=130/16/3589" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1242/dev.00528&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=12835378&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-44" data-original="44" data-author="urbach, rolf" data-title="molecular markers for identified neuroblasts in developing brain of drosophila" data-date="2003-08-15" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="44">44</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=130/16/3621" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-45">Molecular markers for identified neuroblasts in the developing brain of <span class="nlm-italic" data-toggle="yes">Drosophila</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Rolf Urbach&quot;">Rolf Urbach</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Gerhard M. Technau&quot;">Gerhard M. Technau</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-44">Development</span></span>, <span class="elife-reflink-details-volume">130</span>, <span class="elife-reflink-details-pages">3621-3637</span>, <span class="elife-reflink-details-year">2003</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1242/dev.00533">http://dx.doi.org/10.1242/dev.00533</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=130/16/3621" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1242/dev.00533&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=12835380&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-45" data-original="45" data-author="urbach, r" data-title="neuroblast formation and patterning during early brain development in drosophila" data-date="2004" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="45">45</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1002/bies.20062&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-46">Neuroblast formation and patterning during early brain development in <span class="nlm-italic" data-toggle="yes">Drosophila</span>
            </span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:R Urbach&quot;">R Urbach</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:GM Technau&quot;">GM Technau</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-45">Bioessays</span></span>, <span class="elife-reflink-details-volume">26</span>, <span class="elife-reflink-details-pages">739-751</span>, <span class="elife-reflink-details-year">2004</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1002/bies.20062">http://dx.doi.org/10.1002/bies.20062</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1002/bies.20062&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=15221856&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-46" data-original="46" data-author="urbach, rolf" data-title="segment polarity and dv patterning gene expression reveals segmental organization of drosophilbrain" data-date="2003-08-15" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="46">46</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=130/16/3607" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-47">Segment polarity and DV patterning gene expression reveals segmental organization of the <span class="nlm-italic" data-toggle="yes">Drosophila</span> brain</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Rolf Urbach&quot;">Rolf Urbach</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Gerhard M. Technau&quot;">Gerhard M. Technau</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-46">Development</span></span>, <span class="elife-reflink-details-volume">130</span>, <span class="elife-reflink-details-pages">3607-3620</span>, <span class="elife-reflink-details-year">2003</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1242/dev.00532">http://dx.doi.org/10.1242/dev.00532</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-ijlink"><a href="/lookup/ijlink?linkType=ABST&amp;journalCode=develop&amp;resid=130/16/3607" target="_blank" rel="nofollow">HighWire</a></span><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1242/dev.00532&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=12835379&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-47" data-original="47" data-author="wang, jw" data-title="two-photon calcium imaging reveals odor-evoked map of activity in fly brain" data-date="2003" data-cited="2"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="47">47</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/S0092-8674(03)00004-7&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-48">Two-photon calcium imaging reveals an odor-evoked map of activity in the fly brain</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JW Wang&quot;">JW Wang</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:AM Wong&quot;">AM Wong</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:J Flores&quot;">J Flores</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:LB Vosshall&quot;">LB Vosshall</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:R Axel&quot;">R Axel</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-47">Cell</span></span>, <span class="elife-reflink-details-volume">112</span>, <span class="elife-reflink-details-pages">271-282</span>, <span class="elife-reflink-details-year">2003</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(03)00004-7">http://dx.doi.org/10.1016/S0092-8674(03)00004-7</a></span>  — <span class="elife-reflink-details-cited">cited 2 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/S0092-8674(03)00004-7&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=12553914&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-48" data-original="48" data-author="wong, dc" data-title="postembryonic lineages of drosophilbrain: ii. identification of lineage projection patterns based on marcm clones" data-date="2013" data-cited="9"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="48">48</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/j.ydbio.2013.07.009&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-49">Postembryonic lineages of the <span class="nlm-italic" data-toggle="yes">Drosophila</span> brain: II. Identification of lineage projection patterns based on MARCM clones</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:DC Wong&quot;">DC Wong</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JK Lovick&quot;">JK Lovick</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:KT Ngo&quot;">KT Ngo</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:W Borisuthirattana&quot;">W Borisuthirattana</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JJ Omoto&quot;">JJ Omoto</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:V Hartenstein&quot;">V Hartenstein</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-48">Developmental Biology</span></span>, <span class="elife-reflink-details-volume">384</span>, <span class="elife-reflink-details-pages">258-289</span>, <span class="elife-reflink-details-year">2013</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/j.ydbio.2013.07.009">http://dx.doi.org/10.1016/j.ydbio.2013.07.009</a></span>  — <span class="elife-reflink-details-cited">cited 9 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/j.ydbio.2013.07.009&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=23872236&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-49" data-original="49" data-author="yu, hh" data-title="clonal development and organization of adult drosophilcentral brain" data-date="2013" data-cited="8"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="49">49</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1016/j.cub.2013.02.057&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-50">Clonal development and organization of the adult <span class="nlm-italic" data-toggle="yes">Drosophila</span> central brain</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:HH Yu&quot;">HH Yu</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:T Awasaki&quot;">T Awasaki</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:MD Schroeder&quot;">MD Schroeder</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:F Long&quot;">F Long</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JS Yang&quot;">JS Yang</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Y He&quot;">Y He</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:P Ding&quot;">P Ding</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:JC Kao&quot;">JC Kao</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:GY Wu&quot;">GY Wu</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:H Peng&quot;">H Peng</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:G Myers&quot;">G Myers</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:T Lee&quot;">T Lee</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-49">Current Biology</span></span>, <span class="elife-reflink-details-volume">23</span>, <span class="elife-reflink-details-pages">633-643</span>, <span class="elife-reflink-details-year">2013</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1016/j.cub.2013.02.057">http://dx.doi.org/10.1016/j.cub.2013.02.057</a></span>  — <span class="elife-reflink-details-cited">cited 8 times in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1016/j.cub.2013.02.057&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=23541733&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-50" data-original="50" data-author="yu, hh" data-title="twin-spot marcm to reveal developmental origin and identity of neurons" data-date="2009a" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="50">50</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/nn.2345&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-51">Twin-spot MARCM to reveal the developmental origin and identity of neurons</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:HH Yu&quot;">HH Yu</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:CH Chen&quot;">CH Chen</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:L Shi&quot;">L Shi</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:Y Huang&quot;">Y Huang</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:T Lee&quot;">T Lee</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-50">Nature Neuroscience</span></span>, <span class="elife-reflink-details-volume">12</span>, <span class="elife-reflink-details-pages">947-953</span>, <span class="elife-reflink-details-year">2015</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1038/nn.2345">http://dx.doi.org/10.1038/nn.2345</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1038/nn.2345&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=19525942&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article><article class="elife-reflinks-reflink" id="ref-51" data-original="51" data-author="yu, yc" data-title="specific synapses develop preferentially among sister excitatory neurons in neocortex" data-date="2009b" data-cited="1"><div class="elife-reflink-indicators"><div class="elife-reflink-indicators-left"><div class="elife-reflink-indicator-number"><span data-counter="51">51</span></div><div class="elife-reflink-indicator-cm"><a href="#"></a></div></div><div class="elife-reflink-indicators-right"><div class="elife-reflink-indicator-elife"></div><div class="elife-reflink-indicator-oa"></div></div></div><div class="elife-reflink-main"><cite class="elife-reflink-title"><a href="/lookup/external-ref/doi?access_num=10.1038/nature07722&amp;link_type=DOI" target="_blank"><span class="nlm-article-title" data-hwp-id="article-title-52">Specific synapses develop preferentially among sister excitatory neurons in the neocortex</span></a></cite><div class="elife-reflink-authors"><span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:YC Yu&quot;">YC Yu</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:RS Bultje&quot;">RS Bultje</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:X Wang&quot;">X Wang</a></span>, <span class="elife-reflink-author"><a href="http://scholar.google.com/scholar?q=&quot;author:SH Shi&quot;">SH Shi</a></span></div><div class="elife-reflink-details"><span class="elife-reflink-details-journal"><span class="nlm-source" data-hwp-id="source-51">Nature</span></span>, <span class="elife-reflink-details-volume">458</span>, <span class="elife-reflink-details-pages">501-504</span>, <span class="elife-reflink-details-year">2015</span><div class="elife-reflink-doi-cited-wrapper"><span class="elife-reflink-details-doi"><a href="http://dx.doi.org/10.1038/nature07722">http://dx.doi.org/10.1038/nature07722</a></span>  — <span class="elife-reflink-details-cited">cited 1 time in paper</span></div><div class="elife-reflink-links-wrapper"><span class="elife-reflink-link life-reflink-link-doi"><a href="/lookup/external-ref/doi?access_num=10.1038/nature07722&amp;link_type=DOI" target="_blank" rel="nofollow">CrossRef</a></span><span class="elife-reflink-link life-reflink-link-medline"><a href="/lookup/external-ref/medline?access_num=19204731&amp;link_type=MED" target="_blank" rel="nofollow">PubMed</a></span></div></div></div></article></div></div></div><div class="panel-separator"></div><div class="ctools-collapsible-container"><h2 class="pane-title ctools-collapsible-handle">Acknowledgements</h2><div class="ctools-collapsible-content"><div xmlns="http://www.w3.org/1999/xhtml" class="section ack" id="ack-1" xmlns:xhtml="http://www.w3.org/1999/xhtml"><p id="p-103">We dedicate this work to the memory of our late friend and colleague Veronica Rodrigues (1953–2010). We acknowledge DST, Govt. of India–Centre for Nanotechnology (no. SR/S5/NM-36/2005) for CIFF at NCBS. We are grateful to Henry Sun and Tiffany Cook for their generous gift of reagents.</p></div></div></div><div class="panel-separator"></div><div id="decision-letter" class="ctools-collapsible-container"><h2 class="pane-title ctools-collapsible-handle">Decision letter</h2><div class="ctools-collapsible-content"><div id="decision_letter"><div class="elife-article-editors"><div class='elife-article-decision-revewingeditor'><span class="nlm-given-names">Graeme W</span> <span class="nlm-surname">Davis</span>, <span class="nlm-role">Reviewing editor</span>, <span class="nlm-institution" data-hwp-id="institution-9">University of California, San Francisco</span>, <span class="nlm-country">United States</span></div></div><div class="elife-article-decision-letter"><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="boxed-text" id="boxed-text-1"><p id="p-113">eLife posts the editorial decision letter and author response on a selection of the published articles (subject to the approval of the authors). An edited version of the letter sent to the authors after peer review is shown, indicating the substantive concerns or comments; minor concerns are not usually shown. Reviewers have the opportunity to discuss the decision before the letter is sent (see <a href="http://elifesciences.org/review-process">review process</a>). Similarly, the author response typically shows only responses to the major concerns raised by the reviewers.</p></div><p id="p-114">Thank you for sending your work entitled “Rewiring the brain: Genetic transformation of structural and functional circuitry in the <em>Drosophila</em> brain” for consideration at <em>eLife</em>. Your article has been favorably evaluated by Fiona Watt (Senior editor), a Reviewing editor, and 2 reviewers.</p><p id="p-115">This paper provides a comprehensive analysis of a genetic manipulation, consisting in removing or ectopically expressing the gene orthodenticle (otd), which resulted in the fate transformation of a neural lineage in the Drosophila brain. The authors show convincingly that manipulating otd at the stage of the stem cell (neuroblast) determines the fate of the lineage; loss of otd in a lineage that normally innervates the central complex (LALv1) results in the transformation of its neurons into a lineage that normally develops at a close by position, associated with the antennal lobe). The data are clearly presented and documented. The findings are of great significance for the field. They underline the early global function of transcription factors in fate specification. They also shed light on the interplay between intrinsic (“hard wired”) factors and plasticity; for example, otd-mutant axons adopting the antennal projection fate are able to branch and properly innervate the antennal lobe, even though they start out at a different location (the location of the LALv1 lineage).</p><p id="p-116">Substantial concerns:</p><p id="p-117">1) The interpretations depend on the ability to recognize the same lineage after it has been profoundly transformed. It was difficult for me to be assured of this because of how the paper was organized. The study involves the use of a number of different driver lines that are employed in various contexts throughout the paper. In most cases we are not told why a given driver line was selected for a particular experiment or, importantly, the normal expression pattern for each line with respect to LALv1 and ALad1 and other lineages with which they might be confused. For example, for the RNAi experiments, why select insc-GAL4? What is the selectivity of the line? Also, the Per-Gal4 line was used when examining the single neuron otd-/- clones of LALv1 but GH146-Gal4 was used for the neuroblast clones of the lineage. It is only when we get to <a id="xref-fig-8-6" class="xref-fig" href="#F8">Figure 6</a> do we see that two drivers that express in wild-type LALv1 cells (Per-Gal4 and OK371-Gal4) no longer express in these neurons after they are transformed by otd removal. This information needs to be brought up earlier in the paper.</p><p id="p-118">2) Comparison of 4B and F shows a single cluster of otd+ cells that coincide with LALv1 in “B”, but in “F” part of the weakened otd+ cluster is now ascribed to two different clusters: part to LALv1 [yellow outline] and the remainder to ALv1 [which should not express otd?]. I am not convinced that we are seeing 2 distinct clusters rather than a split version of ALv1. It would be good to have a more convincing example of the appearance of a distinct GH146 cluster with the reduction in otd.</p><p id="p-119">3) The ALv1 lineage is a potential complication because it is a ventral lineage of antennal lobe projection neurons that is situated next to LALv1 in the adult. These cells have an anatomy that is similar to the cells of the transformed LALv1 and the GH146 driver expresses in these cells. The authors should have recovered many ALv1 clones in their experiments. It is important to know how they distinguished ALv1 clones from the transformed LALv1.</p><p id="p-120">4) There are other pieces of information that would help establish the correspondence of the LALv1 cells to the LALv1otd- neurons: 1) The observation that the LEp tract is missing when the lineage is transformed is a compelling piece of information. In addition to the reconstruction [or instead of it], it would be good to show an appropriate confocal section showing the loss of the tract on the transformed side and its presence on the control side. 2) If they have them, it would be good to show images of the transformed clones in the larval stage. The position of the neuroblast and its cell cluster should be in its normal position but its axon bundle would then be taking the transformed trajectory.</p><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div><div class="elife-article-decision-letter-doi"><strong>DOI: </strong><a href="/lookup/doi/10.7554/eLife.04407.021">http://dx.doi.org/10.7554/eLife.04407.021</a></div></div></div></div><div class="panel-separator"></div><div id="author-response" class="ctools-collapsible-container"><h2 class="pane-title ctools-collapsible-handle">Author response</h2><div class="ctools-collapsible-content"><div id="author_response"><div class="elife-article-author-response"><!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html><body><div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block" xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><p id="p-121"><em>1) The interpretations depend on the ability to recognize the same lineage after it has been profoundly transformed. It was difficult for me to be assured of this because of how the paper was organized. The study involves the use of a number of different driver lines that are employed in various contexts throughout the paper. In most cases we are not told why a given driver line was selected for a particular experiment or, importantly, the normal expression pattern for each line with respect to LALv1 and ALad1 and other lineages with which they might be confused. For example, for the RNAi experiments, why select insc-GAL4? What is the selectivity of the line? Also, the Per-Gal4 line was used when examining the single neuron otd-/- clones of LALv1 but GH146-Gal4 was used for the neuroblast clones of the lineage. It is only when we get to</em> <a id="xref-fig-8-7" class="xref-fig" href="#F8" data-fragment-nid="22874"><em>Figure 6</em></a> <em>do we see that two drivers that express in wild-type LALv1 cells (Per-Gal4 and OK371-Gal4) no longer express in these neurons after they are transformed by otd removal. This information needs to be brought up earlier in the paper</em>.</p><p id="p-122">We thank the reviewers for bringing this to our attention. We have now reorganized the manuscript so as to first mention the different Gal4 drivers used (see paragraph 4 under the sub heading, &lsquo;Development, morphogenesis and differential expression of Otd in two identified central brain neuroblast lineages, LALv1 and ALad1&rsquo;). We have also taken care to mention the Gal4 drivers used for each of the experiments when we describe them. For example, see second sentence in paragraph 2 and third sentence in paragraph 3 under the sub heading, <em>&lsquo;</em>Loss of Otd from the LALv1 neuroblast results in lineage identity transformation.<em>&rsquo;</em></p><p id="p-123">The use of <em>Insc-Gal4</em> was necessary because MARCM experiments suggested that Otd is required in the neuroblast to maintain the central complex identity of the LALv1 lineage. None of the other drivers used in the study were able to drive RNAi expression as early. We have now mentioned this in the text (see second sentence in paragraph 4 under the sub heading, &lsquo;Determining the lineage identity of the transformed neurons&rsquo;).</p><p id="p-124"><em>2) Comparison of 4B and F shows a single cluster of otd+ cells that coincide with LALv1 in &ldquo;B&rdquo;, but in &ldquo;F&rdquo; part of the weakened otd+ cluster is now ascribed to two different clusters: part to LALv1 [yellow outline] and the remainder to ALv1 [which should not express otd?]. I am not convinced that we are seeing 2 distinct clusters rather than a split version of ALv1. It would be good to have a more convincing example of the appearance of a distinct GH146 cluster with the reduction in otd</em>.</p><p id="p-125">In this experiment we used the <em>Insc-Gal4</em> to drive and Otd RNAi in all neuroblasts and used <em>GH146-QF&gt;QUAS-Tomato::HA</em> to visualize the neurons. However, as RNAi is not always completely penetrant, we often did not observe a complete loss of Otd in the LALv1 lineage. The brain shown in the previous <a id="xref-fig-4-8" class="xref-fig" href="#F4" data-fragment-nid="22872">Figure 4</a> (current <a id="xref-fig-7-8" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5</a>) shows the strongest reduction in Otd in the LALv1 lineage. We have now additionally shown an <em>otd</em><sup><em>-/-</em></sup> MARCM clonal brain in which all four neuroblasts lineages have been recovered. Please see <a id="xref-fig-7-9" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5 A-C</a>.</p><p id="p-126">Regarding splitting of the cell clusters, please see response 3 below.</p><p id="p-127"><em>3) The ALv1 lineage is a potential complication because it is a ventral lineage of antennal lobe projection neurons that is situated next to LALv1 in the adult. These cells have an anatomy that is similar to the cells of the transformed LALv1 and the GH146 driver expresses in these cells. The authors should have recovered many ALv1 clones in their experiments. It is important to know how they distinguished ALv1 clones from the transformed LALv1</em>.</p><p id="p-128">We thank the reviewers for pointing out this potential complication. The GH146 driver labels only about 6-10 cells of the ALv1 lineage. These neurons are indeed antennal lobe projection interneurons, however, their neuroanatomy is very different from the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage. The entry point of the axons of these two lineages into the antennal lobe is diametrically opposite to each other; while the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage enters the lobe medially, ALv1 (as well as the <em>otd</em><sup><em>-/-</em></sup> ALv1) enters it laterally (see asterisks in <a id="xref-fig-15-1" class="xref-fig" href="#F15" data-fragment-nid="22882">Author response image 1A,B</a>). Most strikingly, their axons exit the lobe towards the protocerebrum using different tracts; while the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage uses the medial antennal lobe tract, the ALv1 (as well as the <em>otd</em><sup><em>-/-</em></sup> ALv1) use the mediolateral antennal lobe tract (see arrow in Author response 1A,B). Both these tracts are visible in <a id="xref-fig-7-10" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5 A-C</a>. Finally, while the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage first innervates the calyx of the mushroom body and then the lateral horn, the GH146-labelled neurons of ALv1 (as well as the <em>otd</em><sup><em>-/-</em></sup> ALv1) largely innervate only the lateral horn (see <a id="xref-fig-15-2" class="xref-fig" href="#F15" data-fragment-nid="22882">Author response image 1A,B</a>). We have brought out these points in the main text as well. Please see paragraph 3 under the subheading, <em>&lsquo;</em>Determining the lineage identity of the transformed neurons&rsquo; (also see <a id="xref-fig-15-3" class="xref-fig" href="#F15" data-fragment-nid="22882">Author response image 1</a>).</p><p id="p-129">The ALv1 lineage does not express Otd and, as expected, we do not see any gross defects in any of the <em>otd</em><sup><em>-/-</em></sup> ALv1 clones we have recovered (see <a id="xref-fig-15-4" class="xref-fig" href="#F15" data-fragment-nid="22882">Author response image 1</a>). Moreover, although we have recovered the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage along with other neuroblast clones of the antennal lobe including the ALv1 lineage, we frequently recover it with no other clones in the brain. In these clones, we observe no splitting of the cell cluster and the mutant lineage never uses the mediolateral antennal lobe tract.</p><p id="p-130">Given these data we conclude that the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage and the <em>otd</em><sup><em>-/-</em></sup> ALv1 lineage have two distinct clonal origins (are two distinct neuroblast lineages) and are not a result of splitting of the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage.</p><div id="F15" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22882"><div xmlns="http://www.w3.org/1999/xhtml" xhtml="http://www.w3.org/1999/xhtml" id="content-block"><div class="fig-expansion " id="F15"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/2/F15.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22882" title="Author response image 1." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Author response image 1.&amp;lt;/span&amp;gt; &amp;lt;p id=&amp;quot;p-131&amp;quot;&amp;gt;WT (A) and &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;-/-&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; (B) ALv1 lineage. Note the position of the cell body (yellow dotted lines), the lateral point of entry of axon tract into the antennal lobe (indicated by the asterisk), the use of the mediolateral antennal lobe tract (indicated by the arrow) and the predominant innervation of the lateral horn of both the WT (A) and the &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;-/-&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; ALv1 lineage (B). Midline is represented by the yellow line. Genotype in A: &amp;lt;em&amp;gt;FRT19A/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+.&amp;lt;/em&amp;gt; Genotype in B&amp;lt;em&amp;gt;: FRT19A, oc&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;2&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+.&amp;lt;/em&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22882"><img data-img="/content/elife/3/e04407/2/F15.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/2/F15.medium.gif" alt="Author response image 1."></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/2/F15.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/2/F15.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22882">Download powerpoint</a></span></span><span class="fig-label">Author response image 1.</span> <p id="p-131">WT (A) and <em>otd</em><sup><em>-/-</em></sup> (B) ALv1 lineage. Note the position of the cell body (yellow dotted lines), the lateral point of entry of axon tract into the antennal lobe (indicated by the asterisk), the use of the mediolateral antennal lobe tract (indicated by the arrow) and the predominant innervation of the lateral horn of both the WT (A) and the <em>otd</em><sup><em>-/-</em></sup> ALv1 lineage (B). Midline is represented by the yellow line. Genotype in A: <em>FRT19A/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+.</em> Genotype in B<em>: FRT19A, oc</em><sup><em>2</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+.</em></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><p id="p-132"><em>4) There are other pieces of information that would help establish the correspondence of the LALv1 cells to the LALv1otd- neurons: 1) The observation that the LEp tract is missing when the lineage is transformed is a compelling piece of information. In addition to the reconstruction [or instead of it], it would be good to show an appropriate confocal section showing the loss of the tract on the transformed side and its presence on the control side. 2) If they have them, it would be good to show images of the transformed clones in the larval stage. The position of the neuroblast and its cell cluster should be in its normal position but its axon bundle would then be taking the transformed trajectory</em>.</p><p id="p-133">We thank the reviewers for pointing this out. We have reorganized the text and added an entire new results section under the sub heading, &lsquo;Determining the lineage identity of the transformed neurons&rsquo; in which we systematically list out three different ways by which it is possible to infer correspondence between the LALv1 and the transformed LALv1 neurons. 1) The appearance of an extra antennal lobe lineage (see <a id="xref-fig-7-11" class="xref-fig" href="#F7" data-fragment-nid="22873">Figure 5</a>). 2) The loss of the LEp tract (we have shown the confocal sections instead of the 3D reconstruction; see <a id="xref-fig-8-8" class="xref-fig" href="#F8" data-fragment-nid="22874">Figure 6</a>). 3) The use of a molecular marker, Acj6 that allows the unambiguous identification of the LALv1 lineage in both the WT and mutant conditions (see <a id="xref-fig-9-6" class="xref-fig" href="#F9" data-fragment-nid="22875">Figure 7</a>).</p><p id="p-134">We have also attempted to trace the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage into the larval brain. In these experiments we recovered <em>GH146-Gal4</em> labelled <em>otd</em><sup><em>-/-</em></sup> LALv1 clones, however these clones were largely indistinguishable from the ALad1 lineage (except for the loss of Otd immunolabelling corresponding to the larval LALv1 lineage). This is because in the larval brain, the cell bodies of LALv1 and ALad1 lie closely apposed to each other and when the LALv1 lineage is transformed, they also share similar trajectories. However, in a rare clone, the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage retained a few projections into the loVM tract, allowing us to unambiguously identify it as the LALv1 lineage. In this brain preparation we observed that the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage sent a majority of its projections via the medial antennal lobe tract (see <a id="xref-fig-16-1" class="xref-fig" href="#F16" data-fragment-nid="22883">Author response image 2</a>), confirming that the <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage indeed transforms to an ALad1 neuroblast lineage like fate.</p><div id="F16" class="fig-inline-img-set fig-inline-img-set-single"><div class="elife-fig-image-caption-wrapper"><div xhtml="http://www.w3.org/1999/xhtml" class="highwire-markup" data-fragment-nid="22883"><div xmlns="http://www.w3.org/1999/xhtml" xhtml="http://www.w3.org/1999/xhtml" id="content-block"><div class="fig-expansion " id="F16"><span class="highwire-journal-article-marker-start"></span><div class="fig-inline-img"><a href="/content/elife/3/e04407/2/F16.large.jpg" class="colorbox colorbox-load figure-expand-popup" rel="gallery-fragment-images-22883" title="Author response image 2." data-figure-caption="&amp;lt;div class=&amp;quot;fig-caption&amp;quot; xmlns:xhtml=&amp;quot;http://www.w3.org/1999/xhtml&amp;quot;&amp;gt;&amp;lt;span class=&amp;quot;fig-label&amp;quot;&amp;gt;Author response image 2.&amp;lt;/span&amp;gt; &amp;lt;p id=&amp;quot;p-135&amp;quot;&amp;gt;Note the larval &amp;lt;em&amp;gt;otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;-/-&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt; LALv1 lineage on the right hemisphere, which shows a loss of Otd immunolabelling in B. A few neurites of this lineage enter the loVM tract (cyan arrow in A,C), while most take the medial antennal lobe tract towards the protocerebrum (magenta arrow in A,C). Midline is represented by the yellow line. Genotype: &amp;lt;em&amp;gt;FRT19A, otd&amp;lt;/em&amp;gt;&amp;lt;sup&amp;gt;&amp;lt;em&amp;gt;YH13&amp;lt;/em&amp;gt;&amp;lt;/sup&amp;gt;&amp;lt;em&amp;gt;/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+./+.&amp;lt;/em&amp;gt;&amp;lt;/p&amp;gt;&amp;lt;div class=&amp;quot;sb-div caption-clear&amp;quot;&amp;gt;&amp;lt;/div&amp;gt;&amp;lt;/div&amp;gt;" data-powerpoint-url="/highwire/powerpoint/22883"><img data-img="/content/elife/3/e04407/2/F16.small.gif" src="http://dex3165296d6d.cloudfront.net//content/elife/3/e04407/2/F16.medium.gif" alt="Author response image 2."></a></div><div class="fig-caption"><span class="elife-figure-links"><span class="elife-figure-link elife-figure-link-download"><a href="/content/elife/3/e04407/2/F16.large.jpg?download=true">Download figure</a></span><span class="elife-figure-link elife-figure-link-newtab"><a href="/content/elife/3/e04407/2/F16.large.jpg" target="_new">Open in new tab</a></span><span class="elife-figure-link elife-figure-link-ppt"><a href="/highwire/powerpoint/22883">Download powerpoint</a></span></span><span class="fig-label">Author response image 2.</span> <p id="p-135">Note the larval <em>otd</em><sup><em>-/-</em></sup> LALv1 lineage on the right hemisphere, which shows a loss of Otd immunolabelling in B. A few neurites of this lineage enter the loVM tract (cyan arrow in A,C), while most take the medial antennal lobe tract towards the protocerebrum (magenta arrow in A,C). Midline is represented by the yellow line. Genotype: <em>FRT19A, otd</em><sup><em>YH13</em></sup><em>/FRT19A,Tub-Gal80,hsFLP; GH146-Gal4,UAS-mCD8::GFP/+./+.</em></p><div class="sb-div caption-clear"></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></div></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div></div></body></html>
</div><div class="elife-article-author-response-doi"><strong>DOI: </strong><a href="http://dx.doi.org/10.7554/eLife.04407.022">http://dx.doi.org/10.7554/eLife.04407.022</a></div></div></div></div><div class="panel-separator"></div><div id="comments" class="ctools-collapsible-container"><h2 class="pane-title ctools-collapsible-handle">Comments</h2><div class="ctools-collapsible-content">Comments are checked by a moderator (and/or an <i>eLife</i> editor) before they appear. Comments should be constructive, relevant to the article, conform to our <a href="/terms-and-conditions-of-use/">terms and conditions</a>, and include any pertinent <a href="http://submit.elifesciences.org/html/elife_author_instructions.html#Competing_interests">competing interests</a>.<div id="disqus_thread"><noscript><p><a href="http://hwelifesciences.disqus.com/?url=http%3A%2F%2Felifesciences.org%2Fcontent%2F3%2Fe04407">View the discussion thread.</a></p></noscript></div></div></div></div>
  </div>
</div>
</div>  </div>

  
  </div>
</div>
				</div>
					</div>	
		
					<div class="sidebar-wrapper grid-9 omega">
                  <div class="panel-panel panel-region-sidebar-lens">
            <div class="panel-pane pane-elife-article-lens-icon hidden-small" >
  
      
  
  <div class="pane-content">
    <a href="http://lens.elifesciences.org/04407/index.html"><img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_article/images/lens.png" alt="View in eLife Lens" title="View in eLife Lens" /></a>  </div>

  
  </div>
          </div>
                								  <div class="panel-panel panel-region-sidebar-top">
				    <div class="inside"><div class="panel-pane pane-elife-article-toolbox" >
  
      
  
  <div class="pane-content">
    <section class="article-tools"><h2 class="pane-title">Downloads:</h2><ul class="article-tools__list inline"><li class="article-tools__list_item first last"><a href="/elife/download-pdf/content/3/e04407/n/Genetic%20transformation%20of%20structural%20and%20functional%20circuitry%20rewires%20the%20Drosophila%20brain.pdf/1" title="Article PDF" type="application/pdf" rel="alternate" class="article-tools__link article-tools__link--pdf-article">Article</a></li>
</ul></section><div class="panel-separator"></div><div class="elife-share-wrapper"><div class="sharethis-wrapper"><span st_url="http://elifesciences.org/content/3/e04407" st_title="Genetic transformation of structural and functional circuitry rewires the Drosophila brain" class="st_email_large" displaytext="email"></span>
<span st_url="http://elifesciences.org/content/3/e04407" st_title="Genetic transformation of structural and functional circuitry rewires the Drosophila brain" class="st_citeulike_large" displaytext="citeulike"></span>
<span st_url="http://elifesciences.org/content/3/e04407" st_title="Genetic transformation of structural and functional circuitry rewires the Drosophila brain" class="st_facebook_large" displaytext="facebook"></span>
<span st_url="http://elifesciences.org/content/3/e04407" st_title="Genetic transformation of structural and functional circuitry rewires the Drosophila brain" class="st_googleplus_large" displaytext="googleplus"></span>
<span st_url="http://elifesciences.org/content/3/e04407" st_title="Genetic transformation of structural and functional circuitry rewires the Drosophila brain" class="st_twitter_large" displaytext="twitter"></span>
<span st_url="http://elifesciences.org/content/3/e04407" st_title="Genetic transformation of structural and functional circuitry rewires the Drosophila brain" class="st_reddit_large" displaytext="reddit"></span>
<span st_url="http://elifesciences.org/content/3/e04407" st_title="Genetic transformation of structural and functional circuitry rewires the Drosophila brain" class="st_sharethis_large" displaytext="sharethis"></span>
<script type='text/javascript'>var switchTo5x = true;</script><script type='text/javascript' src='http://w.sharethis.com/button/buttons.js'></script><script type='text/javascript'>stLight.options({"publisher":"dr-7ec628c4-4d9d-bf5-2fd2-4e9770c7a1d"});</script></div><script type='text/javascript'>stLight.options({onhover:false , doNotHash: true, doNotCopy: true, hashAddressBar: false});</script></div><div class="panel-separator"></div><h2 class="pane-title">Reference tools:</h2><div class="item-list"><ul class="citation-links links inline hidden-small"><li id="download" class="first"><a href="#api_box">Download</a><div class="item-list"><ul id="download"><li class="first last"><div id="citations_options" class="article-toolbox-wrapper"><h5 class="citation-tooltip-head">Download Citations</h5><div class="citation-col"><div class="item-list"><ul class="citations-list"><li class="first"><a href="/highwire/citation/22851/bibtex"><span class="icon-citation-download icon-citation-bibtex"></span><span class="name">BibTeX</span></a></li>
<li><a href="/highwire/citation/22851/tagged"><span class="icon-citation-download icon-citation-endnote"></span><span class="name">EndNote</span></a></li>
<li class="last"><a href="/highwire/citation/22851/endnote8"><span class="icon-citation-download icon-citation-endnote-8"></span><span class="name">EndNote 8</span></a></li>
</ul></div></div><div class="citation-col"><div class="item-list"><ul class="citations-list"><li class="first"><a href="/highwire/citation/22851/refworks"><span class="icon-citation-download icon-citation-refworks-tagged"></span><span class="name">RefWorks Tagged</span></a></li>
<li><a href="/highwire/citation/22851/ris"><span class="icon-citation-download icon-citation-ris"></span><span class="name">RIS</span></a></li>
<li class="last"><a href="/highwire/citation/22851/medlars"><span class="icon-citation-download icon-citation-medlars"></span><span class="name">Medlars</span></a></li>
</ul></div></div></div></li>
</ul></div></li>
<li id="open" class="last"><a href="#api_box">Open</a><div class="item-list"><ul id="open"><li class="first last"><div id="open_options" class="article-toolbox-wrapper"><h5 class="citation-tooltip-head">Open Citations In...</h5><div class="citation-col"><ul class="citations-list"><li class="0 first"><a href="/lookup/external-ref?tag_url=http%3A//elifesciences.org/content/3/e04407&amp;title=Genetic%20transformation%20of%20structural%20and%20functional%20circuitry%20rewires%20the%20Drosophila%20brain&amp;link_type=MENDELEY&amp;doi=10.7554/eLife.04407" class="mendeley">Mendeley</a></li><li class="1"><a href="http://www.readcube.com/articles/10.7554/eLife.04407" class="readcube">Readcube</a></li><li class="2 last"><a href="papers2://url/http://elifesciences.org/content/3/e04407?title=Genetic%20transformation%20of%20structural%20and%20functional%20circuitry%20rewires%20the%20Drosophila%20brain" class="papersapp">Papers App</a></li></ul></div><div class="citation-col"><ul class="citations-list"><li class="0 first last"><a href="http://www.citeulike.org/posturl?url=http%3A//elifesciences.org/content/3/e04407&amp;ourl=http%3A//elifesciences.org/content/3/e04407&amp;title=Genetic%20transformation%20of%20structural%20and%20functional%20circuitry%20rewires%20the%20Drosophila%20brain&amp;doi=10.7554/eLife.04407" class="citeulike">Citeulike</a></li></ul></div></div></li>
</ul></div></li>
</ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-elife-article-jumpto hidden-small" >
  
        <h2 class="pane-title">Jump to:</h2>
    
  
  <div class="pane-content">
    <div class='elife-jumpto'><div class="item-list"><ul><li class="first"><a href="#article" class="elife-article-jumplink first" data-panel-ajax-tab="jnl_elife_article_article">Article</a><div class="item-list"><ul><li class="first"><a href="#abstract" class="elife-article-jumplink first" data-panel-ajax-tab="jnl_elife_article_article">Abstract</a></li>
<li><a href="#elife-digest" class="elife-article-jumplink" data-panel-ajax-tab="jnl_elife_article_article">eLife digest</a></li>
<li><a href="#main-text" class="elife-article-jumplink" data-panel-ajax-tab="jnl_elife_article_article">Main text</a><div class="item-list"><ul><li class="first"><a href="#sec-1" class="elife-article-jumplink first" data-panel-ajax-tab="jnl_elife_article_article">Introduction</a></li>
<li><a href="#sec-2" class="elife-article-jumplink" data-panel-ajax-tab="jnl_elife_article_article">Results</a></li>
<li><a href="#sec-12" class="elife-article-jumplink" data-panel-ajax-tab="jnl_elife_article_article">Discussion</a></li>
<li class="last"><a href="#sec-13" class="elife-article-jumplink last" data-panel-ajax-tab="jnl_elife_article_article">Materials and methods</a></li>
</ul></div></li>
<li><a href="#references" class="elife-article-jumplink" data-panel-ajax-tab="jnl_elife_article_article">References</a></li>
<li><a href="#decision-letter" class="elife-article-jumplink" data-panel-ajax-tab="jnl_elife_article_article">Decision letter</a></li>
<li><a href="#author-response" class="elife-article-jumplink" data-panel-ajax-tab="jnl_elife_article_article">Author response</a></li>
<li class="last"><a href="#disqus_thread" data-disqus-identifier="node/22851" class="elife-article-jumplink last" data-panel-ajax-tab="jnl_elife_article_article">Leave a comment</a></li>
</ul></div></li>
<li><a href="/content/3/e04407/article-data#fig-data-figures" class="elife-article-jumplink" data-panel-ajax-tab="jnl_elife_article_figdata">Figures &amp; data</a></li>
<li><a href="/content/3/e04407/article-metrics#elife-article-metrics-summary" class="elife-article-jumplink" data-panel-ajax-tab="jnl_elife_article_metrics">Metrics</a></li>
<li class="last"><a href="/content/3/e04407/article-info#article-n-author-info" class="elife-article-jumplink last" data-panel-ajax-tab="jnl_elife_article_author">Article &amp; author info</a></li>
</ul></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-elife-article-categories hidden-small js-jumpblock-trigger" >
  
        <h2 class="pane-title">Categories &amp; tags</h2>
    
  
  <div class="pane-content">
    <div class="item-list"><ul class="elife-article-categories"><li class="first"><a href="/category/research-article" class="category-display-channel">Research article</a></li>
<li><a href="/category/developmental-biology-and-stem-cells" class="category-heading">Developmental biology and stem cells</a></li>
<li><a href="/category/neuroscience" class="category-heading">Neuroscience</a></li>
<li><a href="/browse?keys=%22neuroblast%20lineage%22" class="keyword">neuroblast lineage</a></li>
<li><a href="/browse?keys=%22central%20complex%22" class="keyword">central complex</a></li>
<li><a href="/browse?keys=%22olfactory%20projection%20neurons%22" class="keyword">olfactory projection neurons</a></li>
<li><a href="/browse?keys=%22functional%20imaging%22" class="keyword">functional imaging</a></li>
<li><a href="/browse?keys=%22orthodenticle%22" class="keyword">orthodenticle</a></li>
<li class="last"><a href="/browse?keys=%22D.%20melanogaster%22" class="keyword">D. melanogaster</a></li>
</ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-elife-core-sign-up hidden-small" >
  
        <h2 class="pane-title">Sign up for <em>eLife</em> alerts</h2>
    
  
  <div class="pane-content">
    <form class="elife-sign-up-form" action="http://crm.elifesciences.org/crm/civicrm/profile/create?reset=1&amp;gid=18" method="post" id="elife-core-sign-up-form" accept-charset="UTF-8"><div><input type="hidden" name="reset" value="1" />
<input type="hidden" name="gid" value="18" />
<input type="hidden" name="custom_6" value="article_rev140402" />
<input type="hidden" name="_qf_Edit_next" value="Save" />
<div class="form-item form-type-textfield form-item-email-3">
 <input placeholder="type your email address..." type="text" id="edit-email-3" name="email-3" value="" size="60" maxlength="128" class="form-text" />
</div>
<input type="submit" id="edit-submit" name="op" value="SIGN UP" class="form-submit" /><input type="hidden" name="form_build_id" value="form-DT01qRSyGyqEEhVFxSivomApIwrnCBxFjkIsiUJIpp4" />
<input type="hidden" name="form_id" value="elife_core_sign_up_form" />
</div></form>  </div>

  
  </div>
</div>
				  </div>
													<div class="panel-panel panel-region-sidebar-bottom" id="article-jumplinks-anchor-region">
				    <div class="inside"><div class="panel-pane pane-custom pane-1 hidden-small"  id="jump-share-container">
  
      
  
  <div class="pane-content">
    placeholder-text  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2 hidden-small"  id="jump-podcast-container">
  
      
  
  <div class="pane-content">
    <p>placeholder-text</p>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-3 hidden-small"  id="jump-links-anchor-container">
  
      
  
  <div class="pane-content">
    <p>placeholder-text</p>
  </div>

  
  </div>
</div>
				  </div>
			  			</div>
				
	</div>
</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</div></section>    
  
      <footer id="section-footer" class="section section-footer">
  <div id="zone-footer-wrapper" class="zone-wrapper zone-footer-wrapper clearfix">  
  <div id="zone-footer" class="zone zone-footer clearfix container-32">
    <div class="grid-30 region region-footer-first push-2" id="region-footer-first">
  <div class="region-inner region-footer-first-inner">
    <div class="block block-elife-footer block-elife-footer block-elife-footer-elife-footer odd block-without-title" id="block-elife-footer-elife-footer">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="site-footer">
  <section class="site-footer__funders">
    <div class="site-footer__funders__inner">
      <span class="site-footer__funders_intro">
        eLife is supported by      </span>
      <span class="site-footer__funder_img">
      <img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_footer/images/Funder_logos_2014_106.png" alt="eLife funder logos" />      </span>

    </div>
  </section><!-- /.site-footer__funders -->
  <section class="site-footer__main">
    <section class="site-footer__text_and_links">
      <div class="site-footer__identity">
        <img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_footer/images/eLife-logo.png" alt="eLife visual identity" />      </div>
      <div class="h-card vcard">
                  <p class="site-footer__statement_text">
            eLife Sciences is a unique, non-profit collaboration between the funders and practitioners of research to improve the way important results are presented and shared. The open-access <i>eLife</i> journal is the first step in this initiative to make science publishing more effectively benefit science and scientists.          </p>
                          <p class="site-footer__company-info">
            eLife Sciences Publications, Ltd is a limited liability non-profit non-stock corporation incorporated in the State of Delaware, USA, with company number 5030732, and is registered in the UK with company number FC030576 and branch number BR015634 at the address:          </p>
                <address class="site-footer__address">
          <div class="p-name fn">eLife Sciences Publications, Ltd</div>
          <div class="p-street-address street-address">1st Floor, 24 Hills Road</div>
          <div><span class="p-locality locality">Cambridge</span>
            <span class="p-postal-code postal-code">CB2 1JP</span>
          </div>
          <div class="p-country-name country-name">UK</div>
        </address>
      </div> <!-- /.h-card /.v-card-->
    </section> <!-- /.site-footer__text_and_links -->

    <section class="site-footer__social">
      <h2 class="site-footer__social_heading">Follow us</h2>
      <div class="item-list"><ol><li class="site-footer__social_icon first"><a href="https://www.facebook.com/elifesciences" class="site-footer__social_link"><img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_footer/images/facebook-icon.png" alt="Facebook icon" /></a></li>
<li class="site-footer__social_icon"><a href="https://twitter.com/elife" class="site-footer__social_link"><img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_footer/images/twitter-icon.png" alt="Twitter icon" /></a></li>
<li class="site-footer__social_icon"><a href="http://www.linkedin.com/company/elife-sciences-publications-ltd" class="site-footer__social_link"><img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_footer/images/linkedin-icon.png" alt="LinkedIn icon" /></a></li>
<li class="site-footer__social_icon"><a href="https://plus.google.com/102129675554093758550/posts" class="site-footer__social_link"><img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_footer/images/google-plus-icon.png" alt="Google plus icon" /></a></li>
<li class="site-footer__social_icon last"><a href="https://medium.com/@eLife" class="site-footer__social_link"><img src="http://dex3165296d6d.cloudfront.net/sites/default/modules/elife/elife_footer/images/medium-icon.png" alt="Medium icon" /></a></li>
</ol></div>    </section> <!-- /.site_footer__social -->

    <section class="site-footer__links">
      <div class="item-list"><ol class="site-footer__misc_links"><li class="first"><a href="/" class="site-footer__misc_link">Home</a></li>
<li><a href="/about" class="site-footer__misc_link">About</a></li>
<li><a href="http://submit.elifesciences.org" class="site-footer__misc_link">Submit</a></li>
<li><a href="/careers" class="site-footer__misc_link">Careers</a></li>
<li><a href="/contact" class="site-footer__misc_link">Contact</a></li>
<li><a href="/for-the-press" class="site-footer__misc_link">For the press</a></li>
<li><a href="/alerts-and-feeds" class="site-footer__misc_link">Sign up for alerts</a></li>
<li><a href="/resources" class="site-footer__misc_link">Downloads</a></li>
<li><a href="/annual-reports" class="site-footer__misc_link">Annual reports</a></li>
<li><a href="/who-we-work-with" class="site-footer__misc_link">Who we work with</a></li>
<li><a href="/terms-and-conditions-of-use" class="site-footer__misc_link">Terms &amp; conditions</a></li>
<li class="last"><a href="/privacy-policy" class="site-footer__misc_link">Privacy policy</a></li>
</ol></div>      <div class="item-list"><ol class="site-footer__section_links"><li class="site-footer__section_list_item site-footer__section_list_item--research first"><a href="/category/research-article" class="site-footer__misc_link">Research articles</a></li>
<li class="site-footer__section_list_item site-footer__section_list_item--short"><a href="/category/short-report" class="site-footer__misc_link">Short reports</a></li>
<li class="site-footer__section_list_item site-footer__section_list_item--advance"><a href="/category/research-advance" class="site-footer__misc_link">Research advances</a></li>
<li class="site-footer__section_list_item site-footer__section_list_item--reproducibility-project"><a href="/collections/reproducibility-project-cancer-biology" class="site-footer__misc_link">Reproducibility Project</a></li>
<li class="site-footer__section_list_item site-footer__section_list_item--editorials"><a href="/category/editorial" class="site-footer__misc_link">Editorials</a></li>
<li class="site-footer__section_list_item site-footer__section_list_item--insights"><a href="/category/insight" class="site-footer__misc_link">Insights</a></li>
<li class="site-footer__section_list_item site-footer__section_list_item--features"><a href="/category/feature-article" class="site-footer__misc_link">Features</a></li>
<li class="site-footer__section_list_item site-footer__section_list_item--podcasts"><a href="/podcast" class="site-footer__misc_link">Podcasts</a></li>
<li class="site-footer__section_list_item site-footer__section_list_item--news last"><a href="/elife-news" class="site-footer__misc_link">eLife news</a></li>
</ol></div>    </section> <!-- /.site-footer__links -->
  </section>

  <section class="site-footer__copyright-line">
    &copy; 2015 eLife Sciences Publications Ltd. Subject to a <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution license</a>, except where otherwise noted. ISSN: 2050-084X  </section>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div></footer>  </div>  <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
    
    <!--[if (IE 6)|(IE 7)|(IE 8)]>
      <script type="text/javascript">
        var IE6UPDATE_OPTIONS = {
          icons_path: "http://elifesciences.org/sites/default/modules/contrib/ie6update/images/",
          message: "Internet Explorer is missing updates required to view this site. Click here to update... ",
          url: "http://www.microsoft.com/windows/internet-explorer/default.aspx"
        }
      </script>
      <script type="text/javascript" src="http://elifesciences.org/sites/default/modules/contrib/ie6update/ie6update.js"></script>
    <![endif]-->
    </div>
</div><script type="text/javascript" src="http://elifesciences.org/sites/default/files/js/js_xtJ-TIkgQnqlMSFSJhwQHAFz3eGQmnkL8RfqoKmGUvU.js"></script>
</body>
</html>
